edam_ontology = {"format_3240": {"class_id": "http://edamontology.org/format_3240", "definitions": "CellML, the format for mathematical models of biological and other networks.", "label": "CellML"}, "topic_3299": {"class_id": "http://edamontology.org/topic_3299", "definitions": "The evolutionary processes, from the genetic to environmental scale, that produced life in all its diversity.", "label": "Evolutionary biology"}, "topic_0607": {"class_id": "http://edamontology.org/topic_0607", "definitions": "Laboratory management and resources, for example, catalogues of biological resources for use in the lab including cell lines, viruses, plasmids, phages, DNA probes and primers and so on.", "label": "Laboratory information management"}, "topic_0749": {"class_id": "http://edamontology.org/topic_0749", "definitions": "Proteins that bind to DNA and control transcription of DNA to mRNA (transcription factors) and also transcriptional regulatory sites, elements and regions (such as promoters, enhancers, silencers and boundary elements / insulators) in nucleotide sequences.|Transcription factor proteins either promote (as an activator) or block (as a repressor) the binding to DNA of RNA polymerase.  Regulatory sites including transcription factor binding site as well as promoters, enhancers, silencers and boundary elements / insulators.|This includes CpG rich regions (isochores) in a nucleotide sequence.|This includes promoters, CAAT signals, TATA signals, -35 signals, -10 signals, GC signals, primer binding sites for initiation of transcription or reverse transcription, enhancer, attenuator, terminators and ribosome binding sites.", "label": "Transcription factors and regulatory sites"}, "topic_0748": {"class_id": "http://edamontology.org/topic_0748", "definitions": "The detection, identification and analysis of positional features in proteins, such as functional sites.", "label": "Protein sites and features"}, "topic_1304": {"class_id": "http://edamontology.org/topic_1304", "definitions": "CpG rich regions (isochores) in a nucleotide sequence.", "label": "CpG island and isochores"}, "topic_0219": {"class_id": "http://edamontology.org/topic_0219", "definitions": "Deposition and curation of database accessions, including annotation, typically with terms from a controlled vocabulary.", "label": "Data submission, annotation and curation"}, "topic_0218": {"class_id": "http://edamontology.org/topic_0218", "definitions": "The processing and analysis of natural language, such as scientific literature in English, in order to extract data and information, or to enable human-computer interaction.", "label": "Natural language processing"}, "data_1566": {"class_id": "http://edamontology.org/data_1566", "definitions": "An informative report on protein-ligand (small molecule) interaction(s).", "label": "Protein-ligand interaction report"}, "data_1567": {"class_id": "http://edamontology.org/data_1567", "definitions": "protein-DNA/RNA interaction(s).", "label": "Protein-nucleic acid interactions report"}, "data_1564": {"class_id": "http://edamontology.org/data_1564", "definitions": "A report on known protein structural domains or folds that are recognised (identified) in protein sequence(s).|Methods use some type of mapping between sequence and fold, for example secondary structure prediction and alignment, profile comparison, sequence properties, homologous sequence search, kernel machines etc. Domains and folds might be taken from SCOP or CATH.", "label": "Protein fold recognition report"}, "data_1565": {"class_id": "http://edamontology.org/data_1565", "definitions": "protein-protein interaction(s), including interactions between protein domains.", "label": "Protein-protein interaction report"}, "topic_0213": {"class_id": "http://edamontology.org/topic_0213", "definitions": "Information on a specific mouse or rat genome including molecular sequences, genes and annotation.|The resource may be specific to a group of mice / rats or all mice / rats.", "label": "Mice or rats"}, "data_2236": {"class_id": "http://edamontology.org/data_2236", "definitions": "Raw CATH domain classification data files.|These are the parsable data files provided by CATH.", "label": "Raw CATH domain classification"}, "data_1560": {"class_id": "http://edamontology.org/data_1560", "definitions": "Information on a protein 'structurally similar group' node from the CATH database.", "label": "CATH structurally similar group"}, "data_1561": {"class_id": "http://edamontology.org/data_1561", "definitions": "Information on a protein 'functional category' node from the CATH database.", "label": "CATH functional category"}, "format_3688": {"class_id": "http://edamontology.org/format_3688", "definitions": "SBtab is a tabular format for biochemical network models.", "label": "SBtab"}, "format_3689": {"class_id": "http://edamontology.org/format_3689", "definitions": "Biological Connection Markup Language (BCML) is an XML format for biological pathways.", "label": "BCML"}, "format_3681": {"class_id": "http://edamontology.org/format_3681", "definitions": "mzTab is a tab-delimited format for mass spectrometry-based proteomics and metabolomics results.", "label": "mzTab"}, "format_3682": {"class_id": "http://edamontology.org/format_3682", "definitions": "imzML is a data format for mass spectrometry imaging data. NB.: See comment.|Data is recorded in 2 files: '.imzXML' is a metadata XML file based on mzML by HUPO-PSI, and '.ibd' is a binary file containing the mass spectra.", "label": "imzML"}, "format_3683": {"class_id": "http://edamontology.org/format_3683", "definitions": "qcML is an XML format for quality-related data of mass spectrometry and other high-throughput measurements.|The focus of qcML is towards mass spectrometry based proteomics, but the format is suitable for metabolomics and sequencing as well.", "label": "qcML"}, "format_3684": {"class_id": "http://edamontology.org/format_3684", "definitions": "PRIDE XML is an XML format for mass spectra, peptide and protein identifications, and metadata about a corresponding measurement, sample, experiment.", "label": "PRIDE XML"}, "format_3685": {"class_id": "http://edamontology.org/format_3685", "definitions": "Simulation Experiment Description Markup Language (SED-ML) is an XML format for encoding simulation setups, according to the MIASE (Minimum Information About a Simulation Experiment) requirements.", "label": "SED-ML"}, "format_3686": {"class_id": "http://edamontology.org/format_3686", "definitions": "Open Modeling EXchange format (OMEX) is a ZIPped format for encapsulating all information necessary for a modeling and simulation project in systems biology.|An OMEX file is a ZIP container that includes a manifest file, listing the content of the archive, an optional metadata file adding information about the archive and its content, and the files describing the model. OMEX is one of the standardised formats within COMBINE (Computational Modeling in Biology Network).", "label": "COMBINE OMEX"}, "format_3687": {"class_id": "http://edamontology.org/format_3687", "definitions": "The Investigation / Study / Assay (ISA) tab-delimited (TAB) format incorporates metadata from \nexperiments employing a combination of technologies.|ISA-TAB is based on MAGE-TAB. Other than tabular, the ISA model can also be represented in RDF, and in JSON (compliable with a set of defined JSON Schemata).", "label": "ISA-TAB"}, "topic_2820": {"class_id": "http://edamontology.org/topic_2820", "definitions": "Vertebrates, e.g. information on a specific vertebrate genome including molecular sequences, genes and annotation.|The resource may be specific to a vertebrate, a group of vertebrates or all vertebrates.", "label": "Vertebrates"}, "topic_0698": {"class_id": "http://edamontology.org/topic_0698", "definitions": "Protein tertiary structures.", "label": "Protein tertiary structure"}, "topic_3321": {"class_id": "http://edamontology.org/topic_3321", "definitions": "The structure and function of genes at a molecular level.", "label": "Molecular genetics"}, "topic_0697": {"class_id": "http://edamontology.org/topic_0697", "definitions": "RNA secondary or tertiary structure and alignments.", "label": "RNA structure"}, "format_3595": {"class_id": "http://edamontology.org/format_3595", "definitions": "PCX is an image file format that uses a simple form of run-length encoding. It is lossless.", "label": "pcx"}, "topic_0694": {"class_id": "http://edamontology.org/topic_0694", "definitions": "Protein secondary structure or secondary structure alignments.|This includes assignment, analysis, comparison, prediction, rendering etc. of secondary structure data.", "label": "Protein secondary structure"}, "data_1896": {"class_id": "http://edamontology.org/data_1896", "definitions": "Identifier for loci from ASPGD (Aspergillus Genome Database).", "label": "Locus ID (ASPGD)"}, "data_1897": {"class_id": "http://edamontology.org/data_1897", "definitions": "Identifier for loci from Magnaporthe grisea Database at the Broad Institute.", "label": "Locus ID (MGG)"}, "data_1895": {"class_id": "http://edamontology.org/data_1895", "definitions": "Locus identifier for Arabidopsis Genome Initiative (TAIR, TIGR and MIPS databases)", "label": "Locus ID (AGI)"}, "data_1892": {"class_id": "http://edamontology.org/data_1892", "definitions": "Name of a gene from the GeneFarm database.", "label": "Gene name (GeneFarm)"}, "data_1893": {"class_id": "http://edamontology.org/data_1893", "definitions": "A unique name or other identifier of a genetic locus, typically conforming to a scheme that names loci (such as predicted genes) depending on their position in a molecular sequence, for example a completely sequenced genome or chromosome.", "label": "Locus ID"}, "data_1890": {"class_id": "http://edamontology.org/data_1890", "definitions": "A unique identifier for an Arabidopsis gene, which is an acronym or abbreviation of the gene name.", "label": "Gene name (Arabidopsis)"}, "data_1891": {"class_id": "http://edamontology.org/data_1891", "definitions": "A unique identifier of a protein or gene used in the iHOP database.", "label": "iHOP symbol"}, "data_3490": {"class_id": "http://edamontology.org/data_3490", "definitions": "A sketch of a small molecule made with some specialised drawing package.|Chemical structure sketches are used for presentational purposes but also as inputs to various analysis software.", "label": "Chemical structure sketch"}, "data_3492": {"class_id": "http://edamontology.org/data_3492", "definitions": "An informative report about a specific or conserved nucleic acid sequence pattern.", "label": "Nucleic acid signature"}, "data_3031": {"class_id": "http://edamontology.org/data_3031", "definitions": "A type of data that (typically) corresponds to entries from the primary biological databases and which is (typically) the primary input or output of a tool, i.e. the data the tool processes or generates, as distinct from metadata and identifiers which describe and identify such core data, parameters that control the behaviour of tools, reports of derivative data generated by tools and annotation.|Core data entities typically have a format and may be identified by an accession number.", "label": "Core data"}, "data_3494": {"class_id": "http://edamontology.org/data_3494", "definitions": "A DNA sequence.", "label": "DNA sequence"}, "data_3495": {"class_id": "http://edamontology.org/data_3495", "definitions": "", "label": "RNA sequence"}, "data_1898": {"class_id": "http://edamontology.org/data_1898", "definitions": "Identifier for loci from CGD (Candida Genome Database).", "label": "Locus ID (CGD)"}, "data_1899": {"class_id": "http://edamontology.org/data_1899", "definitions": "Locus identifier for Comprehensive Microbial Resource at the J. Craig Venter Institute.", "label": "Locus ID (CMR)"}, "format_2559": {"class_id": "http://edamontology.org/format_2559", "definitions": "A format resembling GenBank entry (plain text) format.|This concept may be used for the non-standard GenBank-like formats.", "label": "GenBank-like format"}, "data_2098": {"class_id": "http://edamontology.org/data_2098", "definitions": "An identifier of a submitted job.", "label": "Job identifier"}, "format_2001": {"class_id": "http://edamontology.org/format_2001", "definitions": "EMBOSS simple multiple alignment format.", "label": "EMBOSS simple format"}, "format_2000": {"class_id": "http://edamontology.org/format_2000", "definitions": "SELEX format for (aligned) sequences.", "label": "selex"}, "format_2003": {"class_id": "http://edamontology.org/format_2003", "definitions": "Simple sequence pair (alignment) format for SRS.", "label": "srspair"}, "format_2002": {"class_id": "http://edamontology.org/format_2002", "definitions": "Simple multiple sequence (alignment) format for SRS.", "label": "srs format"}, "format_2005": {"class_id": "http://edamontology.org/format_2005", "definitions": "Treecon format for (aligned) sequences.", "label": "TreeCon-seq"}, "format_2004": {"class_id": "http://edamontology.org/format_2004", "definitions": "T-Coffee program alignment format.", "label": "T-Coffee format"}, "format_2555": {"class_id": "http://edamontology.org/format_2555", "definitions": "XML format for molecular sequence alignment information.", "label": "Alignment format (XML)"}, "format_2006": {"class_id": "http://edamontology.org/format_2006", "definitions": "Data format for a phylogenetic tree.", "label": "Phylogenetic tree format"}, "data_1412": {"class_id": "http://edamontology.org/data_1412", "definitions": "Sequence identity is the number (%) of matches (identical characters) in positions from an alignment of two molecular sequences.", "label": "Sequence identity"}, "data_1413": {"class_id": "http://edamontology.org/data_1413", "definitions": "Sequence similarity is the similarity (expressed as a percentage) of two molecular sequences calculated from their alignment, a scoring matrix for scoring characters substitutions and penalties for gap insertion and extension.|Data Type is float probably.", "label": "Sequence similarity"}, "data_0949": {"class_id": "http://edamontology.org/data_0949", "definitions": "Basic information, annotation or documentation concerning a workflow (but not the workflow itself).", "label": "Workflow metadata"}, "data_0948": {"class_id": "http://edamontology.org/data_0948", "definitions": "A definition of a data resource serving one or more types of data, including metadata and links to the resource or data proper.", "label": "Data resource definition"}, "data_1416": {"class_id": "http://edamontology.org/data_1416", "definitions": "Data on correlations between sites in a molecular sequence alignment, typically to identify possible covarying positions and predict contacts or structural constraints in protein structures.", "label": "Sequence alignment report (site correlation)"}, "data_1417": {"class_id": "http://edamontology.org/data_1417", "definitions": "Alignment of molecular sequences to a Domainatrix signature (representing a sequence alignment).", "label": "Sequence-profile alignment (Domainatrix signature)"}, "data_1414": {"class_id": "http://edamontology.org/data_1414", "definitions": "Data on molecular sequence alignment quality (estimated accuracy).", "label": "Sequence alignment metadata (quality report)"}, "data_1415": {"class_id": "http://edamontology.org/data_1415", "definitions": "Data on character conservation in a molecular sequence alignment.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation. Use this concept for calculated substitution rates, relative site variability, data on sites with biased properties, highly conserved or very poorly conserved sites, regions, blocks etc.", "label": "Sequence alignment report (site conservation)"}, "data_0943": {"class_id": "http://edamontology.org/data_0943", "definitions": "Spectra from mass spectrometry.", "label": "Mass spectrometry spectra"}, "data_0942": {"class_id": "http://edamontology.org/data_0942", "definitions": "Two-dimensional gel electrophoresis image", "label": "2D PAGE image"}, "data_0941": {"class_id": "http://edamontology.org/data_0941", "definitions": "Annotation on a structural 3D model (volume map) from electron microscopy.", "label": "Electron microscopy model"}, "data_0940": {"class_id": "http://edamontology.org/data_0940", "definitions": "Volume map data from electron microscopy.", "label": "Electron microscopy volume map"}, "data_0947": {"class_id": "http://edamontology.org/data_0947", "definitions": "A map (typically a diagram) of a biological pathway.", "label": "Biological pathway map"}, "data_0946": {"class_id": "http://edamontology.org/data_0946", "definitions": "An informative report about a specific biological pathway or network, typically including a map (diagram) of the pathway.", "label": "Pathway or network annotation"}, "data_0945": {"class_id": "http://edamontology.org/data_0945", "definitions": "Protein or peptide identifications with evidence supporting the identifications, for example from comparing a peptide mass fingerprint (from mass spectrometry) to a sequence database, or  the set of typical spectra one obtains when running a protein through a mass spectrometer.", "label": "Peptide identification"}, "data_0944": {"class_id": "http://edamontology.org/data_0944", "definitions": "A set of peptide masses (peptide mass fingerprint) from mass spectrometry.|A molecular weight standard fingerprint is standard protonated molecular masses e.g. from trypsin (modified porcine trypsin, Promega) and keratin peptides.", "label": "Peptide mass fingerprint"}, "topic_0109": {"class_id": "http://edamontology.org/topic_0109", "definitions": "Methods that aims to identify, predict, model or analyse genes or gene structure in DNA sequences.|This includes the study of promoters, coding regions, splice sites, etc. Methods for gene prediction might be ab initio, based on phylogenetic comparisons, use motifs, sequence features, support vector machine, alignment etc.", "label": "Gene finding"}, "topic_0191": {"class_id": "http://edamontology.org/topic_0191", "definitions": "The reconstruction of a phylogeny (evolutionary relatedness amongst organisms), for example, by building a phylogenetic tree.|Currently too specific for the topic sub-ontology (but might be unobsoleted).", "label": "Phylogeny reconstruction"}, "operation_0246": {"class_id": "http://edamontology.org/operation_0246", "definitions": "Identify structural domains in a protein structure from first principles (for example calculations on structural compactness).", "label": "Protein domain recognition"}, "operation_0247": {"class_id": "http://edamontology.org/operation_0247", "definitions": "Analyse the architecture (spatial arrangement of secondary structure) of protein structure(s).", "label": "Protein architecture analysis"}, "operation_0244": {"class_id": "http://edamontology.org/operation_0244", "definitions": "Analyse flexibility and motion in protein structure.|Use this concept for analysis of flexible and rigid residues, local chain deformability, regions undergoing conformational change, molecular vibrations or fluctuational dynamics, domain motions or other large-scale structural transitions in a protein structure.", "label": "Protein flexibility and motion analysis"}, "operation_0245": {"class_id": "http://edamontology.org/operation_0245", "definitions": "Identify or screen for 3D structural motifs in protein structure(s).|This includes conserved substructures and conserved geometry, such as spatial arrangement of secondary structure or protein backbone. Methods might use structure alignment, structural templates, searches for similar electrostatic potential and molecular surface shape, surface-mapping of phylogenetic information etc.", "label": "Protein structural motif recognition"}, "operation_0242": {"class_id": "http://edamontology.org/operation_0242", "definitions": "Identify common, conserved (homologous) or synonymous transcriptional regulatory motifs (transcription factor binding sites).", "label": "Conserved transcription regulatory sequence identification"}, "operation_0243": {"class_id": "http://edamontology.org/operation_0243", "definitions": "Extract, calculate or predict non-positional (physical or chemical) properties of a protein from processing a protein (3D) structure.", "label": "Protein property calculation (from structure)"}, "operation_0240": {"class_id": "http://edamontology.org/operation_0240", "definitions": "Find motifs shared by molecular sequences.", "label": "Sequence motif comparison"}, "operation_0241": {"class_id": "http://edamontology.org/operation_0241", "definitions": "Analyse the sequence, conformational or physicochemical properties of transcription regulatory elements in DNA sequences.|For example transcription factor binding sites (TFBS) analysis to predict accessibility of DNA to binding factors.", "label": "Transcription regulatory sequence analysis"}, "topic_3414": {"class_id": "http://edamontology.org/topic_3414", "definitions": "The branch of medicine that treats body wounds or shock produced by sudden physical injury, as from violence or accident.", "label": "Trauma medicine"}, "operation_0248": {"class_id": "http://edamontology.org/operation_0248", "definitions": "Calculate or extract inter-atomic, inter-residue or residue-atom contacts, distances and interactions in protein structure(s).", "label": "Residue interaction calculation"}, "operation_0249": {"class_id": "http://edamontology.org/operation_0249", "definitions": "Calculate, visualise or analyse phi/psi angles of a protein structure.", "label": "Protein geometry calculation"}, "data_2048": {"class_id": "http://edamontology.org/data_2048", "definitions": "A human-readable collection of information including annotation on a biological entity or phenomena, computer-generated reports of analysis of primary data (e.g. sequence or structural), and metadata (data about primary data) or any other free (essentially unformatted) text, as distinct from the primary data itself.|You can use this term by default for any textual report, in case you can't find another, more specific term. Reports may be generated automatically or collated by hand and can include metadata on the origin, source, history, ownership or location of some thing.", "label": "Report"}, "format_3330": {"class_id": "http://edamontology.org/format_3330", "definitions": "EMBOSS simple sequence pair alignment format.", "label": "PO"}, "format_3331": {"class_id": "http://edamontology.org/format_3331", "definitions": "XML format as produced by the NCBI Blast package", "label": "BLAST XML results format"}, "data_2041": {"class_id": "http://edamontology.org/data_2041", "definitions": "Information on a genome version.", "label": "Genome version information"}, "data_2042": {"class_id": "http://edamontology.org/data_2042", "definitions": "Typically a statement about some data or results, including evidence or the source of a statement, which may include computational prediction, laboratory experiment, literature reference etc.", "label": "Evidence"}, "data_2043": {"class_id": "http://edamontology.org/data_2043", "definitions": "A molecular sequence and minimal metadata, typically an identifier of the sequence and/or a comment.", "label": "Sequence record lite"}, "data_2044": {"class_id": "http://edamontology.org/data_2044", "definitions": "One or more molecular sequences, possibly with associated annotation.|This concept is a placeholder of concepts for primary sequence data including raw sequences and sequence records.  It should not normally be used for derivatives such as sequence alignments, motifs or profiles.", "label": "Sequence"}, "topic_0215": {"class_id": "http://edamontology.org/topic_0215", "definitions": "Information on a specific worm genome including molecular sequences, genes and annotation.", "label": "Worms"}, "data_2046": {"class_id": "http://edamontology.org/data_2046", "definitions": "A nucleic acid sequence and minimal metadata, typically an identifier of the sequence and/or a comment.", "label": "Nucleic acid sequence record (lite)"}, "data_2047": {"class_id": "http://edamontology.org/data_2047", "definitions": "A protein sequence and minimal metadata, typically an identifier of the sequence and/or a comment.", "label": "Protein sequence record (lite)"}, "data_3546": {"class_id": "http://edamontology.org/data_3546", "definitions": "Any data concerning a specific biological or biomedical image.|This can include basic provenance and technical information about the image, scientific annotation and so on.", "label": "Image metadata"}, "format_2310": {"class_id": "http://edamontology.org/format_2310", "definitions": "FASTA format wrapped in HTML elements.", "label": "FASTA-HTML"}, "operation_0338": {"class_id": "http://edamontology.org/operation_0338", "definitions": "Search a sequence database by sequence comparison and retrieve similar sequences.\nsequences matching a given sequence motif or pattern, such as a Prosite pattern or regular expression.|This excludes direct retrieval methods (e.g. the dbfetch program).", "label": "Sequence database search"}, "operation_0339": {"class_id": "http://edamontology.org/operation_0339", "definitions": "Search a tertiary structure database, typically by sequence and/or structure comparison, or some other means, and retrieve structures and associated data.", "label": "Structure database search"}, "data_2235": {"class_id": "http://edamontology.org/data_2235", "definitions": "Raw SCOP domain classification data files.|These are the parsable data files provided by SCOP.", "label": "Raw SCOP domain classification"}, "operation_0332": {"class_id": "http://edamontology.org/operation_0332", "definitions": "Design molecules that elicit an immune response (immunogens).", "label": "Immunogen design"}, "operation_0333": {"class_id": "http://edamontology.org/operation_0333", "definitions": "Predict and optimise zinc finger protein domains for DNA/RNA binding (for example for transcription factors and nucleases).", "label": "Zinc finger prediction"}, "operation_0330": {"class_id": "http://edamontology.org/operation_0330", "definitions": "Map and model the effects of single nucleotide polymorphisms (SNPs) on protein structure(s).", "label": "Protein SNP mapping"}, "operation_0331": {"class_id": "http://edamontology.org/operation_0331", "definitions": "Predict the effect of point mutation on a protein structure, in terms of strucural effects and protein folding, stability and function.|This includes 1) rotamer likelihood prediction: the prediction of rotamer likelihoods for all 20 amino acid types at each position in a protein structure, where output typically includes, for each residue position, the likelihoods for the 20 amino acid types with estimated reliability of the 20 likelihoods.  2) Protein SNP mapping, which maps and modesl the effects of single nucleotide polymorphisms (SNPs) on protein structure(s).  Methods might predict silent or pathological mutations.", "label": "Protein modelling (mutation)"}, "operation_0336": {"class_id": "http://edamontology.org/operation_0336", "definitions": "Test and validate the format and content of a data file.", "label": "Format validation"}, "operation_0337": {"class_id": "http://edamontology.org/operation_0337", "definitions": "Visualise, plot or render (graphically) biomolecular data such as molecular sequences or structures.", "label": "Visualisation"}, "operation_0334": {"class_id": "http://edamontology.org/operation_0334", "definitions": "Calculate Km, Vmax and derived data for an enzyme reaction.", "label": "Enzyme kinetics calculation"}, "operation_0335": {"class_id": "http://edamontology.org/operation_0335", "definitions": "Reformat a file of data (or equivalent entity in memory).", "label": "Formatting"}, "format_3726": {"class_id": "http://edamontology.org/format_3726", "definitions": "PMML uses XML to represent mining models. The structure of the models is described by an XML Schema.| One or more mining models can be contained in a PMML document.", "label": "PMML"}, "operation_3680": {"class_id": "http://edamontology.org/operation_3680", "definitions": "Analyze data from RNA-seq experiments.", "label": "RNA-Seq analysis"}, "operation_1829": {"class_id": "http://edamontology.org/operation_1829", "definitions": "Detect cysteine bridges (from coordinate data) in a protein structure.", "label": "Cysteine bridge detection"}, "operation_1828": {"class_id": "http://edamontology.org/operation_1828", "definitions": "For each amino acid in a protein structure calculate the backbone angle tau.|Tau is the backbone angle N-Calpha-C (angle over the C-alpha).", "label": "Tau angle calculation"}, "operation_1825": {"class_id": "http://edamontology.org/operation_1825", "definitions": "Calculate for each residue in a protein structure all its backbone torsion angles.", "label": "Backbone torsion angle calculation"}, "operation_1824": {"class_id": "http://edamontology.org/operation_1824", "definitions": "Calculate the solvent accessibility ('accessible surface') for a structure as a whole.", "label": "Protein surface calculation (accessible)"}, "operation_1827": {"class_id": "http://edamontology.org/operation_1827", "definitions": "Calculate for each cysteine (bridge) all its torsion angles.", "label": "Cysteine torsion angle calculation"}, "operation_1826": {"class_id": "http://edamontology.org/operation_1826", "definitions": "Calculate for each residue in a protein structure all its torsion angles.", "label": "Full torsion angle calculation"}, "operation_1821": {"class_id": "http://edamontology.org/operation_1821", "definitions": "Calculate the solvent accessibility ('accessible molecular surface') for each residue in a structure.|Solvent accessibility might be calculated for the backbone, sidechain and total (backbone plus sidechain).", "label": "Protein residue surface calculation (accessible molecular)"}, "operation_1820": {"class_id": "http://edamontology.org/operation_1820", "definitions": "Calculate the solvent accessibility ('vacuum accessible surface') for each residue in a structure. This is the accessibility of the residue when taken out of the protein together with the backbone atoms of any residue it is covalently bound to.|Solvent accessibility might be calculated for the backbone, sidechain and total (backbone plus sidechain).", "label": "Protein residue surface calculation (vacuum accessible)"}, "operation_1823": {"class_id": "http://edamontology.org/operation_1823", "definitions": "Calculate the solvent accessibility ('accessible molecular surface') for a structure as a whole.", "label": "Protein surface calculation (accessible molecular)"}, "operation_1822": {"class_id": "http://edamontology.org/operation_1822", "definitions": "Calculate the solvent accessibility ('vacuum molecular surface') for each residue in a structure. This is the accessibility of the residue when taken out of the protein together with the backbone atoms of any residue it is covalently bound to.|Solvent accessibility might be calculated for the backbone, sidechain and total (backbone plus sidechain).", "label": "Protein residue surface calculation (vacuum molecular)"}, "operation_3791": {"class_id": "http://edamontology.org/operation_3791", "definitions": "A method whereby data on several variants are \"collapsed\" into a single covariate based on regions such as genes.|Genome-wide association studies (GWAS) analyse a genome-wide set of genetic variants in different individuals to see if any variant is associated with a trait. Traditional association techniques can lack the power to detect the significance of rare variants individually, or measure their compound effect (rare variant burden).  \"Collapsing methods\" were developed to overcome these problems. ", "label": "Collapsing methods"}, "topic_3489": {"class_id": "http://edamontology.org/topic_3489", "definitions": "The general handling of data stored in digital archives such as databanks, databases proper, web portals and other data resources.|This includes databases for the results of scientific experiments, the application of high-throughput technology, computational analysis and the scientific literature.  It covers the management and manipulation of digital documents, including database records, files and reports.", "label": "Database management"}, "operation_0484": {"class_id": "http://edamontology.org/operation_0484", "definitions": "Find single nucleotide polymorphisms (SNPs) - single nucleotide change in base positions - between sequences. Typically done for sequences from a high-throughput sequencing experiment that differ from a reference genome and which might, especially by reference to population frequency or functional data, indicate a polymorphism.|This includes functional SNPs for large-scale genotyping purposes, disease-associated non-synonymous SNPs etc.", "label": "SNP detection"}, "operation_0485": {"class_id": "http://edamontology.org/operation_0485", "definitions": "Generate a physical (radiation hybrid) map of genetic markers in a DNA sequence using provided radiation hybrid (RH) scores for one or more markers.", "label": "Radiation Hybrid Mapping"}, "operation_0486": {"class_id": "http://edamontology.org/operation_0486", "definitions": "Map the genetic architecture of dynamic complex traits.|This can involve characterisation of the underlying quantitative trait loci (QTLs) or nucleotides (QTNs).", "label": "Functional mapping"}, "operation_0487": {"class_id": "http://edamontology.org/operation_0487", "definitions": "Infer haplotypes, either alleles at multiple loci that are transmitted together on the same chromosome, or a set of single nucleotide polymorphisms (SNPs) on a single chromatid that are statistically associated.|Haplotype inference can help in population genetic studies and the identification of complex disease genes, , and is typically based on aligned single nucleotide polymorphism (SNP) fragments. Haplotype comparison is a useful way to characterize the genetic variation between individuals. An individual's haplotype describes which nucleotide base occurs at each position for a set of common SNPs. Tools might use combinatorial functions (for example parsimony) or a likelihood function or model with optimisation such as minimum error correction (MEC) model, expectation-maximisation algorithm (EM), genetic algorithm or Markov chain Monte Carlo (MCMC).", "label": "Haplotype mapping"}, "operation_0480": {"class_id": "http://edamontology.org/operation_0480", "definitions": "Model, analyse or edit amino acid side chain conformation in protein structure, optimize side-chain packing, hydrogen bonding etc.|Methods might use a residue rotamer library.", "label": "Protein modelling (side chains)"}, "operation_0481": {"class_id": "http://edamontology.org/operation_0481", "definitions": "Model loop conformation in protein structures.", "label": "Protein modelling (loops)"}, "operation_0482": {"class_id": "http://edamontology.org/operation_0482", "definitions": "Model protein-ligand (for example protein-peptide) binding using comparative modelling or other techniques.|Methods aim to predict the position and orientation of a ligand bound to a protein receptor or enzyme.", "label": "Protein-ligand docking"}, "operation_0483": {"class_id": "http://edamontology.org/operation_0483", "definitions": "Predict or optimise RNA sequences (sequence pools) with likely secondary and tertiary structure for in vitro selection.", "label": "Structured RNA prediction and optimisation"}, "data_2387": {"class_id": "http://edamontology.org/data_2387", "definitions": "Identifier of an entry from the TAIR database.", "label": "TAIR accession"}, "operation_0488": {"class_id": "http://edamontology.org/operation_0488", "definitions": "Calculate linkage disequilibrium; the non-random association of alleles or polymorphisms at two or more loci (not necessarily on the same chromosome).|Linkage disequilibrium is identified where a combination of alleles (or genetic markers) occurs more or less frequently in a population than expected by chance formation of haplotypes.", "label": "Linkage disequilibrium calculation"}, "operation_0489": {"class_id": "http://edamontology.org/operation_0489", "definitions": "Predict genetic code from analysis of codon usage data.", "label": "Genetic code prediction"}, "format_3242": {"class_id": "http://edamontology.org/format_3242", "definitions": "Tabular Molecular Interaction format (MITAB), standardised by HUPO PSI MI.", "label": "PSI MI TAB (MITAB)"}, "topic_3318": {"class_id": "http://edamontology.org/topic_3318", "definitions": "The study of matter, space and time, and related concepts such as energy and force.", "label": "Physics"}, "operation_3465": {"class_id": "http://edamontology.org/operation_3465", "definitions": "Identify a correlation, i.e. a statistical relationship between two random variables or two sets of data.", "label": "Correlation"}, "operation_3463": {"class_id": "http://edamontology.org/operation_3463", "definitions": "Analyse the correlation patterns among genes across across a variety of experiments, microarray samples etc.", "label": "Gene expression correlation"}, "format_1208": {"class_id": "http://edamontology.org/format_1208", "definitions": "Alphabet for a protein sequence with possible ambiguity, unknown positions and non-sequence characters.|Non-sequence characters may be used for gaps and translation stop.", "label": "protein"}, "format_1209": {"class_id": "http://edamontology.org/format_1209", "definitions": "Alphabet for the consensus of two or more molecular sequences.", "label": "consensus"}, "format_1206": {"class_id": "http://edamontology.org/format_1206", "definitions": "Alphabet for a molecular sequence with possible unknown positions but without ambiguity or non-sequence characters.", "label": "unambiguous pure"}, "format_1207": {"class_id": "http://edamontology.org/format_1207", "definitions": "Alphabet for a nucleotide sequence with possible ambiguity, unknown positions and non-sequence characters.|Non-sequence characters may be used for example for gaps.", "label": "nucleotide"}, "format_1200": {"class_id": "http://edamontology.org/format_1200", "definitions": "SMILES ARbitrary Target Specification (SMARTS) format for chemical structure specification, which is a subset of the SMILES line notation.", "label": "smarts"}, "operation_0468": {"class_id": "http://edamontology.org/operation_0468", "definitions": "Predict helical secondary structure of protein sequences.", "label": "Protein secondary structure prediction (helices)"}, "operation_0469": {"class_id": "http://edamontology.org/operation_0469", "definitions": "Predict turn structure (for example beta hairpin turns) of protein sequences.", "label": "Protein secondary structure prediction (turns)"}, "data_3154": {"class_id": "http://edamontology.org/data_3154", "definitions": "An alignment of protein sequences and/or structures.", "label": "Protein alignment"}, "operation_0464": {"class_id": "http://edamontology.org/operation_0464", "definitions": "Identify or predict tRNA genes in genomic sequences (tRNA).", "label": "tRNA gene prediction"}, "operation_0465": {"class_id": "http://edamontology.org/operation_0465", "definitions": "Assess binding specificity of putative siRNA sequence(s), for example for a functional assay, typically with respect to designing specific siRNA sequences.", "label": "siRNA binding specificity prediction"}, "operation_0463": {"class_id": "http://edamontology.org/operation_0463", "definitions": "Identify or predict microRNA sequences (miRNA) and precursors or microRNA targets / binding sites in a DNA sequence.", "label": "microRNA detection"}, "data_3153": {"class_id": "http://edamontology.org/data_3153", "definitions": "An image of a protein.", "label": "Protein image"}, "operation_0461": {"class_id": "http://edamontology.org/operation_0461", "definitions": "Calculate curvature and flexibility / stiffness of a nucleotide sequence.|This includes properties such as.", "label": "Nucleic acid curvature calculation"}, "topic_2840": {"class_id": "http://edamontology.org/topic_2840", "definitions": "Toxins and the adverse effects of these chemical substances on living organisms.", "label": "Toxicology"}, "topic_2842": {"class_id": "http://edamontology.org/topic_2842", "definitions": "Parallelised sequencing processes that are capable of sequencing many thousands of sequences simultaneously.", "label": "High-throughput sequencing"}, "topic_2844": {"class_id": "http://edamontology.org/topic_2844", "definitions": "The comparison and grouping together of molecular structures on the basis of similarity; generate, process or analyse structural clusters.", "label": "Structural clustering"}, "topic_2846": {"class_id": "http://edamontology.org/topic_2846", "definitions": "Gene regulatory networks.", "label": "Gene regulatory networks"}, "topic_2847": {"class_id": "http://edamontology.org/topic_2847", "definitions": "Informatics resources dedicated to one or more specific diseases (not diseases in general).", "label": "Disease (specific)"}, "data_3496": {"class_id": "http://edamontology.org/data_3496", "definitions": "A raw RNA sequence.", "label": "RNA sequence (raw)"}, "data_1368": {"class_id": "http://edamontology.org/data_1368", "definitions": "A protein signature of the type used in the EMBASSY Signature package.", "label": "Domainatrix signature"}, "topic_3448": {"class_id": "http://edamontology.org/topic_3448", "definitions": "The study of matter by studying the diffraction pattern from firing neutrons at a sample, typically to determine atomic and/or magnetic structure.", "label": "Neutron diffraction"}, "data_1361": {"class_id": "http://edamontology.org/data_1361", "definitions": "A profile (typically representing a sequence alignment) that is a simple matrix of nucleotide (or amino acid) counts per position.", "label": "Position frequency matrix"}, "data_1362": {"class_id": "http://edamontology.org/data_1362", "definitions": "A profile (typically representing a sequence alignment) that is weighted matrix of nucleotide (or amino acid) counts per position.|Contributions of individual sequences to the matrix might be uneven (weighted).", "label": "Position weight matrix"}, "data_1363": {"class_id": "http://edamontology.org/data_1363", "definitions": "A profile (typically representing a sequence alignment) derived from a matrix of nucleotide (or amino acid) counts per position that reflects information content at each position.", "label": "Information content matrix"}, "data_1364": {"class_id": "http://edamontology.org/data_1364", "definitions": "A statistical Markov model of a system which is assumed to be a Markov process with unobserved (hidden) states.  For example, a hidden Markov model representation of a set or alignment of sequences.", "label": "Hidden Markov model"}, "data_1365": {"class_id": "http://edamontology.org/data_1365", "definitions": "One or more fingerprints (sequence classifiers) as used in the PRINTS database.", "label": "Fingerprint"}, "operation_3260": {"class_id": "http://edamontology.org/operation_3260", "definitions": "Infer a transcriptome sequence by mapping short reads to a reference genome.", "label": "Transcriptome assembly (mapping)"}, "format_1370": {"class_id": "http://edamontology.org/format_1370", "definitions": "Format of a hidden Markov model representation used by the HMMER package.", "label": "HMMER format"}, "operation_3267": {"class_id": "http://edamontology.org/operation_3267", "definitions": "Convert one set of sequence coordinates to another, e.g. convert coordinates of one assembly to another, cDNA to genomic, CDS to genomic, protein translation to genomic etc.", "label": "Sequence coordinate conversion"}, "operation_0448": {"class_id": "http://edamontology.org/operation_0448", "definitions": "Analyse character conservation in a molecular sequence alignment, for example to derive a consensus sequence.|Use this concept for methods that calculate substitution rates, estimate relative site variability, identify sites with biased properties, derive a consensus sequence, or identify highly conserved or very poorly conserved sites, regions, blocks etc.", "label": "Sequence alignment analysis (conservation)"}, "format_1571": {"class_id": "http://edamontology.org/format_1571", "definitions": "Entry format for the textual abstract of signatures in an InterPro entry and its protein matches.|References are included and a functional inference is made where possible.", "label": "InterPro entry abstract format"}, "format_3752": {"class_id": "http://edamontology.org/format_3752", "definitions": "Tabular data represented as comma-separated values in a text file.", "label": "CSV"}, "operation_3793": {"class_id": "http://edamontology.org/operation_3793", "definitions": "Counting and summarising the number of short sequence reads that map to genomic features.", "label": "Read summarisation"}, "operation_2408": {"class_id": "http://edamontology.org/operation_2408", "definitions": "Analyse features in molecular sequences.", "label": "Sequence feature analysis"}, "operation_2409": {"class_id": "http://edamontology.org/operation_2409", "definitions": "Basic (non-analytical) operations of some data, either a file or equivalent entity in memory, such that the same basic type of data is consumed as input and generated as output.", "label": "Data handling"}, "operation_2403": {"class_id": "http://edamontology.org/operation_2403", "definitions": "Analyse one or more known molecular sequences.", "label": "Sequence analysis"}, "operation_2406": {"class_id": "http://edamontology.org/operation_2406", "definitions": "Analyse protein structural data.", "label": "Protein structure analysis"}, "operation_2407": {"class_id": "http://edamontology.org/operation_2407", "definitions": "Process (read and / or write) annotation of some type, typically annotation on an entry from a biological or biomedical database entity.", "label": "Annotation processing"}, "operation_2404": {"class_id": "http://edamontology.org/operation_2404", "definitions": "Analyse molecular sequence motifs.", "label": "Sequence motif analysis"}, "operation_2405": {"class_id": "http://edamontology.org/operation_2405", "definitions": "Process (read and / or write) protein interaction data.", "label": "Protein interaction data processing"}, "data_3498": {"class_id": "http://edamontology.org/data_3498", "definitions": "Data on gene sequence variations resulting large-scale genotyping and DNA sequencing projects.|Variations are stored along with a reference genome.", "label": "Sequence variations"}, "operation_3797": {"class_id": "http://edamontology.org/operation_3797", "definitions": "The calculation of species richness for a number of individual samples, based on plots of the number of species as a function of the number of samples (rarefaction curves).", "label": "Rarefaction"}, "data_1098": {"class_id": "http://edamontology.org/data_1098", "definitions": "Accession number of a RefSeq database entry.", "label": "RefSeq accession"}, "data_1099": {"class_id": "http://edamontology.org/data_1099", "definitions": "Accession number of a UniProt (protein sequence) database entry. May contain version or isoform number.", "label": "UniProt accession (extended)"}, "data_1096": {"class_id": "http://edamontology.org/data_1096", "definitions": "Accession number of a protein sequence database entry.", "label": "Sequence accession (protein)"}, "data_1097": {"class_id": "http://edamontology.org/data_1097", "definitions": "Accession number of a nucleotide sequence database entry.", "label": "Sequence accession (nucleic acid)"}, "data_1094": {"class_id": "http://edamontology.org/data_1094", "definitions": "A label (text token) describing a type of molecular sequence.|Sequence type might reflect the molecule (protein, nucleic acid etc) or the sequence itself (gapped, ambiguous etc).", "label": "Sequence type"}, "data_1095": {"class_id": "http://edamontology.org/data_1095", "definitions": "The name of a sequence-based entity adhering to the standard sequence naming scheme used by all EMBOSS applications.", "label": "EMBOSS Uniform Sequence Address"}, "data_1092": {"class_id": "http://edamontology.org/data_1092", "definitions": "Class of an object from the WormBase database.|A WormBase class describes the type of object such as 'sequence' or 'protein'.", "label": "WormBase class"}, "data_1093": {"class_id": "http://edamontology.org/data_1093", "definitions": "A persistent, unique identifier of a molecular sequence database entry.", "label": "Sequence accession"}, "data_1091": {"class_id": "http://edamontology.org/data_1091", "definitions": "Name of an object from the WormBase database, usually a human-readable name.", "label": "WormBase name"}, "operation_2222": {"class_id": "http://edamontology.org/operation_2222", "definitions": "Search and retrieve documentation on a bioinformatics ontology.", "label": "Data retrieval (ontology annotation)"}, "operation_0440": {"class_id": "http://edamontology.org/operation_0440", "definitions": "Identify or predict whole promoters or promoter elements (transcription start sites, RNA polymerase binding site, transcription factor binding sites, promoter enhancers etc) in DNA sequences.|Methods might recognize CG content, CpG islands, splice sites, polyA signals etc.", "label": "Promoter prediction"}, "operation_2224": {"class_id": "http://edamontology.org/operation_2224", "definitions": "Query an ontology and retrieve concepts or relations.", "label": "Data retrieval (ontology concept)"}, "operation_0441": {"class_id": "http://edamontology.org/operation_0441", "definitions": "Identify, predict or analyse cis-regulatory elements  in DNA sequences (TATA box, Pribnow box, SOS box, CAAT box, CCAAT box, operator etc.) or in RNA sequences (e.g. riboswitches).|Cis-regulatory elements (cis-elements) regulate the expression of genes located on the same strand from which the element was transcribed. Cis-elements are found in the 5' promoter region of the gene, in an intron, or in the 3' untranslated region. Cis-elements are often binding sites of one or more trans-acting factors. They also occur in RNA sequences, e.g. a riboswitch is a region of an mRNA molecule that bind a small target molecule that regulates the gene's activity.", "label": "cis-regulatory element prediction"}, "format_1974": {"class_id": "http://edamontology.org/format_1974", "definitions": "General Feature Format (GFF) of sequence features.", "label": "GFF2"}, "format_1975": {"class_id": "http://edamontology.org/format_1975", "definitions": "Generic Feature Format version 3 (GFF3) of sequence features.", "label": "GFF3"}, "format_1976": {"class_id": "http://edamontology.org/format_1976", "definitions": "PIR feature format.", "label": "pir"}, "format_1977": {"class_id": "http://edamontology.org/format_1977", "definitions": "Swiss-Prot feature format.", "label": "swiss feature"}, "format_1970": {"class_id": "http://edamontology.org/format_1970", "definitions": "Jackknifer output sequence non-interleaved format.", "label": "jackknifernon"}, "format_1971": {"class_id": "http://edamontology.org/format_1971", "definitions": "Mega non-interleaved output sequence format.", "label": "meganon sequence format"}, "format_1972": {"class_id": "http://edamontology.org/format_1972", "definitions": "NCBI FASTA sequence format with NCBI-style IDs.|There are several variants of this.", "label": "NCBI format"}, "format_1973": {"class_id": "http://edamontology.org/format_1973", "definitions": "Nexus/paup non-interleaved sequence format.", "label": "nexusnon"}, "format_1978": {"class_id": "http://edamontology.org/format_1978", "definitions": "DAS GFF (XML) feature format.", "label": "DASGFF"}, "format_1979": {"class_id": "http://edamontology.org/format_1979", "definitions": "EMBOSS debugging trace feature format of full internal data content.", "label": "debug-feat"}, "operation_0445": {"class_id": "http://edamontology.org/operation_0445", "definitions": "Identify or predict transcription factor binding sites in DNA sequences.", "label": "Transcription factor binding site prediction"}, "topic_1317": {"class_id": "http://edamontology.org/topic_1317", "definitions": "The molecular structure of biological molecules, particularly macromolecules such as proteins and nucleic acids.|This includes experimental methods for biomolecular structure determination, such as X-ray crystallography, nuclear magnetic resonance (NMR), circular dichroism (CD) spectroscopy, microscopy etc., including the assignment or modelling of molecular structure from such data.", "label": "Structural biology"}, "operation_1768": {"class_id": "http://edamontology.org/operation_1768", "definitions": "Identify folding families of related RNAs.", "label": "Nucleic acid folding family identification"}, "data_1783": {"class_id": "http://edamontology.org/data_1783", "definitions": "Name of a gene from Aspergillus Genome Database.", "label": "Gene name (ASPGD)"}, "data_1786": {"class_id": "http://edamontology.org/data_1786", "definitions": "Primary name of a gene from EcoGene Database.", "label": "Gene name (EcoGene primary)"}, "data_1787": {"class_id": "http://edamontology.org/data_1787", "definitions": "Name of a gene from MaizeGDB (maize genes) database.", "label": "Gene name (MaizeGDB)"}, "data_1784": {"class_id": "http://edamontology.org/data_1784", "definitions": "Name of a gene from Candida Genome Database.", "label": "Gene name (CGD)"}, "data_1785": {"class_id": "http://edamontology.org/data_1785", "definitions": "Name of a gene from dictyBase database.", "label": "Gene name (dictyBase)"}, "data_1788": {"class_id": "http://edamontology.org/data_1788", "definitions": "Name of a gene from Saccharomyces Genome Database.", "label": "Gene name (SGD)"}, "data_1789": {"class_id": "http://edamontology.org/data_1789", "definitions": "Name of a gene from Tetrahymena Genome Database.", "label": "Gene name (TGD)"}, "data_1140": {"class_id": "http://edamontology.org/data_1140", "definitions": "Unique identifier (number) of a hidden Markov model from the Superfamily database.", "label": "Superfamily hidden Markov model number"}, "data_1141": {"class_id": "http://edamontology.org/data_1141", "definitions": "Accession number of an entry (family) from the TIGRFam database.", "label": "TIGRFam ID"}, "data_1142": {"class_id": "http://edamontology.org/data_1142", "definitions": "A ProDom domain family accession number.|ProDom is a protein domain family database.", "label": "ProDom accession number"}, "data_1143": {"class_id": "http://edamontology.org/data_1143", "definitions": "Identifier of an entry from the TRANSFAC database.", "label": "TRANSFAC accession number"}, "data_1144": {"class_id": "http://edamontology.org/data_1144", "definitions": "Accession number of an entry from the ArrayExpress database.", "label": "ArrayExpress accession number"}, "data_1145": {"class_id": "http://edamontology.org/data_1145", "definitions": "PRIDE experiment accession number.", "label": "PRIDE experiment accession number"}, "data_1146": {"class_id": "http://edamontology.org/data_1146", "definitions": "Identifier of an entry from the EMDB electron microscopy database.", "label": "EMDB ID"}, "data_1147": {"class_id": "http://edamontology.org/data_1147", "definitions": "Accession number of an entry from the GEO database.", "label": "GEO accession number"}, "data_1148": {"class_id": "http://edamontology.org/data_1148", "definitions": "Identifier of an entry from the GermOnline database.", "label": "GermOnline ID"}, "data_1149": {"class_id": "http://edamontology.org/data_1149", "definitions": "Identifier of an entry from the EMAGE database.", "label": "EMAGE ID"}, "topic_2809": {"class_id": "http://edamontology.org/topic_2809", "definitions": "A general area of bioinformatics study, typically the broad scope or category of content of a bioinformatics journal or conference proceeding.", "label": "Study topic"}, "format_2558": {"class_id": "http://edamontology.org/format_2558", "definitions": "An XML format resembling EMBL entry format.|This concept may be used for the any non-standard EMBL-like XML formats.", "label": "EMBL-like (XML)"}, "format_3239": {"class_id": "http://edamontology.org/format_3239", "definitions": "CopasiML, the native format of COPASI.", "label": "CopasiML"}, "data_3807": {"class_id": "http://edamontology.org/data_3807", "definitions": "Raw DDD movie acquisition from electron microscopy.", "label": "EM Movie"}, "data_3806": {"class_id": "http://edamontology.org/data_3806", "definitions": "Annotation on a structural 3D EM Map from electron microscopy. This might include one or several locations in the map of the known features of a particular macromolecule.", "label": "3D EM Mask"}, "data_3805": {"class_id": "http://edamontology.org/data_3805", "definitions": "Structural 3D model (volume map) from electron microscopy.", "label": "3D EM Map"}, "data_3808": {"class_id": "http://edamontology.org/data_3808", "definitions": "Raw acquistion from electron microscopy or average of an aligned DDD movie.", "label": "EM Micrograph"}, "format_3235": {"class_id": "http://edamontology.org/format_3235", "definitions": "Cytoband format for chromosome cytobands.|Reflects a UCSC Browser DB table.", "label": "Cytoband format"}, "topic_0598": {"class_id": "http://edamontology.org/topic_0598", "definitions": "Sequence motifs, or sequence profiles derived from an alignment of molecular sequences of a particular type.|This includes comparison, discovery, recognition etc. of sequence motifs.", "label": "Sequence motif or profile"}, "format_2553": {"class_id": "http://edamontology.org/format_2553", "definitions": "XML format for a sequence feature table.", "label": "Sequence feature table format (XML)"}, "format_2552": {"class_id": "http://edamontology.org/format_2552", "definitions": "Data format for a molecular sequence record.", "label": "Sequence record format (XML)"}, "topic_0593": {"class_id": "http://edamontology.org/topic_0593", "definitions": "An analytical technique that exploits the magenetic properties of certain atomic nuclei to provide information on the structure, dynamics, reaction state and chemical environment of molecules.", "label": "NMR"}, "topic_0594": {"class_id": "http://edamontology.org/topic_0594", "definitions": "The classification of molecular sequences based on some measure of their similarity.|Methods including sequence motifs, profile and other diagnostic elements which (typically) represent conserved patterns (of residues or properties) in molecular sequences.", "label": "Sequence classification"}, "format_2551": {"class_id": "http://edamontology.org/format_2551", "definitions": "Data format for a molecular sequence record.", "label": "Sequence record format (text)"}, "format_2550": {"class_id": "http://edamontology.org/format_2550", "definitions": "OBO ontology XML format.", "label": "OBO-XML"}, "format_2557": {"class_id": "http://edamontology.org/format_2557", "definitions": "XML format for a phylogenetic tree.", "label": "Phylogenetic tree format (XML)"}, "format_2556": {"class_id": "http://edamontology.org/format_2556", "definitions": "Text format for a phylogenetic tree.", "label": "Phylogenetic tree format (text)"}, "topic_3794": {"class_id": "http://edamontology.org/topic_3794", "definitions": "An antibody-based technique used to map in vivo RNA-protein interactions.", "label": "RNA immunoprecipitation"}, "format_2554": {"class_id": "http://edamontology.org/format_2554", "definitions": "Text format for molecular sequence alignment information.", "label": "Alignment format (text)"}, "topic_3068": {"class_id": "http://edamontology.org/topic_3068", "definitions": "The scientific literature, language processing, reference information, and documentation.|This includes the documentation of resources such as tools, services and databases, user support, how to get help etc.", "label": "Literature and language"}, "topic_0175": {"class_id": "http://edamontology.org/topic_0175", "definitions": "The modelling of the three-dimensional structure of a protein using known sequence and structural data.", "label": "Homology modelling"}, "topic_0102": {"class_id": "http://edamontology.org/topic_0102", "definitions": "The mapping of complete (typically nucleotide) sequences. Mapping (in the sense of short read alignment, or more generally, just alignment) has application in RNA-Seq analysis (mapping of transcriptomics reads), variant discovery (e.g. mapping of exome capture), and re-sequencing (mapping of WGS reads). |This includes resources that aim to identify, map or analyse genetic markers in DNA sequences, for example to produce a genetic (linkage) map of a chromosome or genome or to analyse genetic linkage and synteny. It also includes resources for physical (sequence) maps of a DNA sequence showing the physical distance (base pairs) between features or landmarks such as restriction sites, cloned DNA fragments, genes and other genetic markers.  It also covers for example the alignment of sequences of (typically millions) of short reads to a reference genome.  ", "label": "Mapping"}, "data_1410": {"class_id": "http://edamontology.org/data_1410", "definitions": "A number defining the penalty for opening gaps at the termini of an alignment, either from the N/C terminal of protein or 5'/3' terminal of nucleotide sequences.", "label": "Terminal gap opening penalty"}, "operation_1769": {"class_id": "http://edamontology.org/operation_1769", "definitions": "Compute energies of nucleic acid folding, e.g. minimum folding energies for DNA or RNA sequences or energy landscape of RNA mutants.", "label": "Nucleic acid folding energy calculation"}, "data_1553": {"class_id": "http://edamontology.org/data_1553", "definitions": "Information on a node from the CATH database.|The report (for example http://www.cathdb.info/cathnode/1.10.10.10) includes CATH code (of the node and upper levels in the hierarchy), classification text (of appropriate levels in hierarchy), list of child nodes, representative domain and other relevant data and links.", "label": "CATH node"}, "data_2958": {"class_id": "http://edamontology.org/data_2958", "definitions": "A melting curve of a double-stranded nucleic acid molecule (DNA or DNA/RNA).|Shows the proportion of nucleic acid which are double-stranded versus temperature.", "label": "Nucleic acid melting curve"}, "data_1411": {"class_id": "http://edamontology.org/data_1411", "definitions": "A number defining the penalty for extending gaps at the termini of an alignment, either from the N/C terminal of protein or 5'/3' terminal of nucleotide sequences.", "label": "Terminal gap extension penalty"}, "data_1557": {"class_id": "http://edamontology.org/data_1557", "definitions": "Information on a protein 'architecture' node from the CATH database.", "label": "CATH architecture"}, "data_1556": {"class_id": "http://edamontology.org/data_1556", "definitions": "Information on a protein 'class' node from the CATH database.", "label": "CATH class"}, "data_1555": {"class_id": "http://edamontology.org/data_1555", "definitions": "An EMBASSY domain classification file (DCF) of classification and other data for domains from SCOP or CATH, in EMBL-like format.", "label": "EMBASSY domain classification"}, "data_1554": {"class_id": "http://edamontology.org/data_1554", "definitions": "Information on a node from the SCOP database.", "label": "SCOP node"}, "topic_0107": {"class_id": "http://edamontology.org/topic_0107", "definitions": "The study of codon usage in nucleotide sequence(s), genetic codes and so on.", "label": "Genetic codes and codon usage"}, "data_1559": {"class_id": "http://edamontology.org/data_1559", "definitions": "Information on a protein 'homologous superfamily' node from the CATH database.", "label": "CATH homologous superfamily"}, "data_1558": {"class_id": "http://edamontology.org/data_1558", "definitions": "Information on a protein 'topology' node from the CATH database.", "label": "CATH topology"}, "data_2956": {"class_id": "http://edamontology.org/data_2956", "definitions": "An informative report about the properties or features of one or more protein secondary structures.", "label": "Protein secondary structure report"}, "data_2957": {"class_id": "http://edamontology.org/data_2957", "definitions": "A Hopp and Woods plot of predicted antigenicity of a peptide or protein.", "label": "Hopp and Woods plot"}, "data_2954": {"class_id": "http://edamontology.org/data_2954", "definitions": "Data derived from the analysis of a scientific text such as a full text article from a scientific journal.", "label": "Article report"}, "data_2223": {"class_id": "http://edamontology.org/data_2223", "definitions": "Data concerning a biological ontology.", "label": "Ontology metadata"}, "data_2309": {"class_id": "http://edamontology.org/data_2309", "definitions": "Identifier of a biological reaction from the SABIO-RK reactions database.", "label": "Reaction ID (SABIO-RK)"}, "data_2308": {"class_id": "http://edamontology.org/data_2308", "definitions": "An informative report on the taxonomy of a specific virus.", "label": "Virus annotation (taxonomy)"}, "operation_3470": {"class_id": "http://edamontology.org/operation_3470", "definitions": "Predict RNA secondary structure by analysis, e.g. probabilistic analysis, of the shape of RNA folds.", "label": "RNA secondary structure prediction (shape-based)"}, "operation_2934": {"class_id": "http://edamontology.org/operation_2934", "definitions": "Generate a view of clustered quantitative data, annotated with textual information.", "label": "Cluster textual view generation"}, "data_2307": {"class_id": "http://edamontology.org/data_2307", "definitions": "An informative report on a specific virus.", "label": "Virus annotation"}, "data_1418": {"class_id": "http://edamontology.org/data_1418", "definitions": "Alignment of molecular sequence(s) to a hidden Markov model(s).", "label": "Sequence-profile alignment (HMM)"}, "format_3160": {"class_id": "http://edamontology.org/format_3160", "definitions": "NeXML is a standardised XML format for rich phyloinformatic data.", "label": "NeXML"}, "topic_0108": {"class_id": "http://edamontology.org/topic_0108", "definitions": "The translation of mRNA into protein and subsequent protein processing in the cell.", "label": "Protein expression"}, "operation_2008": {"class_id": "http://edamontology.org/operation_2008", "definitions": "Identify or predict siRNA duplexes in RNA.", "label": "siRNA duplex prediction"}, "data_2302": {"class_id": "http://edamontology.org/data_2302", "definitions": "Unique identifier of an entry from the STRING database of protein-protein interactions.", "label": "STRING ID"}, "format_2548": {"class_id": "http://edamontology.org/format_2548", "definitions": "Format for a sequence feature table.", "label": "Sequence feature table format"}, "format_2549": {"class_id": "http://edamontology.org/format_2549", "definitions": "OBO ontology text format.", "label": "OBO"}, "data_2301": {"class_id": "http://edamontology.org/data_2301", "definitions": "A specification of a chemical structure in SMILES format.", "label": "SMILES string"}, "format_2017": {"class_id": "http://edamontology.org/format_2017", "definitions": "Data format for an amino acid index.", "label": "Amino acid index format"}, "format_2014": {"class_id": "http://edamontology.org/format_2014", "definitions": "Data format for a sequence-profile alignment.", "label": "Sequence-profile alignment format"}, "data_2300": {"class_id": "http://edamontology.org/data_2300", "definitions": "Name of an entry (gene) from the NCBI genes database.", "label": "Gene name (NCBI)"}, "format_2013": {"class_id": "http://edamontology.org/format_2013", "definitions": "Data format for a biological pathway or network.", "label": "Biological pathway or network format"}, "format_2546": {"class_id": "http://edamontology.org/format_2546", "definitions": "A format resembling FASTA format.|This concept may be used for the many non-standard FASTA-like formats.", "label": "FASTA-like"}, "format_2547": {"class_id": "http://edamontology.org/format_2547", "definitions": "A sequence format resembling uniprotkb entry format.", "label": "uniprotkb-like format"}, "data_1427": {"class_id": "http://edamontology.org/data_1427", "definitions": "Character data with discrete states that may be read during phylogenetic tree calculation.", "label": "Phylogenetic discrete data"}, "data_1426": {"class_id": "http://edamontology.org/data_1426", "definitions": "Continuous quantitative data that may be read during phylogenetic tree calculation.", "label": "Phylogenetic continuous quantitative data"}, "operation_3084": {"class_id": "http://edamontology.org/operation_3084", "definitions": "Predict general (non-positional) functional properties of a protein from analysing its sequence.|For functional properties that are positional, use 'Protein site detection' instead.", "label": "Protein function prediction (from sequence)"}, "data_2313": {"class_id": "http://edamontology.org/data_2313", "definitions": "Annotation on or information derived from one or more specific carbohydrate 3D structure(s).", "label": "Carbohydrate report"}, "data_0958": {"class_id": "http://edamontology.org/data_0958", "definitions": "Basic information about one or more bioinformatics applications or packages, such as name, type, description, or other documentation.", "label": "Tool metadata"}, "data_2315": {"class_id": "http://edamontology.org/data_2315", "definitions": "An identifier assigned to sequence records processed by NCBI, made of the accession number of the database record followed by a dot and a version number.|Nucleotide sequence version contains two letters followed by six digits, a dot, and a version number (or for older nucleotide sequence records, the format is one letter followed by five digits, a dot, and a version number). Protein sequence version contains three letters followed by five digits, a dot, and a version number.", "label": "NCBI version"}, "data_2316": {"class_id": "http://edamontology.org/data_2316", "definitions": "The name of a cell line.", "label": "Cell line name"}, "data_1420": {"class_id": "http://edamontology.org/data_1420", "definitions": "Alignment of molecular sequences to a protein fingerprint from the PRINTS database.", "label": "Sequence-profile alignment (fingerprint)"}, "data_0954": {"class_id": "http://edamontology.org/data_0954", "definitions": "A mapping of the accession numbers (or other database identifier) of entries between (typically) two biological or biomedical databases.|The cross-mapping is typically a table where each row is an accession number and each column is a database being cross-referenced. The cells give the accession number or identifier of the corresponding entry in a database. If a cell in the table is not filled then no mapping could be found for the database. Additional information might be given on version, date etc.", "label": "Database cross-mapping"}, "data_0955": {"class_id": "http://edamontology.org/data_0955", "definitions": "An index of data of biological relevance.", "label": "Data index"}, "data_0956": {"class_id": "http://edamontology.org/data_0956", "definitions": "A report of an analysis of an index of biological data.", "label": "Data index report"}, "data_0957": {"class_id": "http://edamontology.org/data_0957", "definitions": "Basic information on bioinformatics database(s) or other data sources such as name, type, description, URL etc.", "label": "Database metadata"}, "data_0950": {"class_id": "http://edamontology.org/data_0950", "definitions": "A biological model represented in mathematical terms.", "label": "Mathematical model"}, "data_0951": {"class_id": "http://edamontology.org/data_0951", "definitions": "A value representing estimated statistical significance of some observed data; typically sequence database hits.", "label": "Statistical estimate score"}, "data_1429": {"class_id": "http://edamontology.org/data_1429", "definitions": "Phylogenetic invariants data for testing alternative tree topologies.", "label": "Phylogenetic invariants"}, "data_1428": {"class_id": "http://edamontology.org/data_1428", "definitions": "One or more cliques of mutually compatible characters that are generated, for example from analysis of discrete character data, and are used to generate a phylogeny.", "label": "Phylogenetic character cliques"}, "topic_2229": {"class_id": "http://edamontology.org/topic_2229", "definitions": "Cells, such as key genes and proteins involved in the cell cycle.", "label": "Cell biology"}, "operation_1780": {"class_id": "http://edamontology.org/operation_1780", "definitions": "Submit a molecular sequence to a database.", "label": "Sequence submission"}, "operation_1781": {"class_id": "http://edamontology.org/operation_1781", "definitions": "Analyse a known network of gene regulation.", "label": "Gene regulatory network analysis"}, "topic_1308": {"class_id": "http://edamontology.org/topic_1308", "definitions": "Matrix/scaffold attachment regions (MARs/SARs) in a DNA sequence.", "label": "Matrix/scaffold attachment sites"}, "operation_0255": {"class_id": "http://edamontology.org/operation_0255", "definitions": "Query the features (in a feature table) of molecular sequence(s).", "label": "Feature table query"}, "operation_0254": {"class_id": "http://edamontology.org/operation_0254", "definitions": "Extract a sequence feature table from a sequence database entry.", "label": "Data retrieval (feature table)"}, "operation_0257": {"class_id": "http://edamontology.org/operation_0257", "definitions": "Display basic information about a sequence alignment.", "label": "Data retrieval (sequence alignment)"}, "operation_0256": {"class_id": "http://edamontology.org/operation_0256", "definitions": "Compare the feature tables of two or more molecular sequences.", "label": "Sequence feature comparison"}, "operation_0250": {"class_id": "http://edamontology.org/operation_0250", "definitions": "Extract, calculate or predict non-positional (physical or chemical) properties of a protein, including any non-positional properties of the molecular sequence, from processing a protein sequence or 3D structure.|This includes methods to render and visualise the properties of a protein sequence, and a residue-level search for properties such as solvent accessibility, hydropathy, secondary structure, ligand-binding etc.", "label": "Protein property calculation"}, "operation_0253": {"class_id": "http://edamontology.org/operation_0253", "definitions": "Predict, recognise and identify positional features in molecular sequences such as key functional sites or regions.|Look at \"Protein feature detection\" (http://edamontology.org/operation_3092) and \"Nucleic acid feature detection\" (http://edamontology.org/operation_0415) in case more specific terms are needed.", "label": "Sequence feature detection"}, "operation_0252": {"class_id": "http://edamontology.org/operation_0252", "definitions": "Predict antigenicity, allergenicity / immunogenicity, allergic cross-reactivity etc of peptides and proteins.|This includes methods that generate a graphical rendering of antigenicity of a protein, such as a Hopp and Woods plot.|This is usually done in the development of peptide-specific antibodies or multi-epitope vaccines. Methods might use sequence data (for example motifs) and / or structural data.", "label": "Peptide immunogenicity prediction"}, "operation_0259": {"class_id": "http://edamontology.org/operation_0259", "definitions": "Compare (typically by aligning) two molecular sequence alignments.|See also 'Sequence profile alignment'.", "label": "Sequence alignment comparison"}, "operation_0258": {"class_id": "http://edamontology.org/operation_0258", "definitions": "Analyse a molecular sequence alignment.", "label": "Sequence alignment analysis"}, "format_3250": {"class_id": "http://edamontology.org/format_3250", "definitions": "spML is the format for describing proteomics sample processing, other than using gels, prior to mass spectrometric protein identification, standardised by HUPO PSI PS. It may also be applicable for metabolomics.", "label": "spML"}, "format_2303": {"class_id": "http://edamontology.org/format_2303", "definitions": "Entry format (HTML) for the STRING database of protein interaction.", "label": "STRING entry format (HTML)"}, "format_2306": {"class_id": "http://edamontology.org/format_2306", "definitions": "Gene Transfer Format (GTF), a restricted version of GFF.", "label": "GTF"}, "format_2304": {"class_id": "http://edamontology.org/format_2304", "definitions": "Entry format (XML) for the STRING database of protein interaction.", "label": "STRING entry format (XML)"}, "format_2305": {"class_id": "http://edamontology.org/format_2305", "definitions": "GFF feature format (of indeterminate version).", "label": "GFF"}, "topic_0003": {"class_id": "http://edamontology.org/topic_0003", "definitions": "A category denoting a rather broad domain or field of interest, of study, application, work, data, or technology. Topics have no clearly defined borders between each other.", "label": "Topic"}, "operation_0329": {"class_id": "http://edamontology.org/operation_0329", "definitions": "Predict the folding pathway(s) or non-native structural intermediates of a protein.", "label": "Protein folding pathway prediction"}, "operation_0328": {"class_id": "http://edamontology.org/operation_0328", "definitions": "Simulate the folding of a protein.", "label": "Protein folding simulation"}, "format_3257": {"class_id": "http://edamontology.org/format_3257", "definitions": "A shorthand non-XML serialisation of Resource Description Framework model, designed with human-readability in mind.", "label": "Notation3"}, "operation_0321": {"class_id": "http://edamontology.org/operation_0321", "definitions": "Evaluate the quality or correctness a protein three-dimensional model.|This includes methods that calculate poor quality residues. The scoring function to identify poor quality residues may consider residues with bad atoms or atoms with high B-factor, residues in the N- or C-terminal position, adjacent to an unstructured residue, non-canonical residues, glycine and proline (or adjacent to these such residues).|Model validation might involve checks for atomic packing, steric clashes (bumps), volume irregularities, agreement with electron density maps, number of amino acid residues, percentage of residues with missing or bad atoms, irregular Ramachandran Z-scores, irregular Chi-1 / Chi-2 normality scores, RMS-Z score on bonds and angles etc.|The PDB file format has had difficulties, inconsistencies and errors. Corrections can include identifying a meaningful sequence, removal of alternate atoms, correction of nomenclature problems, removal of incomplete residues and spurious waters, addition or removal of water, modelling of missing side chains, optimisation of cysteine bonds, regularisation of bond lengths, bond angles and planarities etc.", "label": "Protein model validation"}, "operation_0320": {"class_id": "http://edamontology.org/operation_0320", "definitions": "Assign a protein tertiary structure (3D coordinates), or other aspects of protein structure, from raw experimental data.", "label": "Protein structure assignment"}, "operation_0323": {"class_id": "http://edamontology.org/operation_0323", "definitions": "Construct a phylogenetic tree.|Phylogenetic trees are usually constructed from a set of sequences from which an alignment (or data matrix) is calculated.", "label": "Phylogenetic tree generation"}, "operation_0322": {"class_id": "http://edamontology.org/operation_0322", "definitions": "Refine (after evaluation) a model of a molecular structure (typically a protein structure) to reduce steric clashes, volume irregularities etc.", "label": "Molecular model refinement"}, "operation_0325": {"class_id": "http://edamontology.org/operation_0325", "definitions": "Compare two or more phylogenetic trees.|For example, to produce a consensus tree, subtrees, supertrees, calculate distances between trees or test topological similarity between trees (e.g. a congruence index) etc.", "label": "Phylogenetic tree comparison"}, "operation_0324": {"class_id": "http://edamontology.org/operation_0324", "definitions": "Analyse an existing phylogenetic tree or trees, typically to detect features or make predictions.", "label": "Phylogenetic tree analysis"}, "operation_0327": {"class_id": "http://edamontology.org/operation_0327", "definitions": "Infer a phylogenetic tree by comparing orthologous sequences in different species, particularly many closely related species (phylogenetic shadowing).|A phylogenetic 'shadow' represents the additive differences between individual sequences. By masking or 'shadowing' variable positions a conserved sequence is produced with few or none of the variations, which is then compared to the sequences of interest to identify significant regions of conservation.", "label": "Phylogenetic footprinting / shadowing"}, "operation_0326": {"class_id": "http://edamontology.org/operation_0326", "definitions": "Edit a phylogenetic tree.", "label": "Phylogenetic tree editing"}, "operation_3677": {"class_id": "http://edamontology.org/operation_3677", "definitions": "Differential binding analysis identifies binding sites in nucleic acid sequences that are statistically significantly differentially bound between sample groups.", "label": "Differential binding analysis"}, "operation_3675": {"class_id": "http://edamontology.org/operation_3675", "definitions": "Variant filtering is used to eliminate false positive variants based for example on base calling quality, strand and position information, and mapping info.", "label": "Variant filtering"}, "operation_3672": {"class_id": "http://edamontology.org/operation_3672", "definitions": "Annotate one or more sequences with functional information, such as cellular processes or metaobolic pathways, by reference to a controlled vocabulary - invariably the Gene Ontology (GO).", "label": "Gene functional annotation"}, "operation_2935": {"class_id": "http://edamontology.org/operation_2935", "definitions": "Visualise clustered quantitative data as set of different profiles, where each profile is plotted versus different entities or samples on the X-axis.", "label": "Clustering profile plotting"}, "data_2783": {"class_id": "http://edamontology.org/data_2783", "definitions": "Accession number of an entry (family) from the PANTHER database.", "label": "Protein family ID (PANTHER)"}, "data_2782": {"class_id": "http://edamontology.org/data_2782", "definitions": "Name of a domain from the SMART database.", "label": "SMART domain name"}, "data_2781": {"class_id": "http://edamontology.org/data_2781", "definitions": "Identifier of an entry from the RNA editing database (REDIdb).", "label": "REDIdb ID"}, "data_2780": {"class_id": "http://edamontology.org/data_2780", "definitions": "A stock number from The Arabidopsis information resource (TAIR).", "label": "Stock number (TAIR)"}, "data_2787": {"class_id": "http://edamontology.org/data_2787", "definitions": "A unique identifier of a whole genome assigned by the NCBI.", "label": "NCBI genome accession"}, "data_2786": {"class_id": "http://edamontology.org/data_2786", "definitions": "An identifier of a genome project assigned by NCBI.", "label": "NCBI Genome Project ID"}, "data_2785": {"class_id": "http://edamontology.org/data_2785", "definitions": "An accession of annotation on a (group of) viruses (catalogued in a database).", "label": "Virus ID"}, "data_2784": {"class_id": "http://edamontology.org/data_2784", "definitions": "A unique identifier for a virus from the RNAVirusDB database.|Could list (or reference) other taxa here from https://www.phenoscape.org/wiki/Taxonomic_Rank_Vocabulary.", "label": "RNAVirusDB ID"}, "data_2789": {"class_id": "http://edamontology.org/data_2789", "definitions": "Unique identifier for a membrane protein from the TopDB database.", "label": "Protein ID (TopDB)"}, "data_2788": {"class_id": "http://edamontology.org/data_2788", "definitions": "Data concerning, extracted from, or derived from the analysis of a sequence profile, such as its name, length, technical details about the profile or it's construction, the biological role or annotation, and so on.", "label": "Sequence profile data"}, "operation_1814": {"class_id": "http://edamontology.org/operation_1814", "definitions": "Query a tertiary structure data resource (typically a database) and retrieve structures, structure-related data and annotation.|This includes direct retrieval methods but not those that perform calculations on the sequence or structure.", "label": "Structure retrieval"}, "operation_1816": {"class_id": "http://edamontology.org/operation_1816", "definitions": "Calculate the positions of dots that are homogeneously distributed over the surface of a molecule.|A dot has three coordinates (x,y,z) and (typically) a color.", "label": "Surface rendering"}, "operation_1817": {"class_id": "http://edamontology.org/operation_1817", "definitions": "Calculate the solvent accessibility ('accessible surface') for each atom in a structure.|Waters are not considered.", "label": "Protein atom surface calculation (accessible)"}, "data_2141": {"class_id": "http://edamontology.org/data_2141", "definitions": "Size of the incremental 'step' a sequence window is moved over a sequence.", "label": "Window step size"}, "data_2140": {"class_id": "http://edamontology.org/data_2140", "definitions": "The concentration of a chemical compound.", "label": "Concentration"}, "data_2143": {"class_id": "http://edamontology.org/data_2143", "definitions": "An application report generated by the EMBOSS suite.", "label": "EMBOSS report"}, "data_2142": {"class_id": "http://edamontology.org/data_2142", "definitions": "An image of a graph generated by the EMBOSS suite.", "label": "EMBOSS graph"}, "data_2145": {"class_id": "http://edamontology.org/data_2145", "definitions": "An offset for a single-point sequence position.", "label": "Sequence offset"}, "data_2147": {"class_id": "http://edamontology.org/data_2147", "definitions": "An informative report on a transcription factor protein.|This might include conformational or physicochemical properties, as well as sequence information for transcription factor(s) binding sites.", "label": "Protein report (transcription factor)"}, "data_2146": {"class_id": "http://edamontology.org/data_2146", "definitions": "A value that serves as a threshold for a tool (usually to control scoring or output).", "label": "Threshold"}, "data_2149": {"class_id": "http://edamontology.org/data_2149", "definitions": "The name of a category of biological or bioinformatics database.", "label": "Database category name"}, "operation_3081": {"class_id": "http://edamontology.org/operation_3081", "definitions": "Edit, convert or otherwise change a molecular sequence alignment, either randomly or specifically.", "label": "Sequence alignment editing"}, "operation_3080": {"class_id": "http://edamontology.org/operation_3080", "definitions": "Edit, convert or otherwise change a  molecular tertiary structure, either randomly or specifically.", "label": "Structure editing"}, "operation_0493": {"class_id": "http://edamontology.org/operation_0493", "definitions": "Locally align exactly two molecular sequences.|Local alignment methods identify regions of local similarity.", "label": "Pairwise sequence alignment generation (local)"}, "operation_0492": {"class_id": "http://edamontology.org/operation_0492", "definitions": "Align more than two molecular sequences.|This includes methods that use an existing alignment, for example to incorporate sequences into an alignment, or combine several multiple alignments into a single, improved alignment.", "label": "Multiple sequence alignment"}, "operation_0491": {"class_id": "http://edamontology.org/operation_0491", "definitions": "Align exactly two molecular sequences.|Methods might perform one-to-one, one-to-many or many-to-many comparisons.", "label": "Pairwise sequence alignment"}, "operation_0490": {"class_id": "http://edamontology.org/operation_0490", "definitions": "Render a representation of a distribution that consists of group of data points plotted on a simple scale.|Dot plots are useful when having not too many (e.g. 20) data points for each category. Example: draw a dotplot of sequence similarities identified from word-matching or character comparison.", "label": "Dot plot plotting"}, "operation_0497": {"class_id": "http://edamontology.org/operation_0497", "definitions": "Align two or more molecular sequences with user-defined constraints.", "label": "Constrained sequence alignment"}, "operation_0496": {"class_id": "http://edamontology.org/operation_0496", "definitions": "Globally align two or more molecular sequences.|Global alignment methods identify similarity across the entire length of the sequences.", "label": "Global alignment"}, "operation_0495": {"class_id": "http://edamontology.org/operation_0495", "definitions": "Locally align two or more molecular sequences.|Local alignment methods identify regions of local similarity.", "label": "Local alignment"}, "operation_0494": {"class_id": "http://edamontology.org/operation_0494", "definitions": "Globally align exactly two molecular sequences.|Global alignment methods identify similarity across the entire length of the sequences.", "label": "Pairwise sequence alignment generation (global)"}, "operation_0499": {"class_id": "http://edamontology.org/operation_0499", "definitions": "Align multiple sequences using relative gap costs calculated from neighbors in a supplied phylogenetic tree.|This is supposed to give a more biologically meaningful alignment than standard alignments.", "label": "Tree-based sequence alignment"}, "operation_0498": {"class_id": "http://edamontology.org/operation_0498", "definitions": "Align two or more molecular sequences using multiple methods to achieve higher quality.", "label": "Consensus-based sequence alignment"}, "topic_3678": {"class_id": "http://edamontology.org/topic_3678", "definitions": "The design of an experiment intended to test a hypothesis, and describe or explain empirical data obtained under various experimental conditions.", "label": "Experimental design and studies"}, "topic_0111": {"class_id": "http://edamontology.org/topic_0111", "definitions": "Promoters in DNA sequences (region of DNA that facilitates the transcription of a particular gene by binding RNA polymerase and transcription factor proteins).", "label": "Promoters"}, "format_1761": {"class_id": "http://edamontology.org/format_1761", "definitions": "Format of CATH domain classification information for a protein PDB file.|The report (for example http://www.cathdb.info/pdb/1cuk) includes chain identifiers, domain identifiers and CATH codes for domains in a given PDB file.", "label": "CATH PDB report format"}, "format_1760": {"class_id": "http://edamontology.org/format_1760", "definitions": "Format of CATH domain classification information for a polypeptide chain.|The report (for example http://www.cathdb.info/chain/1cukA) includes chain identifiers, domain identifiers and CATH codes for domains in a given protein chain.", "label": "CATH chain report format"}, "format_2543": {"class_id": "http://edamontology.org/format_2543", "definitions": "A format resembling EMBL entry (plain text) format.|This concept may be used for the many non-standard EMBL-like formats.", "label": "EMBL-like format"}, "format_3254": {"class_id": "http://edamontology.org/format_3254", "definitions": "A superset of the \"Description-Logic Knowledge Representation System Specification from the KRSS Group of the ARPA Knowledge Sharing Effort\".|This format is used in Protege 4.", "label": "KRSS2 Syntax"}, "topic_3452": {"class_id": "http://edamontology.org/topic_3452", "definitions": "Imaging in sections (sectioning), through the use of a wave-generating device (tomograph) that generates an image (a tomogram).", "label": "Tomography"}, "format_1563": {"class_id": "http://edamontology.org/format_1563", "definitions": "Format of SMART domain assignment data.|The SMART output file includes data on genetically mobile domains / analysis of domain architectures, including phyletic distributions, functional class, tertiary structures and functionally important residues.", "label": "SMART domain assignment report format"}, "operation_0479": {"class_id": "http://edamontology.org/operation_0479", "definitions": "Model protein backbone conformation.|Methods might require a preliminary C(alpha) trace.", "label": "Protein modelling (backbone)"}, "operation_0478": {"class_id": "http://edamontology.org/operation_0478", "definitions": "Model the structure of a protein in complex with a small molecule or another macromolecule.|This includes protein-protein interactions, protein-nucleic acid, protein-ligand binding etc. Methods might predict whether the molecules are likely to bind in vivo, their conformation when bound, the strength of the interaction, possible mutations to achieve bonding and so on.", "label": "Molecular docking"}, "data_3148": {"class_id": "http://edamontology.org/data_3148", "definitions": "An informative report on a particular family of genes, typically a set of genes with similar sequence that originate from duplication of a common ancestor gene, or any other classification of nucleic acid sequences or structures that reflects gene structure.|This includes reports on on gene homologues between species.", "label": "Gene family report"}, "data_3146": {"class_id": "http://edamontology.org/data_3146", "definitions": "Information on a 'species' node from the SCOP database.", "label": "SCOP species"}, "data_3147": {"class_id": "http://edamontology.org/data_3147", "definitions": "mass spectrometry experiments.", "label": "Mass spectrometry experiment"}, "data_3144": {"class_id": "http://edamontology.org/data_3144", "definitions": "Information on a 'family' node from the SCOP database.", "label": "SCOP family"}, "data_3145": {"class_id": "http://edamontology.org/data_3145", "definitions": "Information on a 'protein' node from the SCOP database.", "label": "SCOP protein"}, "data_3142": {"class_id": "http://edamontology.org/data_3142", "definitions": "Information on a 'fold' node from the SCOP database.", "label": "SCOP fold"}, "data_3143": {"class_id": "http://edamontology.org/data_3143", "definitions": "Information on a 'superfamily' node from the SCOP database.", "label": "SCOP superfamily"}, "data_3140": {"class_id": "http://edamontology.org/data_3140", "definitions": "A report on predicted or actual immunoglobulin gene structure including constant, switch and variable regions and diversity, joining and variable segments.", "label": "Nucleic acid features (immunoglobulin gene structure)"}, "data_3141": {"class_id": "http://edamontology.org/data_3141", "definitions": "Information on a 'class' node from the SCOP database.", "label": "SCOP class"}, "data_3786": {"class_id": "http://edamontology.org/data_3786", "definitions": "A structured query, in form of a script, that defines a database search task.", "label": "Query script"}, "topic_3520": {"class_id": "http://edamontology.org/topic_3520", "definitions": "Proteomics experiments.|This includes two-dimensional gel electrophoresis (2D PAGE) experiments, gels or spots in a gel.  Also mass spectrometry - an analytical chemistry technique that measures the mass-to-charge ratio and abundance of ions in the gas phase.  Also Northern blot experiments.", "label": "Proteomics experiment"}, "format_1625": {"class_id": "http://edamontology.org/format_1625", "definitions": "Format of a record from the HIVDB database of genotypes and phenotypes.", "label": "HIVDB entry format"}, "topic_3078": {"class_id": "http://edamontology.org/topic_3078", "definitions": "Specific genes and/or their encoded proteins or a family or other grouping of related genes and proteins.", "label": "Genes and proteins resources"}, "topic_3077": {"class_id": "http://edamontology.org/topic_3077", "definitions": "The acquisition of data, typically measurements of physical systems using any type of sampling system, or by another other means.", "label": "Data acquisition"}, "topic_3074": {"class_id": "http://edamontology.org/topic_3074", "definitions": "The detection, identification and analysis of positional protein sequence features, such as functional sites.", "label": "Protein feature detection"}, "topic_3075": {"class_id": "http://edamontology.org/topic_3075", "definitions": "Topic for modelling biological systems in mathematical terms.", "label": "Biological system modelling"}, "topic_3072": {"class_id": "http://edamontology.org/topic_3072", "definitions": "The detection of the positional features, such as functional and other key sites, in molecular sequences.", "label": "Sequence feature detection"}, "topic_3073": {"class_id": "http://edamontology.org/topic_3073", "definitions": "The detection of positional features such as functional sites in nucleotide sequences.", "label": "Nucleic acid feature detection"}, "topic_3070": {"class_id": "http://edamontology.org/topic_3070", "definitions": "A particular biological science, especially observable traits such as aspects of biochemistry, physiology, morphology, anatomy, development and so on.", "label": "Biology"}, "topic_3071": {"class_id": "http://edamontology.org/topic_3071", "definitions": "The development and use of architectures, policies, practices and procedures for management of data.", "label": "Data management"}, "format_2545": {"class_id": "http://edamontology.org/format_2545", "definitions": "A format resembling FASTQ short read format.|This concept may be used for non-standard FASTQ short read-like formats.", "label": "FASTQ-like format"}, "operation_3258": {"class_id": "http://edamontology.org/operation_3258", "definitions": "Infer a transcriptome sequence by analysis of short sequence reads.", "label": "Transcriptome assembly"}, "operation_3259": {"class_id": "http://edamontology.org/operation_3259", "definitions": "Infer a transcriptome sequence without the aid of a reference genome, i.e. by comparing short sequences (reads) to each other.", "label": "Transcriptome assembly (de novo)"}, "data_1315": {"class_id": "http://edamontology.org/data_1315", "definitions": "transcription factor binding sites (TFBS) in a DNA sequence.", "label": "Transcription factor binding sites"}, "data_1314": {"class_id": "http://edamontology.org/data_1314", "definitions": "A report on selenocysteine insertion sequence (SECIS) element in a DNA sequence.", "label": "Gene features (SECIS element)"}, "data_1311": {"class_id": "http://edamontology.org/data_1311", "definitions": "operons (operators, promoters and genes) from a bacterial genome.", "label": "Gene features report (operon)"}, "data_1310": {"class_id": "http://edamontology.org/data_1310", "definitions": "A report on microRNA sequence (miRNA) or precursor, microRNA targets, miRNA binding sites in an RNA sequence etc.", "label": "Nucleic acid features (microRNA)"}, "data_1313": {"class_id": "http://edamontology.org/data_1313", "definitions": "protein-coding regions including coding sequences (CDS), exons, translation initiation sites and open reading frames.", "label": "Coding region"}, "data_1312": {"class_id": "http://edamontology.org/data_1312", "definitions": "whole promoters or promoter elements (transcription start sites, RNA polymerase binding site, transcription factor binding sites, promoter enhancers etc) in a DNA sequence.", "label": "Nucleic acid features report (promoters)"}, "operation_3096": {"class_id": "http://edamontology.org/operation_3096", "definitions": "Edit a data entity, either randomly or specifically.", "label": "Editing"}, "operation_3094": {"class_id": "http://edamontology.org/operation_3094", "definitions": "Predict a network of protein interactions.", "label": "Protein interaction network prediction"}, "operation_2932": {"class_id": "http://edamontology.org/operation_2932", "definitions": "Generate a Hopp and Woods plot of antigenicity of a protein.", "label": "Hopp and Woods plotting"}, "operation_3092": {"class_id": "http://edamontology.org/operation_3092", "definitions": "Predict, recognise and identify positional features in proteins from analysing protein sequences or structures.|Features includes functional sites or regions, secondary structure, structural domains and so on. Methods might use fingerprints, motifs, profiles, hidden Markov models, sequence alignment etc to provide a mapping of a query protein sequence to a discriminatory element. This includes methods that search a secondary protein database (Prosite, Blocks, ProDom, Prints, Pfam etc.) to assign a protein sequence(s) to a known protein family or group.", "label": "Protein feature detection"}, "operation_3093": {"class_id": "http://edamontology.org/operation_3093", "definitions": "Screen a molecular sequence(s) against a database (of some type) to identify similarities between the sequence and database entries.", "label": "Database search (by sequence)"}, "operation_3090": {"class_id": "http://edamontology.org/operation_3090", "definitions": "Predict, recognise and identify positional features in proteins from analysing protein structure.", "label": "Protein feature prediction (from structure)"}, "operation_2936": {"class_id": "http://edamontology.org/operation_2936", "definitions": "Generate a dendrograph of raw, preprocessed or clustered expression (e.g. microarray) data.", "label": "Dendrograph plotting"}, "operation_2937": {"class_id": "http://edamontology.org/operation_2937", "definitions": "Generate a plot of distances (distance or correlation matrix) between expression values.", "label": "Proximity map plotting"}, "data_2563": {"class_id": "http://edamontology.org/data_2563", "definitions": "Single letter amino acid identifier, e.g. G.", "label": "Amino acid name (single letter)"}, "operation_2426": {"class_id": "http://edamontology.org/operation_2426", "definitions": "Model or simulate some biological entity or system, typically using mathematical techniques including  dynamical systems, statistical models, differential equations, and game theoretic models.", "label": "Modelling and simulation"}, "data_2565": {"class_id": "http://edamontology.org/data_2565", "definitions": "Full name of an amino acid, e.g. Glycine.", "label": "Amino acid name (full name)"}, "data_2564": {"class_id": "http://edamontology.org/data_2564", "definitions": "Three letter amino acid identifier, e.g. GLY.", "label": "Amino acid name (three letter)"}, "data_2769": {"class_id": "http://edamontology.org/data_2769", "definitions": "Identifier of a RNA transcript.", "label": "Transcript ID"}, "data_2768": {"class_id": "http://edamontology.org/data_2768", "definitions": "Annotation about a gene symbol.", "label": "Gene symbol annotation"}, "operation_2427": {"class_id": "http://edamontology.org/operation_2427", "definitions": "Perform basic operations on some data or a database.", "label": "Data handling"}, "topic_3297": {"class_id": "http://edamontology.org/topic_3297", "definitions": "The exploitation of biological process, structure and function for industrial purposes, for example the genetic manipulation of microorganisms for the antibody production.", "label": "Biotechnology"}, "topic_3295": {"class_id": "http://edamontology.org/topic_3295", "definitions": "Topic concerning the study of heritable changes, for example in gene expression or phenotype, caused by mechanisms other than changes in the DNA sequence.|This includes sub-topics such as histone modification and DNA methylation (methylation sites and analysis, for example of patterns and profiles of DNA methylation in a population, tissue etc.)", "label": "Epigenetics"}, "data_2761": {"class_id": "http://edamontology.org/data_2761", "definitions": "Identifier of an entry from the UTRSite database of regulatory motifs in eukaryotic UTRs.", "label": "UTRSite ID"}, "operation_2410": {"class_id": "http://edamontology.org/operation_2410", "definitions": "Analyse gene expression and regulation data.", "label": "Gene expression analysis"}, "data_2763": {"class_id": "http://edamontology.org/data_2763", "definitions": "An informative report on a particular locus.", "label": "Locus annotation"}, "data_2762": {"class_id": "http://edamontology.org/data_2762", "definitions": "An informative report about a specific or conserved pattern in a molecular sequence, such as its context in genes or proteins, its role, origin or method of construction, etc.", "label": "Sequence signature report"}, "data_2765": {"class_id": "http://edamontology.org/data_2765", "definitions": "One or more terms from one or more controlled vocabularies which are annotations on an entity.|The concepts are typically provided as a persistent identifier or some other link the source ontologies. Evidence of the validity of the annotation might be included.", "label": "Term ID list"}, "data_2764": {"class_id": "http://edamontology.org/data_2764", "definitions": "Official name of a protein as used in the UniProt database.", "label": "Protein name (UniProt)"}, "data_2767": {"class_id": "http://edamontology.org/data_2767", "definitions": "Basic information concerning an identifier of data (typically including the identifier itself).  For example, a gene symbol with information concerning its provenance.", "label": "Identifier with metadata"}, "data_2766": {"class_id": "http://edamontology.org/data_2766", "definitions": "Name of a protein family from the HAMAP database.", "label": "HAMAP ID"}, "operation_2930": {"class_id": "http://edamontology.org/operation_2930", "definitions": "Compare the physicochemical properties of two or more proteins (or reference data).", "label": "Protein property comparison"}, "operation_2931": {"class_id": "http://edamontology.org/operation_2931", "definitions": "Compare two or more molecular secondary structures.", "label": "Secondary structure comparison"}, "format_1949": {"class_id": "http://edamontology.org/format_1949", "definitions": "Nexus/paup interleaved sequence format.", "label": "nexus-seq"}, "format_1948": {"class_id": "http://edamontology.org/format_1948", "definitions": "NBRF/PIR entry sequence format.", "label": "nbrf/pir"}, "format_1419": {"class_id": "http://edamontology.org/format_1419", "definitions": "Format for alignment of molecular sequences to MEME profiles (position-dependent scoring matrices) as generated by the MAST tool from the MEME package.", "label": "Sequence-MEME profile alignment"}, "operation_2234": {"class_id": "http://edamontology.org/operation_2234", "definitions": "Perform basic (non-analytical) operations on a file of molecular tertiary structural data.", "label": "Structure file processing"}, "operation_2237": {"class_id": "http://edamontology.org/operation_2237", "definitions": "Query a profile data resource and retrieve one or more profile(s) and / or associated annotation.|This includes direct retrieval methods that retrieve a profile by, e.g. the profile name.", "label": "Data retrieval (sequence profile)"}, "format_1941": {"class_id": "http://edamontology.org/format_1941", "definitions": "Hennig86 output sequence format.", "label": "hennig86"}, "format_1940": {"class_id": "http://edamontology.org/format_1940", "definitions": "FASTA sequence format including NCBI-style GIs.", "label": "giFASTA format"}, "format_1943": {"class_id": "http://edamontology.org/format_1943", "definitions": "Intelligenetics sequence format (strict version).", "label": "igstrict"}, "format_1942": {"class_id": "http://edamontology.org/format_1942", "definitions": "Intelligenetics sequence format.", "label": "ig"}, "format_1945": {"class_id": "http://edamontology.org/format_1945", "definitions": "Mase program sequence format.", "label": "mase format"}, "format_1944": {"class_id": "http://edamontology.org/format_1944", "definitions": "Jackknifer interleaved and non-interleaved sequence format.", "label": "jackknifer"}, "format_1947": {"class_id": "http://edamontology.org/format_1947", "definitions": "GCG MSF (multiple sequence file) file format.", "label": "GCG MSF"}, "format_1946": {"class_id": "http://edamontology.org/format_1946", "definitions": "Mega interleaved and non-interleaved sequence format.", "label": "mega-seq"}, "format_1782": {"class_id": "http://edamontology.org/format_1782", "definitions": "Entry (gene) format of the NCBI database.", "label": "NCBI gene report format"}, "operation_2429": {"class_id": "http://edamontology.org/operation_2429", "definitions": "Map properties to positions on an biological entity (typically a molecular sequence or structure), or assemble such an entity from constituent parts.", "label": "Mapping"}, "operation_2938": {"class_id": "http://edamontology.org/operation_2938", "definitions": "Visualise clustered expression data using a tree diagram.", "label": "Dendrogram visualisation"}, "format_3466": {"class_id": "http://edamontology.org/format_3466", "definitions": "Encapsulated PostScript format", "label": "EPS"}, "operation_2939": {"class_id": "http://edamontology.org/operation_2939", "definitions": "Visualize the results of a principal component analysis (orthogonal data transformation).", "label": "Principal component plotting"}, "format_3499": {"class_id": "http://edamontology.org/format_3499", "definitions": "Ensembl standard format for variation data.", "label": "Ensembl variation file format"}, "format_2311": {"class_id": "http://edamontology.org/format_2311", "definitions": "EMBL entry format wrapped in HTML elements.", "label": "EMBL-HTML"}, "topic_3531": {"class_id": "http://edamontology.org/topic_3531", "definitions": "Super-secondary structure of protein sequence(s).", "label": "Protein super-secondary structure"}, "topic_3656": {"class_id": "http://edamontology.org/topic_3656", "definitions": "Experimental techniques to purify a protein-DNA crosslinked complex.  Usually sequencing follows e.g. in the techniques ChIP-chip, ChIP-seq and MeDIP-seq.", "label": "Immunoprecipitation experiment"}, "operation_2121": {"class_id": "http://edamontology.org/operation_2121", "definitions": "Perform basic (non-analytical) operations on a report or file of sequences (which might include features), such as file concatenation, removal or ordering of sequences, creation of subset or a new file of sequences.", "label": "Sequence file editing"}, "operation_2120": {"class_id": "http://edamontology.org/operation_2120", "definitions": "Process an EMBOSS listfile (list of EMBOSS Uniform Sequence Addresses).", "label": "Listfile processing"}, "operation_2123": {"class_id": "http://edamontology.org/operation_2123", "definitions": "Process (read and / or write) physicochemical property data for small molecules.", "label": "Small molecule data processing"}, "operation_2122": {"class_id": "http://edamontology.org/operation_2122", "definitions": "Perform basic (non-analytical) operations on a sequence alignment file, such as copying or removal and ordering of sequences.", "label": "Sequence alignment file processing"}, "data_1135": {"class_id": "http://edamontology.org/data_1135", "definitions": "Unique identifier of an entry from the Gene3D database.", "label": "Gene3D ID"}, "data_1134": {"class_id": "http://edamontology.org/data_1134", "definitions": "Secondary accession number of an InterPro entry.", "label": "InterPro secondary accession"}, "data_1137": {"class_id": "http://edamontology.org/data_1137", "definitions": "The unique identifier of an entry in the PRINTS database.", "label": "PRINTS code"}, "data_1136": {"class_id": "http://edamontology.org/data_1136", "definitions": "Unique identifier of an entry from the PIRSF database.", "label": "PIRSF ID"}, "data_1131": {"class_id": "http://edamontology.org/data_1131", "definitions": "Name of a protein family.", "label": "Protein family name"}, "data_1130": {"class_id": "http://edamontology.org/data_1130", "definitions": "Accession number of an entry from the IntAct database.", "label": "IntAct accession number"}, "data_1133": {"class_id": "http://edamontology.org/data_1133", "definitions": "Primary accession number of an InterPro entry.|Every InterPro entry has a unique accession number to provide a persistent citation of database records.", "label": "InterPro accession"}, "data_1132": {"class_id": "http://edamontology.org/data_1132", "definitions": "Name of an InterPro entry, usually indicating the type of protein matches for that entry.", "label": "InterPro entry name"}, "topic_3530": {"class_id": "http://edamontology.org/topic_3530", "definitions": "Genetic information processing pathways.", "label": "Genetic information processing pathways"}, "data_1139": {"class_id": "http://edamontology.org/data_1139", "definitions": "Accession number of an entry from the SMART database.", "label": "SMART accession number"}, "data_1138": {"class_id": "http://edamontology.org/data_1138", "definitions": "Accession number of a Pfam entry.", "label": "Pfam accession number"}, "operation_1774": {"class_id": "http://edamontology.org/operation_1774", "definitions": "Retrieve existing annotation (or documentation), typically annotation on a database entity.|Use this concepts for tools which retrieve pre-existing annotations, not for example prediction methods that might make annotations.", "label": "Annotation retrieval"}, "operation_1777": {"class_id": "http://edamontology.org/operation_1777", "definitions": "Predict general functional properties of a protein.|For functional properties that can be mapped to a sequence, use 'Sequence feature detection (protein)' instead.", "label": "Protein function prediction"}, "data_3210": {"class_id": "http://edamontology.org/data_3210", "definitions": "An index of a genome sequence.|Many sequence alignment tasks involving many or very large sequences rely on a precomputed index of the sequence to accelerate the alignment.", "label": "Genome index"}, "data_2812": {"class_id": "http://edamontology.org/data_2812", "definitions": "Identifier of a lipid.", "label": "Lipid identifier"}, "format_3594": {"class_id": "http://edamontology.org/format_3594", "definitions": "Photo CD format, which is the highest resolution format for images on a CD.|PCD was developed by Kodak. A PCD file contains five different resolution (ranging from low to high) of a slide or film negative. Due to it PCD is often used by many photographers and graphics professionals for high-end printed applications.", "label": "pcd"}, "topic_3409": {"class_id": "http://edamontology.org/topic_3409", "definitions": "The branch of medicine that deals with disorders of the oesophagus, stomach, duodenum, jejenum, ileum, large intestine, sigmoid colon and rectum.", "label": "Gastroenterology"}, "data_1049": {"class_id": "http://edamontology.org/data_1049", "definitions": "The name of a directory.", "label": "Directory name"}, "data_1048": {"class_id": "http://edamontology.org/data_1048", "definitions": "An identifier of a biological or bioinformatics database.", "label": "Database ID"}, "data_1041": {"class_id": "http://edamontology.org/data_1041", "definitions": "A SCOP concise classification string (sccs) is a compact representation of a SCOP domain classification.|An scss includes the class (alphabetical), fold, superfamily and family (all numerical) to which a given domain belongs.", "label": "SCOP concise classification string (sccs)"}, "data_1040": {"class_id": "http://edamontology.org/data_1040", "definitions": "Identifier of a protein domain from CATH.", "label": "CATH domain ID"}, "data_1043": {"class_id": "http://edamontology.org/data_1043", "definitions": "A code number identifying a node from the CATH database.", "label": "CATH node ID"}, "data_1042": {"class_id": "http://edamontology.org/data_1042", "definitions": "Unique identifier (number) of an entry in the SCOP hierarchy, for example 33229.|A sunid uniquely identifies an entry in the SCOP hierarchy, including leaves (the SCOP domains) and higher level nodes including entries corresponding to the protein level.", "label": "SCOP sunid"}, "data_1045": {"class_id": "http://edamontology.org/data_1045", "definitions": "The name of a species (typically a taxonomic group) of organism.", "label": "Species name"}, "data_1044": {"class_id": "http://edamontology.org/data_1044", "definitions": "The name of a biological kingdom (Bacteria, Archaea, or Eukaryotes).", "label": "Kingdom name"}, "data_1047": {"class_id": "http://edamontology.org/data_1047", "definitions": "A string of characters that name or otherwise identify a resource on the Internet.", "label": "URI"}, "data_1046": {"class_id": "http://edamontology.org/data_1046", "definitions": "The name of a strain of an organism variant, typically a plant, virus or bacterium.", "label": "Strain name"}, "topic_0767": {"class_id": "http://edamontology.org/topic_0767", "definitions": "", "label": "Protein and peptide identification"}, "format_1958": {"class_id": "http://edamontology.org/format_1958", "definitions": "Refseq protein entry sequence format.|Currently identical to genpept format", "label": "refseqp"}, "format_2159": {"class_id": "http://edamontology.org/format_2159", "definitions": "Format used for report on coding regions in nucleotide sequences.", "label": "Gene features (coding region) format"}, "format_2158": {"class_id": "http://edamontology.org/format_2158", "definitions": "Format used for report on restriction enzyme recognition sites in nucleotide sequences.", "label": "Nucleic acid features (restriction sites) format"}, "format_3596": {"class_id": "http://edamontology.org/format_3596", "definitions": "The PPM format is a lowest common denominator color image file format.", "label": "ppm"}, "format_2155": {"class_id": "http://edamontology.org/format_2155", "definitions": "Format used for map of repeats in molecular (typically nucleotide) sequences.", "label": "Sequence features (repeats) format"}, "format_1959": {"class_id": "http://edamontology.org/format_1959", "definitions": "Selex sequence format.", "label": "selex sequence format"}, "operation_3762": {"class_id": "http://edamontology.org/operation_3762", "definitions": "An operation supporting the aggregation of other services (at least two) into a funtional unit, for the automation of some task.", "label": "Service composition"}, "data_1544": {"class_id": "http://edamontology.org/data_1544", "definitions": "Phi/psi angle data or a Ramachandran plot of a protein structure.", "label": "Ramachandran plot"}, "data_1545": {"class_id": "http://edamontology.org/data_1545", "definitions": "Data on the net charge distribution (dipole moment) of a protein structure.", "label": "Protein dipole moment"}, "data_1546": {"class_id": "http://edamontology.org/data_1546", "definitions": "A matrix of distances between amino acid residues (for example the C-alpha atoms) in a protein structure.", "label": "Protein distance matrix"}, "data_1547": {"class_id": "http://edamontology.org/data_1547", "definitions": "An amino acid residue contact map for a protein structure.", "label": "Protein contact map"}, "data_1540": {"class_id": "http://edamontology.org/data_1540", "definitions": "Data on inter-atomic or inter-residue contacts, distances and interactions in protein structure(s) or on the interactions of protein atoms or residues with non-protein groups.", "label": "Protein non-covalent interactions report"}, "data_1541": {"class_id": "http://edamontology.org/data_1541", "definitions": "Informative report on flexibility or motion of a protein structure.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Protein flexibility or motion report"}, "data_1542": {"class_id": "http://edamontology.org/data_1542", "definitions": "Data on the solvent accessible or buried surface area of a protein structure.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation. This concept covers definitions of the protein surface, interior and interfaces, accessible and buried residues, surface accessible pockets, interior inaccessible cavities etc.", "label": "Protein solvent accessibility report"}, "data_2968": {"class_id": "http://edamontology.org/data_2968", "definitions": "Biological or biomedical data has been rendered into an image, typically for display on screen.", "label": "Image"}, "data_2967": {"class_id": "http://edamontology.org/data_2967", "definitions": "An image from a microarray experiment which (typically) allows a visualisation of probe hybridisation and gene-expression data.", "label": "Microarray image"}, "data_2966": {"class_id": "http://edamontology.org/data_2966", "definitions": "General annotation on a set of oligonucleotide probes, such as the gene name with which the probe set is associated and which probes belong to the set.", "label": "Oligonucleotide probe sets annotation"}, "data_2965": {"class_id": "http://edamontology.org/data_2965", "definitions": "An informative report on a two-dimensional (2D PAGE) gel.", "label": "2D PAGE gel report"}, "data_0896": {"class_id": "http://edamontology.org/data_0896", "definitions": "An informative human-readable report about one or more specific protein molecules or protein structural domains, derived from analysis of primary (sequence or structural) data.", "label": "Protein report"}, "data_1548": {"class_id": "http://edamontology.org/data_1548", "definitions": "Report on clusters of contacting residues in protein structures such as a key structural residue network.", "label": "Protein residue 3D cluster"}, "data_1549": {"class_id": "http://edamontology.org/data_1549", "definitions": "Patterns of hydrogen bonding in protein structures.", "label": "Protein hydrogen bonds"}, "data_0893": {"class_id": "http://edamontology.org/data_0893", "definitions": "An alignment of molecular sequence to structure (from threading sequence(s) through 3D structure or representation of structure(s)).", "label": "Sequence-structure alignment"}, "data_2960": {"class_id": "http://edamontology.org/data_2960", "definitions": "A temperature profile of a double-stranded nucleic acid molecule (DNA or DNA/RNA).|Plots melting temperature versus base position.", "label": "Nucleic acid temperature profile"}, "format_1955": {"class_id": "http://edamontology.org/format_1955", "definitions": "Phylip interleaved sequence format.", "label": "phylip sequence format"}, "topic_3365": {"class_id": "http://edamontology.org/topic_3365", "definitions": "The development of policies, models and standards that cover data acquisition, storage and integration, such that it can be put to use, typically through a process of systematically applying statistical and / or logical techniques to describe, illustrate, summarise or evaluate data.", "label": "Data architecture, analysis and design"}, "operation_2496": {"class_id": "http://edamontology.org/operation_2496", "definitions": "Process (read and / or write) a network of gene regulation.", "label": "Gene regulatory network processing"}, "topic_3368": {"class_id": "http://edamontology.org/topic_3368", "definitions": "Any matter, surface or construct that interacts with a biological system.", "label": "Biomaterials"}, "topic_3369": {"class_id": "http://edamontology.org/topic_3369", "definitions": "The use of synthetic chemistry to study and manipulate biological systems.", "label": "Chemical biology"}, "format_3772": {"class_id": "http://edamontology.org/format_3772", "definitions": "BioJSON is a BioXSD-schema-based JSON format of sequence-based data and some other common data - sequence records, alignments, feature records, references to resources, and more - optimised for integrative bioinformatics, web applications and APIs, and object-oriented programming.", "label": "BioJSON (BioXSD)"}, "data_1438": {"class_id": "http://edamontology.org/data_1438", "definitions": "A report of data concerning or derived from a phylogenetic tree, or from comparing two or more phylogenetic trees.|This is a broad data type and is used  for example for reports on confidence, shape or stratigraphic (age) data derived from phylogenetic tree analysis.", "label": "Phylogenetic report"}, "data_1439": {"class_id": "http://edamontology.org/data_1439", "definitions": "A model of DNA substitution that explains a DNA sequence alignment, derived from phylogenetic tree analysis.", "label": "DNA substitution model"}, "data_2984": {"class_id": "http://edamontology.org/data_2984", "definitions": "An informative report concerning or derived from the analysis of a biological pathway or network, such as a map (diagram) or annotation.", "label": "Pathway or network report"}, "format_2571": {"class_id": "http://edamontology.org/format_2571", "definitions": "Format of a raw molecular sequence (i.e. the alphabet used).", "label": "Raw sequence format"}, "format_2570": {"class_id": "http://edamontology.org/format_2570", "definitions": "Alphabet for an RNA sequence (characters ACGU only) without unknown positions, ambiguity or non-sequence characters.", "label": "completely unambiguous pure rna sequence"}, "format_2573": {"class_id": "http://edamontology.org/format_2573", "definitions": "Sequence Alignment/Map (SAM) format for alignment of nucleotide sequences (e.g. sequencing reads) to (a) reference sequence(s). May contain base-call and alignment qualities and other data.|The format supports short and long reads (up to 128Mbp) produced by different sequencing platforms and is used to hold mapped data within the GATK and across the Broad Institute, the Sanger Centre, and throughout the 1000 Genomes project.", "label": "SAM"}, "format_2572": {"class_id": "http://edamontology.org/format_2572", "definitions": "BAM format, the binary, BGZF-formatted compressed version of SAM format for alignment of nucleotide sequences (e.g. sequencing reads) to (a) reference sequence(s). May contain base-call and alignment qualities and other data.", "label": "BAM"}, "data_2365": {"class_id": "http://edamontology.org/data_2365", "definitions": "A persistent, unique identifier of a biological pathway or network (typically a database entry).", "label": "Pathway or network accession"}, "data_2364": {"class_id": "http://edamontology.org/data_2364", "definitions": "two-dimensional gel electrophoresis experiments, gels or spots in a gel.", "label": "2D PAGE report"}, "data_2367": {"class_id": "http://edamontology.org/data_2367", "definitions": "Identifier of an object from the ASTD database.", "label": "ASTD ID"}, "data_2366": {"class_id": "http://edamontology.org/data_2366", "definitions": "Alignment of the (1D representations of) secondary structure of two or more molecules.", "label": "Secondary structure alignment"}, "format_1577": {"class_id": "http://edamontology.org/format_1577", "definitions": "Entry format for the SMART protein secondary database.", "label": "SMART entry format"}, "data_2360": {"class_id": "http://edamontology.org/data_2360", "definitions": "Data on indirect protein domain-protein domain interaction(s).", "label": "Domain-domain interaction (indirect)"}, "data_2363": {"class_id": "http://edamontology.org/data_2363", "definitions": "Data concerning two-dimensional polygel electrophoresis.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "2D PAGE data"}, "data_2362": {"class_id": "http://edamontology.org/data_2362", "definitions": "Accession number of a nucleotide or protein sequence database entry.", "label": "Sequence accession (hybrid)"}, "topic_0781": {"class_id": "http://edamontology.org/topic_0781", "definitions": "Study of viruses, e.g. sequence and structural data, interactions of viral proteins, or a viral genome including molecular sequences, genes and annotation.", "label": "Virology"}, "data_2369": {"class_id": "http://edamontology.org/data_2369", "definitions": "Identifier of an intron from the ASTD database.", "label": "ASTD ID (intron)"}, "data_2368": {"class_id": "http://edamontology.org/data_2368", "definitions": "Identifier of an exon from the ASTD database.", "label": "ASTD ID (exon)"}, "topic_0780": {"class_id": "http://edamontology.org/topic_0780", "definitions": "Plants, e.g. information on a specific plant genome including molecular sequences, genes and annotation.|The resource may be specific to a plant, a group of plants or all plants.", "label": "Plant biology"}, "topic_3529": {"class_id": "http://edamontology.org/topic_3529", "definitions": "Environmental information processing pathways.", "label": "Environmental information processing pathways"}, "topic_0783": {"class_id": "http://edamontology.org/topic_0783", "definitions": "Pathogens, e.g. information on a specific vertebrate genome including molecular sequences, genes and annotation.|The resource may be specific to a pathogen, a group of pathogens or all pathogens.", "label": "Pathogens"}, "topic_0782": {"class_id": "http://edamontology.org/topic_0782", "definitions": "Fungi and molds, e.g. information on a specific fungal genome including molecular sequences, genes and annotation.|The resource may be specific to a fungus, a group of fungi or all fungi.", "label": "Fungi"}, "operation_0260": {"class_id": "http://edamontology.org/operation_0260", "definitions": "Convert a molecular sequence alignment from one type to another (for example amino acid to coding nucleotide sequence).", "label": "Sequence alignment conversion"}, "operation_0261": {"class_id": "http://edamontology.org/operation_0261", "definitions": "Process (read and / or write) physicochemical property data of nucleic acids.", "label": "Nucleic acid property processing"}, "operation_0262": {"class_id": "http://edamontology.org/operation_0262", "definitions": "Calculate or predict physical or chemical properties of nucleic acid molecules, including any non-positional properties of the molecular sequence.", "label": "Nucleic acid property calculation"}, "operation_0264": {"class_id": "http://edamontology.org/operation_0264", "definitions": "Predict splicing alternatives or transcript isoforms from analysis of sequence data.", "label": "Splice transcript prediction"}, "operation_0265": {"class_id": "http://edamontology.org/operation_0265", "definitions": "Detect frameshifts in DNA sequences, including frameshift sites and signals, and frameshift errors from sequencing projects.|Methods include sequence alignment (if related sequences are available) and word-based sequence comparison.", "label": "Frameshift detection"}, "operation_0266": {"class_id": "http://edamontology.org/operation_0266", "definitions": "Detect vector sequences in nucleotide sequence, typically by comparison to a set of known vector sequences.", "label": "Vector sequence detection"}, "operation_0267": {"class_id": "http://edamontology.org/operation_0267", "definitions": "Predict secondary structure of protein sequences.|Methods might use amino acid composition, local sequence information, multiple sequence alignments, physicochemical features, estimated energy content, statistical algorithms, hidden Markov models, support vector machines, kernel machines, neural networks etc.", "label": "Protein secondary structure prediction"}, "operation_0268": {"class_id": "http://edamontology.org/operation_0268", "definitions": "Predict super-secondary structure of protein sequence(s).|Super-secondary structures include leucine zippers, coiled coils, Helix-Turn-Helix etc.", "label": "Protein super-secondary structure prediction"}, "operation_0269": {"class_id": "http://edamontology.org/operation_0269", "definitions": "Predict and/or classify transmembrane proteins or transmembrane (helical) domains or regions in protein sequences.", "label": "Transmembrane protein prediction"}, "topic_0121": {"class_id": "http://edamontology.org/topic_0121", "definitions": "Protein and peptide identification, especially in the study of whole proteomes of organisms.|Includes metaproteomics: proteomics analysis of an environmental sample.|Proteomics includes any methods (especially high-throughput) that separate, characterize and identify expressed proteins such as mass spectrometry, two-dimensional gel electrophoresis and protein microarrays, as well as in-silico methods that perform proteolytic or mass calculations on a protein sequence and other analyses of protein production data, for example in different cells or tissues.", "label": "Proteomics"}, "data_2028": {"class_id": "http://edamontology.org/data_2028", "definitions": "Raw data from or annotation on laboratory experiments.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Experimental data"}, "data_2026": {"class_id": "http://edamontology.org/data_2026", "definitions": "A plot based on the Michaelis Menten equation of enzyme kinetics plotting the ratio of the initial substrate concentration (S) against the reaction velocity (v).", "label": "Hanes Woolf plot"}, "data_2024": {"class_id": "http://edamontology.org/data_2024", "definitions": "Data concerning chemical reaction(s) catalysed by enzyme(s).|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Enzyme kinetics data"}, "data_2025": {"class_id": "http://edamontology.org/data_2025", "definitions": "A plot giving an approximation of the kinetics of an enzyme-catalysed reaction, assuming simple kinetics (i.e. no intermediate or product inhibition, allostericity or cooperativity). It plots initial reaction rate to the substrate concentration (S) from which the maximum rate (vmax) is apparent.", "label": "Michaelis Menten plot"}, "data_2022": {"class_id": "http://edamontology.org/data_2022", "definitions": "Data used by the Vienna RNA analysis package.", "label": "Vienna RNA structural data"}, "data_2023": {"class_id": "http://edamontology.org/data_2023", "definitions": "Data used to replace (mask) characters in a molecular sequence.", "label": "Sequence mask parameter"}, "format_2376": {"class_id": "http://edamontology.org/format_2376", "definitions": "A serialisation format conforming to the Resource Description Framework (RDF) model.", "label": "RDF format"}, "data_3567": {"class_id": "http://edamontology.org/data_3567", "definitions": "A report about a biosample.", "label": "Reference sample report"}, "data_3568": {"class_id": "http://edamontology.org/data_3568", "definitions": "Accession number of an entry from the Gene Expression Atlas.", "label": "Gene Expression Atlas Experiment ID"}, "operation_0314": {"class_id": "http://edamontology.org/operation_0314", "definitions": "The measurement of the gene expression of multiple genes in a cell, tissue, sample etc., in order to get an impression of biological function.|Metagenomic inference is the profiling of phylogenetic marker genes in order to predict metagenome function.|Gene expression profiling generates some sort of gene expression profile, for example from microarray data.", "label": "Gene expression profiling"}, "operation_0315": {"class_id": "http://edamontology.org/operation_0315", "definitions": "Comparison of gene expression profiles.", "label": "Gene expression comparison"}, "operation_0316": {"class_id": "http://edamontology.org/operation_0316", "definitions": "Interpret (in functional terms) and annotate gene expression data.", "label": "Functional profiling"}, "operation_0317": {"class_id": "http://edamontology.org/operation_0317", "definitions": "Analyse EST or cDNA sequences.|For example, identify full-length cDNAs from EST sequences or detect potential EST antisense transcripts.", "label": "EST and cDNA sequence analysis"}, "operation_0310": {"class_id": "http://edamontology.org/operation_0310", "definitions": "Combine (align and merge) overlapping fragments of a DNA sequence to reconstruct the original sequence.|For example, assemble overlapping reads from paired-end sequencers into contigs (a contiguous sequence corresponding to read overlaps). Or assemble contigs, for example ESTs and genomic DNA fragments, depending on the detected fragment overlaps.", "label": "Sequence assembly"}, "operation_0311": {"class_id": "http://edamontology.org/operation_0311", "definitions": "Standardize or normalize microarray data.", "label": "Microarray data standardisation and normalisation"}, "operation_0312": {"class_id": "http://edamontology.org/operation_0312", "definitions": "Process (read and / or write) SAGE, MPSS or SBS experimental data.", "label": "Sequencing-based expression profile data processing"}, "operation_0313": {"class_id": "http://edamontology.org/operation_0313", "definitions": "Perform cluster analysis of gene expression data to identify groups with similar gene-expression profiles, for example by clustering.", "label": "Gene expression clustering"}, "operation_0318": {"class_id": "http://edamontology.org/operation_0318", "definitions": "Identify and select targets for protein structural determination.|Methods will typically navigate a graph of protein families of known structure.", "label": "Structural genomics target selection"}, "operation_0319": {"class_id": "http://edamontology.org/operation_0319", "definitions": "Assign secondary structure from protein coordinate or experimental data.", "label": "Protein secondary structure assignment"}, "operation_2483": {"class_id": "http://edamontology.org/operation_2483", "definitions": "Compare two or more molecular tertiary structures.", "label": "Structure comparison"}, "operation_3664": {"class_id": "http://edamontology.org/operation_3664", "definitions": "Construction of a statistical model, or a set of assumptions around some observed data, usually by describing a set of probability distributions which approximate the distribution of data.", "label": "Statistical modelling"}, "format_1954": {"class_id": "http://edamontology.org/format_1954", "definitions": "Plain old FASTA sequence format (unspecified format for IDs).", "label": "Pearson format"}, "operation_3666": {"class_id": "http://edamontology.org/operation_3666", "definitions": "Compare two or more molecular surfaces.", "label": "Molecular surface comparison"}, "operation_3661": {"class_id": "http://edamontology.org/operation_3661", "definitions": "Predict the effect or function of an individual single nucleotide polymorphism (SNP).", "label": "SNP annotation"}, "operation_3660": {"class_id": "http://edamontology.org/operation_3660", "definitions": "Model a metabolic network, for example, to reconstruct pathways or to simulate metabolism.", "label": "Metabolic network modelling"}, "operation_3663": {"class_id": "http://edamontology.org/operation_3663", "definitions": "Prediction of genes or gene components by reference to homologous genes.", "label": "Homology-based gene prediction"}, "operation_3662": {"class_id": "http://edamontology.org/operation_3662", "definitions": "Prediction of genes or gene components from first principles, i.e. without reference to existing genes.", "label": "Ab-initio gene prediction"}, "operation_3095": {"class_id": "http://edamontology.org/operation_3095", "definitions": "Design (or predict) nucleic acid sequences with specific chemical or physical properties.", "label": "Nucleic acid design"}, "data_2794": {"class_id": "http://edamontology.org/data_2794", "definitions": "Accession of an open reading frame (catalogued in a database).", "label": "ORF ID"}, "data_2795": {"class_id": "http://edamontology.org/data_2795", "definitions": "An identifier of an open reading frame.", "label": "ORF identifier"}, "data_2796": {"class_id": "http://edamontology.org/data_2796", "definitions": "Identifier of an entry from the GlycosciencesDB database.", "label": "Linucs ID"}, "data_2797": {"class_id": "http://edamontology.org/data_2797", "definitions": "Unique identifier for a ligand-gated ion channel protein from the LGICdb database.", "label": "Protein ID (LGICdb)"}, "data_2790": {"class_id": "http://edamontology.org/data_2790", "definitions": "Identifier of a two-dimensional (protein) gel.", "label": "Gel ID"}, "data_2791": {"class_id": "http://edamontology.org/data_2791", "definitions": "Name of a reference map gel from the SWISS-2DPAGE database.", "label": "Reference map name (SWISS-2DPAGE)"}, "data_2792": {"class_id": "http://edamontology.org/data_2792", "definitions": "Unique identifier for a peroxidase protein from the PeroxiBase database.", "label": "Protein ID (PeroxiBase)"}, "data_2793": {"class_id": "http://edamontology.org/data_2793", "definitions": "Identifier of an entry from the SISYPHUS database of tertiary structure alignments.", "label": "SISYPHUS ID"}, "data_2798": {"class_id": "http://edamontology.org/data_2798", "definitions": "Identifier of an EST sequence from the MaizeDB database.", "label": "MaizeDB ID"}, "data_2799": {"class_id": "http://edamontology.org/data_2799", "definitions": "A unique identifier of gene in the MfunGD database.", "label": "Gene ID (MfunGD)"}, "topic_3400": {"class_id": "http://edamontology.org/topic_3400", "definitions": "Health issues related to the immune system and their prevention, diagnosis and mangement.", "label": "Allergy, clinical immunology and immunotherapeutics"}, "data_2152": {"class_id": "http://edamontology.org/data_2152", "definitions": "A parameter that is used to control rendering (drawing) to a device or image.", "label": "Rendering parameter"}, "data_0998": {"class_id": "http://edamontology.org/data_0998", "definitions": "IUPAC recommended name of a chemical compound.", "label": "Chemical name (IUPAC)"}, "data_2150": {"class_id": "http://edamontology.org/data_2150", "definitions": "Name of a sequence profile.", "label": "Sequence profile name"}, "data_2151": {"class_id": "http://edamontology.org/data_2151", "definitions": "Specification of one or more colors.", "label": "Color"}, "data_2156": {"class_id": "http://edamontology.org/data_2156", "definitions": "A temporal date.", "label": "Date"}, "data_2157": {"class_id": "http://edamontology.org/data_2157", "definitions": "Word composition data for a molecular sequence.", "label": "Word composition"}, "data_2154": {"class_id": "http://edamontology.org/data_2154", "definitions": "Any arbitrary name of a molecular sequence.", "label": "Sequence name"}, "data_0999": {"class_id": "http://edamontology.org/data_0999", "definitions": "International Non-proprietary Name (INN or 'generic name') of a chemical compound, assigned by the World Health Organisation (WHO).", "label": "Chemical name (INN)"}, "format_2203": {"class_id": "http://edamontology.org/format_2203", "definitions": "Data format for a molecular sequence record 'lite', typically molecular sequence and minimal metadata, such as an identifier of the sequence and/or a comment.", "label": "Sequence record lite format"}, "format_2202": {"class_id": "http://edamontology.org/format_2202", "definitions": "Data format for a molecular sequence record, typically corresponding to a full entry from a molecular sequence database.", "label": "Sequence record full format"}, "format_2200": {"class_id": "http://edamontology.org/format_2200", "definitions": "A text format resembling FASTA format.|This concept may also be used for the many non-standard FASTA-like formats.", "label": "FASTA-like (text)"}, "format_2206": {"class_id": "http://edamontology.org/format_2206", "definitions": "Text format for a sequence feature table.", "label": "Sequence feature table format (text)"}, "format_2205": {"class_id": "http://edamontology.org/format_2205", "definitions": "A text format resembling GenBank entry (plain text) format.|This concept may be used for the non-standard GenBank-like text formats.", "label": "GenBank-like format (text)"}, "format_2204": {"class_id": "http://edamontology.org/format_2204", "definitions": "An XML format for EMBL entries.|This is a placeholder for other more specific concepts. It should not normally be used for annotation.", "label": "EMBL format (XML)"}, "format_3580": {"class_id": "http://edamontology.org/format_3580", "definitions": "Reporter Code Count-A data file (.csv) output by the Nanostring nCounter Digital Analyzer, which contains gene sample information, probe information and probe counts.", "label": "rcc"}, "data_2358": {"class_id": "http://edamontology.org/data_2358", "definitions": "An informative report on protein domain-DNA/RNA interaction(s).", "label": "Domain-nucleic acid interaction report"}, "data_2359": {"class_id": "http://edamontology.org/data_2359", "definitions": "An informative report on protein domain-protein domain interaction(s).", "label": "Domain-domain interactions"}, "format_3585": {"class_id": "http://edamontology.org/format_3585", "definitions": "BED file format where each feature is described by chromosome, start, end, name, score, and strand.|Tab delimited data in strict BED format - no non-standard columns allowed; column count forced to 6", "label": "bed6"}, "topic_3444": {"class_id": "http://edamontology.org/topic_3444", "definitions": "Techniques that uses magnetic fields and radiowaves to form images, typically to investigate the anatomy and physiology of the human body.", "label": "MRI"}, "topic_3118": {"class_id": "http://edamontology.org/topic_3118", "definitions": "Topological domains such as cytoplasmic regions in a protein.", "label": "Protein topological domains"}, "format_3584": {"class_id": "http://edamontology.org/format_3584", "definitions": "Browser Extensible Data (BED) format of sequence annotation track that strictly does not contain non-standard fields beyond the first 3 columns.  |Galaxy allows BED files to contain non-standard fields beyond the first 3 columns, some other implementations do not.  ", "label": "bedstrict"}, "operation_3443": {"class_id": "http://edamontology.org/operation_3443", "definitions": "The analysis of a image (typically a digital image) of some type in order to extract information from it.", "label": "Image analysis"}, "format_1570": {"class_id": "http://edamontology.org/format_1570", "definitions": "Entry format for the InterPro database of protein signatures (sequence classifiers) and classified sequences.|This includes signature metadata, sequence references and a reference to the signature itself. There is normally a header (entry accession numbers and name), abstract, taxonomy information, example proteins etc. Each entry also includes a match list which give a number of different views of the signature matches for the sequences in each InterPro entry.", "label": "InterPro entry format"}, "data_2354": {"class_id": "http://edamontology.org/data_2354", "definitions": "An informative report on a specific RNA family or other group of classified RNA sequences.", "label": "RNA family report"}, "format_1572": {"class_id": "http://edamontology.org/format_1572", "definitions": "Entry format for the Gene3D protein secondary database.", "label": "Gene3D entry format"}, "format_1573": {"class_id": "http://edamontology.org/format_1573", "definitions": "Entry format for the PIRSF protein secondary database.", "label": "PIRSF entry format"}, "format_1574": {"class_id": "http://edamontology.org/format_1574", "definitions": "Entry format for the PRINTS protein secondary database.", "label": "PRINTS entry format"}, "format_1575": {"class_id": "http://edamontology.org/format_1575", "definitions": "Entry format for the Panther library of protein families and subfamilies.", "label": "Panther Families and HMMs entry format"}, "format_1576": {"class_id": "http://edamontology.org/format_1576", "definitions": "Entry format for the Pfam protein secondary database.", "label": "Pfam entry format"}, "data_0991": {"class_id": "http://edamontology.org/data_0991", "definitions": "Unique registry number of a chemical compound.", "label": "Chemical registry number"}, "format_1578": {"class_id": "http://edamontology.org/format_1578", "definitions": "Entry format for the Superfamily protein secondary database.", "label": "Superfamily entry format"}, "format_1579": {"class_id": "http://edamontology.org/format_1579", "definitions": "Entry format for the TIGRFam protein secondary database.", "label": "TIGRFam entry format"}, "operation_0442": {"class_id": "http://edamontology.org/operation_0442", "definitions": "Identify, predict or analyse cis-regulatory elements (for example riboswitches) in RNA sequences.", "label": "Transcriptional regulatory element prediction (RNA-cis)"}, "operation_0443": {"class_id": "http://edamontology.org/operation_0443", "definitions": "Identify or predict functional RNA sequences with a gene regulatory role (trans-regulatory elements) or targets.|Trans-regulatory elements regulate genes distant from the gene from which they were transcribed.", "label": "trans-regulatory element prediction"}, "operation_0444": {"class_id": "http://edamontology.org/operation_0444", "definitions": "Identify matrix/scaffold attachment regions (MARs/SARs) in DNA sequences.|MAR/SAR sites often flank a gene or gene cluster and are found nearby cis-regulatory sequences. They might contribute to transcription regulation.", "label": "S/MAR prediction"}, "data_0992": {"class_id": "http://edamontology.org/data_0992", "definitions": "Code word for a ligand, for example from a PDB file.", "label": "Ligand identifier"}, "operation_0446": {"class_id": "http://edamontology.org/operation_0446", "definitions": "Identify or predict exonic splicing enhancers (ESE) in exons.|An exonic splicing enhancer (ESE) is 6-base DNA sequence motif in an exon that enhances or directs splicing of pre-mRNA or hetero-nuclear RNA (hnRNA) into mRNA.", "label": "Exonic splicing enhancer prediction"}, "operation_0447": {"class_id": "http://edamontology.org/operation_0447", "definitions": "Evaluate molecular sequence alignment accuracy.|Evaluation might be purely sequence-based or use structural information.", "label": "Sequence alignment validation"}, "data_1468": {"class_id": "http://edamontology.org/data_1468", "definitions": "3D coordinate and associated data for the tertiary (3D) structure of a protein domain.", "label": "Protein domain"}, "topic_3316": {"class_id": "http://edamontology.org/topic_3316", "definitions": "The theory and practical use of computer systems.", "label": "Computer science"}, "data_3779": {"class_id": "http://edamontology.org/data_3779", "definitions": "A text (such as a scientific article), annotated with notes, data and metadata, such as recognised entities, concepts, and their relations.", "label": "Annotated text"}, "topic_3315": {"class_id": "http://edamontology.org/topic_3315", "definitions": "The study of numbers (quantity) and other topics including structure, space, and change.", "label": "Mathematics"}, "data_1505": {"class_id": "http://edamontology.org/data_1505", "definitions": "Molecular weights of amino acids.", "label": "Amino acid index (molecular weight)"}, "topic_3314": {"class_id": "http://edamontology.org/topic_3314", "definitions": "The composition and properties of matter, reactions, and the use of reactions to create new substances.", "label": "Chemistry"}, "topic_0602": {"class_id": "http://edamontology.org/topic_0602", "definitions": "Molecular interactions, biological pathways, networks and other models.", "label": "Molecular interactions, pathways and networks"}, "topic_3065": {"class_id": "http://edamontology.org/topic_3065", "definitions": "The development of organisms between the one-cell stage (typically the zygote) and the end of the embryonic stage.", "label": "Embryology"}, "topic_3064": {"class_id": "http://edamontology.org/topic_3064", "definitions": "How organisms grow and develop.", "label": "Developmental biology"}, "topic_0608": {"class_id": "http://edamontology.org/topic_0608", "definitions": "General cell culture or data on a specific cell lines.", "label": "Cell and tissue culture"}, "data_0997": {"class_id": "http://edamontology.org/data_0997", "definitions": "Unique name from Chemical Entities of Biological Interest (ChEBI) of a chemical compound.|This is the recommended chemical name for use for example in database annotation.", "label": "Chemical name (ChEBI)"}, "topic_3061": {"class_id": "http://edamontology.org/topic_3061", "definitions": "The documentation of resources such as tools, services and databases and how to get help.", "label": "Documentation and help"}, "topic_3060": {"class_id": "http://edamontology.org/topic_3060", "definitions": "Regulatory RNA sequences including microRNA (miRNA) and small interfering RNA (siRNA).", "label": "Regulatory RNA"}, "topic_3063": {"class_id": "http://edamontology.org/topic_3063", "definitions": "The application of information technology to health, disease and biomedicine.", "label": "Medical informatics"}, "topic_3062": {"class_id": "http://edamontology.org/topic_3062", "definitions": "The structural and functional organisation of genes and other genetic elements.", "label": "Genetic organisation"}, "format_3464": {"class_id": "http://edamontology.org/format_3464", "definitions": "JavaScript Object Notation format; a lightweight, text-based format to represent tree-structured data using key-value pairs.", "label": "JSON"}, "topic_2225": {"class_id": "http://edamontology.org/topic_2225", "definitions": "Protein data resources.", "label": "Protein databases"}, "data_1308": {"class_id": "http://edamontology.org/data_1308", "definitions": "matrix/scaffold attachment regions (MARs/SARs) in a DNA sequence.", "label": "Nucleic acid features report (matrix/scaffold attachment sites)"}, "data_1309": {"class_id": "http://edamontology.org/data_1309", "definitions": "A report on exonic splicing enhancers (ESE) in an exon.", "label": "Gene features (exonic splicing enhancer)"}, "data_1306": {"class_id": "http://edamontology.org/data_1306", "definitions": "Report on nucleosome formation potential or exclusion sequence(s).", "label": "Nucleosome exclusion sequences"}, "data_1307": {"class_id": "http://edamontology.org/data_1307", "definitions": "splice sites in a nucleotide sequence or alternative RNA splicing events.", "label": "Nucleic acid features report (splice sites)"}, "data_1304": {"class_id": "http://edamontology.org/data_1304", "definitions": "CpG rich regions (isochores) in a nucleotide sequence.", "label": "Nucleic acid features report (CpG island and isochore)"}, "data_1305": {"class_id": "http://edamontology.org/data_1305", "definitions": "restriction enzyme recognition sites (restriction sites) in a nucleic acid sequence.", "label": "Nucleic acid features report (restriction sites)"}, "data_1302": {"class_id": "http://edamontology.org/data_1302", "definitions": "regions or sites in a eukaryotic and eukaryotic viral RNA sequence which directs endonuclease cleavage or polyadenylation of an RNA transcript.", "label": "Nucleic acid features report (PolyA signal or site)"}, "data_1303": {"class_id": "http://edamontology.org/data_1303", "definitions": "A report on quadruplex-forming motifs in a nucleotide sequence.", "label": "Nucleic acid features (quadruplexes)"}, "data_1300": {"class_id": "http://edamontology.org/data_1300", "definitions": "A report on predicted or actual gene structure, regions which make an RNA product and features such as promoters, coding regions, splice sites etc.", "label": "Gene and transcript structure (report)"}, "data_1301": {"class_id": "http://edamontology.org/data_1301", "definitions": "regions of a nucleic acid sequence containing mobile genetic elements.", "label": "Mobile genetic elements"}, "format_1316": {"class_id": "http://edamontology.org/format_1316", "definitions": "Format of a report on exon-intron structure generated by EMBOSS est2genome.", "label": "est2genome format"}, "format_3467": {"class_id": "http://edamontology.org/format_3467", "definitions": "Graphics Interchange Format.", "label": "GIF"}, "operation_3088": {"class_id": "http://edamontology.org/operation_3088", "definitions": "Calculate (or predict) physical or chemical properties of a protein, including any non-positional properties of the molecular sequence, from processing a protein sequence.", "label": "Protein property calculation (from sequence)"}, "operation_3434": {"class_id": "http://edamontology.org/operation_3434", "definitions": "Convert a data set from one form to another.", "label": "Conversion"}, "format_3749": {"class_id": "http://edamontology.org/format_3749", "definitions": "JSON-LD, or JavaScript Object Notation for Linked Data, is a method of encoding Linked Data using JSON.", "label": "JSON-LD"}, "operation_3087": {"class_id": "http://edamontology.org/operation_3087", "definitions": "Predict, recognise and identify functional or other key sites within protein sequences, typically by scanning for known motifs, patterns and regular expressions.", "label": "Protein sequence feature detection"}, "format_1318": {"class_id": "http://edamontology.org/format_1318", "definitions": "Report format for restriction enzyme recognition sites used by EMBOSS restrict program.", "label": "restrict format"}, "format_1319": {"class_id": "http://edamontology.org/format_1319", "definitions": "Report format for restriction enzyme recognition sites used by EMBOSS restover program.", "label": "restover format"}, "operation_3083": {"class_id": "http://edamontology.org/operation_3083", "definitions": "Render (visualise) a biological pathway or network.", "label": "Pathway or network visualisation"}, "data_1916": {"class_id": "http://edamontology.org/data_1916", "definitions": "An alignment of molecular sequences, structures or profiles derived from them.", "label": "Alignment"}, "data_2578": {"class_id": "http://edamontology.org/data_2578", "definitions": "Unique identifier of a toxin from the ArachnoServer database.", "label": "ArachnoServer ID"}, "data_2579": {"class_id": "http://edamontology.org/data_2579", "definitions": "A simple summary of expressed genes.", "label": "Expressed gene list"}, "data_2576": {"class_id": "http://edamontology.org/data_2576", "definitions": "Identifier of a toxin.", "label": "Toxin identifier"}, "operation_2424": {"class_id": "http://edamontology.org/operation_2424", "definitions": "Compare two or more things to identify similarities.", "label": "Comparison"}, "operation_2425": {"class_id": "http://edamontology.org/operation_2425", "definitions": "Refine or optimise some data model.", "label": "Optimisation and refinement"}, "data_2778": {"class_id": "http://edamontology.org/data_2778", "definitions": "The number of a strain of algae and protozoa from the CCAP database.", "label": "CCAP strain number"}, "data_2779": {"class_id": "http://edamontology.org/data_2779", "definitions": "An identifier of stock from a catalogue of biological resources.", "label": "Stock number"}, "operation_2420": {"class_id": "http://edamontology.org/operation_2420", "definitions": "Process (read and / or write) data of a specific type, for example applying analytical methods.", "label": "Operation (typed)"}, "operation_2421": {"class_id": "http://edamontology.org/operation_2421", "definitions": "Search a database (or other data resource) with a supplied query and retrieve entries (or parts of entries) that are similar to the query.|Typically the query is compared to each entry and high scoring matches (hits) are returned. For example, a BLAST search of a sequence database.", "label": "Database search"}, "operation_2422": {"class_id": "http://edamontology.org/operation_2422", "definitions": "Retrieve an entry (or part of an entry) from a data resource that matches a supplied query. This might include some primary data and annotation. The query is a data identifier or other indexed term. For example, retrieve a sequence record with the specified accession number, or matching supplied keywords.", "label": "Data retrieval"}, "operation_2423": {"class_id": "http://edamontology.org/operation_2423", "definitions": "Predict, recognise, detect or identify some properties of a biomolecule.", "label": "Prediction and recognition"}, "data_2772": {"class_id": "http://edamontology.org/data_2772", "definitions": "Identifier of a antibody from the HPA database.", "label": "HPA antibody id"}, "data_2773": {"class_id": "http://edamontology.org/data_2773", "definitions": "Identifier of a human major histocompatibility complex (HLA) or other protein from the IMGT/HLA database.", "label": "IMGT/HLA ID"}, "data_2770": {"class_id": "http://edamontology.org/data_2770", "definitions": "Identifier of an RNA transcript from the H-InvDB database.", "label": "HIT ID"}, "data_2771": {"class_id": "http://edamontology.org/data_2771", "definitions": "A unique identifier of gene cluster in the H-InvDB database.", "label": "HIX ID"}, "data_2776": {"class_id": "http://edamontology.org/data_2776", "definitions": "Identifier of a physical entity from the ConsensusPathDB database.", "label": "ConsensusPathDB entity ID"}, "data_2777": {"class_id": "http://edamontology.org/data_2777", "definitions": "Name of a physical entity from the ConsensusPathDB database.", "label": "ConsensusPathDB entity name"}, "data_2774": {"class_id": "http://edamontology.org/data_2774", "definitions": "A unique identifier of gene assigned by the J. Craig Venter Institute (JCVI).", "label": "Gene ID (JCVI)"}, "data_2775": {"class_id": "http://edamontology.org/data_2775", "definitions": "The name of a kinase protein.", "label": "Kinase name"}, "format_3468": {"class_id": "http://edamontology.org/format_3468", "definitions": "Microsoft Excel spreadsheet format.", "label": "xls"}, "topic_3418": {"class_id": "http://edamontology.org/topic_3418", "definitions": "The branch of medicine that deals with the medical care of infants, children and adolescents.", "label": "Paediatrics"}, "data_1233": {"class_id": "http://edamontology.org/data_1233", "definitions": "Any collection of multiple protein sequences and associated metadata that do not (typically) correspond to common sequence database records or database entries.", "label": "Sequence set (protein)"}, "operation_3436": {"class_id": "http://edamontology.org/operation_3436", "definitions": "Combine multiple files or data items into a single file or object.", "label": "Aggregation"}, "data_1236": {"class_id": "http://edamontology.org/data_1236", "definitions": "A file of intermediate results from a PSIBLAST search that is used for priming the search in the next PSIBLAST iteration.|A Psiblast checkpoint file uses ASN.1 Binary Format and usually has the extension '.asn'.", "label": "Psiblast checkpoint file"}, "data_1237": {"class_id": "http://edamontology.org/data_1237", "definitions": "Sequences generated by HMMER package in FASTA-style format.", "label": "HMMER synthetic sequences set"}, "data_1234": {"class_id": "http://edamontology.org/data_1234", "definitions": "Any collection of multiple nucleotide sequences and associated metadata that do not (typically) correspond to common sequence database records or database entries.", "label": "Sequence set (nucleic acid)"}, "data_1235": {"class_id": "http://edamontology.org/data_1235", "definitions": "A set of sequences that have been clustered or otherwise classified as belonging to a group including (typically) sequence cluster information.|The cluster might include sequences identifiers, short descriptions, alignment and summary information.", "label": "Sequence cluster"}, "format_1956": {"class_id": "http://edamontology.org/format_1956", "definitions": "Phylip non-interleaved sequence format.", "label": "phylipnon sequence format"}, "format_1957": {"class_id": "http://edamontology.org/format_1957", "definitions": "Raw sequence format with no non-sequence characters.", "label": "raw"}, "data_1238": {"class_id": "http://edamontology.org/data_1238", "definitions": "A protein sequence cleaved into peptide fragments (by enzymatic or chemical cleavage) with fragment masses.", "label": "Proteolytic digest"}, "data_1239": {"class_id": "http://edamontology.org/data_1239", "definitions": "Restriction digest fragments from digesting a nucleotide sequence with restriction sites using a restriction endonuclease.", "label": "Restriction digest"}, "format_1952": {"class_id": "http://edamontology.org/format_1952", "definitions": "PDB nucleotide sequence format (SEQRES lines).|pdbnucseq format in EMBOSS.", "label": "pdbseqresnuc"}, "format_1953": {"class_id": "http://edamontology.org/format_1953", "definitions": "PDB sequence format (SEQRES lines).|pdbseq format in EMBOSS.", "label": "pdbseqres"}, "format_1950": {"class_id": "http://edamontology.org/format_1950", "definitions": "PDB sequence format (ATOM lines).|pdb format in EMBOSS.", "label": "pdbatom"}, "format_1951": {"class_id": "http://edamontology.org/format_1951", "definitions": "PDB nucleotide sequence format (ATOM lines).|pdbnuc format in EMBOSS.", "label": "pdbatomnuc"}, "data_3358": {"class_id": "http://edamontology.org/data_3358", "definitions": "An identifier of a data format.", "label": "Format identifier"}, "operation_3461": {"class_id": "http://edamontology.org/operation_3461", "definitions": "The prediction of the degree of pathogenicity of a microorganism from analysis of molecular sequences.", "label": "Virulence prediction"}, "operation_3799": {"class_id": "http://edamontology.org/operation_3799", "definitions": "Counting and measuring experimentally determined observations into quantities.", "label": "Quantification"}, "operation_3460": {"class_id": "http://edamontology.org/operation_3460", "definitions": "Classifiication (typically of molecular sequences) by assignment to some taxonomic hierarchy.", "label": "Taxonomic classification"}, "data_3356": {"class_id": "http://edamontology.org/data_3356", "definitions": "A statistical Markov model of a system which is assumed to be a Markov process with unobserved (hidden) states.", "label": "Hidden Markov model"}, "data_3355": {"class_id": "http://edamontology.org/data_3355", "definitions": "A HMM emission matrix holds the probabilities of choosing the four nucleotides (A, C, G and T) in each of the states of a HMM.|Consider for example an HMM with two states (AT-rich and GC-rich).  The emission matrix holds the probabilities of choosing each of the four nucleotides (A, C, G and T) in the AT-rich state and in the GC-rich state.", "label": "Emission matrix"}, "data_3354": {"class_id": "http://edamontology.org/data_3354", "definitions": "A HMM transition matrix contains the probabilities of switching from one HMM state to another.|Consider for example an HMM with two states (AT-rich and GC-rich).  The transition matrix will hold the probabilities of switching from the AT-rich to the GC-rich state, and vica versa.", "label": "Transition matrix"}, "operation_3437": {"class_id": "http://edamontology.org/operation_3437", "definitions": "Compare two or more scientific articles.", "label": "Article comparison"}, "format_3582": {"class_id": "http://edamontology.org/format_3582", "definitions": "AFG is a single text-based file assembly format  that holds read and consensus information together", "label": "afg"}, "format_3583": {"class_id": "http://edamontology.org/format_3583", "definitions": "The bedGraph format allows display of continuous-valued data in track format. This display type is useful for probability scores and transcriptome data|Holds a tab-delimited chromosome /start /end / datavalue dataset.", "label": "bedgraph"}, "operation_3435": {"class_id": "http://edamontology.org/operation_3435", "definitions": "Standardize or normalize data by some statistical method.| In the simplest normalisation means adjusting values measured on different scales to a common scale (often between 0.0 and 1.0), but can refer to more sophisticated adjustment whereby entire probability distributions of adjusted values are brought into alignment. Standardisation typically refers to an operation whereby a range of values are standardised to measure how many standard deviations a value is from its mean. ", "label": "Standardisation and normalisation"}, "format_3033": {"class_id": "http://edamontology.org/format_3033", "definitions": "Format of a matrix (array) of numerical values.", "label": "Matrix format"}, "operation_3430": {"class_id": "http://edamontology.org/operation_3430", "definitions": "Predict, recognise and identify functional or other key sites within nucleic acid sequences, typically by scanning for known motifs, patterns and regular expressions.", "label": "Nucleic acid sequence feature detection"}, "topic_0605": {"class_id": "http://edamontology.org/topic_0605", "definitions": "The study and practice of information processing and use of computer information systems.", "label": "Informatics"}, "operation_3469": {"class_id": "http://edamontology.org/operation_3469", "definitions": "Compute the covariance model for (a family of) RNA secondary structures.", "label": "RNA structure covariance model generation"}, "topic_3525": {"class_id": "http://edamontology.org/topic_3525", "definitions": "Protein-DNA/RNA interaction(s).", "label": "Protein-nucleic acid interactions"}, "data_1126": {"class_id": "http://edamontology.org/data_1126", "definitions": "Unique name or identifier of a comparison matrix.|See for example http://www.ebi.ac.uk/Tools/webservices/help/matrix.", "label": "Comparison matrix name"}, "data_1127": {"class_id": "http://edamontology.org/data_1127", "definitions": "An identifier of an entry from the PDB database.|A PDB identification code which consists of 4 characters, the first of which is a digit in the range 0 - 9; the remaining 3 are alpha-numeric, and letters are upper case only. (source: https://cdn.rcsb.org/wwpdb/docs/documentation/file-format/PDB_format_1996.pdf)", "label": "PDB ID"}, "data_1124": {"class_id": "http://edamontology.org/data_1124", "definitions": "Accession number of an entry from the TreeFam database.", "label": "TreeFam accession number"}, "data_1125": {"class_id": "http://edamontology.org/data_1125", "definitions": "A label (text token) describing the type of a comparison matrix.|For example 'blosum', 'pam', 'gonnet', 'id' etc. Comparison matrix type may be required where a series of matrices of a certain type are used.", "label": "Comparison matrix type"}, "data_1122": {"class_id": "http://edamontology.org/data_1122", "definitions": "A label (text token) describing the type of a phylogenetic tree.|For example 'nj', 'upgmp' etc.", "label": "Phylogenetic tree type"}, "data_1123": {"class_id": "http://edamontology.org/data_1123", "definitions": "Accession number of an entry from the TreeBASE database.", "label": "TreeBASE study accession number"}, "data_1120": {"class_id": "http://edamontology.org/data_1120", "definitions": "A label (text token) describing the type of a sequence alignment.|Possible values include for example the EMBOSS alignment types, BLAST alignment types and so on.", "label": "Sequence alignment type"}, "data_1121": {"class_id": "http://edamontology.org/data_1121", "definitions": "The type of a BLAST sequence alignment.", "label": "BLAST sequence alignment type"}, "operation_3431": {"class_id": "http://edamontology.org/operation_3431", "definitions": "Deposit some data in a database or some other type of repository or software system.|For non-analytical operations, see the 'Processing' branch.", "label": "Deposition"}, "data_1128": {"class_id": "http://edamontology.org/data_1128", "definitions": "Identifier of an entry from the AAindex database.", "label": "AAindex ID"}, "data_1129": {"class_id": "http://edamontology.org/data_1129", "definitions": "Accession number of an entry from the BIND database.", "label": "BIND accession number"}, "format_1921": {"class_id": "http://edamontology.org/format_1921", "definitions": "Data format for molecular sequence alignment information.", "label": "Alignment format"}, "format_3581": {"class_id": "http://edamontology.org/format_3581", "definitions": "ARFF (Attribute-Relation File Format) is an ASCII text file format that describes a list of instances sharing a set of attributes.|This file format is for machine learning.", "label": "arff"}, "format_3781": {"class_id": "http://edamontology.org/format_3781", "definitions": "JSON format of annotated scientific text used by PubAnnotations and other tools.", "label": "PubAnnotation format"}, "format_3780": {"class_id": "http://edamontology.org/format_3780", "definitions": "Data format of an annotated text, e.g. with recognised entities, concepts, and relations.", "label": "Annotated text format"}, "format_3783": {"class_id": "http://edamontology.org/format_3783", "definitions": "Native textual export format of annotated scientific text from PubTator.", "label": "PubTator format"}, "format_1920": {"class_id": "http://edamontology.org/format_1920", "definitions": "Data format for molecular sequence feature information.", "label": "Sequence feature annotation format"}, "format_3785": {"class_id": "http://edamontology.org/format_3785", "definitions": "A family of similar formats of text annotation, used by BRAT and other tools, known as BioNLP Shared Task format (BioNLP 2009 Shared Task on Event Extraction, BioNLP Shared Task 2011, BioNLP Shared Task 2013), BRAT format, BRAT standoff format, and similar.", "label": "BioNLP Shared Task format"}, "format_3784": {"class_id": "http://edamontology.org/format_3784", "definitions": "A format of text annotation using the linked-data Open Annotation Data Model, serialised typically in RDF or JSON-LD.", "label": "Open Annotation format"}, "format_3787": {"class_id": "http://edamontology.org/format_3787", "definitions": "A query language (format) for structured database queries.", "label": "Query language"}, "data_2804": {"class_id": "http://edamontology.org/data_2804", "definitions": "Unique identifier for a cone snail toxin protein from the ConoServer database.", "label": "Protein ID (ConoServer)"}, "data_2805": {"class_id": "http://edamontology.org/data_2805", "definitions": "Identifier of a GeneSNP database entry.", "label": "GeneSNP ID"}, "data_2800": {"class_id": "http://edamontology.org/data_2800", "definitions": "An identifier of a disease from the Orpha database.", "label": "Orpha number"}, "data_2802": {"class_id": "http://edamontology.org/data_2802", "definitions": "Unique identifier for a protein from the EcID database.", "label": "Protein ID (EcID)"}, "data_2803": {"class_id": "http://edamontology.org/data_2803", "definitions": "A unique identifier of a cDNA molecule catalogued in the RefSeq database.", "label": "Clone ID (RefSeq)"}, "format_1925": {"class_id": "http://edamontology.org/format_1925", "definitions": "Codata entry format.", "label": "codata"}, "operation_0467": {"class_id": "http://edamontology.org/operation_0467", "definitions": "Predict secondary structure of protein sequence(s) using multiple methods to achieve better predictions.", "label": "Protein secondary structure prediction (integrated)"}, "format_1924": {"class_id": "http://edamontology.org/format_1924", "definitions": "Clustalw output format.", "label": "clustal sequence format"}, "data_1058": {"class_id": "http://edamontology.org/data_1058", "definitions": "The name of a file (of any type) with restricted possible values.", "label": "Enumerated file name"}, "data_1059": {"class_id": "http://edamontology.org/data_1059", "definitions": "The extension of a file name.|A file extension is the characters appearing after the final '.' in the file name.", "label": "File name extension"}, "data_1052": {"class_id": "http://edamontology.org/data_1052", "definitions": "A Uniform Resource Locator (URL).", "label": "URL"}, "data_1053": {"class_id": "http://edamontology.org/data_1053", "definitions": "A Uniform Resource Name (URN).", "label": "URN"}, "data_1050": {"class_id": "http://edamontology.org/data_1050", "definitions": "The name (or part of a name) of a file (of any type).", "label": "File name"}, "data_1051": {"class_id": "http://edamontology.org/data_1051", "definitions": "Name of an ontology of biological or bioinformatics concepts and relations.", "label": "Ontology name"}, "data_1056": {"class_id": "http://edamontology.org/data_1056", "definitions": "The name of a biological or bioinformatics database.", "label": "Database name"}, "data_1057": {"class_id": "http://edamontology.org/data_1057", "definitions": "The name of a molecular sequence database.", "label": "Sequence database name"}, "data_1055": {"class_id": "http://edamontology.org/data_1055", "definitions": "A Life Science Identifier (LSID) - a unique identifier of some data.|LSIDs provide a standard way to locate and describe data. An LSID is represented as a Uniform Resource Name (URN) with the following format: URN:LSID:<Authority>:<Namespace>:<ObjectID>[:<Version>]", "label": "LSID"}, "operation_0460": {"class_id": "http://edamontology.org/operation_0460", "definitions": "Calculate and plot a DNA or DNA/RNA temperature profile.", "label": "Nucleic acid temperature profile plotting"}, "format_3587": {"class_id": "http://edamontology.org/format_3587", "definitions": "Tabular format of chromosome names and sizes used by Galaxy.|Galaxy allows BED files to contain non-standard fields beyond the first 3 columns, some other implementations do not.  ", "label": "chrominfo"}, "topic_0601": {"class_id": "http://edamontology.org/topic_0601", "definitions": "Protein chemical modifications, e.g. post-translational modifications.|EDAM does not describe all possible protein modifications. For fine-grained annotation of protein modification use the Gene Ontology (children of concept GO:0006464) and/or the Protein Modifications ontology (children of concept MOD:00000)", "label": "Protein modifications"}, "data_2970": {"class_id": "http://edamontology.org/data_2970", "definitions": "A report on protein properties concerning hydropathy.", "label": "Protein hydropathy data"}, "data_2971": {"class_id": "http://edamontology.org/data_2971", "definitions": "Data concerning a computational workflow.", "label": "Workflow data"}, "data_2972": {"class_id": "http://edamontology.org/data_2972", "definitions": "A computational workflow.", "label": "Workflow"}, "data_2973": {"class_id": "http://edamontology.org/data_2973", "definitions": "Data concerning molecular secondary structure data.", "label": "Secondary structure data"}, "data_2974": {"class_id": "http://edamontology.org/data_2974", "definitions": "A raw protein sequence (string of characters).", "label": "Protein sequence (raw)"}, "data_2975": {"class_id": "http://edamontology.org/data_2975", "definitions": "A raw nucleic acid sequence.", "label": "Nucleic acid sequence (raw)"}, "data_2976": {"class_id": "http://edamontology.org/data_2976", "definitions": "One or more protein sequences, possibly with associated annotation.", "label": "Protein sequence"}, "data_2977": {"class_id": "http://edamontology.org/data_2977", "definitions": "One or more nucleic acid sequences, possibly with associated annotation.", "label": "Nucleic acid sequence"}, "data_2978": {"class_id": "http://edamontology.org/data_2978", "definitions": "Data concerning a biochemical reaction, typically data and more general annotation on the kinetics of enzyme-catalysed reaction.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Reaction data"}, "data_2979": {"class_id": "http://edamontology.org/data_2979", "definitions": "Data concerning small peptides.", "label": "Peptide property"}, "data_0884": {"class_id": "http://edamontology.org/data_0884", "definitions": "An entry from a molecular tertiary (3D) structure database.", "label": "Tertiary structure record"}, "data_0885": {"class_id": "http://edamontology.org/data_0885", "definitions": "Results (hits) from searching a database of tertiary structure.", "label": "Structure database search results"}, "data_0882": {"class_id": "http://edamontology.org/data_0882", "definitions": "An informative report of RNA secondary structure alignment-derived data or metadata.", "label": "Secondary structure alignment metadata (RNA)"}, "data_0883": {"class_id": "http://edamontology.org/data_0883", "definitions": "3D coordinate and associated data for a macromolecular tertiary (3D) structure or part of a structure.|The coordinate data may be predicted or real.", "label": "Structure"}, "data_0880": {"class_id": "http://edamontology.org/data_0880", "definitions": "An informative report of secondary structure (predicted or real) of an RNA molecule.|This includes thermodynamically stable or evolutionarily conserved structures such as knots, pseudoknots etc.", "label": "RNA secondary structure"}, "data_0881": {"class_id": "http://edamontology.org/data_0881", "definitions": "Alignment of the (1D representations of) secondary structure of two or more RNA molecules.", "label": "Secondary structure alignment (RNA)"}, "operation_2963": {"class_id": "http://edamontology.org/operation_2963", "definitions": "Generate a codon usage bias plot.", "label": "Codon usage bias plotting"}, "operation_2962": {"class_id": "http://edamontology.org/operation_2962", "definitions": "Calculate codon usage bias.", "label": "Codon usage bias calculation"}, "topic_0803": {"class_id": "http://edamontology.org/topic_0803", "definitions": "Human diseases, typically describing the genes, mutations and proteins implicated in disease.", "label": "Human disease"}, "operation_2964": {"class_id": "http://edamontology.org/operation_2964", "definitions": "Calculate the differences in codon usage fractions between two sequences, sets of sequences, codon usage tables etc.", "label": "Codon usage fraction calculation"}, "format_2562": {"class_id": "http://edamontology.org/format_2562", "definitions": "Text format (representation) of amino acid residues.", "label": "Amino acid identifier format"}, "topic_0159": {"class_id": "http://edamontology.org/topic_0159", "definitions": "The comparison of two or more molecular sequences, for example sequence alignment and clustering.|The comparison might be on the basis of sequence, physico-chemical or some other properties of the sequences.", "label": "Sequence comparison"}, "format_2560": {"class_id": "http://edamontology.org/format_2560", "definitions": "Entry format for the STRING database of protein interaction.", "label": "STRING entry format"}, "format_2561": {"class_id": "http://edamontology.org/format_2561", "definitions": "Text format for sequence assembly data.", "label": "Sequence assembly format (text)"}, "data_1449": {"class_id": "http://edamontology.org/data_1449", "definitions": "Matrix of integer or floating point numbers for amino acid comparison.", "label": "Comparison matrix (amino acid)"}, "data_1448": {"class_id": "http://edamontology.org/data_1448", "definitions": "Matrix of integer or floating point numbers for nucleotide comparison.", "label": "Comparison matrix (nucleotide)"}, "topic_0150": {"class_id": "http://edamontology.org/topic_0150", "definitions": "The design of proteins with specific properties, typically by designing changes (via site-directed mutagenesis) to an existing protein.", "label": "Protein design"}, "data_1444": {"class_id": "http://edamontology.org/data_1444", "definitions": "Independent contrasts for characters used in a phylogenetic tree, or covariances, regressions and correlations between characters for those contrasts.", "label": "Phylogenetic character contrasts"}, "data_1447": {"class_id": "http://edamontology.org/data_1447", "definitions": "Matrix of floating point numbers for sequence comparison.", "label": "Comparison matrix (floats)"}, "data_1446": {"class_id": "http://edamontology.org/data_1446", "definitions": "Matrix of integer numbers for sequence comparison.", "label": "Comparison matrix (integers)"}, "data_1441": {"class_id": "http://edamontology.org/data_1441", "definitions": "Data on the confidence of a phylogenetic tree.", "label": "Phylogenetic tree report (tree evaluation)"}, "data_1440": {"class_id": "http://edamontology.org/data_1440", "definitions": "Data about the shape of a phylogenetic tree.", "label": "Phylogenetic tree report (tree shape)"}, "data_1443": {"class_id": "http://edamontology.org/data_1443", "definitions": "Molecular clock and stratigraphic (age) data derived from phylogenetic tree analysis.", "label": "Phylogenetic tree report (tree stratigraphic)"}, "data_1442": {"class_id": "http://edamontology.org/data_1442", "definitions": "Distances, such as Branch Score distance, between two or more phylogenetic trees.", "label": "Phylogenetic tree distances"}, "data_1689": {"class_id": "http://edamontology.org/data_1689", "definitions": "A username on a computer system.", "label": "Username"}, "data_1688": {"class_id": "http://edamontology.org/data_1688", "definitions": "EMBOSS vectorstrip log file.", "label": "EMBOSS vectorstrip log file"}, "data_2372": {"class_id": "http://edamontology.org/data_2372", "definitions": "An informative report on individual spot(s) from a two-dimensional (2D PAGE) gel.", "label": "2D PAGE spot report"}, "data_2373": {"class_id": "http://edamontology.org/data_2373", "definitions": "Unique identifier of a spot from a two-dimensional (protein) gel.", "label": "Spot ID"}, "data_2370": {"class_id": "http://edamontology.org/data_2370", "definitions": "Identifier of a polyA signal from the ASTD database.", "label": "ASTD ID (polya)"}, "data_2371": {"class_id": "http://edamontology.org/data_2371", "definitions": "Identifier of a transcription start site from the ASTD database.", "label": "ASTD ID (tss)"}, "data_1683": {"class_id": "http://edamontology.org/data_1683", "definitions": "EMBOSS (EMBASSY) domainatrix application log file.", "label": "EMBOSS domainatrix log file"}, "data_1682": {"class_id": "http://edamontology.org/data_1682", "definitions": "EMBOSS wordfinder log file.", "label": "EMBOSS wordfinder log file"}, "data_1681": {"class_id": "http://edamontology.org/data_1681", "definitions": "NACCESS log file.", "label": "NACCESS log file"}, "data_1680": {"class_id": "http://edamontology.org/data_1680", "definitions": "STRIDE log file.", "label": "STRIDE log file"}, "data_1687": {"class_id": "http://edamontology.org/data_1687", "definitions": "EMBOSS megamerger log file.", "label": "EMBOSS whichdb log file"}, "data_1686": {"class_id": "http://edamontology.org/data_1686", "definitions": "EMBOSS megamerger log file.", "label": "EMBOSS megamerger log file"}, "data_2378": {"class_id": "http://edamontology.org/data_2378", "definitions": "Data on the interaction of a protein (or protein domain) with specific structural (3D) and/or sequence motifs.", "label": "Protein-motif interaction"}, "data_2379": {"class_id": "http://edamontology.org/data_2379", "definitions": "Identifier of a strain of an organism variant, typically a plant, virus or bacterium.", "label": "Strain identifier"}, "format_3789": {"class_id": "http://edamontology.org/format_3789", "definitions": "XQuery (XML Query) is a query language (format of queries) for querying and manipulating structured and unstructured data, usually in the form of XML, text, and with vendor-specific extensions for other data formats (JSON, binary, etc.).", "label": "XQuery"}, "format_3788": {"class_id": "http://edamontology.org/format_3788", "definitions": "SQL (Structured Query Language) is the de-facto standard query language (format of queries) for querying and manipulating data in relational databases.", "label": "SQL"}, "data_2012": {"class_id": "http://edamontology.org/data_2012", "definitions": "A position in a map (for example a genetic map), either a single position (point) or a region / interval.|This includes positions in genomes based on a reference sequence. A position may be specified for any mappable object, i.e. anything that may have positional information such as a physical position in a chromosome. Data might include sequence region name, strand, coordinate system name, assembly name, start position and end position.", "label": "Sequence coordinates"}, "topic_3543": {"class_id": "http://edamontology.org/topic_3543", "definitions": "Short repetitive subsequences (repeat sequences) in a protein sequence.", "label": "Protein sequence repeats"}, "data_2016": {"class_id": "http://edamontology.org/data_2016", "definitions": "Data concerning the intrinsic physical (e.g. structural) or chemical properties of one, more or all amino acids.", "label": "Amino acid property"}, "topic_3169": {"class_id": "http://edamontology.org/topic_3169", "definitions": "The analysis of protein-DNA interactions where chromatin immunoprecipitation (ChIP) is used in combination with massively parallel DNA sequencing to identify the binding sites of DNA-associated proteins.", "label": "ChIP-seq"}, "data_2019": {"class_id": "http://edamontology.org/data_2019", "definitions": "A molecular map (genetic or physical), an attribute of such a map, or data extracted from or derived from the analysis of such a map.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation. It includes concepts that are best described as scientific text or closely concerned with or derived from text.", "label": "Map data"}, "data_2018": {"class_id": "http://edamontology.org/data_2018", "definitions": "A human-readable collection of information which (typically) is generated or collated by hand and which describes a biological entity, phenomena or associated primary (e.g. sequence or structural) data, as distinct from the primary data itself and computer-generated reports derived from it.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Annotation"}, "format_3589": {"class_id": "http://edamontology.org/format_3589", "definitions": "Color space FASTA format sequence variant.|FASTA format extended for color space information.", "label": "csfasta"}, "operation_0303": {"class_id": "http://edamontology.org/operation_0303", "definitions": "Recognize (predict and identify) known protein structural domains or folds in protein sequence(s) which (typically) are not accompanied by any significant sequence similarity to know structures.|Methods use some type of mapping between sequence and fold, for example secondary structure prediction and alignment, profile comparison, sequence properties, homologous sequence search, kernel machines etc. Domains and folds might be taken from SCOP or CATH.", "label": "Protein fold recognition"}, "operation_0302": {"class_id": "http://edamontology.org/operation_0302", "definitions": "Align molecular sequence to structure in 3D space (threading).|Use this concept for methods that evaluate sequence-structure compatibility by assessing residue interactions in 3D. Methods might perform one-to-one, one-to-many or many-to-many comparisons.", "label": "Protein threading"}, "operation_0301": {"class_id": "http://edamontology.org/operation_0301", "definitions": "Align molecular sequence(s) to structural (3D) profile(s) or template(s) (representing a structure or structure alignment).", "label": "Sequence-to-3D-profile alignment"}, "operation_0300": {"class_id": "http://edamontology.org/operation_0300", "definitions": "Align molecular sequence(s) to sequence profile(s), or profiles to other profiles.  A profile typically represents a sequence alignment.|A sequence profile typically represents a sequence alignment. Methods might perform one-to-one, one-to-many or many-to-many comparisons.", "label": "Sequence profile alignment"}, "operation_0307": {"class_id": "http://edamontology.org/operation_0307", "definitions": "Perform in-silico (virtual) PCR.", "label": "Virtual PCR"}, "operation_0306": {"class_id": "http://edamontology.org/operation_0306", "definitions": "Process and analyse text (typically scientific literature) to extract information from it.", "label": "Text mining"}, "data_1599": {"class_id": "http://edamontology.org/data_1599", "definitions": "A simple measure of synonymous codon usage bias often used to predict gene expression levels.", "label": "Codon adaptation index"}, "data_1598": {"class_id": "http://edamontology.org/data_1598", "definitions": "A genetic code for an organism.|A genetic code need not include detailed codon usage information.", "label": "Genetic code"}, "data_1597": {"class_id": "http://edamontology.org/data_1597", "definitions": "Table of codon usage data calculated from one or more nucleic acid sequences.|A codon usage table might include the codon usage table name, optional comments and a table with columns for codons and corresponding codon usage data. A genetic code can be extracted from or represented by a codon usage table.", "label": "Codon usage table"}, "data_1596": {"class_id": "http://edamontology.org/data_1596", "definitions": "A report on an analysis of RNA/DNA folding, minimum folding energies for DNA or RNA sequences, energy landscape of RNA mutants etc.", "label": "Nucleic acid folding report"}, "data_1595": {"class_id": "http://edamontology.org/data_1595", "definitions": "Dotplot of RNA base pairing probability matrix.|Such as generated by the Vienna package.", "label": "Base pairing probability matrix dotplot"}, "data_1594": {"class_id": "http://edamontology.org/data_1594", "definitions": "RNA calculated energy data generated by the Vienna package.", "label": "Vienna RNA calculated energy"}, "data_1593": {"class_id": "http://edamontology.org/data_1593", "definitions": "RNA concentration data used by the Vienna package.", "label": "Vienna RNA concentration data"}, "data_1592": {"class_id": "http://edamontology.org/data_1592", "definitions": "Structure constraints used by the Vienna package.", "label": "Vienna RNA structure constraints"}, "data_1591": {"class_id": "http://edamontology.org/data_1591", "definitions": "RNA parameters used by the Vienna package.", "label": "Vienna RNA parameters"}, "data_1590": {"class_id": "http://edamontology.org/data_1590", "definitions": "DNA base trimer roll angles data.", "label": "DNA base trimer roll angles data"}, "format_1478": {"class_id": "http://edamontology.org/format_1478", "definitions": "Entry format of PDB database in PDBML (XML) format.", "label": "PDBML"}, "operation_3658": {"class_id": "http://edamontology.org/operation_3658", "definitions": "Analyse data in order to deduce properties of an underlying distribution or population.", "label": "Statistical inference"}, "operation_3659": {"class_id": "http://edamontology.org/operation_3659", "definitions": "A statistical calculation to estimate the relationships among variables. ", "label": "Regression analysis"}, "format_3588": {"class_id": "http://edamontology.org/format_3588", "definitions": "Custom Sequence annotation track format used by Galaxy.|Used for tracks/track views within galaxy.", "label": "customtrack"}, "topic_3541": {"class_id": "http://edamontology.org/topic_3541", "definitions": "Post-translation modifications in a protein sequence, typically describing the specific sites involved.", "label": "Protein post-translational modifications"}, "topic_3415": {"class_id": "http://edamontology.org/topic_3415", "definitions": "The branch of medicine that deals with the diagnosis, management and prevention of poisoning and other adverse health effects caused by medications, occupational and environmental toxins, and biological agents.", "label": "Medical toxicology"}, "data_2167": {"class_id": "http://edamontology.org/data_2167", "definitions": "Density plot (of base composition) for a nucleotide sequence.", "label": "Nucleic acid density plot"}, "data_2166": {"class_id": "http://edamontology.org/data_2166", "definitions": "A plot of character or word composition / frequency of a molecular sequence.", "label": "Sequence composition plot"}, "data_2165": {"class_id": "http://edamontology.org/data_2165", "definitions": "A plot of pK versus pH for a protein.", "label": "Protein ionisation curve"}, "data_2164": {"class_id": "http://edamontology.org/data_2164", "definitions": "A plot of general physicochemical properties of a protein sequence.", "label": "Protein sequence properties plot"}, "data_2163": {"class_id": "http://edamontology.org/data_2163", "definitions": "An image of peptide sequence sequence in a simple 3,4,3,4 repeating pattern that emulates at a simple level the arrangement of residues around an alpha helix.|Useful for highlighting amphipathicity and other properties.", "label": "Helical net"}, "data_2162": {"class_id": "http://edamontology.org/data_2162", "definitions": "An image of peptide sequence sequence looking down the axis of the helix for highlighting amphipathicity and other properties.", "label": "Helical wheel"}, "data_2161": {"class_id": "http://edamontology.org/data_2161", "definitions": "A plot of sequence similarities identified from word-matching or character comparison.|Use this concept for calculated substitution rates, relative site variability, data on sites with biased properties, highly conserved or very poorly conserved sites, regions, blocks etc.", "label": "Sequence similarity plot"}, "data_2160": {"class_id": "http://edamontology.org/data_2160", "definitions": "A plot of Fickett testcode statistic (identifying protein coding regions) in a nucleotide sequences.", "label": "Fickett testcode plot"}, "format_2210": {"class_id": "http://edamontology.org/format_2210", "definitions": "Format of a report on organism strain data / cell line.", "label": "Strain data format"}, "format_2211": {"class_id": "http://edamontology.org/format_2211", "definitions": "Format for a report of strain data as used for CIP database entries.", "label": "CIP strain data format"}, "data_3425": {"class_id": "http://edamontology.org/data_3425", "definitions": "Data concerning the intrinsic physical (e.g. structural) or chemical properties of one, more or all carbohydrates.", "label": "Carbohydrate property"}, "data_1865": {"class_id": "http://edamontology.org/data_1865", "definitions": "A feature which may mapped (positioned) on a genetic or other type of map.|Mappable features may be based on Gramene's notion of map features; see http://www.gramene.org/db/cmap/feature_type_info.", "label": "Map feature"}, "data_2169": {"class_id": "http://edamontology.org/data_2169", "definitions": "A report on siRNA duplexes in mRNA.", "label": "Nucleic acid features (siRNA)"}, "data_2168": {"class_id": "http://edamontology.org/data_2168", "definitions": "Image of a sequence trace (nucleotide sequence versus probabilities of each of the 4 bases).", "label": "Sequence trace image"}, "format_3777": {"class_id": "http://edamontology.org/format_3777", "definitions": "The FAO/Bioversity/IPGRI Multi-Crop Passport Descriptors (MCPD) is an international standard format for exchange of germplasm information.", "label": "MCPD"}, "data_0849": {"class_id": "http://edamontology.org/data_0849", "definitions": "A molecular sequence and associated metadata.", "label": "Sequence record"}, "operation_0279": {"class_id": "http://edamontology.org/operation_0279", "definitions": "Analyse some aspect of RNA/DNA folding, typically by processing sequence and/or structural data.", "label": "Nucleic acid folding analysis"}, "operation_0278": {"class_id": "http://edamontology.org/operation_0278", "definitions": "Predict RNA secondary structure (for example knots, pseudoknots, alternative structures etc).|Methods might use RNA motifs, predicted intermolecular contacts, or RNA sequence-structure compatibility (inverse RNA folding).", "label": "RNA secondary structure prediction"}, "operation_0277": {"class_id": "http://edamontology.org/operation_0277", "definitions": "Compare two or more biological pathways or networks.", "label": "Pathway or network comparison"}, "operation_0276": {"class_id": "http://edamontology.org/operation_0276", "definitions": "Analyse a network of protein interactions.", "label": "Protein interaction network analysis"}, "operation_0275": {"class_id": "http://edamontology.org/operation_0275", "definitions": "Identify or predict protein-protein interactions, interfaces, binding sites etc in protein structures.", "label": "Protein-protein interaction prediction (from protein structure)"}, "operation_0274": {"class_id": "http://edamontology.org/operation_0274", "definitions": "Identify or predict protein-protein interactions, interfaces, binding sites etc in protein sequences.", "label": "Protein-protein interaction prediction (from protein sequence)"}, "operation_0273": {"class_id": "http://edamontology.org/operation_0273", "definitions": "Analyse experimental protein-protein interaction data from for example yeast two-hybrid analysis, protein microarrays, immunoaffinity chromatography followed by mass spectrometry, phage display etc.", "label": "Protein interaction raw data analysis"}, "operation_0272": {"class_id": "http://edamontology.org/operation_0272", "definitions": "Predict contacts, non-covalent interactions and distance (constraints) between amino acids in protein sequences.|Methods usually involve multiple sequence alignment analysis.", "label": "Residue interaction prediction"}, "operation_0271": {"class_id": "http://edamontology.org/operation_0271", "definitions": "Predict tertiary structure of a molecular (biopolymer) sequence.", "label": "Structure prediction"}, "operation_0270": {"class_id": "http://edamontology.org/operation_0270", "definitions": "Analyse transmembrane protein(s), typically by processing sequence and / or structural data, and write an informative report for example about the protein and its transmembrane domains / regions.|Use this (or child) concept for analysis of transmembrane domains (buried and exposed faces), transmembrane helices, helix topology, orientation, inter-helical contacts, membrane dipping (re-entrant) loops and other secondary structure etc. Methods might use pattern discovery, hidden Markov models, sequence alignment, structural profiles, amino acid property analysis, comparison to known domains or some combination (hybrid methods).", "label": "Transmembrane protein analysis"}, "topic_3125": {"class_id": "http://edamontology.org/topic_3125", "definitions": "Nucleic acids binding to some other molecule.|This includes sites involved with DNA replication and recombination.  This includes binding sites for initiation of replication (origin of replication), regions where transfer is initiated during the conjugation or mobilisation (origin of transfer), starting sites for DNA duplication (origin of replication) and regions which are eliminated through any of kind of recombination.  Also nucleosome exclusion regions, i.e. specific patterns or regions which exclude nucleosomes (the basic structural units of eukaryotic chromatin which play a significant role in regulating gene expression).|This includes ribosome binding sites (Shine-Dalgarno sequence in prokaryotes), restriction enzyme recognition sites (restriction sites) etc.", "label": "DNA binding sites"}, "topic_2885": {"class_id": "http://edamontology.org/topic_2885", "definitions": "DNA polymorphism.|Includes microsatellite polymorphism in a DNA sequence.  A microsatellite polymorphism is a very short subsequence that is repeated a variable number of times between individuals. These repeats consist of the nucleotides cytosine and adenosine.|Includes restriction fragment length polymorphisms (RFLP) in a DNA sequence.  An RFLP is defined by the presence or absence of a specific restriction site of a bacterial restriction enzyme.|Includes single nucleotide polymorphisms (SNP) and associated data, for example, the discovery and annotation of SNPs.  A SNP is a DNA sequence variation where a single nucleotide differs between members of a species or paired chromosomes in an individual.|Includes variable number of tandem repeat (VNTR) polymorphism in a DNA sequence.  VNTRs occur in non-coding regions of DNA and consists sub-sequence that is repeated a multiple (and varied) number of times.", "label": "DNA polymorphism"}, "topic_3127": {"class_id": "http://edamontology.org/topic_3127", "definitions": "DNA replication or recombination.", "label": "DNA replication and recombination"}, "topic_3120": {"class_id": "http://edamontology.org/topic_3120", "definitions": "Protein sequence variants produced e.g. from alternative splicing, alternative promoter usage, alternative initiation and ribosomal frameshifting.", "label": "Protein variants"}, "topic_3123": {"class_id": "http://edamontology.org/topic_3123", "definitions": "Regions within a nucleic acid sequence containing a signal that alters a biological function.", "label": "Expression signals"}, "operation_0457": {"class_id": "http://edamontology.org/operation_0457", "definitions": "Calculate and plot a DNA or DNA/RNA stitch profile.|A stitch profile represents the alternative conformations that partly melted DNA can adopt in a temperature range.", "label": "Nucleic acid stitch profile plotting"}, "data_3165": {"class_id": "http://edamontology.org/data_3165", "definitions": "sequencing experiment, including samples, sampling, preparation, sequencing, and analysis.", "label": "NGS experiment"}, "operation_0455": {"class_id": "http://edamontology.org/operation_0455", "definitions": "Calculate a thermodynamic property of DNA or DNA/RNA, such as melting temperature, enthalpy and entropy.", "label": "Nucleic acid thermodynamic property calculation"}, "operation_0453": {"class_id": "http://edamontology.org/operation_0453", "definitions": "Predict nucleosome formation potential of DNA sequences.", "label": "Nucleosome formation potential prediction"}, "operation_0452": {"class_id": "http://edamontology.org/operation_0452", "definitions": "Identify insertion, deletion and duplication events from a sequence alignment.|Tools might use a genetic algorithm, quartet-mapping, bootscanning, graphical methods, random forest model and so on.", "label": "Indel detection"}, "operation_0451": {"class_id": "http://edamontology.org/operation_0451", "definitions": "Detect recombination (hotspots and coldspots) and identify recombination breakpoints in a sequence alignment.|Tools might use a genetic algorithm, quartet-mapping, bootscanning, graphical methods, random forest model and so on.", "label": "Recombination detection"}, "operation_0450": {"class_id": "http://edamontology.org/operation_0450", "definitions": "Detects chimeric sequences (chimeras) from a sequence alignment.|A chimera includes regions from two or more phylogenetically distinct sequences. They are usually artifacts of PCR and are thought to occur when a prematurely terminated amplicon reanneals to another DNA strand and is subsequently copied to completion in later PCR cycles.", "label": "Chimeric sequence detection"}, "operation_0459": {"class_id": "http://edamontology.org/operation_0459", "definitions": "Calculate and plot a DNA or DNA/RNA probability profile.", "label": "Nucleic acid probability profile plotting"}, "operation_0458": {"class_id": "http://edamontology.org/operation_0458", "definitions": "Calculate and plot a DNA or DNA/RNA melting curve.", "label": "Nucleic acid melting curve plotting"}, "topic_0770": {"class_id": "http://edamontology.org/topic_0770", "definitions": "Structuring data into basic types and (computational) objects.", "label": "Data types and objects"}, "data_3768": {"class_id": "http://edamontology.org/data_3768", "definitions": "Groupings of gene expression profiles according to a clustering algorithm.", "label": "Clustered gene expression profiles"}, "data_3769": {"class_id": "http://edamontology.org/data_3769", "definitions": "An identifier of a concept from the BRENDA ontology.", "label": "BRENDA ontology concept ID"}, "topic_3050": {"class_id": "http://edamontology.org/topic_3050", "definitions": "The degree of variation of life forms within a given ecosystem, biome or an entire planet.", "label": "Biodiversity"}, "topic_3052": {"class_id": "http://edamontology.org/topic_3052", "definitions": "The comparison, grouping together and classification of macromolecules on the basis of sequence similarity.|This includes the results of sequence clustering, ortholog identification, assignment to families, annotation etc.", "label": "Sequence clusters and classification"}, "topic_3053": {"class_id": "http://edamontology.org/topic_3053", "definitions": "The study of genes, genetic variation and heredity in living organisms.", "label": "Genetics"}, "data_1869": {"class_id": "http://edamontology.org/data_1869", "definitions": "A unique identifier of a (group of) organisms.", "label": "Organism identifier"}, "topic_3056": {"class_id": "http://edamontology.org/topic_3056", "definitions": "The distribution of allele frequencies in a population of organisms and its change subject to evolutionary processes including natural selection, genetic drift, mutation and gene flow.", "label": "Population genetics"}, "data_1868": {"class_id": "http://edamontology.org/data_1868", "definitions": "The name of a group of organisms belonging to the same taxonomic rank.|For a complete list of taxonomic ranks see https://www.phenoscape.org/wiki/Taxonomic_Rank_Vocabulary.", "label": "Taxon"}, "topic_0771": {"class_id": "http://edamontology.org/topic_0771", "definitions": "", "label": "Theoretical biology"}, "data_1331": {"class_id": "http://edamontology.org/data_1331", "definitions": "A report or plot of PEST sites in a protein sequence.|'PEST' motifs target proteins for proteolytic degradation and reduce the half-lives of proteins dramatically.", "label": "Protein features (PEST sites)"}, "data_1330": {"class_id": "http://edamontology.org/data_1330", "definitions": "A report on predicted epitopes that bind to MHC class II molecules.", "label": "MHC Class II epitopes report"}, "topic_2232": {"class_id": "http://edamontology.org/topic_2232", "definitions": "Lipoproteins (protein-lipid assemblies).", "label": "Lipoproteins"}, "topic_2230": {"class_id": "http://edamontology.org/topic_2230", "definitions": "Topic focused on identifying, grouping, or naming things in a structured way according to some schema based on observable relationships.", "label": "Classification"}, "topic_3542": {"class_id": "http://edamontology.org/topic_3542", "definitions": "Secondary structure (predicted or real) of a protein, including super-secondary structure.|Super-secondary structures include leucine zippers, coiled coils, Helix-Turn-Helix etc.|The location and size of the secondary structure elements and intervening loop regions is typically given.  The report can include disulphide bonds and post-translationally formed peptide bonds (crosslinks).", "label": "Protein secondary structure"}, "format_1320": {"class_id": "http://edamontology.org/format_1320", "definitions": "Report format for restriction enzyme recognition sites used by REBASE database.", "label": "REBASE restriction sites"}, "data_1339": {"class_id": "http://edamontology.org/data_1339", "definitions": "Alignments from a sequence database search (for example a BLAST search).", "label": "Sequence database hits alignments list"}, "data_1338": {"class_id": "http://edamontology.org/data_1338", "definitions": "Scores from a sequence database search (for example a BLAST search).", "label": "Sequence database hits scores list"}, "operation_3504": {"class_id": "http://edamontology.org/operation_3504", "definitions": "Identify and map patterns of genomic variations.|Methods often utilise a database of aligned reads.", "label": "Variant pattern analysis"}, "topic_3544": {"class_id": "http://edamontology.org/topic_3544", "definitions": "Signal peptides or signal peptide cleavage sites in protein sequences.", "label": "Protein signal peptides"}, "data_2581": {"class_id": "http://edamontology.org/data_2581", "definitions": "The name of a concept from the GO ontology.", "label": "GO concept name"}, "data_2580": {"class_id": "http://edamontology.org/data_2580", "definitions": "Unique identifier of a monomer from the BindingDB database.", "label": "BindingDB Monomer ID"}, "data_2583": {"class_id": "http://edamontology.org/data_2583", "definitions": "An identifier of a 'molecular function' concept from the the Gene Ontology.", "label": "GO concept ID (molecular function)"}, "data_2582": {"class_id": "http://edamontology.org/data_2582", "definitions": "An identifier of a 'biological process' concept from the the Gene Ontology.", "label": "GO concept ID (biological process)"}, "data_2584": {"class_id": "http://edamontology.org/data_2584", "definitions": "The name of a concept for a cellular component from the GO ontology.", "label": "GO concept name (cellular component)"}, "data_2587": {"class_id": "http://edamontology.org/data_2587", "definitions": "Unique identifier of a blot from a Northern Blot.", "label": "Blot ID"}, "data_2586": {"class_id": "http://edamontology.org/data_2586", "definitions": "An image arising from a Northern Blot experiment.", "label": "Northern blot image"}, "data_2589": {"class_id": "http://edamontology.org/data_2589", "definitions": "Raw data on a biological hierarchy, describing the hierarchy proper, hierarchy components and possibly associated annotation.", "label": "Hierarchy"}, "data_2588": {"class_id": "http://edamontology.org/data_2588", "definitions": "Unique identifier of a blot from a Northern Blot from the BlotBase database.", "label": "BlotBase blot ID"}, "operation_2433": {"class_id": "http://edamontology.org/operation_2433", "definitions": "Process (read and / or write) a codon usage table.", "label": "Codon usage table processing"}, "operation_2432": {"class_id": "http://edamontology.org/operation_2432", "definitions": "Process (read and / or write) microarray data.", "label": "Microarray data processing"}, "operation_2430": {"class_id": "http://edamontology.org/operation_2430", "definitions": "Design a biological entity (typically a molecular sequence or structure) with specific properties.", "label": "Design"}, "operation_2437": {"class_id": "http://edamontology.org/operation_2437", "definitions": "Predict a network of gene regulation.", "label": "Gene regulatory network prediction"}, "operation_2436": {"class_id": "http://edamontology.org/operation_2436", "definitions": "Analysis of a set of genes, corresponding e.g. to a gene expression profile, annotated with Gene Ontology (GO) concepts, where eventual over-/under-representation of certain GO concept within the studied set of genes is revealed.|The Gene Ontology (GO) is typically used, the input is a set of Gene IDs, and the output of the analysis is typically a ranked list of GO concepts, each associated with a p-value.", "label": "Gene-set enrichment analysis"}, "data_2749": {"class_id": "http://edamontology.org/data_2749", "definitions": "An identifier of a particular genome.", "label": "Genome identifier"}, "data_2748": {"class_id": "http://edamontology.org/data_2748", "definitions": "The name of a subdivision of the Osteogenesis database.", "label": "Database name (Osteogenesis)"}, "data_2747": {"class_id": "http://edamontology.org/data_2747", "definitions": "The name of a subdivision of the Collagen Mutation Database (CMD) database.", "label": "Database name (CMD)"}, "data_2746": {"class_id": "http://edamontology.org/data_2746", "definitions": "Unique identifier of a monosaccharide from the MonosaccharideDB database.", "label": "MonosaccharideDB ID"}, "data_2745": {"class_id": "http://edamontology.org/data_2745", "definitions": "Identifier of a locus from the UTR database.", "label": "Locus ID (UTR)"}, "data_2744": {"class_id": "http://edamontology.org/data_2744", "definitions": "Identifier of a locus from the PseudoCAP database.", "label": "Locus ID (PseudoCAP)"}, "data_2743": {"class_id": "http://edamontology.org/data_2743", "definitions": "Name of an entry (gene) from the HUGO database.", "label": "Gene name (HUGO)"}, "data_2742": {"class_id": "http://edamontology.org/data_2742", "definitions": "Identifier of a transcription factor from the AraC-XylS database.", "label": "AraC-XylS ID"}, "data_2741": {"class_id": "http://edamontology.org/data_2741", "definitions": "Identifier of an entry (promoter) from the ABS database.", "label": "ABS ID"}, "data_2740": {"class_id": "http://edamontology.org/data_2740", "definitions": "Name of an entry (gene) from the Genolist genes database.", "label": "Gene name (Genolist)"}, "operation_1850": {"class_id": "http://edamontology.org/operation_1850", "definitions": "Assign cysteine bonding state and disulfide bond partners in protein structures.", "label": "Protein cysteine and disulfide bond assignment"}, "data_1205": {"class_id": "http://edamontology.org/data_1205", "definitions": "A QSAR molecular descriptor.", "label": "QSAR descriptor (molecular)"}, "data_1204": {"class_id": "http://edamontology.org/data_1204", "definitions": "A QSAR topological descriptor.", "label": "QSAR descriptor (topological)"}, "data_1203": {"class_id": "http://edamontology.org/data_1203", "definitions": "A QSAR geometrical descriptor.", "label": "QSAR descriptor (geometrical)"}, "data_1202": {"class_id": "http://edamontology.org/data_1202", "definitions": "A QSAR electronic descriptor.", "label": "QSAR descriptor (electronic)"}, "data_1201": {"class_id": "http://edamontology.org/data_1201", "definitions": "A QSAR constitutional descriptor.", "label": "QSAR descriptor (constitutional)"}, "format_1435": {"class_id": "http://edamontology.org/format_1435", "definitions": "Phylogenetic tree data format used by the PHYLIP program.", "label": "Phylip tree format"}, "format_1434": {"class_id": "http://edamontology.org/format_1434", "definitions": "Format of PHYLIP cliques data.", "label": "Phylip cliques format"}, "format_1437": {"class_id": "http://edamontology.org/format_1437", "definitions": "The format of an entry from the TreeFam database of phylogenetic data.", "label": "TreeFam format"}, "format_1436": {"class_id": "http://edamontology.org/format_1436", "definitions": "The format of an entry from the TreeBASE database of phylogenetic data.", "label": "TreeBASE format"}, "format_1431": {"class_id": "http://edamontology.org/format_1431", "definitions": "Format of phylogenetic property data.", "label": "Phylogenetic property values format"}, "format_1430": {"class_id": "http://edamontology.org/format_1430", "definitions": "PHYLIP file format for continuous quantitative character data.", "label": "Phylip continuous quantitative characters"}, "format_1433": {"class_id": "http://edamontology.org/format_1433", "definitions": "Format of PHYLIP discrete states data.", "label": "Phylip discrete states format"}, "format_1432": {"class_id": "http://edamontology.org/format_1432", "definitions": "PHYLIP file format for phylogenetics character frequency data.", "label": "Phylip character frequencies format"}, "topic_0632": {"class_id": "http://edamontology.org/topic_0632", "definitions": "Molecular probes (e.g. a peptide probe or DNA microarray probe) or PCR primers and hybridisation oligos in a nucleic acid sequence.|This includes the design of primers for PCR and DNA amplification or the design of molecular probes.", "label": "Probes and primers"}, "data_3181": {"class_id": "http://edamontology.org/data_3181", "definitions": "An informative report about a DNA sequence assembly.|This might include an overall quality assement of the assembly and summary statistics including counts, average length and number of bases for reads, matches and non-matches, contigs, reads in pairs etc.", "label": "Sequence assembly report"}, "topic_3473": {"class_id": "http://edamontology.org/topic_3473", "definitions": "The discovery of patterns in large data sets and the extraction and trasnsformation of those patterns into a useful format.", "label": "Data mining"}, "format_3020": {"class_id": "http://edamontology.org/format_3020", "definitions": "BCF, the binary version of Variant Call Format (VCF) for sequence variation (indels, polymorphisms, structural variation).", "label": "BCF"}, "data_1119": {"class_id": "http://edamontology.org/data_1119", "definitions": "Unique identifier or name of a profile from the JASPAR database.", "label": "JASPAR profile ID"}, "data_1118": {"class_id": "http://edamontology.org/data_1118", "definitions": "Unique identifier or name of a HMMER hidden Markov model.", "label": "HMMER hidden Markov model ID"}, "data_1113": {"class_id": "http://edamontology.org/data_1113", "definitions": "Unique identifier of an entry from the COG database.", "label": "Sequence cluster ID (COG)"}, "data_1112": {"class_id": "http://edamontology.org/data_1112", "definitions": "An identifier of a cluster of molecular sequence(s).", "label": "Sequence cluster ID"}, "data_1111": {"class_id": "http://edamontology.org/data_1111", "definitions": "List of EMBOSS Uniform Sequence Addresses (EMBOSS listfile).", "label": "EMBOSS listfile"}, "data_1110": {"class_id": "http://edamontology.org/data_1110", "definitions": "The EMBOSS type of a molecular sequence.|See the EMBOSS documentation (http://emboss.sourceforge.net/) for a definition of what this includes.", "label": "EMBOSS sequence type"}, "data_1117": {"class_id": "http://edamontology.org/data_1117", "definitions": "Accession number of an entry from the Prosite database.", "label": "Prosite accession number"}, "data_1116": {"class_id": "http://edamontology.org/data_1116", "definitions": "Identifier of an entry from the ELMdb database of protein functional sites.", "label": "ELM ID"}, "data_1115": {"class_id": "http://edamontology.org/data_1115", "definitions": "Identifier of a sequence profile.|A sequence profile typically represents a sequence alignment.", "label": "Sequence profile ID"}, "data_1114": {"class_id": "http://edamontology.org/data_1114", "definitions": "Identifier of a sequence motif, for example an entry from a motif database.", "label": "Sequence motif identifier"}, "data_3231": {"class_id": "http://edamontology.org/data_3231", "definitions": "Report concerning genome-wide association study experiments.", "label": "GWAS report"}, "data_3236": {"class_id": "http://edamontology.org/data_3236", "definitions": "The position of a cytogenetic band in a genome.|Information might include start and end position in a chromosome sequence, chromosome identifier, name of band and so on.", "label": "Cytoband position"}, "data_3238": {"class_id": "http://edamontology.org/data_3238", "definitions": "Cell type ontology concept ID.", "label": "Cell type ontology ID"}, "operation_0305": {"class_id": "http://edamontology.org/operation_0305", "definitions": "Query scientific literature, in search for articles, article data, concepts, named entities, or for statistics.", "label": "Literature search"}, "format_3790": {"class_id": "http://edamontology.org/format_3790", "definitions": "SPARQL (SPARQL Protocol and RDF Query Language) is a semantic query language for querying and manipulating data stored in Resource Description Framework (RDF) format.", "label": "SPARQL"}, "topic_1811": {"class_id": "http://edamontology.org/topic_1811", "definitions": "Specific bacteria or archaea, e.g. information on a specific prokaryote genome including molecular sequences, genes and annotation.|The resource may be specific to a prokaryote, a group of prokaryotes or all prokaryotes.", "label": "Prokaryotes and Archaea"}, "format_3155": {"class_id": "http://edamontology.org/format_3155", "definitions": "Systems Biology Result Markup Language (SBRML), the standard XML format for simulated or calculated results (e.g. trajectories) of systems biology models.", "label": "SBRML"}, "format_3156": {"class_id": "http://edamontology.org/format_3156", "definitions": "BioPAX is an exchange format for pathway data, with its data model defined in OWL.", "label": "BioPAX"}, "format_3157": {"class_id": "http://edamontology.org/format_3157", "definitions": "EBI Application Result XML is a format returned by sequence similarity search Web services at EBI.", "label": "EBI Application Result XML"}, "format_3158": {"class_id": "http://edamontology.org/format_3158", "definitions": "XML Molecular Interaction Format (MIF), standardised by HUPO PSI MI.", "label": "PSI MI XML (MIF)"}, "format_3159": {"class_id": "http://edamontology.org/format_3159", "definitions": "phyloXML is a standardised XML format for phylogenetic trees, networks, and associated data.", "label": "phyloXML"}, "format_3248": {"class_id": "http://edamontology.org/format_3248", "definitions": "mzQuantML is the format for quantitation values associated with peptides, proteins and small molecules from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of quantitation software for proteomics.", "label": "mzQuantML"}, "data_1069": {"class_id": "http://edamontology.org/data_1069", "definitions": "An identifier of a comparison matrix.", "label": "Comparison matrix identifier"}, "data_1068": {"class_id": "http://edamontology.org/data_1068", "definitions": "Identifier of a phylogenetic tree for example from a phylogenetic tree database.", "label": "Phylogenetic tree ID"}, "data_1067": {"class_id": "http://edamontology.org/data_1067", "definitions": "Identifier of a phylogenetic distance matrix.", "label": "Phylogenetic distance matrix identifier"}, "data_1066": {"class_id": "http://edamontology.org/data_1066", "definitions": "Identifier of a molecular sequence alignment, for example a record from an alignment database.", "label": "Sequence alignment ID"}, "data_1065": {"class_id": "http://edamontology.org/data_1065", "definitions": "Identifier of a sequence signature (motif or profile) for example from a database of sequence patterns.", "label": "Sequence signature identifier"}, "data_1064": {"class_id": "http://edamontology.org/data_1064", "definitions": "An identifier of a set of molecular sequence(s).", "label": "Sequence set ID"}, "data_1063": {"class_id": "http://edamontology.org/data_1063", "definitions": "An identifier of molecular sequence(s) or entries from a molecular sequence database.", "label": "Sequence identifier"}, "data_1062": {"class_id": "http://edamontology.org/data_1062", "definitions": "An identifier of an entry from a database where the same type of identifier is used for objects (data) of different semantic type.|This concept is required for completeness. It should never have child concepts.", "label": "Database entry identifier"}, "data_1061": {"class_id": "http://edamontology.org/data_1061", "definitions": "Name of a QSAR descriptor.", "label": "QSAR descriptor name"}, "data_1060": {"class_id": "http://edamontology.org/data_1060", "definitions": "The base name of a file.|A file base name is the file name stripped of its directory specification and extension.", "label": "File base name"}, "format_3491": {"class_id": "http://edamontology.org/format_3491", "definitions": "Bowtie format for indexed reference genome for \"large\" genomes.", "label": "ebwtl"}, "format_3804": {"class_id": "http://edamontology.org/format_3804", "definitions": "XML format for XML Schema.", "label": "xsd"}, "format_2171": {"class_id": "http://edamontology.org/format_2171", "definitions": "Format used for clusters of protein sequences.", "label": "Sequence cluster format (protein)"}, "format_2170": {"class_id": "http://edamontology.org/format_2170", "definitions": "Format used for clusters of molecular sequences.", "label": "Sequence cluster format"}, "format_2172": {"class_id": "http://edamontology.org/format_2172", "definitions": "Format used for clusters of nucleotide sequences.", "label": "Sequence cluster format (nucleic acid)"}, "format_2175": {"class_id": "http://edamontology.org/format_2175", "definitions": "Format used for clusters of genes.", "label": "Gene cluster format"}, "data_0888": {"class_id": "http://edamontology.org/data_0888", "definitions": "A value representing molecular structure similarity, measured from structure alignment or some other type of structure comparison.", "label": "Structure similarity score"}, "data_0889": {"class_id": "http://edamontology.org/data_0889", "definitions": "Some type of structural (3D) profile or template (representing a structure or structure alignment).", "label": "Structural profile"}, "format_3486": {"class_id": "http://edamontology.org/format_3486", "definitions": "Some format based on the GCG format.", "label": "GCG format variant"}, "format_3487": {"class_id": "http://edamontology.org/format_3487", "definitions": "Bioinformatics Sequence Markup Language format.", "label": "BSML"}, "format_3484": {"class_id": "http://edamontology.org/format_3484", "definitions": "Bowtie format for indexed reference genome for \"small\" genomes.", "label": "ebwt"}, "format_3485": {"class_id": "http://edamontology.org/format_3485", "definitions": "Rich sequence format.|RSF-format files contain one or more sequences that may or may not be related. In addition to the sequence data, each sequence can be annotated with descriptive sequence information (from the GCG manual).", "label": "RSF"}, "data_2905": {"class_id": "http://edamontology.org/data_2905", "definitions": "Accession of an entry from a database of lipids.", "label": "Lipid accession"}, "data_2904": {"class_id": "http://edamontology.org/data_2904", "definitions": "An accession of a map of a molecular sequence (deposited in a database).", "label": "Map accession"}, "data_2907": {"class_id": "http://edamontology.org/data_2907", "definitions": "Accession of a protein deposited in a database.", "label": "Protein accession"}, "data_2906": {"class_id": "http://edamontology.org/data_2906", "definitions": "Accession of a peptide deposited in a database.", "label": "Peptide ID"}, "data_2901": {"class_id": "http://edamontology.org/data_2901", "definitions": "Accession of a specific molecule (catalogued in a database).", "label": "Molecule accession"}, "data_2900": {"class_id": "http://edamontology.org/data_2900", "definitions": "Accession of an entry from a database of carbohydrates.", "label": "Carbohydrate accession"}, "data_2903": {"class_id": "http://edamontology.org/data_2903", "definitions": "An accession of a particular genome (in a database).", "label": "Genome accession"}, "data_2902": {"class_id": "http://edamontology.org/data_2902", "definitions": "Accession of a data definition (catalogued in a database).", "label": "Data resource definition accession"}, "topic_0635": {"class_id": "http://edamontology.org/topic_0635", "definitions": "A particular protein, protein family or other group of proteins.", "label": "Specific protein resources"}, "data_2909": {"class_id": "http://edamontology.org/data_2909", "definitions": "The name of an organism (or group of organisms).", "label": "Organism name"}, "data_2873": {"class_id": "http://edamontology.org/data_2873", "definitions": "Gene frequencies data that may be read during phylogenetic tree calculation.", "label": "Phylogenetic gene frequencies data"}, "topic_3474": {"class_id": "http://edamontology.org/topic_3474", "definitions": "A topic concerning the application of artificial intelligence methods to algorithms, in order to create methods that can learn from data in order to generate an ouput, rather than relying on explicitly encoded information only.", "label": "Machine learning"}, "format_3243": {"class_id": "http://edamontology.org/format_3243", "definitions": "Protein affinity format (PSI-PAR), standardised by HUPO PSI MI. It is compatible with PSI MI XML (MIF) and uses the same XML Schema.", "label": "PSI-PAR"}, "format_3244": {"class_id": "http://edamontology.org/format_3244", "definitions": "mzML format for raw spectrometer output data, standardised by HUPO PSI MSS.|mzML is the successor and unifier of the mzData format developed by PSI and mzXML developed at the Seattle Proteome Center.", "label": "mzML"}, "format_3245": {"class_id": "http://edamontology.org/format_3245", "definitions": "Format for mass pectra and derived data, include peptide sequences etc.", "label": "Mass spectrometry data format"}, "format_3246": {"class_id": "http://edamontology.org/format_3246", "definitions": "TraML (Transition Markup Language) is the format for mass spectrometry transitions, standardised by HUPO PSI MSS.", "label": "TraML"}, "topic_0634": {"class_id": "http://edamontology.org/topic_0634", "definitions": "Diseases, including diseases in general and the genes, gene variations and proteins involved in one or more specific diseases.", "label": "Pathology"}, "format_3711": {"class_id": "http://edamontology.org/format_3711", "definitions": "Output format used by X! series search engines that is based on the XML language BIOML.", "label": "X!Tandem XML"}, "topic_0639": {"class_id": "http://edamontology.org/topic_0639", "definitions": "Archival, processing and analysis of protein sequences and sequence-based entities such as alignments, motifs and profiles.", "label": "Protein sequence analysis"}, "data_2349": {"class_id": "http://edamontology.org/data_2349", "definitions": "Unique identifier of an entry from the UniRef50 database.", "label": "Sequence cluster ID (UniRef50)"}, "data_2348": {"class_id": "http://edamontology.org/data_2348", "definitions": "Unique identifier of an entry from the UniRef90 database.", "label": "Sequence cluster ID (UniRef90)"}, "topic_0637": {"class_id": "http://edamontology.org/topic_0637", "definitions": "Organism classification, identification and naming.", "label": "Taxonomy"}, "format_3578": {"class_id": "http://edamontology.org/format_3578", "definitions": "Proprietary file format for (raw) BeadArray data used by genomewide profiling platforms from Illumina Inc. This format is output directly from the scanner and stores summary intensities for each probe-type on an array.", "label": "IDAT"}, "data_1459": {"class_id": "http://edamontology.org/data_1459", "definitions": "3D coordinate and associated data for a nucleic acid tertiary (3D) structure.", "label": "Nucleic acid structure"}, "data_2343": {"class_id": "http://edamontology.org/data_2343", "definitions": "Identifier of a pathway from the KEGG pathway database.", "label": "Pathway ID (KEGG)"}, "data_2342": {"class_id": "http://edamontology.org/data_2342", "definitions": "The name of a biological pathway or network.", "label": "Pathway or network name"}, "data_2340": {"class_id": "http://edamontology.org/data_2340", "definitions": "An identifier of a build of a particular genome.", "label": "Genome build identifier"}, "data_2347": {"class_id": "http://edamontology.org/data_2347", "definitions": "Unique identifier of an entry from the UniRef100 database.", "label": "Sequence cluster ID (UniRef100)"}, "data_2346": {"class_id": "http://edamontology.org/data_2346", "definitions": "Unique identifier of an entry from the UniRef database.", "label": "Sequence cluster ID (UniRef)"}, "data_2345": {"class_id": "http://edamontology.org/data_2345", "definitions": "Identifier of a pathway from the ConsensusPathDB pathway database.", "label": "Pathway ID (ConsensusPathDB)"}, "data_2344": {"class_id": "http://edamontology.org/data_2344", "definitions": "Identifier of a pathway from the NCI-Nature pathway database.", "label": "Pathway ID (NCI-Nature)"}, "data_0987": {"class_id": "http://edamontology.org/data_0987", "definitions": "Name of a chromosome.", "label": "Chromosome name"}, "data_0986": {"class_id": "http://edamontology.org/data_0986", "definitions": "Unique identifier of a chemical compound.", "label": "Chemical identifier"}, "data_0985": {"class_id": "http://edamontology.org/data_0985", "definitions": "A label (text token) describing the type a molecule.|For example, 'Protein', 'DNA', 'RNA' etc.", "label": "Molecule type"}, "data_0984": {"class_id": "http://edamontology.org/data_0984", "definitions": "Name of a specific molecule.", "label": "Molecule name"}, "data_0983": {"class_id": "http://edamontology.org/data_0983", "definitions": "Identifier (e.g. character symbol) of a specific atom.", "label": "Atom ID"}, "data_0982": {"class_id": "http://edamontology.org/data_0982", "definitions": "Name or other identifier of a molecule.", "label": "Molecule identifier"}, "data_0981": {"class_id": "http://edamontology.org/data_0981", "definitions": "Name or other identifier of a physical, observable biological occurrence or event.", "label": "Phenomenon identifier"}, "data_0980": {"class_id": "http://edamontology.org/data_0980", "definitions": "Name or other identifier of a collection of discrete biological entities.", "label": "Entity collection identifier"}, "data_1694": {"class_id": "http://edamontology.org/data_1694", "definitions": "Number of entities (for example database hits, sequences, alignments etc) to write to an output file.", "label": "Number of output entities"}, "data_1695": {"class_id": "http://edamontology.org/data_1695", "definitions": "Controls the order of hits (reported matches) in an output file from a database search.", "label": "Hit sort order"}, "data_1696": {"class_id": "http://edamontology.org/data_1696", "definitions": "An informative report on a specific drug.", "label": "Drug report"}, "format_3773": {"class_id": "http://edamontology.org/format_3773", "definitions": "BioYAML is a BioXSD-schema-based YAML format of sequence-based data and some other common data - sequence records, alignments, feature records, references to resources, and more - optimised for integrative bioinformatics, web APIs, human readability and editting, and object-oriented programming.", "label": "BioYAML"}, "data_1690": {"class_id": "http://edamontology.org/data_1690", "definitions": "A password on a computer system.", "label": "Password"}, "data_1691": {"class_id": "http://edamontology.org/data_1691", "definitions": "A valid email address of an end-user.", "label": "Email address"}, "data_0989": {"class_id": "http://edamontology.org/data_0989", "definitions": "Identifier of a protein.", "label": "Protein identifier"}, "data_1693": {"class_id": "http://edamontology.org/data_1693", "definitions": "Number of iterations of an algorithm.", "label": "Number of iterations"}, "data_2357": {"class_id": "http://edamontology.org/data_2357", "definitions": "A label (text token) describing a type of protein family signature (sequence classifier) from the InterPro database.", "label": "Protein signature type"}, "data_2007": {"class_id": "http://edamontology.org/data_2007", "definitions": "A word or phrase that can appear in the keywords field (KW line) of entries from the UniProt database.", "label": "UniProt keyword"}, "format_2350": {"class_id": "http://edamontology.org/format_2350", "definitions": "A placeholder concept for visual navigation by dividing data formats by the content of the data that is represented.|This concept exists only to assist EDAM maintenance and navigation in graphical browsers.  It does not add semantic information. The concept branch under 'Format (typed)' provides an alternative organisation of the concepts nested under the other top-level branches ('Binary', 'HTML', 'RDF', 'Text' and 'XML'. All concepts under here are already included under those branches.", "label": "Format (by type of data)"}, "format_2352": {"class_id": "http://edamontology.org/format_2352", "definitions": "BioXSD-schema-based XML format of sequence-based data and some other common data - sequence records, alignments, feature records, references to resources, and more - optimised for integrative bioinformatics, Web services, and object-oriented programming.", "label": "BioXSD (XML)"}, "data_2009": {"class_id": "http://edamontology.org/data_2009", "definitions": "A name for a genetic locus conforming to a scheme that names loci (such as predicted genes) depending on their position in a molecular sequence, for example a completely sequenced genome or chromosome.", "label": "Ordered locus name"}, "data_3505": {"class_id": "http://edamontology.org/data_3505", "definitions": "A list of publications such as scientic papers or books.", "label": "Bibliography"}, "operation_3741": {"class_id": "http://edamontology.org/operation_3741", "definitions": "The analysis, using proteomics techniques, to identify proteins whose encoding genes are differentially expressed under a given experimental setup.", "label": "Differential protein expression analysis"}, "format_2197": {"class_id": "http://edamontology.org/format_2197", "definitions": "A serialisation format conforming to the Web Ontology Language (OWL) model.", "label": "OWL format"}, "format_2196": {"class_id": "http://edamontology.org/format_2196", "definitions": "A serialisation format conforming to the Open Biomedical Ontologies (OBO) model.", "label": "OBO format"}, "format_2195": {"class_id": "http://edamontology.org/format_2195", "definitions": "Format used for ontologies.", "label": "Ontology format"}, "data_3509": {"class_id": "http://edamontology.org/data_3509", "definitions": "A mapping of supplied textual terms or phrases to ontology concepts (URIs).", "label": "Ontology mapping"}, "data_0859": {"class_id": "http://edamontology.org/data_0859", "definitions": "Data files used by motif or profile methods.", "label": "Sequence signature model"}, "data_0858": {"class_id": "http://edamontology.org/data_0858", "definitions": "Report on the location of matches in one or more sequences to profiles, motifs (conserved or functional patterns) or other signatures.|This ncluding reports of hits from a search of a protein secondary or domain database.", "label": "Sequence signature matches"}, "data_1583": {"class_id": "http://edamontology.org/data_1583", "definitions": "Data on the dissociation characteristics of a double-stranded nucleic acid molecule (DNA or a DNA/RNA hybrid) during heating.|A melting (stability) profile calculated the free energy required to unwind and separate the nucleic acid strands, plotted for sliding windows over a sequence.", "label": "Nucleic acid melting profile"}, "data_1584": {"class_id": "http://edamontology.org/data_1584", "definitions": "Enthalpy of hybridised or double stranded nucleic acid (DNA or RNA/DNA).", "label": "Nucleic acid enthalpy"}, "data_1585": {"class_id": "http://edamontology.org/data_1585", "definitions": "Entropy of hybridised or double stranded nucleic acid (DNA or RNA/DNA).", "label": "Nucleic acid entropy"}, "data_1586": {"class_id": "http://edamontology.org/data_1586", "definitions": "Melting temperature of hybridised or double stranded nucleic acid (DNA or RNA/DNA).", "label": "Nucleic acid melting temperature"}, "data_1587": {"class_id": "http://edamontology.org/data_1587", "definitions": "Stitch profile of hybridised or double stranded nucleic acid (DNA or RNA/DNA).|A stitch profile diagram shows partly melted DNA conformations (with probabilities) at a range of temperatures. For example, a stitch profile might show possible loop openings with their location, size, probability and fluctuations at a given temperature.", "label": "Nucleic acid stitch profile"}, "data_0851": {"class_id": "http://edamontology.org/data_0851", "definitions": "A character used to replace (mask) other characters in a molecular sequence.", "label": "Sequence mask character"}, "data_0850": {"class_id": "http://edamontology.org/data_0850", "definitions": "A collection of multiple molecular sequences and associated metadata that do not (typically) correspond to molecular sequence database records or entries and which (typically) are derived from some analytical method.|This concept may be used for arbitrary sequence sets and associated data arising from processing.", "label": "Sequence set"}, "data_0853": {"class_id": "http://edamontology.org/data_0853", "definitions": "The strand of a DNA sequence (forward or reverse).|The forward or 'top' strand might specify a sequence is to be used as given, the reverse or 'bottom' strand specifying the reverse complement of the sequence is to be used.", "label": "DNA sense specification"}, "data_0852": {"class_id": "http://edamontology.org/data_0852", "definitions": "A label (text token) describing the type of sequence masking to perform.|Sequence masking is where specific characters or positions in a molecular sequence are masked (replaced) with an another (mask character). The mask type indicates what is masked, for example regions that are not of interest or which are information-poor including acidic protein regions, basic protein regions, proline-rich regions, low compositional complexity regions, short-periodicity internal repeats, simple repeats and low complexity regions. Masked sequences are used in database search to eliminate statistically significant but biologically uninteresting hits.", "label": "Sequence mask type"}, "data_0855": {"class_id": "http://edamontology.org/data_0855", "definitions": "Basic or general information concerning molecular sequences.|This is used for such things as a report including the sequence identifier, type and length.", "label": "Sequence metadata"}, "data_0854": {"class_id": "http://edamontology.org/data_0854", "definitions": "A specification of sequence length(s).", "label": "Sequence length specification"}, "data_0857": {"class_id": "http://edamontology.org/data_0857", "definitions": "A report of sequence hits and associated data from searching a database of sequences (for example a BLAST search). This will typically include a list of scores (often with statistical evaluation) and a set of alignments for the hits.|The score list includes the alignment score, percentage of the query sequence matched, length of the database sequence entry in this alignment, identifier of the database sequence entry, excerpt of the database sequence entry description etc.", "label": "Sequence search results"}, "data_0856": {"class_id": "http://edamontology.org/data_0856", "definitions": "How the annotation of a sequence feature (for example in EMBL or Swiss-Prot) was derived.|This might be the name and version of a software tool, the name of a database, or 'curated' to indicate a manual annotation (made by a human).", "label": "Sequence feature source"}, "operation_0516": {"class_id": "http://edamontology.org/operation_0516", "definitions": "Search and retrieve names of or documentation on bioinformatics databases or query terms, for example by keyword.", "label": "Data retrieval (database metadata)"}, "operation_0517": {"class_id": "http://edamontology.org/operation_0517", "definitions": "Predict primers for large scale sequencing.", "label": "PCR primer design (for large scale sequencing)"}, "operation_0514": {"class_id": "http://edamontology.org/operation_0514", "definitions": "Align two or more molecular 3D profiles.", "label": "Structural profile alignment generation (multiple)"}, "operation_0515": {"class_id": "http://edamontology.org/operation_0515", "definitions": "Search and retrieve names of or documentation on bioinformatics tools, for example by keyword or which perform a particular function.", "label": "Data retrieval (tool metadata)"}, "operation_0512": {"class_id": "http://edamontology.org/operation_0512", "definitions": "Align two or more molecular profiles.", "label": "Sequence alignment generation (multiple profile)"}, "operation_0513": {"class_id": "http://edamontology.org/operation_0513", "definitions": "Align exactly two molecular Structural (3D) profiles.", "label": "3D profile-to-3D profile alignment (pairwise)"}, "operation_0510": {"class_id": "http://edamontology.org/operation_0510", "definitions": "Globally align (superimpose) two or more molecular tertiary structures.|Global alignment methods identify similarity across the entire structures.", "label": "Global structure alignment"}, "operation_0511": {"class_id": "http://edamontology.org/operation_0511", "definitions": "Align exactly two molecular profiles.|Methods might perform one-to-one, one-to-many or many-to-many comparisons.", "label": "Profile-profile alignment (pairwise)"}, "operation_0518": {"class_id": "http://edamontology.org/operation_0518", "definitions": "Predict primers for genotyping polymorphisms, for example single nucleotide polymorphisms (SNPs).", "label": "PCR primer design (for genotyping polymorphisms)"}, "operation_0519": {"class_id": "http://edamontology.org/operation_0519", "definitions": "Predict primers for gene transcription profiling.", "label": "PCR primer design (for gene transcription profiling)"}, "data_2178": {"class_id": "http://edamontology.org/data_2178", "definitions": "One or more things.", "label": "1 or more"}, "data_2179": {"class_id": "http://edamontology.org/data_2179", "definitions": "Exactly two things.", "label": "Exactly 2"}, "topic_0741": {"class_id": "http://edamontology.org/topic_0741", "definitions": "Protein sequence alignments.|A sequence profile typically represents a sequence alignment.", "label": "Protein sequence alignment"}, "operation_3191": {"class_id": "http://edamontology.org/operation_3191", "definitions": "Trim sequences (typically from an automated DNA sequencer) to remove the sequence ends that extend beyond an assembled reference sequence.", "label": "Trim to reference"}, "data_2173": {"class_id": "http://edamontology.org/data_2173", "definitions": "A collection of multiple molecular sequences and (typically) associated metadata that is intended for sequential processing.|This concept may be used for sequence sets that are expected to be read and processed a single sequence at a time.", "label": "Sequence set (stream)"}, "data_2174": {"class_id": "http://edamontology.org/data_2174", "definitions": "Secondary identifier of an object from the FlyBase database.|Secondary identifier are used to handle entries that were merged with or split from other entries in the database.", "label": "FlyBase secondary identifier"}, "data_2176": {"class_id": "http://edamontology.org/data_2176", "definitions": "The number of a certain thing.", "label": "Cardinality"}, "data_2177": {"class_id": "http://edamontology.org/data_2177", "definitions": "A single thing.", "label": "Exactly 1"}, "topic_2533": {"class_id": "http://edamontology.org/topic_2533", "definitions": "DNA mutation.", "label": "DNA mutation"}, "topic_0740": {"class_id": "http://edamontology.org/topic_0740", "definitions": "Nucleotide sequence alignments.", "label": "Nucleic acid sequence alignment"}, "topic_3126": {"class_id": "http://edamontology.org/topic_3126", "definitions": "Repetitive elements within a nucleic acid sequence.|This includes long terminal repeats (LTRs); sequences (typically retroviral) directly repeated at  both ends of a defined sequence and other types of repeating unit.", "label": "Nucleic acid repeats"}, "topic_3421": {"class_id": "http://edamontology.org/topic_3421", "definitions": "The use of operative, manual and instrumental techniques on a patient to investigate and/or treat a pathological condition or help improve bodily function or appearance.", "label": "Surgery"}, "topic_3420": {"class_id": "http://edamontology.org/topic_3420", "definitions": "The health of the reproductive processes, functions and systems at all stages of life.", "label": "Reproductive health"}, "topic_3423": {"class_id": "http://edamontology.org/topic_3423", "definitions": "Medical therapies that fall beyond the scope of conventional medicine but may be used alongside it in the treatment of disease and ill health.", "label": "Complementary medicine"}, "topic_3422": {"class_id": "http://edamontology.org/topic_3422", "definitions": "The branches of medicine and physiology focussing on the function and disorders of the urinary system in males and females, the reproductive system in males, and the kidney.", "label": "Urology and nephrology"}, "format_1552": {"class_id": "http://edamontology.org/format_1552", "definitions": "Format of output of the ProQ protein model quality predictor.|ProQ is a neural network-based predictor that predicts the quality of a protein model based on the number of structural features.", "label": "ProQ report format"}, "data_3270": {"class_id": "http://edamontology.org/data_3270", "definitions": "Unique identifier for a gene tree from the Ensembl database.", "label": "Ensembl gene tree ID"}, "format_1551": {"class_id": "http://edamontology.org/format_1551", "definitions": "Format of output of the Pcons Model Quality Assessment Program (MQAP).|Pcons ranks protein models by assessing their quality based on the occurrence of recurring common three-dimensional structural patterns. Pcons returns a score reflecting the overall global quality and a score for each individual residue in the protein reflecting the local residue quality.", "label": "Pcons report format"}, "data_3271": {"class_id": "http://edamontology.org/data_3271", "definitions": "A phylogenetic tree that is an estimate of the character's phylogeny.", "label": "Gene tree"}, "operation_0422": {"class_id": "http://edamontology.org/operation_0422", "definitions": "Detect or predict cleavage sites (enzymatic or chemical) in protein sequences.", "label": "Protein cleavage site prediction"}, "operation_0423": {"class_id": "http://edamontology.org/operation_0423", "definitions": "Predict epitopes that bind to MHC class I molecules.", "label": "Epitope mapping (MHC Class I)"}, "operation_0420": {"class_id": "http://edamontology.org/operation_0420", "definitions": "Predict RNA and DNA-binding binding sites in protein sequences.|This includes methods that predict and optimise zinc finger protein domains for DNA/RNA binding (for example for transcription factors and nucleases).", "label": "Protein-nucleic acid binding prediction"}, "operation_0421": {"class_id": "http://edamontology.org/operation_0421", "definitions": "Predict protein sites that are key to protein folding, such as possible sites of nucleation or stabilisation.", "label": "Protein folding site prediction"}, "operation_0426": {"class_id": "http://edamontology.org/operation_0426", "definitions": "Detect, predict and identify genetic elements such as promoters, coding regions, splice sites, etc in DNA sequences.|Methods for gene prediction might be ab initio, based on phylogenetic comparisons, use motifs, sequence features, support vector machine, alignment etc.", "label": "Gene component prediction"}, "operation_0427": {"class_id": "http://edamontology.org/operation_0427", "definitions": "Detect or predict transposons, retrotransposons / retrotransposition signatures etc.", "label": "Transposon prediction"}, "operation_0424": {"class_id": "http://edamontology.org/operation_0424", "definitions": "Predict epitopes that bind to MHC class II molecules.", "label": "Epitope mapping (MHC Class II)"}, "operation_0425": {"class_id": "http://edamontology.org/operation_0425", "definitions": "Detect, predict and identify whole gene structure in DNA sequences. This includes protein coding regions, exon-intron structure, regulatory regions etc.", "label": "Whole gene prediction"}, "operation_1914": {"class_id": "http://edamontology.org/operation_1914", "definitions": "Query a tertiary structure database and retrieve water molecules.", "label": "Structure retrieval (water)"}, "operation_0428": {"class_id": "http://edamontology.org/operation_0428", "definitions": "Detect polyA signals in nucleotide sequences.", "label": "PolyA signal detection"}, "operation_0429": {"class_id": "http://edamontology.org/operation_0429", "definitions": "Detect quadruplex-forming motifs in nucleotide sequences.|Quadruplex (4-stranded) structures are formed by guanine-rich regions and are implicated in various important biological processes and as therapeutic targets.", "label": "Quadruplex formation site detection"}, "operation_1913": {"class_id": "http://edamontology.org/operation_1913", "definitions": "Identify poor quality amino acid positions in protein structures.", "label": "Residue validation"}, "data_3757": {"class_id": "http://edamontology.org/data_3757", "definitions": "Identifier of a protein modification catalogued in the Unimod database.", "label": "Unimod ID"}, "data_3756": {"class_id": "http://edamontology.org/data_3756", "definitions": "Score for localization of one or more post-translational modifications in peptide sequence measured by mass spectrometry.", "label": "Localisation score"}, "data_3754": {"class_id": "http://edamontology.org/data_3754", "definitions": "An informative report containing a ranked list of Gene Ontology concepts, each associated with a p-value, concerning or derived from the analysis of e.g. a set of genes or proteins.", "label": "GO-term enrichment report"}, "data_3753": {"class_id": "http://edamontology.org/data_3753", "definitions": "An informative report containing a ranked list of categories (usually ontology concepts), each associated with a statistical metric of over-/under-representation within the studied data.", "label": "Over-representation report"}, "data_3759": {"class_id": "http://edamontology.org/data_3759", "definitions": "Identifier for mass spectrometry proteomics data in the proteomexchange.org repository.", "label": "ProteomeXchange ID"}, "topic_3047": {"class_id": "http://edamontology.org/topic_3047", "definitions": "The molecular basis of biological activity, particularly the macromolecules (e.g. proteins and nucleic acids) that are essential to life.", "label": "Molecular biology"}, "topic_3044": {"class_id": "http://edamontology.org/topic_3044", "definitions": "", "label": "Protein interaction networks"}, "topic_3043": {"class_id": "http://edamontology.org/topic_3043", "definitions": "Protein sequences and associated concepts such as sequence sites, alignments, motifs and profiles.", "label": "Protein sequences"}, "topic_3042": {"class_id": "http://edamontology.org/topic_3042", "definitions": "Nucleotide sequences and associated concepts such as sequence sites, alignments, motifs and profiles.", "label": "Nucleic acid sequences"}, "operation_3649": {"class_id": "http://edamontology.org/operation_3649", "definitions": "Statistical estimation of false discovery rate from score distribution for peptide-spectrum-matches, following a peptide database search, and by comparison to search results with a database containing incorrect information.", "label": "Target-Decoy"}, "operation_3648": {"class_id": "http://edamontology.org/operation_3648", "definitions": "Statistical estimation of false discovery rate from score distribution for peptide-spectrum-matches, following a peptide database search.", "label": "Validation of peptide-spectrum matches"}, "operation_3647": {"class_id": "http://edamontology.org/operation_3647", "definitions": "Peptide database search for identification of known and unknown PTMs looking for mass difference mismatches.", "label": "Blind peptide database search"}, "operation_3646": {"class_id": "http://edamontology.org/operation_3646", "definitions": "Determination of best matches between MS/MS spectrum and a database of protein or nucleic acid sequences.", "label": "Peptide database search"}, "operation_3645": {"class_id": "http://edamontology.org/operation_3645", "definitions": "Identification of post-translational modifications (PTMs) of peptides/proteins in mass spectrum.", "label": "PTM identification"}, "operation_3644": {"class_id": "http://edamontology.org/operation_3644", "definitions": "Analytical process that derives a peptide\u2019s amino acid sequence from its tandem mass spectrum (MS/MS) without the assistance of a sequence database.", "label": "de Novo sequencing"}, "operation_3643": {"class_id": "http://edamontology.org/operation_3643", "definitions": "Peptide sequence tags are used as piece of information about a peptide obtained by tandem mass spectrometry.", "label": "Tag-based peptide identification"}, "operation_3642": {"class_id": "http://edamontology.org/operation_3642", "definitions": "Quantification analysis using chemical labeling by stable isotope dimethylation", "label": "Dimethyl"}, "operation_3641": {"class_id": "http://edamontology.org/operation_3641", "definitions": "Quantification analysis using the Thermo Fisher tandem mass tag labelling workflow.", "label": "TMT-tag"}, "operation_3640": {"class_id": "http://edamontology.org/operation_3640", "definitions": "Quantification analysis using labeling based on 18O-enriched H2O.", "label": "18O labeling"}, "data_1324": {"class_id": "http://edamontology.org/data_1324", "definitions": "post-translation modifications in a protein sequence, typically describing the specific sites involved.", "label": "Protein features (post-translation modifications)"}, "data_1325": {"class_id": "http://edamontology.org/data_1325", "definitions": "catalytic residues (active site) of an enzyme.", "label": "Protein features report (active sites)"}, "data_1326": {"class_id": "http://edamontology.org/data_1326", "definitions": "ligand-binding (non-catalytic) residues of a protein, such as sites that bind metal, prosthetic groups or lipids.", "label": "Protein features report (binding sites)"}, "data_1327": {"class_id": "http://edamontology.org/data_1327", "definitions": "A report on antigenic determinant sites (epitopes) in proteins, from sequence and / or structural data.|Epitope mapping is commonly done during vaccine design.", "label": "Protein features (epitopes)"}, "data_1321": {"class_id": "http://edamontology.org/data_1321", "definitions": "A report on predicted or known key residue positions (sites) in a protein sequence, such as binding or functional sites.|Use this concept for collections of specific sites which are not necessarily contiguous, rather than contiguous stretches of amino acids.", "label": "Protein features (sites)"}, "data_1322": {"class_id": "http://edamontology.org/data_1322", "definitions": "signal peptides or signal peptide cleavage sites in protein sequences.", "label": "Protein features report (signal peptides)"}, "data_1323": {"class_id": "http://edamontology.org/data_1323", "definitions": "cleavage sites (for a proteolytic enzyme or agent) in a protein sequence.", "label": "Protein features report (cleavage sites)"}, "format_1336": {"class_id": "http://edamontology.org/format_1336", "definitions": "Format of EMBASSY domain hits file (DHF) of hits (sequences) with domain classification information.|The hits are relatives to a SCOP or CATH family and are found from a search of a sequence database.", "label": "dhf"}, "format_1337": {"class_id": "http://edamontology.org/format_1337", "definitions": "Format of EMBASSY ligand hits file (LHF) of database hits (sequences) with ligand classification information.|The hits are putative ligand-binding sequences and are found from a search of a sequence database.", "label": "lhf"}, "format_1334": {"class_id": "http://edamontology.org/format_1334", "definitions": "Format of results of a sequence database search using some variant of MSPCrunch.", "label": "mspcrunch"}, "format_1335": {"class_id": "http://edamontology.org/format_1335", "definitions": "Format of results of a sequence database search using some variant of Smith Waterman.", "label": "Smith-Waterman format"}, "data_1328": {"class_id": "http://edamontology.org/data_1328", "definitions": "RNA and DNA-binding proteins and binding sites in protein sequences.", "label": "Protein features report (nucleic acid binding sites)"}, "data_1329": {"class_id": "http://edamontology.org/data_1329", "definitions": "A report on epitopes that bind to MHC class I molecules.", "label": "MHC Class I epitopes report"}, "topic_3557": {"class_id": "http://edamontology.org/topic_3557", "definitions": "An experiment for studying protein-protein interactions.", "label": "Protein interaction experiment"}, "data_2592": {"class_id": "http://edamontology.org/data_2592", "definitions": "A type (represented as a string) of cancer.", "label": "Cancer type"}, "data_2593": {"class_id": "http://edamontology.org/data_2593", "definitions": "A unique identifier for an organism used in the BRENDA database.", "label": "BRENDA organism ID"}, "data_2590": {"class_id": "http://edamontology.org/data_2590", "definitions": "Identifier of an entry from a database of biological hierarchies.", "label": "Hierarchy identifier"}, "data_2591": {"class_id": "http://edamontology.org/data_2591", "definitions": "Identifier of an entry from the Brite database of biological hierarchies.", "label": "Brite hierarchy ID"}, "data_2596": {"class_id": "http://edamontology.org/data_2596", "definitions": "An identifier of a catalogue of biological resources.", "label": "Catalogue ID"}, "data_2597": {"class_id": "http://edamontology.org/data_2597", "definitions": "The name of a catalogue of biological resources from the CABRI database.", "label": "CABRI catalogue name"}, "data_2594": {"class_id": "http://edamontology.org/data_2594", "definitions": "The name of a taxon using the controlled vocabulary of the UniGene database.", "label": "UniGene taxon"}, "data_2595": {"class_id": "http://edamontology.org/data_2595", "definitions": "The name of a taxon using the controlled vocabulary of the UTRdb database.", "label": "UTRdb taxon"}, "data_2598": {"class_id": "http://edamontology.org/data_2598", "definitions": "An informative report on protein secondary structure alignment-derived data or metadata.", "label": "Secondary structure alignment metadata"}, "data_2599": {"class_id": "http://edamontology.org/data_2599", "definitions": "An informative report on the physical, chemical or other information concerning the interaction of two or more molecules (or parts of molecules).", "label": "Molecule interaction report"}, "operation_3194": {"class_id": "http://edamontology.org/operation_3194", "definitions": "Compare the features of two genome sequences.|Genomic elements that might be compared include genes, indels, single nucleotide polymorphisms (SNPs), retrotransposons, tandem repeats and so on.", "label": "Genome feature comparison"}, "data_2751": {"class_id": "http://edamontology.org/data_2751", "definitions": "An identifier of a particular genome.", "label": "GenomeReviews ID"}, "data_2752": {"class_id": "http://edamontology.org/data_2752", "definitions": "Identifier of an entry from the GlycosciencesDB database.", "label": "GlycoMap ID"}, "data_2753": {"class_id": "http://edamontology.org/data_2753", "definitions": "A conformational energy map of the glycosidic linkages in a carbohydrate molecule.", "label": "Carbohydrate conformational map"}, "data_2754": {"class_id": "http://edamontology.org/data_2754", "definitions": "introns in a nucleotide sequences.", "label": "Gene features report (intron)"}, "data_2755": {"class_id": "http://edamontology.org/data_2755", "definitions": "The name of a transcription factor.", "label": "Transcription factor name"}, "data_2756": {"class_id": "http://edamontology.org/data_2756", "definitions": "Identifier of a membrane transport proteins from the transport classification database (TCDB).", "label": "TCID"}, "data_2757": {"class_id": "http://edamontology.org/data_2757", "definitions": "Name of a domain from the Pfam database.", "label": "Pfam domain name"}, "data_2758": {"class_id": "http://edamontology.org/data_2758", "definitions": "Accession number of a Pfam clan.", "label": "Pfam clan ID"}, "data_2759": {"class_id": "http://edamontology.org/data_2759", "definitions": "Identifier for a gene from the VectorBase database.", "label": "Gene ID (VectorBase)"}, "topic_3135": {"class_id": "http://edamontology.org/topic_3135", "definitions": "Coding sequences for a signal or transit peptide.", "label": "Signal or transit peptide"}, "format_2567": {"class_id": "http://edamontology.org/format_2567", "definitions": "Alphabet for a molecular sequence without unknown positions, ambiguity or non-sequence characters.", "label": "completely unambiguous pure"}, "topic_0747": {"class_id": "http://edamontology.org/topic_0747", "definitions": "The archival, detection, prediction and analysis ofpositional features such as functional sites in nucleotide sequences.", "label": "Nucleic acid sites and features"}, "format_1422": {"class_id": "http://edamontology.org/format_1422", "definitions": "Format used by the HMMER package for of an alignment of a hidden Markov model against a sequence database.", "label": "HMMER profile alignment (HMM versus sequences)"}, "format_1423": {"class_id": "http://edamontology.org/format_1423", "definitions": "Format of PHYLIP phylogenetic distance matrix data.|Data Type must include the distance matrix, probably as pairs of sequence identifiers with a distance (integer or float).", "label": "Phylip distance matrix"}, "operation_3353": {"class_id": "http://edamontology.org/operation_3353", "definitions": "Compare two or more ontologies, e.g. identify differences.", "label": "Ontology comparison"}, "format_1421": {"class_id": "http://edamontology.org/format_1421", "definitions": "Format used by the HMMER package for an alignment of a sequence against a hidden Markov model database.", "label": "HMMER profile alignment (sequences versus HMMs)"}, "format_1424": {"class_id": "http://edamontology.org/format_1424", "definitions": "Dendrogram (tree file) format generated by ClustalW.", "label": "ClustalW dendrogram"}, "format_1425": {"class_id": "http://edamontology.org/format_1425", "definitions": "Raw data file format used by Phylip from which a phylogenetic tree is directly generated or plotted.", "label": "Phylip tree raw"}, "operation_3359": {"class_id": "http://edamontology.org/operation_3359", "definitions": "Split a file containing multiple data items into many files, each containing one item", "label": "Splitting"}, "format_2568": {"class_id": "http://edamontology.org/format_2568", "definitions": "Alphabet for a nucleotide sequence (characters ACGTU only) without unknown positions, ambiguity or non-sequence characters .", "label": "completely unambiguous pure nucleotide"}, "topic_2397": {"class_id": "http://edamontology.org/topic_2397", "definitions": "Exons in a nucleotide sequences.", "label": "Exons"}, "format_2569": {"class_id": "http://edamontology.org/format_2569", "definitions": "Alphabet for a DNA sequence (characters ACGT only) without unknown positions, ambiguity or non-sequence characters.", "label": "completely unambiguous pure dna"}, "topic_0154": {"class_id": "http://edamontology.org/topic_0154", "definitions": "Small molecules of biological significance, typically archival, curation, processing and analysis of structural information.|This concept excludes macromolecules such as proteins and nucleic acids.|This includes the structures of drugs, drug target, their interactions and binding affinities.  Also the structures of reactants or products of metabolism, for example small molecules such as including vitamins, polyols, nucleotides and amino acids.  Also the physicochemical, biochemical or structural properties of amino acids or peptides.  Also structural and associated data for toxic chemical substances.|Small molecules include organic molecules, metal-organic compounds, small polypeptides, small polysaccharides and oligonucleotides.  Structural data is usually included.", "label": "Small molecules"}, "topic_0156": {"class_id": "http://edamontology.org/topic_0156", "definitions": "Edit, convert or otherwise change a molecular sequence, either randomly or specifically.", "label": "Sequence editing"}, "topic_3339": {"class_id": "http://edamontology.org/topic_3339", "definitions": "Collections of microbial cells including bacteria, yeasts and moulds.", "label": "Microbial collection"}, "topic_0157": {"class_id": "http://edamontology.org/topic_0157", "definitions": "The archival, processing and analysis of the basic character composition of molecular sequences, for example character or word frequency, ambiguity, complexity, particularly regions of low complexity, and repeats or the repetitive nature of molecular sequences.|This includes repetitive elements within a nucleic acid sequence, e.g. long terminal repeats (LTRs); sequences (typically retroviral) directly repeated at  both ends of a sequence and other types of repeating unit.|This includes short repetitive subsequences (repeat sequences) in a protein sequence.", "label": "Sequence composition, complexity and repeats"}, "topic_3338": {"class_id": "http://edamontology.org/topic_3338", "definitions": "Laboratory study of mice, for example, phenotyping, and mutagenesis of mouse cell lines.", "label": "Mouse clinic"}, "format_3011": {"class_id": "http://edamontology.org/format_3011", "definitions": "genePred table format for gene prediction tracks.|genePred format has 3 main variations (http://genome.ucsc.edu/FAQ/FAQformat#format9 http://www.broadinstitute.org/software/igv/genePred). They reflect UCSC Browser DB tables.", "label": "genePred"}, "format_3010": {"class_id": "http://edamontology.org/format_3010", "definitions": ".nib (nibble) binary format of a nucleotide sequence using 4 bits per nucleotide (including unknown) and its lower-case 'masking'.", "label": ".nib"}, "format_3013": {"class_id": "http://edamontology.org/format_3013", "definitions": "axt format of alignments, typically produced from BLASTZ.", "label": "axt"}, "format_3012": {"class_id": "http://edamontology.org/format_3012", "definitions": "Personal Genome SNP (pgSnp) format for sequence variation tracks (indels and polymorphisms), supported by the UCSC Genome Browser.", "label": "pgSnp"}, "format_3015": {"class_id": "http://edamontology.org/format_3015", "definitions": "Pileup format of alignment of sequences (e.g. sequencing reads) to (a) reference sequence(s). Contains aligned bases per base of the reference sequence(s).", "label": "Pileup"}, "format_3014": {"class_id": "http://edamontology.org/format_3014", "definitions": "LAV format of alignments generated by BLASTZ and LASTZ.", "label": "LAV"}, "format_3017": {"class_id": "http://edamontology.org/format_3017", "definitions": "Sequence Read Format (SRF) of sequence trace data. Supports submission to the NCBI Short Read Archive.", "label": "SRF"}, "format_3016": {"class_id": "http://edamontology.org/format_3016", "definitions": "Variant Call Format (VCF) for sequence variation (indels, polymorphisms, structural variation).", "label": "VCF"}, "format_3019": {"class_id": "http://edamontology.org/format_3019", "definitions": "Genome Variation Format (GVF). A GFF3-compatible format with defined header and attribute tags for sequence variation.", "label": "GVF"}, "format_3018": {"class_id": "http://edamontology.org/format_3018", "definitions": "ZTR format for storing chromatogram data from DNA sequencing instruments.", "label": "ZTR"}, "topic_1770": {"class_id": "http://edamontology.org/topic_1770", "definitions": "The comparison of two or more molecular structures, for example structure alignment and clustering.|This might involve comparison of secondary or tertiary (3D) structural information.", "label": "Structure comparison"}, "data_2374": {"class_id": "http://edamontology.org/data_2374", "definitions": "Unique identifier of a spot from a two-dimensional (protein) gel in the SWISS-2DPAGE database.", "label": "Spot serial number"}, "topic_1775": {"class_id": "http://edamontology.org/topic_1775", "definitions": "The study of gene and protein function including the prediction of functional properties of a protein.", "label": "Function analysis"}, "data_2375": {"class_id": "http://edamontology.org/data_2375", "definitions": "Unique identifier of a spot from a two-dimensional (protein) gel from a HSC-2DPAGE database.", "label": "Spot ID (HSC-2DPAGE)"}, "data_1104": {"class_id": "http://edamontology.org/data_1104", "definitions": "A unique identifier of an entry (gene cluster) from the NCBI UniGene database.", "label": "Sequence cluster ID (UniGene)"}, "data_1105": {"class_id": "http://edamontology.org/data_1105", "definitions": "Identifier of a dbEST database entry.", "label": "dbEST accession"}, "data_1106": {"class_id": "http://edamontology.org/data_1106", "definitions": "Identifier of a dbSNP database entry.", "label": "dbSNP ID"}, "data_1100": {"class_id": "http://edamontology.org/data_1100", "definitions": "An identifier of PIR sequence database entry.", "label": "PIR identifier"}, "data_1101": {"class_id": "http://edamontology.org/data_1101", "definitions": "Identifier of a TREMBL sequence database entry.", "label": "TREMBL accession"}, "data_1102": {"class_id": "http://edamontology.org/data_1102", "definitions": "Primary identifier of a Gramene database entry.", "label": "Gramene primary identifier"}, "data_1103": {"class_id": "http://edamontology.org/data_1103", "definitions": "Identifier of a (nucleic acid) entry from the EMBL/GenBank/DDBJ databases.", "label": "EMBL/GenBank/DDBJ ID"}, "data_3241": {"class_id": "http://edamontology.org/data_3241", "definitions": "Mathematical model of a network, that contains biochemical kinetics.", "label": "Kinetic model"}, "topic_3332": {"class_id": "http://edamontology.org/topic_3332", "definitions": "", "label": "Computational chemistry"}, "data_0831": {"class_id": "http://edamontology.org/data_0831", "definitions": "Controlled vocabulary from National Library of Medicine. The MeSH thesaurus is used to index articles in biomedical journals for the Medline/PubMED databases.", "label": "MeSH vocabulary"}, "topic_0820": {"class_id": "http://edamontology.org/topic_0820", "definitions": "Lipoproteins (protein-lipid assemblies), and proteins or region of a protein that spans or are associated with a membrane.", "label": "Membrane and lipoproteins"}, "topic_0821": {"class_id": "http://edamontology.org/topic_0821", "definitions": "Proteins that catalyze chemical reaction, the kinetics of enzyme-catalysed reactions, enzyme nomenclature etc.", "label": "Enzymes"}, "operation_2463": {"class_id": "http://edamontology.org/operation_2463", "definitions": "Process (read and / or write) a molecular sequence alignment.", "label": "Sequence alignment processing"}, "topic_0622": {"class_id": "http://edamontology.org/topic_0622", "definitions": "Whole genomes of one or more organisms, or genomes in general, such as meta-information on genomes, genome projects, gene names etc.", "label": "Genomics"}, "format_3164": {"class_id": "http://edamontology.org/format_3164", "definitions": "GTrack is a generic and optimised tabular format for genome or sequence feature tracks. GTrack unifies the power of other track formats (e.g. GFF3, BED, WIG), and while optimised in size, adds more flexibility, customisation, and automation (\"machine understandability\").", "label": "GTrack"}, "format_3167": {"class_id": "http://edamontology.org/format_3167", "definitions": "Data format for annotation on a laboratory experiment.", "label": "Experiment annotation format"}, "format_3166": {"class_id": "http://edamontology.org/format_3166", "definitions": "Data format for a report of information derived from a biological pathway or network.", "label": "Biological pathway or network report format"}, "format_3161": {"class_id": "http://edamontology.org/format_3161", "definitions": "MAGE-ML XML format for microarray expression data, standardised by MGED (now FGED).", "label": "MAGE-ML"}, "data_1685": {"class_id": "http://edamontology.org/data_1685", "definitions": "EMBOSS (EMBASSY) supermatcher error file.", "label": "EMBOSS supermatcher error file"}, "format_3163": {"class_id": "http://edamontology.org/format_3163", "definitions": "GCDML XML format for genome and metagenome metadata according to MIGS/MIMS/MIMARKS information standards, standardised by the Genomic Standards Consortium (GSC).", "label": "GCDML"}, "format_3162": {"class_id": "http://edamontology.org/format_3162", "definitions": "MAGE-TAB textual format for microarray expression data, standardised by MGED (now FGED).", "label": "MAGE-TAB"}, "operation_3566": {"class_id": "http://edamontology.org/operation_3566", "definitions": "Simulate gene expression data, e.g. for purposes of benchmarking.", "label": "Simulated gene expression data generation"}, "data_1684": {"class_id": "http://edamontology.org/data_1684", "definitions": "EMBOSS (EMBASSY) sites application log file.", "label": "EMBOSS sites log file"}, "data_1070": {"class_id": "http://edamontology.org/data_1070", "definitions": "A unique and persistent identifier of a molecular tertiary structure, typically an entry from a structure database.", "label": "Structure ID"}, "data_1071": {"class_id": "http://edamontology.org/data_1071", "definitions": "Identifier or name of a structural (3D) profile or template (representing a structure or structure alignment).", "label": "Structural (3D) profile ID"}, "data_1072": {"class_id": "http://edamontology.org/data_1072", "definitions": "Identifier of an entry from a database of tertiary structure alignments.", "label": "Structure alignment ID"}, "data_1073": {"class_id": "http://edamontology.org/data_1073", "definitions": "Identifier of an index of amino acid physicochemical and biochemical property data.", "label": "Amino acid index ID"}, "data_1074": {"class_id": "http://edamontology.org/data_1074", "definitions": "Identifier of a report of protein interactions from a protein interaction database (typically).", "label": "Protein interaction ID"}, "data_1075": {"class_id": "http://edamontology.org/data_1075", "definitions": "Identifier of a protein family.", "label": "Protein family identifier"}, "data_1076": {"class_id": "http://edamontology.org/data_1076", "definitions": "Unique name of a codon usage table.", "label": "Codon usage table name"}, "data_1077": {"class_id": "http://edamontology.org/data_1077", "definitions": "Identifier of a transcription factor (or a TF binding site).", "label": "Transcription factor identifier"}, "data_1078": {"class_id": "http://edamontology.org/data_1078", "definitions": "Identifier of an entry from a database of microarray data.", "label": "Experiment annotation ID"}, "data_1079": {"class_id": "http://edamontology.org/data_1079", "definitions": "Identifier of an entry from a database of electron microscopy data.", "label": "Electron microscopy model ID"}, "operation_3761": {"class_id": "http://edamontology.org/operation_3761", "definitions": "An operation supporting the browsing or discovery of other tools and services.", "label": "Service discovery"}, "topic_0623": {"class_id": "http://edamontology.org/topic_0623", "definitions": "Particular gene(s), gene family or other gene group or system and their encoded proteins.Primarily the classification of proteins (from sequence or structural data) into clusters, groups, families etc., curation of a particular protein or protein family, or any other proteins that have been classified as members of a common group.|A protein families database might include the classifier (e.g. a sequence profile) used to build the classification.", "label": "Gene and protein families"}, "operation_3445": {"class_id": "http://edamontology.org/operation_3445", "definitions": "Analysis of data from a diffraction experiment.", "label": "Diffraction data analysis"}, "operation_0304": {"class_id": "http://edamontology.org/operation_0304", "definitions": "Search for and retrieve data concerning or describing some core data, as distinct from the primary data that is being described.|This includes documentation, general information and other metadata on entities such as databases, database entries and tools.", "label": "Metadata retrieval"}, "operation_3441": {"class_id": "http://edamontology.org/operation_3441", "definitions": "Generate a graph, or other visual representation, of data, showing the relationship between two or more variables.", "label": "Plotting"}, "topic_0620": {"class_id": "http://edamontology.org/topic_0620", "definitions": "The structures of drugs, drug target, their interactions and binding affinities.", "label": "Drugs and target structures"}, "topic_0736": {"class_id": "http://edamontology.org/topic_0736", "definitions": "Protein tertiary structural domains and folds in a protein or polypeptide chain.|This includes trans- or intra-membrane regions of a protein, typically describing physicochemical properties of the secondary structure elements.  For example, the location and size of the membrane spanning segments and intervening loop regions, transmembrane region IN/OUT orientation relative to the membrane, plus the following data for each amino acid: A Z-coordinate (the distance to the membrane center), the free energy of membrane insertion (calculated in a sliding window over the sequence) and a reliability score. The z-coordinate implies information about re-entrant helices, interfacial helices, the tilt of a transmembrane helix and loop lengths.|This includes topological domains such as cytoplasmic regions in a protein.", "label": "Protein folds and structural domains"}, "operation_3440": {"class_id": "http://edamontology.org/operation_3440", "definitions": "The process of assembling many short DNA sequences together such thay they represent the original chromosomes from which the DNA originated.", "label": "Genome assembly"}, "data_2916": {"class_id": "http://edamontology.org/data_2916", "definitions": "An identifier of an entry from the DDBJ sequence database.", "label": "DDBJ accession"}, "data_2917": {"class_id": "http://edamontology.org/data_2917", "definitions": "An identifier of an entity from the ConsensusPathDB database.", "label": "ConsensusPathDB identifier"}, "data_2914": {"class_id": "http://edamontology.org/data_2914", "definitions": "Metadata on sequence features.", "label": "Sequence features metadata"}, "data_2915": {"class_id": "http://edamontology.org/data_2915", "definitions": "Identifier of a Gramene database entry.", "label": "Gramene identifier"}, "data_2912": {"class_id": "http://edamontology.org/data_2912", "definitions": "Identifier of a strain of an organism variant, typically a plant, virus or bacterium.", "label": "Strain accession"}, "data_2913": {"class_id": "http://edamontology.org/data_2913", "definitions": "An accession of annotation on a (group of) viruses (catalogued in a database).", "label": "Virus identifier"}, "data_2910": {"class_id": "http://edamontology.org/data_2910", "definitions": "Accession of a protein family (that is deposited in a database).", "label": "Protein family accession"}, "data_2911": {"class_id": "http://edamontology.org/data_2911", "definitions": "Accession of an entry from a database of transcription factors or binding sites.", "label": "Transcription factor accession"}, "topic_3408": {"class_id": "http://edamontology.org/topic_3408", "definitions": "The branch of medicine that deals with the blood, blood-forming organs and blood diseases.", "label": "Haematology"}, "operation_3763": {"class_id": "http://edamontology.org/operation_3763", "definitions": "An operation supporting the calling (invocation) of other tools and services.", "label": "Service invocation"}, "topic_0621": {"class_id": "http://edamontology.org/topic_0621", "definitions": "A specific organism, or group of organisms, used to study a particular aspect of biology.|This may include information on the genome (including molecular sequences and map, genes and annotation), proteome, as well as more general information about an organism.", "label": "Model organisms"}, "operation_0309": {"class_id": "http://edamontology.org/operation_0309", "definitions": "Predict and/or optimize oligonucleotide probes for DNA microarrays, for example for transcription profiling of genes, or for genomes and gene families.", "label": "Microarray probe design"}, "data_1463": {"class_id": "http://edamontology.org/data_1463", "definitions": "3D coordinate and associated data for the (3D) structure of a small molecule, such as any common chemical compound.", "label": "Small molecule structure"}, "data_1462": {"class_id": "http://edamontology.org/data_1462", "definitions": "3D coordinate and associated data for a carbohydrate (3D) structure.", "label": "Carbohydrate structure"}, "data_1461": {"class_id": "http://edamontology.org/data_1461", "definitions": "The structure of a protein in complex with a ligand, typically a small molecule such as an enzyme substrate or cofactor, but possibly another macromolecule.|This includes interactions of proteins with atoms, ions and small molecules or macromolecules such as nucleic acids or other polypeptides.  For stable inter-polypeptide interactions use 'Protein complex' instead.", "label": "Protein-ligand complex"}, "data_1460": {"class_id": "http://edamontology.org/data_1460", "definitions": "3D coordinate and associated data for a protein tertiary (3D) structure.", "label": "Protein structure"}, "data_1467": {"class_id": "http://edamontology.org/data_1467", "definitions": "3D coordinate and associated data for the tertiary (3D) structure of a polypeptide chain.", "label": "Protein chain"}, "data_1466": {"class_id": "http://edamontology.org/data_1466", "definitions": "3D coordinate and associated data for a tRNA tertiary (3D) structure, including tmRNA, snoRNAs etc.", "label": "tRNA structure"}, "data_1465": {"class_id": "http://edamontology.org/data_1465", "definitions": "3D coordinate and associated data for an RNA tertiary (3D) structure.", "label": "RNA structure"}, "data_1464": {"class_id": "http://edamontology.org/data_1464", "definitions": "3D coordinate and associated data for a DNA tertiary (3D) structure.", "label": "DNA structure"}, "data_0990": {"class_id": "http://edamontology.org/data_0990", "definitions": "Unique name of a chemical compound.", "label": "Compound name"}, "data_2355": {"class_id": "http://edamontology.org/data_2355", "definitions": "Identifier of an RNA family, typically an entry from a RNA sequence classification database.", "label": "RNA family identifier"}, "data_2356": {"class_id": "http://edamontology.org/data_2356", "definitions": "Stable accession number of an entry (RNA family) from the RFAM database.", "label": "RFAM accession"}, "data_0993": {"class_id": "http://edamontology.org/data_0993", "definitions": "Identifier of a drug.", "label": "Drug identifier"}, "data_0994": {"class_id": "http://edamontology.org/data_0994", "definitions": "Identifier of an amino acid.", "label": "Amino acid identifier"}, "data_0995": {"class_id": "http://edamontology.org/data_0995", "definitions": "Name or other identifier of a nucleotide.", "label": "Nucleotide identifier"}, "data_0996": {"class_id": "http://edamontology.org/data_0996", "definitions": "Identifier of a monosaccharide.", "label": "Monosaccharide identifier"}, "data_2353": {"class_id": "http://edamontology.org/data_2353", "definitions": "Data concerning or derived from an ontology.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Ontology data"}, "data_1661": {"class_id": "http://edamontology.org/data_1661", "definitions": "disease pathways, typically of human disease.", "label": "Disease pathway or network report"}, "data_1660": {"class_id": "http://edamontology.org/data_1660", "definitions": "Topic concernning cellular process pathways.", "label": "Cellular process pathways report"}, "data_1663": {"class_id": "http://edamontology.org/data_1663", "definitions": "networks of protein interactions.", "label": "Protein interaction networks"}, "data_1662": {"class_id": "http://edamontology.org/data_1662", "definitions": "A report typically including a map (diagram) of drug structure relationships.", "label": "Drug structure relationship map"}, "data_1917": {"class_id": "http://edamontology.org/data_1917", "definitions": "Data for an atom (in a molecular structure).", "label": "Atomic property"}, "data_1664": {"class_id": "http://edamontology.org/data_1664", "definitions": "An entry (data type) from the Minimal Information Requested in the Annotation of Biochemical Models (MIRIAM) database of data resources.|A MIRIAM entry describes a MIRIAM data type including the official name, synonyms, root URI, identifier pattern (regular expression applied to a unique identifier of the data type) and documentation. Each data type can be associated with several resources. Each resource is a physical location of a service (typically a database) providing information on the elements of a data type. Several resources may exist for each data type, provided the same (mirrors) or different information. MIRIAM provides a stable and persistent reference to its data types.", "label": "MIRIAM datatype"}, "data_1667": {"class_id": "http://edamontology.org/data_1667", "definitions": "A simple floating point number defining the lower or upper limit of an expectation value (E-value).|An expectation value (E-Value) is the expected number of observations which are at least as extreme as observations expected to occur by random chance. The E-value describes the number of hits with a given score or better that are expected to occur at random when searching a database of a particular size. It decreases exponentially with the score (S) of a hit. A low E value indicates a more significant score.", "label": "E-value"}, "data_1669": {"class_id": "http://edamontology.org/data_1669", "definitions": "The P-value is the probability of obtaining by random chance a result that is at least as extreme as an observed result, assuming a NULL hypothesis is true.|A z-value might be specified as a threshold for reporting hits from database searches.", "label": "P-value"}, "data_1668": {"class_id": "http://edamontology.org/data_1668", "definitions": "The z-value is the number of standard deviations a data value is above or below a mean value.|A z-value might be specified as a threshold for reporting hits from database searches.", "label": "Z-value"}, "format_3747": {"class_id": "http://edamontology.org/format_3747", "definitions": "A format for storage, exchange, and processing of protein identifications created from ms/ms-derived peptide sequence data.|No human-consumable information about this format is available (see http://tools.proteomecenter.org/wiki/index.php?title=Formats:protXML).", "label": "protXML"}, "format_3746": {"class_id": "http://edamontology.org/format_3746", "definitions": "The BIological Observation Matrix (BIOM) is a format for representing biological sample by observation contingency tables in broad areas of comparative omics. The primary use of this format is to represent OTU tables and metagenome tables.|BIOM is a recognised standard for the Earth Microbiome Project, and is a project supported by Genomics Standards Consortium. Supported in QIIME, Mothur, MEGAN, etc.", "label": "BIOM format"}, "topic_0624": {"class_id": "http://edamontology.org/topic_0624", "definitions": "Study of chromosomes.", "label": "Chromosomes"}, "operation_0226": {"class_id": "http://edamontology.org/operation_0226", "definitions": "Annotate an entity (typically a biological or biomedical database entity) with terms from a controlled vocabulary.|This is a broad concept and is used a placeholder for other, more specific concepts.", "label": "Annotation"}, "topic_0100": {"class_id": "http://edamontology.org/topic_0100", "definitions": "Topic for the study of restriction enzymes, their cleavage sites and the restriction of nucleic acids.", "label": "Nucleic acid restriction"}, "format_2341": {"class_id": "http://edamontology.org/format_2341", "definitions": "The format of an entry from the NCI-Nature pathways database.", "label": "NCI-Nature pathway entry format"}, "format_2188": {"class_id": "http://edamontology.org/format_2188", "definitions": "UniProt entry sequence format.", "label": "UniProt format"}, "format_2189": {"class_id": "http://edamontology.org/format_2189", "definitions": "ipi sequence format.", "label": "ipi"}, "topic_2868": {"class_id": "http://edamontology.org/topic_2868", "definitions": "Microsatellite polymorphism in a DNA sequence.", "label": "Microsatellites"}, "format_2181": {"class_id": "http://edamontology.org/format_2181", "definitions": "A text format resembling EMBL entry format.|This concept may be used for the many non-standard EMBL-like text formats.", "label": "EMBL-like (text)"}, "format_2182": {"class_id": "http://edamontology.org/format_2182", "definitions": "A text format resembling FASTQ short read format.|This concept may be used for non-standard FASTQ short read-like formats.", "label": "FASTQ-like format (text)"}, "format_2183": {"class_id": "http://edamontology.org/format_2183", "definitions": "XML format for EMBL entries.", "label": "EMBLXML"}, "format_2184": {"class_id": "http://edamontology.org/format_2184", "definitions": "XML format for EMBL entries.", "label": "cdsxml"}, "format_2185": {"class_id": "http://edamontology.org/format_2185", "definitions": "XML format for EMBL entries.", "label": "insdxml"}, "format_2186": {"class_id": "http://edamontology.org/format_2186", "definitions": "Geneseq sequence format.", "label": "geneseq"}, "format_2187": {"class_id": "http://edamontology.org/format_2187", "definitions": "A text sequence format resembling uniprotkb entry format.", "label": "UniProt-like (text)"}, "data_1867": {"class_id": "http://edamontology.org/data_1867", "definitions": "The name of a protein fold.", "label": "Protein fold name"}, "data_1866": {"class_id": "http://edamontology.org/data_1866", "definitions": "A designation of the type of map (genetic map, physical map, sequence map etc) or map set.|Map types may be based on Gramene's notion of a map type; see http://www.gramene.org/db/cmap/map_type_info.", "label": "Map type"}, "data_0848": {"class_id": "http://edamontology.org/data_0848", "definitions": "A raw molecular sequence (string of characters) which might include ambiguity, unknown positions and non-sequence characters.|Non-sequence characters may be used for example for gaps and translation stop.", "label": "Raw sequence"}, "data_1864": {"class_id": "http://edamontology.org/data_1864", "definitions": "Data describing a set of multiple genetic or physical maps, typically sharing a common set of features which are mapped.", "label": "Map set data"}, "data_1863": {"class_id": "http://edamontology.org/data_1863", "definitions": "A map of haplotypes in a genome or other sequence, describing common patterns of genetic variation.", "label": "Haplotype map"}, "data_1860": {"class_id": "http://edamontology.org/data_1860", "definitions": "A genetic map which shows the approximate location of quantitative trait loci (QTL) between two or more markers.", "label": "QTL map"}, "data_0842": {"class_id": "http://edamontology.org/data_0842", "definitions": "A text token, number or something else which identifies an entity, but which may not be persistent (stable) or unique (the same identifier may identify multiple things).", "label": "Identifier"}, "data_0843": {"class_id": "http://edamontology.org/data_0843", "definitions": "An entry (retrievable via URL) from a biological database.", "label": "Database entry"}, "data_0846": {"class_id": "http://edamontology.org/data_0846", "definitions": "A specification of a chemical structure.", "label": "Chemical formula"}, "data_0847": {"class_id": "http://edamontology.org/data_0847", "definitions": "A QSAR quantitative descriptor (name-value pair) of chemical structure.|QSAR descriptors have numeric values that quantify chemical information encoded in a symbolic representation of a molecule. They are used in quantitative structure activity relationship (QSAR) applications. Many subtypes of individual descriptors (not included in EDAM) cover various types of protein properties.", "label": "QSAR descriptor"}, "data_0844": {"class_id": "http://edamontology.org/data_0844", "definitions": "Mass of a molecule.", "label": "Molecular mass"}, "data_0845": {"class_id": "http://edamontology.org/data_0845", "definitions": "Net charge of a molecule.", "label": "Molecular charge"}, "operation_0505": {"class_id": "http://edamontology.org/operation_0505", "definitions": "Align protein tertiary structures.", "label": "Structure alignment (protein)"}, "operation_0504": {"class_id": "http://edamontology.org/operation_0504", "definitions": "Align (superimpose) more than two molecular tertiary structures.|This includes methods that use an existing alignment.", "label": "Multiple structure alignment"}, "operation_0507": {"class_id": "http://edamontology.org/operation_0507", "definitions": "Locally align (superimpose) exactly two molecular tertiary structures.|Local alignment methods identify regions of local similarity, common substructures etc.", "label": "Pairwise structure alignment generation (local)"}, "operation_0506": {"class_id": "http://edamontology.org/operation_0506", "definitions": "Align RNA tertiary structures.", "label": "Structure alignment (RNA)"}, "operation_0501": {"class_id": "http://edamontology.org/operation_0501", "definitions": "Align protein secondary structures.", "label": "Protein secondary structure alignment generation"}, "operation_0500": {"class_id": "http://edamontology.org/operation_0500", "definitions": "Align molecular secondary structure (represented as a 1D string).", "label": "Secondary structure alignment generation"}, "operation_0503": {"class_id": "http://edamontology.org/operation_0503", "definitions": "Align (superimpose) exactly two molecular tertiary structures.", "label": "Pairwise structure alignment"}, "operation_0502": {"class_id": "http://edamontology.org/operation_0502", "definitions": "Align RNA secondary structures.", "label": "RNA secondary structure alignment"}, "topic_0629": {"class_id": "http://edamontology.org/topic_0629", "definitions": "Gene expression e.g. microarray data, northern blots, gene-indexed expression profiles etc.", "label": "Gene expression and microarray"}, "operation_0509": {"class_id": "http://edamontology.org/operation_0509", "definitions": "Locally align (superimpose) two or more molecular tertiary structures.|Local alignment methods identify regions of local similarity, common substructures etc.", "label": "Local structure alignment"}, "operation_0508": {"class_id": "http://edamontology.org/operation_0508", "definitions": "Globally align (superimpose) exactly two molecular tertiary structures.|Global alignment methods identify similarity across the entire structures.", "label": "Pairwise structure alignment generation (global)"}, "data_2109": {"class_id": "http://edamontology.org/data_2109", "definitions": "An identifier that is re-used for data objects of fundamentally different types (typically served from a single database).|This branch provides an alternative organisation of the concepts nested under 'Accession' and 'Name'. All concepts under here are already included under 'Accession' or 'Name'.", "label": "Identifier (hybrid)"}, "data_2108": {"class_id": "http://edamontology.org/data_2108", "definitions": "Identifier of a biological reaction from a database.", "label": "Reaction ID"}, "data_2105": {"class_id": "http://edamontology.org/data_2105", "definitions": "Identifier of a compound from the BioCyc chemical compounds database.", "label": "Compound ID (BioCyc)"}, "data_2104": {"class_id": "http://edamontology.org/data_2104", "definitions": "Identifier of an object from one of the BioCyc databases.", "label": "BioCyc ID"}, "data_2107": {"class_id": "http://edamontology.org/data_2107", "definitions": "Identifier of an enzyme from the BioCyc enzymes database.", "label": "Enzyme ID (BioCyc)"}, "data_2106": {"class_id": "http://edamontology.org/data_2106", "definitions": "Identifier of a biological reaction from the BioCyc reactions database.", "label": "Reaction ID (BioCyc)"}, "data_2101": {"class_id": "http://edamontology.org/data_2101", "definitions": "An identifier of a software end-user (typically a person).", "label": "User ID"}, "data_2100": {"class_id": "http://edamontology.org/data_2100", "definitions": "A label (text token) describing the type of a thing, typically an enumerated string (a string with one of a limited set of values).", "label": "Type"}, "data_2103": {"class_id": "http://edamontology.org/data_2103", "definitions": "Name of an entry (gene) from the KEGG GENES database.", "label": "Gene name (KEGG GENES)"}, "data_2102": {"class_id": "http://edamontology.org/data_2102", "definitions": "A three-letter code used in the KEGG databases to uniquely identify organisms.", "label": "KEGG organism code"}, "operation_3184": {"class_id": "http://edamontology.org/operation_3184", "definitions": "Render and visualise a DNA sequence assembly.", "label": "Sequence assembly visualisation"}, "operation_3185": {"class_id": "http://edamontology.org/operation_3185", "definitions": "Identify base (nucleobase) sequence from a fluorescence 'trace' data generated by an automated DNA sequencer.", "label": "Base-calling"}, "operation_2516": {"class_id": "http://edamontology.org/operation_2516", "definitions": "Visualise, format or render a protein sequence.|Various protein sequence analysis methods might generate a sequence rendering but are not (for brevity) listed under here.", "label": "Protein sequence visualisation"}, "data_1489": {"class_id": "http://edamontology.org/data_1489", "definitions": "Alignment (superimposition) of exactly two protein tertiary (3D) structures (all atoms considered).", "label": "Multiple protein tertiary structure alignment (all atoms)"}, "data_1488": {"class_id": "http://edamontology.org/data_1488", "definitions": "Alignment (superimposition) of exactly two protein tertiary (3D) structures (typically C-alpha atoms only considered).|C-beta atoms from amino acid side-chains may be included.", "label": "Pairwise protein tertiary structure alignment (C-alpha atoms)"}, "operation_3187": {"class_id": "http://edamontology.org/operation_3187", "definitions": "Identify and filter a (typically large) sequence data set to remove sequences from contaminants in the sample that was sequenced.", "label": "Sequence contamination filtering"}, "data_1481": {"class_id": "http://edamontology.org/data_1481", "definitions": "Alignment (superimposition) of protein tertiary (3D) structures.", "label": "Structure alignment (protein)"}, "data_1480": {"class_id": "http://edamontology.org/data_1480", "definitions": "Alignment (superimposition) of more than two molecular tertiary (3D) structures.", "label": "Structure alignment (multiple)"}, "data_1483": {"class_id": "http://edamontology.org/data_1483", "definitions": "Alignment (superimposition) of exactly two protein tertiary (3D) structures.", "label": "Structure alignment (protein pair)"}, "data_1482": {"class_id": "http://edamontology.org/data_1482", "definitions": "Alignment (superimposition) of nucleic acid tertiary (3D) structures.", "label": "Structure alignment (nucleic acid)"}, "data_1485": {"class_id": "http://edamontology.org/data_1485", "definitions": "Alignment (superimposition) of protein tertiary (3D) structures (all atoms considered).", "label": "Structure alignment (protein all atoms)"}, "data_1484": {"class_id": "http://edamontology.org/data_1484", "definitions": "Alignment (superimposition) of more than two protein tertiary (3D) structures.", "label": "Multiple protein tertiary structure alignment"}, "data_1487": {"class_id": "http://edamontology.org/data_1487", "definitions": "Alignment (superimposition) of exactly two protein tertiary (3D) structures (all atoms considered).", "label": "Pairwise protein tertiary structure alignment (all atoms)"}, "data_1486": {"class_id": "http://edamontology.org/data_1486", "definitions": "Alignment (superimposition) of protein tertiary (3D) structures (typically C-alpha atoms only considered).|C-beta atoms from amino acid side-chains may be considered.", "label": "Structure alignment (protein C-alpha atoms)"}, "topic_3388": {"class_id": "http://edamontology.org/topic_3388", "definitions": "The identification of molecular and genetic causes of disease and the development of interventions to correct them.", "label": "Molecular medicine"}, "operation_3744": {"class_id": "http://edamontology.org/operation_3744", "definitions": "Visualise, format or render data arising from an analysis of multiple samples from a metagenomics/community experiment.", "label": "Multiple sample visualisation"}, "operation_3745": {"class_id": "http://edamontology.org/operation_3745", "definitions": "The extrapolation of empirical characteristics of individuals or populations, backwards in time, to their common ancestors.|Ancestral reconstruction is often used to recover possible ancestral character states of ancient, extinct organisms.", "label": "Ancestral reconstruction"}, "operation_3742": {"class_id": "http://edamontology.org/operation_3742", "definitions": "The analysis, using any of diverse techniques, to identify genes that are differentially expressed under a given experimental setup.", "label": "Differential gene expression analysis"}, "operation_3182": {"class_id": "http://edamontology.org/operation_3182", "definitions": "Align two or more (tpyically huge) molecular sequences that represent genomes.", "label": "Genome alignment"}, "operation_3186": {"class_id": "http://edamontology.org/operation_3186", "definitions": "The mapping of methylation sites in a DNA (genome) sequence.|Bisulfite mapping follows high-throughput sequencing of DNA which has undergone bisulfite treatment followed by PCR amplification; unmethylated cytosines are specifically converted to thymine, allowing the methylation status of cytosine in the DNA to be detected.", "label": "Bisulfite mapping"}, "format_3714": {"class_id": "http://edamontology.org/format_3714", "definitions": "Format of peak list files from Andromeda search engine (MaxQuant) that consist of arbitrarily many spectra.", "label": "MaxQuant APL peaklist format"}, "topic_3382": {"class_id": "http://edamontology.org/topic_3382", "definitions": "The visual representation of an object.|This includes diffraction experiments that are based upon the interference of waves, typically electromagnetic waves such as  X-rays or visible light, by some object being studied, typical in order to produce an image of the object or determine its structure.", "label": "Imaging"}, "operation_3183": {"class_id": "http://edamontology.org/operation_3183", "definitions": "Reconstruction of a sequence assembly in a localised area.", "label": "Localised reassembly"}, "topic_3384": {"class_id": "http://edamontology.org/topic_3384", "definitions": "The use of imaging techniques for clinical purposes for medical research.", "label": "Medical imaging"}, "topic_3385": {"class_id": "http://edamontology.org/topic_3385", "definitions": "The use of optical instruments to magnify the image of an object.", "label": "Light microscopy"}, "topic_3386": {"class_id": "http://edamontology.org/topic_3386", "definitions": "The use of animals and alternatives in experimental research.", "label": "Laboratory animal science"}, "topic_3387": {"class_id": "http://edamontology.org/topic_3387", "definitions": "The study of organisms in the ocean or brackish waters.", "label": "Marine biology"}, "operation_0431": {"class_id": "http://edamontology.org/operation_0431", "definitions": "Find and identify restriction enzyme cleavage sites (restriction sites) in (typically) DNA sequences, for example to generate a restriction map.", "label": "Restriction site recognition"}, "operation_0430": {"class_id": "http://edamontology.org/operation_0430", "definitions": "Find CpG rich regions in a nucleotide sequence or isochores in genome sequences.|An isochore is long region (> 3 KB) of DNA with very uniform GC content, in contrast to the rest of the genome. Isochores tend tends to have more genes, higher local melting or denaturation temperatures, and different flexibility. Methods might calculate fractional GC content or variation of GC content, predict methylation status of CpG islands etc. This includes methods that visualise CpG rich regions in a nucleotide sequence, for example plot isochores in a genome sequence.", "label": "CpG island and isochore detection"}, "operation_0433": {"class_id": "http://edamontology.org/operation_0433", "definitions": "Identify, predict or analyse splice sites in nucleotide sequences.|Methods might require a pre-mRNA or genomic DNA sequence.", "label": "Splice site prediction"}, "operation_0432": {"class_id": "http://edamontology.org/operation_0432", "definitions": "Identify or predict nucleosome exclusion sequences (nucleosome free regions) in DNA.", "label": "Nucleosome formation or exclusion sequence prediction"}, "operation_0435": {"class_id": "http://edamontology.org/operation_0435", "definitions": "Find operons (operators, promoters and genes) in bacteria genes.", "label": "Operon prediction"}, "operation_0434": {"class_id": "http://edamontology.org/operation_0434", "definitions": "Predict whole gene structure using a combination of multiple methods to achieve better predictions.", "label": "Integrated gene prediction"}, "operation_0437": {"class_id": "http://edamontology.org/operation_0437", "definitions": "Predict selenocysteine insertion sequence (SECIS) in a DNA sequence.|SECIS elements are around 60 nucleotides in length with a stem-loop structure directs the cell to translate UGA codons as selenocysteines.", "label": "SECIS element prediction"}, "operation_0436": {"class_id": "http://edamontology.org/operation_0436", "definitions": "Predict protein-coding regions (CDS or exon) or open reading frames in nucleotide sequences.", "label": "Coding region prediction"}, "operation_0439": {"class_id": "http://edamontology.org/operation_0439", "definitions": "Predict translation initiation sites, possibly by searching a database of sites.", "label": "Translation initiation site prediction"}, "operation_0438": {"class_id": "http://edamontology.org/operation_0438", "definitions": "Identify or predict transcriptional regulatory motifs, patterns, elements or regions in DNA sequences.|This includes promoters, enhancers, silencers and boundary elements / insulators, regulatory protein or transcription factor binding sites etc. Methods might be specific to a particular genome and use motifs, word-based / grammatical methods, position-specific frequency matrices, discriminative pattern analysis etc.", "label": "Transcriptional regulatory element prediction"}, "data_3743": {"class_id": "http://edamontology.org/data_3743", "definitions": "A plot in which community data (e.g. species abundance data) is summarised.  Similar species and samples are plotted close together, and dissimilar species and samples are plotted placed far apart.", "label": "Ordination plot"}, "data_2611": {"class_id": "http://edamontology.org/data_2611", "definitions": "An identifier of a disease from the International Classification of Diseases (ICD) database.", "label": "ICD identifier"}, "data_2610": {"class_id": "http://edamontology.org/data_2610", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl database.", "label": "Ensembl ID"}, "data_2613": {"class_id": "http://edamontology.org/data_2613", "definitions": "Unique identifier of a glycan ligand from the KEGG GLYCAN database (a subset of KEGG LIGAND).", "label": "KEGG Glycan ID"}, "data_2612": {"class_id": "http://edamontology.org/data_2612", "definitions": "Unique identifier of a sequence cluster from the CluSTr database.", "label": "Sequence cluster ID (CluSTr)"}, "data_2615": {"class_id": "http://edamontology.org/data_2615", "definitions": "Unique identifier of an entry from the MINT database of protein-protein interactions.", "label": "MINT ID"}, "data_2614": {"class_id": "http://edamontology.org/data_2614", "definitions": "A unique identifier of a family from the transport classification database (TCDB) of membrane transport proteins.|OBO file for regular expression.", "label": "TCDB ID"}, "data_2617": {"class_id": "http://edamontology.org/data_2617", "definitions": "Unique identifier of a protein listed in the UCSD-Nature Signaling Gateway Molecule Pages database.", "label": "Signaling Gateway protein ID"}, "data_2616": {"class_id": "http://edamontology.org/data_2616", "definitions": "Unique identifier of an entry from the DIP database of protein-protein interactions.", "label": "DIP ID"}, "data_2619": {"class_id": "http://edamontology.org/data_2619", "definitions": "Identifier of a protein modification catalogued in the RESID database.", "label": "RESID ID"}, "data_1674": {"class_id": "http://edamontology.org/data_1674", "definitions": "Cross-references from a sequence record to other databases.", "label": "Sequence database cross-references"}, "operation_2519": {"class_id": "http://edamontology.org/operation_2519", "definitions": "Process (read and / or write) nucleic acid tertiary structure data.|http://edamontology.org/operation_2481", "label": "Structure processing (nucleic acid)"}, "operation_3638": {"class_id": "http://edamontology.org/operation_3638", "definitions": "Quantification analysis using stable isotope labeling by amino acids in cell culture.", "label": "SILAC"}, "operation_3639": {"class_id": "http://edamontology.org/operation_3639", "definitions": "Quantification analysis using the AB SCIEX iTRAQ isobaric labelling workflow, wherein 2-8 reporter ions are measured in MS2 spectra near 114 m/z.", "label": "iTRAQ"}, "operation_3632": {"class_id": "http://edamontology.org/operation_3632", "definitions": "Calculate the isotope distribution of a given chemical species.", "label": "Isotopic distributions calculation"}, "operation_3633": {"class_id": "http://edamontology.org/operation_3633", "definitions": "Prediction of retention time in a mass spectrometry experiment based on compositional and structural properties of the separated species.", "label": "Retention time prediction"}, "operation_3630": {"class_id": "http://edamontology.org/operation_3630", "definitions": "Technique for determining the amount of proteins in a sample.", "label": "Protein quantification"}, "operation_3631": {"class_id": "http://edamontology.org/operation_3631", "definitions": "Determination of peptide sequence from mass spectrum.", "label": "Peptide identification"}, "operation_3636": {"class_id": "http://edamontology.org/operation_3636", "definitions": "Quantification by Selected/multiple Reaction Monitoring workflow (XIC quantitation of precursor / fragment mass pair).", "label": "MRM/SRM"}, "operation_3637": {"class_id": "http://edamontology.org/operation_3637", "definitions": "Calculate number of identified MS2 spectra as approximation of peptide / protein quantity.", "label": "Spectral counting"}, "operation_3634": {"class_id": "http://edamontology.org/operation_3634", "definitions": "Quantification without the use of chemical tags.", "label": "Label-free quantification"}, "operation_3635": {"class_id": "http://edamontology.org/operation_3635", "definitions": "Quantification based on the use of chemical tags.", "label": "Labeled quantification"}, "operation_3562": {"class_id": "http://edamontology.org/operation_3562", "definitions": "Simulate the bevaviour of a biological pathway or network.", "label": "Network simulation"}, "operation_3563": {"class_id": "http://edamontology.org/operation_3563", "definitions": "Analyze read counts from RNA-seq experiments.", "label": "RNA-seq read count analysis"}, "operation_3560": {"class_id": "http://edamontology.org/operation_3560", "definitions": "A method for making numerical assessments about the maximum percent of time that a conformer of a flexible macromolecule can exist and still be compatible with the experimental data. ", "label": "Maximum occurence analysis"}, "operation_3180": {"class_id": "http://edamontology.org/operation_3180", "definitions": "Evaluate a DNA sequence assembly, typically for purposes of quality control.", "label": "Sequence assembly validation"}, "format_3712": {"class_id": "http://edamontology.org/format_3712", "definitions": "Proprietary file format for mass spectrometry data from Thermo Scientific.|Proprietary format for which documentation is not available.", "label": "Thermo RAW"}, "topic_3569": {"class_id": "http://edamontology.org/topic_3569", "definitions": "The application of mathematics to specific problems in science, typically by the formulation and analysis of mathematical models.", "label": "Applied mathematics"}, "operation_3564": {"class_id": "http://edamontology.org/operation_3564", "definitions": "Identify and remove redudancy from a set of small molecule structures.", "label": "Chemical redundancy removal"}, "operation_3565": {"class_id": "http://edamontology.org/operation_3565", "definitions": "Analyze time series data from an RNA-seq experiment.", "label": "RNA-seq time series data analysis"}, "format_1633": {"class_id": "http://edamontology.org/format_1633", "definitions": "PHD sequence trace format to store serialised chromatogram data (reads).", "label": "PHD"}, "format_1632": {"class_id": "http://edamontology.org/format_1632", "definitions": "Staden Chromatogram Files format (SCF) of base-called sequence reads, qualities, and other metadata.", "label": "SCF"}, "format_1631": {"class_id": "http://edamontology.org/format_1631", "definitions": "Sequence assembly project file EXP format.", "label": "exp"}, "format_1630": {"class_id": "http://edamontology.org/format_1630", "definitions": "Common Assembly Format (CAF). A sequence assembly format including contigs, base-call qualities, and other metadata.", "label": "CAF"}, "format_1637": {"class_id": "http://edamontology.org/format_1637", "definitions": "Format of Affymetrix data file of raw image data.", "label": "dat"}, "format_1639": {"class_id": "http://edamontology.org/format_1639", "definitions": "Format of affymetrix gene cluster files (hc-genes.txt, hc-chips.txt) from hierarchical clustering.", "label": "affymetrix"}, "format_1638": {"class_id": "http://edamontology.org/format_1638", "definitions": "Format of Affymetrix data file of information about (raw) expression levels of the individual probes.", "label": "cel"}, "format_3713": {"class_id": "http://edamontology.org/format_3713", "definitions": "\"Raw\" result file from Mascot database search.", "label": "Mascot .dat file"}, "topic_3041": {"class_id": "http://edamontology.org/topic_3041", "definitions": "Molecular sequence data resources, including sequence sites, alignments, motifs and profiles.", "label": "Sequence databases"}, "topic_0595": {"class_id": "http://edamontology.org/topic_0595", "definitions": "primarily the classification of proteins (from sequence or structural data) into clusters, groups, families etc.", "label": "Protein classification"}, "data_2725": {"class_id": "http://edamontology.org/data_2725", "definitions": "Unique identifier for a gene transcript from the Ensembl database.", "label": "Ensembl transcript ID"}, "data_2724": {"class_id": "http://edamontology.org/data_2724", "definitions": "Annotation on an embryo or concerning embryological development.", "label": "Embryo report"}, "data_2727": {"class_id": "http://edamontology.org/data_2727", "definitions": "An identifier of a promoter of a gene that is catalogued in a database.", "label": "Promoter ID"}, "data_2726": {"class_id": "http://edamontology.org/data_2726", "definitions": "An informative report on one or more small molecules that are enzyme inhibitors.", "label": "Inhibitor annotation"}, "data_2721": {"class_id": "http://edamontology.org/data_2721", "definitions": "Identifier of an dinucleotide property from the DiProDB database.", "label": "DiProDB ID"}, "data_2720": {"class_id": "http://edamontology.org/data_2720", "definitions": "Physicochemical property data for one or more dinucleotides.", "label": "Dinucleotide property"}, "data_2723": {"class_id": "http://edamontology.org/data_2723", "definitions": "Unique identifier for a protein from the DisProt database.", "label": "Protein ID (DisProt)"}, "data_2722": {"class_id": "http://edamontology.org/data_2722", "definitions": "disordered structure in a protein.", "label": "Protein features report (disordered structure)"}, "topic_3410": {"class_id": "http://edamontology.org/topic_3410", "definitions": "The study of the biological and physiological differences between males and females and how they effect differences in disease presentation and management.", "label": "Gender medicine"}, "topic_3411": {"class_id": "http://edamontology.org/topic_3411", "definitions": "The branch of medicine that deals with the health of the female reproductive system, pregnancy and birth.", "label": "Gynaecology and obstetrics"}, "topic_3412": {"class_id": "http://edamontology.org/topic_3412", "definitions": "The branch of medicine that deals with the liver, gallbladder, bile ducts and bile.", "label": "Hepatic and biliary medicine"}, "topic_3413": {"class_id": "http://edamontology.org/topic_3413", "definitions": "The branch of medicine that deals with the infectious diseases of the tropics. ", "label": "Infectious tropical disease"}, "data_2729": {"class_id": "http://edamontology.org/data_2729", "definitions": "Identifier of an EST sequence from the COGEME database.", "label": "COGEME EST ID"}, "data_2728": {"class_id": "http://edamontology.org/data_2728", "definitions": "Identifier of an EST sequence.", "label": "EST accession"}, "topic_3416": {"class_id": "http://edamontology.org/topic_3416", "definitions": "The branch of medicine that deals with the prevention, diagnosis, and treatment of disorders of the muscle, bone and connective tissue.  It incorporates aspects of orthopaedics, rheumatology, rehabilitation medicine and pain medicine.", "label": "Musculoskeletal medicine"}, "topic_3417": {"class_id": "http://edamontology.org/topic_3417", "definitions": "The branch of medicine that deals with disorders of the eye, including eyelid, optic nerve/visual pathways and occular muscles.", "label": "Opthalmology"}, "format_3710": {"class_id": "http://edamontology.org/format_3710", "definitions": "Mass spectrum file format from QSTAR and QTRAP instruments (ABI/Sciex).", "label": "WIFF format"}, "data_1269": {"class_id": "http://edamontology.org/data_1269", "definitions": "Annotation of a molecular sequence in DAS format.", "label": "DAS sequence feature annotation"}, "data_1268": {"class_id": "http://edamontology.org/data_1268", "definitions": "A table of amino acid word composition of a protein sequence.", "label": "Amino acid word frequencies table"}, "data_1265": {"class_id": "http://edamontology.org/data_1265", "definitions": "A table of base frequencies of a nucleotide sequence.", "label": "Base frequencies table"}, "data_1264": {"class_id": "http://edamontology.org/data_1264", "definitions": "A table of character or word composition / frequency of a molecular sequence.", "label": "Sequence composition table"}, "data_1267": {"class_id": "http://edamontology.org/data_1267", "definitions": "A table of amino acid frequencies of a protein sequence.", "label": "Amino acid frequencies table"}, "data_1266": {"class_id": "http://edamontology.org/data_1266", "definitions": "A table of word composition of a nucleotide sequence.", "label": "Base word frequencies table"}, "data_1261": {"class_id": "http://edamontology.org/data_1261", "definitions": "A report (typically a table) on character or word composition / frequency of a molecular sequence(s).", "label": "Sequence composition report"}, "data_1260": {"class_id": "http://edamontology.org/data_1260", "definitions": "A report on ambiguity in molecular sequence(s).", "label": "Sequence ambiguity report"}, "data_1263": {"class_id": "http://edamontology.org/data_1263", "definitions": "A plot of third base position variability in a nucleotide sequence.", "label": "Base position variability plot"}, "data_1262": {"class_id": "http://edamontology.org/data_1262", "definitions": "A report on peptide fragments of certain molecular weight(s) in one or more protein sequences.", "label": "Peptide molecular weight hits"}, "format_1457": {"class_id": "http://edamontology.org/format_1457", "definitions": "Format of RNA secondary structure in dot-bracket notation, originally generated by the Vienna RNA package/server.", "label": "Dot-bracket format"}, "format_1984": {"class_id": "http://edamontology.org/format_1984", "definitions": "Fasta format for (aligned) sequences.", "label": "FASTA-aln"}, "format_1455": {"class_id": "http://edamontology.org/format_1455", "definitions": "Entry format of the HSSP database (Homology-derived Secondary Structure in Proteins).", "label": "hssp"}, "format_1454": {"class_id": "http://edamontology.org/format_1454", "definitions": "Format of an entry from the DSSP database (Dictionary of Secondary Structure in Proteins).|The DSSP database is built using the DSSP application which defines secondary structure, geometrical features and solvent exposure of proteins, given atomic coordinates in PDB format.", "label": "dssp"}, "format_1981": {"class_id": "http://edamontology.org/format_1981", "definitions": "Genbank feature format.", "label": "GenBank feature"}, "format_1980": {"class_id": "http://edamontology.org/format_1980", "definitions": "EMBL feature format.", "label": "EMBL feature"}, "format_1983": {"class_id": "http://edamontology.org/format_1983", "definitions": "EMBOSS alignment format for debugging trace of full internal data content.", "label": "debug"}, "format_1982": {"class_id": "http://edamontology.org/format_1982", "definitions": "ClustalW format for (aligned) sequences.", "label": "ClustalW format"}, "format_1989": {"class_id": "http://edamontology.org/format_1989", "definitions": "Pearson MARKX3 alignment format.", "label": "markx3"}, "format_1988": {"class_id": "http://edamontology.org/format_1988", "definitions": "Pearson MARKX2 alignment format.", "label": "markx2"}, "format_1458": {"class_id": "http://edamontology.org/format_1458", "definitions": "Format of local RNA secondary structure components with free energy values, generated by the Vienna RNA package/server.", "label": "Vienna local RNA secondary structure format"}, "format_1985": {"class_id": "http://edamontology.org/format_1985", "definitions": "Pearson MARKX0 alignment format.", "label": "markx0"}, "operation_0308": {"class_id": "http://edamontology.org/operation_0308", "definitions": "Design or predict oligonucleotide primers for PCR and DNA amplification etc.|Primer design involves predicting or selecting primers that are specific to a provided PCR template. Primers can be designed with certain properties such as size of product desired, primer size etc. The output might be a minimal or overlapping primer set.|This includes predicting primers based on gene structure, promoters, exon-exon junctions, predicting primers that are conserved across multiple genomes or species, primers for for gene transcription profiling,  for genotyping polymorphisms, for example single nucleotide polymorphisms (SNPs),  for large scale sequencing, or for methylation PCRs.", "label": "PCR primer design"}, "format_1987": {"class_id": "http://edamontology.org/format_1987", "definitions": "Pearson MARKX10 alignment format.", "label": "markx10"}, "format_1986": {"class_id": "http://edamontology.org/format_1986", "definitions": "Pearson MARKX1 alignment format.", "label": "markx1"}, "operation_3203": {"class_id": "http://edamontology.org/operation_3203", "definitions": "Visualise, format or render an image of a Chromatogram.", "label": "Chromatogram visualisation"}, "operation_3202": {"class_id": "http://edamontology.org/operation_3202", "definitions": "Detect mutations in multiple DNA sequences, for example, from the alignment and comparison of the fluorescent traces produced by DNA sequencing hardware.", "label": "Polymorphism detection"}, "format_3006": {"class_id": "http://edamontology.org/format_3006", "definitions": "bigWig format for large sequence annotation tracks that consist of a value for each sequence position. Similar to textual WIG format.", "label": "bigWig"}, "format_3007": {"class_id": "http://edamontology.org/format_3007", "definitions": "PSL format of alignments, typically generated by BLAT or psLayout. Can be displayed in a genome browser like a sequence annotation track.", "label": "PSL"}, "format_3004": {"class_id": "http://edamontology.org/format_3004", "definitions": "bigBed format for large sequence annotation tracks, similar to textual BED format.", "label": "bigBed"}, "format_3005": {"class_id": "http://edamontology.org/format_3005", "definitions": "Wiggle format (WIG) of a sequence annotation track that consists of a value for each sequence position. Typically to be displayed in a genome browser.", "label": "WIG"}, "format_3003": {"class_id": "http://edamontology.org/format_3003", "definitions": "Browser Extensible Data (BED) format of sequence annotation track, typically to be displayed in a genome browser.|BED detail format includes 2 additional columns (http://genome.ucsc.edu/FAQ/FAQformat#format1.7) and BED 15 includes 3 additional columns for experiment scores (http://genomewiki.ucsc.edu/index.php/Microarray_track).", "label": "BED"}, "format_3000": {"class_id": "http://edamontology.org/format_3000", "definitions": "AB1 binary format of raw DNA sequence reads (output of Applied Biosystems' sequencing analysis software). Contains an electropherogram and the DNA base sequence.|AB1 uses the generic binary Applied Biosystems, Inc. Format (ABIF).", "label": "AB1"}, "format_3001": {"class_id": "http://edamontology.org/format_3001", "definitions": "ACE sequence assembly format including contigs, base-call qualities, and other metadata (version Aug 1998 and onwards).", "label": "ACE"}, "operation_3200": {"class_id": "http://edamontology.org/operation_3200", "definitions": "Analyse DNA sequences in order to identify a DNA 'barcode'; marker genes or any short fragment(s) of DNA that are useful to diagnose the taxa of biological organisms.", "label": "Community profiling"}, "format_3008": {"class_id": "http://edamontology.org/format_3008", "definitions": "Multiple Alignment Format (MAF) supporting alignments of whole genomes with rearrangements, directions, multiple pieces to the alignment, and so forth.|Typically generated by Multiz and TBA aligners; can be displayed in a genome browser like a sequence annotation track. This should not be confused with MIRA Assembly Format or Mutation Annotation Format.", "label": "MAF"}, "format_3009": {"class_id": "http://edamontology.org/format_3009", "definitions": "2bit binary format of nucleotide sequences using 2 bits per nucleotide. In addition encodes unknown nucleotides and lower-case 'masking'.", "label": "2bit"}, "operation_3209": {"class_id": "http://edamontology.org/operation_3209", "definitions": "Compare the sequence or features of two or more genomes, for example, to find matching regions.", "label": "Genome comparison"}, "operation_3208": {"class_id": "http://edamontology.org/operation_3208", "definitions": "Visualise, format or render a nucleic acid sequence that is part of (and in context of) a complete genome sequence.", "label": "Genome visualisation"}, "topic_0211": {"class_id": "http://edamontology.org/topic_0211", "definitions": "Information on a specific fly genome including molecular sequences, genes and annotation.", "label": "Flies"}, "topic_0177": {"class_id": "http://edamontology.org/topic_0177", "definitions": "The modelling the structure of proteins in complex with small molecules or other macromolecules.", "label": "Molecular docking"}, "data_1005": {"class_id": "http://edamontology.org/data_1005", "definitions": "3-letter code word for a ligand (HET group) from a PDB file, for example ATP.", "label": "HET group name"}, "data_1004": {"class_id": "http://edamontology.org/data_1004", "definitions": "Gmelin registry number of a chemical compound.", "label": "Chemical registry number (Gmelin)"}, "data_1007": {"class_id": "http://edamontology.org/data_1007", "definitions": "String of one or more ASCII characters representing a nucleotide.", "label": "Nucleotide code"}, "data_1006": {"class_id": "http://edamontology.org/data_1006", "definitions": "String of one or more ASCII characters representing an amino acid.", "label": "Amino acid name"}, "data_1001": {"class_id": "http://edamontology.org/data_1001", "definitions": "Synonymous name of a chemical compound.", "label": "Chemical name (synonymous)"}, "data_1000": {"class_id": "http://edamontology.org/data_1000", "definitions": "Brand name of a chemical compound.", "label": "Chemical name (brand)"}, "data_1003": {"class_id": "http://edamontology.org/data_1003", "definitions": "Beilstein registry number of a chemical compound.", "label": "Chemical registry number (Beilstein)"}, "data_1002": {"class_id": "http://edamontology.org/data_1002", "definitions": "CAS registry number of a chemical compound.", "label": "Chemical registry number (CAS)"}, "topic_3277": {"class_id": "http://edamontology.org/topic_3277", "definitions": "Biological samples and specimens.", "label": "Sample collections"}, "data_1659": {"class_id": "http://edamontology.org/data_1659", "definitions": "A report typically including a map (diagram) of a signal transduction pathway.", "label": "Signal transduction pathway report"}, "data_1009": {"class_id": "http://edamontology.org/data_1009", "definitions": "Name of a protein.", "label": "Protein name"}, "data_1008": {"class_id": "http://edamontology.org/data_1008", "definitions": "Identifier of a polypeptide chain from a protein.|This is typically a character (for the chain) appended to a PDB identifier, e.g. 1cukA", "label": "Polypeptide chain ID"}, "data_2299": {"class_id": "http://edamontology.org/data_2299", "definitions": "The name of a gene, (typically) assigned by a person and/or according to a naming scheme. It may contain white space characters and is typically more intuitive and readable than a gene symbol. It (typically) may be used to identify similar genes in different species and to derive a gene symbol.", "label": "Gene name"}, "data_2298": {"class_id": "http://edamontology.org/data_2298", "definitions": "Identifier for a gene approved by the HUGO Gene Nomenclature Committee.", "label": "Gene ID (HGNC)"}, "data_2295": {"class_id": "http://edamontology.org/data_2295", "definitions": "A unique (and typically persistent) identifier of a gene in a database, that is (typically) different to the gene name/symbol.", "label": "Gene ID"}, "data_2294": {"class_id": "http://edamontology.org/data_2294", "definitions": "An identifier of an entry from a database of molecular sequence variation.", "label": "Sequence variation ID"}, "data_2297": {"class_id": "http://edamontology.org/data_2297", "definitions": "Identifier of an E. coli K-12 gene from EcoGene Database.", "label": "Gene ID (ECK)"}, "data_2296": {"class_id": "http://edamontology.org/data_2296", "definitions": "Name of an entry (gene) from the AceView genes database.", "label": "Gene name (AceView)"}, "data_2927": {"class_id": "http://edamontology.org/data_2927", "definitions": "Data concerning codon usage.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Codon usage"}, "data_2290": {"class_id": "http://edamontology.org/data_2290", "definitions": "An accession number of an entry from the EMBL sequence database.", "label": "EMBL accession"}, "data_2925": {"class_id": "http://edamontology.org/data_2925", "definitions": "Data concerning, extracted from, or derived from the analysis of molecular sequence(s). |This is a broad data type and is used a placeholder for other, more specific types.", "label": "Sequence data"}, "data_2292": {"class_id": "http://edamontology.org/data_2292", "definitions": "Accession number of an entry from the GenBank sequence database.", "label": "GenBank accession"}, "operation_3189": {"class_id": "http://edamontology.org/operation_3189", "definitions": "Trim sequences (typically from an automated DNA sequencer) to remove misleading ends.|For example trim polyA tails, introns and primer sequence flanking the sequence of amplified exons, or other unwanted sequence.", "label": "Trim ends"}, "operation_0288": {"class_id": "http://edamontology.org/operation_0288", "definitions": "Find exact character or word matches between molecular sequences without full sequence alignment.", "label": "Sequence word comparison"}, "data_1719": {"class_id": "http://edamontology.org/data_1719", "definitions": "A term from the NCBI taxonomy vocabulary.", "label": "NCBI taxonomy vocabulary"}, "data_1718": {"class_id": "http://edamontology.org/data_1718", "definitions": "A term from the HGNC controlled vocabulary.", "label": "HGNC"}, "operation_0289": {"class_id": "http://edamontology.org/operation_0289", "definitions": "Calculate a sequence distance matrix or otherwise estimate genetic distances between molecular sequences.", "label": "Sequence distance matrix generation"}, "data_1711": {"class_id": "http://edamontology.org/data_1711", "definitions": "Image of two or more aligned molecular sequences possibly annotated with alignment features.", "label": "Sequence alignment image"}, "data_1710": {"class_id": "http://edamontology.org/data_1710", "definitions": "Image of one or more molecular tertiary (3D) structures.", "label": "Structure image"}, "data_1713": {"class_id": "http://edamontology.org/data_1713", "definitions": " A fate map is a plan of early stage of an embryo such as a blastula, showing areas that are significance to development.", "label": "Fate map"}, "data_1712": {"class_id": "http://edamontology.org/data_1712", "definitions": "An image of the structure of a small chemical compound.|The molecular identifier and formula are typically included.", "label": "Chemical structure image"}, "data_1715": {"class_id": "http://edamontology.org/data_1715", "definitions": "A term from the BioPax ontology.", "label": "BioPax term"}, "data_1714": {"class_id": "http://edamontology.org/data_1714", "definitions": "An image of spots from a microarray experiment.", "label": "Microarray spots image"}, "data_1717": {"class_id": "http://edamontology.org/data_1717", "definitions": "A term from the MeSH vocabulary.", "label": "MeSH"}, "data_1716": {"class_id": "http://edamontology.org/data_1716", "definitions": "A term definition from The Gene Ontology (GO).", "label": "GO"}, "topic_0090": {"class_id": "http://edamontology.org/topic_0090", "definitions": "The search and query of data sources (typically databases or ontologies) in order to retrieve entries or other information.", "label": "Information retrieval"}, "data_3428": {"class_id": "http://edamontology.org/data_3428", "definitions": "biological computational model experiments (simulation), for example the minimum information required in order to permit its correct interpretation and reproduction.", "label": "Simulation experiment report"}, "topic_2258": {"class_id": "http://edamontology.org/topic_2258", "definitions": "The application of information technology to chemistry in biological research environment.", "label": "Cheminformatics"}, "format_2532": {"class_id": "http://edamontology.org/format_2532", "definitions": "Genbank entry format wrapped in HTML elements.", "label": "GenBank-HTML"}, "data_1474": {"class_id": "http://edamontology.org/data_1474", "definitions": "3D coordinate and associated data for a protein domain tertiary (3D) structure (typically C-alpha atoms only).|C-beta atoms from amino acid side-chains may be included.", "label": "Protein domain (C-alpha atoms)"}, "format_3591": {"class_id": "http://edamontology.org/format_3591", "definitions": "A versatile bitmap format.|The TIFF format is perhaps the most versatile and diverse bitmap format in existence. Its extensible nature and support for numerous data compression schemes allow developers to customize the TIFF format to fit any peculiar data storage needs.", "label": "tiff"}, "format_3592": {"class_id": "http://edamontology.org/format_3592", "definitions": "Standard bitmap storage format in the Microsoft Windows environment.|Although it is based on Windows internal bitmap data structures, it is supported by many non-Windows and non-PC applications.", "label": "bmp"}, "format_3593": {"class_id": "http://edamontology.org/format_3593", "definitions": "IM is a format used by LabEye and other applications based on the IFUNC image processing library.|IFUNC library reads and writes most uncompressed interchange versions of this format.", "label": "im"}, "data_1470": {"class_id": "http://edamontology.org/data_1470", "definitions": "3D coordinate and associated data for a protein tertiary (3D) structure (typically C-alpha atoms only).|C-beta atoms from amino acid side-chains may be included.", "label": "C-alpha trace"}, "data_1471": {"class_id": "http://edamontology.org/data_1471", "definitions": "3D coordinate and associated data for a polypeptide chain tertiary (3D) structure (all atoms).", "label": "Protein chain (all atoms)"}, "data_1472": {"class_id": "http://edamontology.org/data_1472", "definitions": "3D coordinate and associated data for a polypeptide chain tertiary (3D) structure (typically C-alpha atoms only).|C-beta atoms from amino acid side-chains may be included.", "label": "Protein chain (C-alpha atoms)"}, "data_1473": {"class_id": "http://edamontology.org/data_1473", "definitions": "3D coordinate and associated data for a protein domain tertiary (3D) structure (all atoms).", "label": "Protein domain (all atoms)"}, "format_3598": {"class_id": "http://edamontology.org/format_3598", "definitions": "X BitMap is a plain text binary image format used by the X Window System used for storing cursor and icon bitmaps used in the X GUI. |The XBM format was replaced by XPM for X11 in 1989.", "label": "xbm"}, "format_3599": {"class_id": "http://edamontology.org/format_3599", "definitions": "X PixMap (XPM) is an image file format used by the X Window System, it is intended primarily for creating icon pixmaps, and supports transparent pixels.|Sequence of segments with markers. Begins with byte of 0xFF and follows by marker type.", "label": "xpm"}, "topic_0618": {"class_id": "http://edamontology.org/topic_0618", "definitions": "A database about scents.", "label": "Scents"}, "data_1479": {"class_id": "http://edamontology.org/data_1479", "definitions": "Alignment (superimposition) of exactly two molecular tertiary (3D) structures.", "label": "Structure alignment (pair)"}, "data_1904": {"class_id": "http://edamontology.org/data_1904", "definitions": "Identifier of a locus from EntrezGene database.", "label": "Locus ID (EntrezGene)"}, "data_1673": {"class_id": "http://edamontology.org/data_1673", "definitions": "Cross-mapping of Swiss-Prot codes to PDB identifiers.", "label": "Swiss-Prot to PDB mapping"}, "data_1906": {"class_id": "http://edamontology.org/data_1906", "definitions": "A stretch of DNA that is closely linked to the genes underlying a quantitative trait (a phenotype that varies in degree and depends upon the interactions between multiple genes and their environment).|A QTL sometimes but does not necessarily correspond to a gene.", "label": "Quantitative trait locus"}, "data_1907": {"class_id": "http://edamontology.org/data_1907", "definitions": "Identifier of a gene from the KOME database.", "label": "Gene ID (KOME)"}, "data_1900": {"class_id": "http://edamontology.org/data_1900", "definitions": "Identifier for loci from NCBI database.", "label": "NCBI locus tag"}, "data_1901": {"class_id": "http://edamontology.org/data_1901", "definitions": "Identifier for loci from SGD (Saccharomyces Genome Database).", "label": "Locus ID (SGD)"}, "data_1902": {"class_id": "http://edamontology.org/data_1902", "definitions": "Identifier of loci from Maize Mapping Project.", "label": "Locus ID (MMP)"}, "data_1903": {"class_id": "http://edamontology.org/data_1903", "definitions": "Identifier of locus from DictyBase (Dictyostelium discoideum).", "label": "Locus ID (DictyBase)"}, "data_1678": {"class_id": "http://edamontology.org/data_1678", "definitions": "A report of tool-specific metadata on some analysis or process performed, for example a log of diagnostic or error messages.", "label": "Tool log"}, "data_1679": {"class_id": "http://edamontology.org/data_1679", "definitions": "DaliLite log file describing all the steps taken by a DaliLite alignment of two protein structures.", "label": "DaliLite log file"}, "data_1908": {"class_id": "http://edamontology.org/data_1908", "definitions": "Identifier of a locus from the Tropgene database.", "label": "Locus ID (Tropgene)"}, "data_2959": {"class_id": "http://edamontology.org/data_2959", "definitions": "A probability profile of a double-stranded nucleic acid molecule (DNA or DNA/RNA).|Shows the probability of a base pair not being melted (i.e. remaining as double-stranded DNA) at a specified temperature", "label": "Nucleic acid probability profile"}, "format_3309": {"class_id": "http://edamontology.org/format_3309", "definitions": "File format of a CT (Connectivity Table) file from the RNAstructure package.", "label": "CT"}, "topic_0779": {"class_id": "http://edamontology.org/topic_0779", "definitions": "Mitochondria, typically of mitochondrial genes and proteins.", "label": "Mitochondria"}, "data_3424": {"class_id": "http://edamontology.org/data_3424", "definitions": "Raw biological or biomedical image generated by some experimental technique.", "label": "Raw image"}, "data_3427": {"class_id": "http://edamontology.org/data_3427", "definitions": "RNAi experiments.", "label": "RNAi report"}, "data_3426": {"class_id": "http://edamontology.org/data_3426", "definitions": "Report concerning proteomics experiments.", "label": "Proteomics experiment report"}, "data_1878": {"class_id": "http://edamontology.org/data_1878", "definitions": "A common misspelling of a word.", "label": "Misspelling"}, "data_1879": {"class_id": "http://edamontology.org/data_1879", "definitions": "An abbreviation of a phrase or word.", "label": "Acronym"}, "data_1870": {"class_id": "http://edamontology.org/data_1870", "definitions": "The name of a genus of organism.", "label": "Genus name"}, "data_2220": {"class_id": "http://edamontology.org/data_2220", "definitions": "Unique identifier of a sequence cluster from the SYSTERS database.", "label": "Sequence cluster ID (SYSTERS)"}, "data_1872": {"class_id": "http://edamontology.org/data_1872", "definitions": "The full name for a group of organisms, reflecting their biological classification and (usually) conforming to a standard nomenclature.|Name components correspond to levels in a taxonomic hierarchy (e.g. 'Genus', 'Species', etc.) Meta information such as a reference where the name was defined and a date might be included.", "label": "Taxonomic classification"}, "data_1873": {"class_id": "http://edamontology.org/data_1873", "definitions": "A unique identifier for an organism used in the iHOP database.", "label": "iHOP organism ID"}, "data_1874": {"class_id": "http://edamontology.org/data_1874", "definitions": "Common name for an organism as used in the GenBank database.", "label": "Genbank common name"}, "data_1875": {"class_id": "http://edamontology.org/data_1875", "definitions": "The name of a taxon from the NCBI taxonomy database.", "label": "NCBI taxon"}, "data_1877": {"class_id": "http://edamontology.org/data_1877", "definitions": "An alternative for a word.", "label": "Synonym"}, "data_2389": {"class_id": "http://edamontology.org/data_2389", "definitions": "Identifier of an entry from the UniSTS database.", "label": "UniSTS accession"}, "data_0879": {"class_id": "http://edamontology.org/data_0879", "definitions": "An informative report on protein secondary structure alignment-derived data or metadata.", "label": "Secondary structure alignment metadata (protein)"}, "data_0878": {"class_id": "http://edamontology.org/data_0878", "definitions": "Alignment of the (1D representations of) secondary structure of two or more proteins.", "label": "Secondary structure alignment (protein)"}, "data_0877": {"class_id": "http://edamontology.org/data_0877", "definitions": "Super-secondary structure of protein sequence(s).|Super-secondary structures include leucine zippers, coiled coils, Helix-Turn-Helix etc.", "label": "Protein features report (super-secondary)"}, "data_0876": {"class_id": "http://edamontology.org/data_0876", "definitions": "Secondary structure (predicted or real) of a protein.", "label": "Protein features report (secondary structure)"}, "data_0875": {"class_id": "http://edamontology.org/data_0875", "definitions": "Predicted or actual protein topology represented as a string of protein secondary structure elements.|The location and size of the secondary structure elements and intervening loop regions is usually indicated.", "label": "Protein topology"}, "data_2388": {"class_id": "http://edamontology.org/data_2388", "definitions": "Identifier of an Arabidopsis thaliana gene from the TAIR database.", "label": "TAIR accession (At gene)"}, "data_0872": {"class_id": "http://edamontology.org/data_0872", "definitions": "The raw data (not just an image) from which a phylogenetic tree is directly generated or plotted, such as topology, lengths (in time or in expected amounts of variance) and a confidence interval for each length.|A phylogenetic tree is usually constructed from a set of sequences from which an alignment (or data matrix) is calculated. See also 'Phylogenetic tree image'.", "label": "Phylogenetic tree"}, "data_0871": {"class_id": "http://edamontology.org/data_0871", "definitions": "Basic character data from which a phylogenetic tree may be generated.|As defined, this concept would also include molecular sequences, microsatellites, polymorphisms (RAPDs, RFLPs, or AFLPs), restriction sites and fragments", "label": "Phylogenetic character data"}, "data_0870": {"class_id": "http://edamontology.org/data_0870", "definitions": "A matrix of estimated evolutionary distance between molecular sequences, such as is suitable for phylogenetic tree calculation.|Methods might perform character compatibility analysis or identify patterns of similarity in an alignment or data matrix.", "label": "Sequence distance matrix"}, "operation_0530": {"class_id": "http://edamontology.org/operation_0530", "definitions": "Process (read and / or write) sequencing by synthesis (SBS) data.", "label": "SBS data processing"}, "operation_0531": {"class_id": "http://edamontology.org/operation_0531", "definitions": "Generate a heat map of expression data from e.g. microarray data.|The heat map usually uses a coloring scheme to represent expression values. They can show how quantitative measurements were influenced by experimental conditions.", "label": "Heat map generation"}, "operation_0532": {"class_id": "http://edamontology.org/operation_0532", "definitions": "Analyse one or more gene expression profiles, typically to interpret them in functional terms.", "label": "Gene expression profile analysis"}, "operation_0533": {"class_id": "http://edamontology.org/operation_0533", "definitions": "Map a gene expression profile to known biological pathways, for example, to identify or reconstruct a pathway.", "label": "Gene to pathway mapping"}, "operation_0534": {"class_id": "http://edamontology.org/operation_0534", "definitions": "Assign secondary structure from protein coordinate data.", "label": "Protein secondary structure assignment (from coordinate data)"}, "operation_0535": {"class_id": "http://edamontology.org/operation_0535", "definitions": "Assign secondary structure from circular dichroism (CD) spectroscopic data.", "label": "Protein secondary structure assignment (from CD data)"}, "operation_0536": {"class_id": "http://edamontology.org/operation_0536", "definitions": "Assign a protein tertiary structure (3D coordinates) from raw X-ray crystallography data.", "label": "Protein structure assignment (from X-ray crystallographic data)"}, "operation_0537": {"class_id": "http://edamontology.org/operation_0537", "definitions": "Assign a protein tertiary structure (3D coordinates) from raw NMR spectroscopy data.", "label": "Protein structure assignment (from NMR data)"}, "operation_0538": {"class_id": "http://edamontology.org/operation_0538", "definitions": "Construct a phylogenetic tree from a specific type of data.", "label": "Phylogenetic tree generation (data centric)"}, "operation_0539": {"class_id": "http://edamontology.org/operation_0539", "definitions": "Construct a phylogenetic tree using a specific method.", "label": "Phylogenetic tree generation (method centric)"}, "operation_1841": {"class_id": "http://edamontology.org/operation_1841", "definitions": "Predict rotamer likelihoods for all 20 amino acid types at each position in a protein structure.|Output typically includes, for each residue position, the likelihoods for the 20 amino acid types with estimated reliability of the 20 likelihoods.", "label": "Rotamer likelihood prediction"}, "operation_1847": {"class_id": "http://edamontology.org/operation_1847", "definitions": "Determine for residue the DSSP determined secondary structure in three-state (HSC).", "label": "DSSP secondary structure assignment"}, "operation_1846": {"class_id": "http://edamontology.org/operation_1846", "definitions": "Identify HET groups in PDB files.|A HET group usually corresponds to ligands, lipids, but might also (not consistently) include groups that are attached to amino acids. Each HET group is supposed to have a unique three letter code and a unique name which might be given in the output.", "label": "HET group detection"}, "operation_1845": {"class_id": "http://edamontology.org/operation_1845", "definitions": "Extract a molecular sequence from a PDB file.", "label": "PDB file sequence retrieval"}, "operation_1844": {"class_id": "http://edamontology.org/operation_1844", "definitions": "Validate protein geometry, for example bond lengths, bond angles, torsion angles, chiralities, planaraties etc.", "label": "Protein geometry validation"}, "data_2118": {"class_id": "http://edamontology.org/data_2118", "definitions": "An identifier of a software end-user (typically a person).", "label": "Person identifier"}, "data_2119": {"class_id": "http://edamontology.org/data_2119", "definitions": "Name or other identifier of a nucleic acid molecule.", "label": "Nucleic acid identifier"}, "data_2116": {"class_id": "http://edamontology.org/data_2116", "definitions": "An informative report on a trinucleotide sequence that encodes an amino acid including the triplet sequence, the encoded amino acid or whether it is a start or stop codon.", "label": "Nucleic acid features (codon)"}, "data_2117": {"class_id": "http://edamontology.org/data_2117", "definitions": "An identifier of a map of a molecular sequence.", "label": "Map identifier"}, "data_2114": {"class_id": "http://edamontology.org/data_2114", "definitions": "Protein identifier used by WormBase database.", "label": "WormBase wormpep ID"}, "data_2112": {"class_id": "http://edamontology.org/data_2112", "definitions": "Primary identifier of an object from the FlyBase database.", "label": "FlyBase primary identifier"}, "data_2113": {"class_id": "http://edamontology.org/data_2113", "definitions": "Identifier of an object from the WormBase database.", "label": "WormBase identifier"}, "data_2110": {"class_id": "http://edamontology.org/data_2110", "definitions": "Identifier of a molecular property.", "label": "Molecular property identifier"}, "data_2111": {"class_id": "http://edamontology.org/data_2111", "definitions": "Identifier of a codon usage table, for example a genetic code.", "label": "Codon usage table ID"}, "format_2243": {"class_id": "http://edamontology.org/format_2243", "definitions": "PHYLIP file format for phylogenetic property data.", "label": "phylip property values"}, "topic_0148": {"class_id": "http://edamontology.org/topic_0148", "definitions": "Protein-ligand (small molecule) interactions.", "label": "Protein-ligand interactions"}, "operation_0224": {"class_id": "http://edamontology.org/operation_0224", "definitions": "Search or query a data resource and retrieve entries and / or annotation.", "label": "Query and retrieval"}, "operation_0225": {"class_id": "http://edamontology.org/operation_0225", "definitions": "Search database to retrieve all relevant references to a particular entity or entry.", "label": "Data retrieval (database cross-reference)"}, "data_1498": {"class_id": "http://edamontology.org/data_1498", "definitions": "A measure of the similarity between two ligand fingerprints.|A ligand fingerprint is derived from ligand structural data from a Protein DataBank file. It reflects the elements or groups present or absent, covalent bonds and bond orders and the bonded environment in terms of SATIS codes and BLEEP atom types.", "label": "Tanimoto similarity score"}, "data_1499": {"class_id": "http://edamontology.org/data_1499", "definitions": "A matrix of 3D-1D scores reflecting the probability of amino acids to occur in different tertiary structural environments.", "label": "3D-1D scoring matrix"}, "data_1492": {"class_id": "http://edamontology.org/data_1492", "definitions": "Alignment (superimposition) of more than two nucleic acid tertiary (3D) structures.", "label": "Multiple nucleic acid tertiary structure alignment"}, "data_1493": {"class_id": "http://edamontology.org/data_1493", "definitions": "Alignment (superimposition) of RNA tertiary (3D) structures.", "label": "Structure alignment (RNA)"}, "data_1490": {"class_id": "http://edamontology.org/data_1490", "definitions": "Alignment (superimposition) of exactly two protein tertiary (3D) structures (typically C-alpha atoms only considered).|C-beta atoms from amino acid side-chains may be included.", "label": "Multiple protein tertiary structure alignment (C-alpha atoms)"}, "data_1491": {"class_id": "http://edamontology.org/data_1491", "definitions": "Alignment (superimposition) of exactly two nucleic acid tertiary (3D) structures.", "label": "Structure alignment (nucleic acid pair)"}, "data_1496": {"class_id": "http://edamontology.org/data_1496", "definitions": "A score reflecting structural similarities of two molecules.", "label": "Molecular similarity score"}, "data_1497": {"class_id": "http://edamontology.org/data_1497", "definitions": "Root-mean-square deviation (RMSD) is calculated to measure the average distance between superimposed macromolecular coordinates.", "label": "Root-mean-square deviation"}, "data_1494": {"class_id": "http://edamontology.org/data_1494", "definitions": "Matrix to transform (rotate/translate) 3D coordinates, typically the transformation necessary to superimpose two molecular structures.", "label": "Structural transformation matrix"}, "data_1495": {"class_id": "http://edamontology.org/data_1495", "definitions": "DaliLite hit table of protein chain tertiary structure alignment data.|The significant and top-scoring hits for regions of the compared structures is shown. Data such as Z-Scores, number of aligned residues, root-mean-square deviation (RMSD) of atoms and sequence identity are given.", "label": "DaliLite hit table"}, "operation_3755": {"class_id": "http://edamontology.org/operation_3755", "definitions": "Site localisation of post-translational modifications in peptide or protein mass spectra.", "label": "PTM localisation"}, "operation_0408": {"class_id": "http://edamontology.org/operation_0408", "definitions": "Predict the stability or globularity of a protein sequence, whether it is intrinsically unfolded etc.", "label": "Protein globularity prediction"}, "operation_0409": {"class_id": "http://edamontology.org/operation_0409", "definitions": "Predict the solubility or atomic solvation energy of a protein sequence.", "label": "Protein solubility prediction"}, "topic_3399": {"class_id": "http://edamontology.org/topic_3399", "definitions": "The branch of medicine dealing with the diagnosis, treatment and prevention of disease in older people, and the problems specific to aging.", "label": "Geriatric medicine"}, "topic_3398": {"class_id": "http://edamontology.org/topic_3398", "definitions": "The application of biological concepts and methods to the analytical and synthetic methodologies of engineering.", "label": "Bioengineering"}, "operation_0404": {"class_id": "http://edamontology.org/operation_0404", "definitions": "Estimate hydrogen exchange rate of a protein sequence.", "label": "Protein hydrogen exchange rate calculation"}, "operation_0405": {"class_id": "http://edamontology.org/operation_0405", "definitions": "Calculate hydrophobic or hydrophilic / charged regions of a protein sequence.", "label": "Protein hydrophobic region calculation"}, "operation_0406": {"class_id": "http://edamontology.org/operation_0406", "definitions": "Calculate aliphatic index (relative volume occupied by aliphatic side chains) of a protein.", "label": "Protein aliphatic index calculation"}, "operation_0407": {"class_id": "http://edamontology.org/operation_0407", "definitions": "Calculate the hydrophobic moment of a peptide sequence and recognize amphiphilicity.|Hydrophobic moment is a peptides hydrophobicity measured for different angles of rotation.", "label": "Protein hydrophobic moment plotting"}, "operation_0400": {"class_id": "http://edamontology.org/operation_0400", "definitions": "Calculate pH-dependent properties from pKa calculations of a protein sequence.", "label": "Protein pKa calculation"}, "operation_0401": {"class_id": "http://edamontology.org/operation_0401", "definitions": "Hydropathy calculation on a protein sequence.", "label": "Protein hydropathy calculation (from sequence)"}, "operation_0402": {"class_id": "http://edamontology.org/operation_0402", "definitions": "Plot a protein titration curve.", "label": "Protein titration curve plotting"}, "operation_0403": {"class_id": "http://edamontology.org/operation_0403", "definitions": "Calculate isoelectric point of a protein sequence.", "label": "Protein isoelectric point calculation"}, "data_3739": {"class_id": "http://edamontology.org/data_3739", "definitions": "The total species diversity in a landscape. ", "label": "Gamma diversity data"}, "data_3738": {"class_id": "http://edamontology.org/data_3738", "definitions": "The ratio between regional and local species diversity.", "label": "Beta diversity data"}, "data_3735": {"class_id": "http://edamontology.org/data_3735", "definitions": "A number corresponding to the number of an analysis performed by a sequencing machine.  For example, if it's the 13th analysis, the run is 13.", "label": "Run number"}, "data_3734": {"class_id": "http://edamontology.org/data_3734", "definitions": "An identifier of a lane within a flow cell of a sequencing machine, within which millions of sequences are immobilised, amplified and sequenced.", "label": "Lane identifier"}, "data_3737": {"class_id": "http://edamontology.org/data_3737", "definitions": "The mean species diversity in sites or habitats at a local scale.", "label": "Alpha diversity data"}, "data_3736": {"class_id": "http://edamontology.org/data_3736", "definitions": "Data concerning ecology; for example measurements and reports from the study of interactions among organisms and their environment.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Ecological data"}, "data_3733": {"class_id": "http://edamontology.org/data_3733", "definitions": "An identifier of a flow cell of a sequencing machine.|A flow cell is used to immobilise, amplify and sequence millions of molecules at once.  In Illumina machines, a flowcell is composed of  8 \"lanes\" which allows 8 experiments in a single analysis.", "label": "Flow cell identifier"}, "data_3732": {"class_id": "http://edamontology.org/data_3732", "definitions": "Data concerning a sequencing experiment, that may be specified as an input to some tool.", "label": "Sequencing metadata name"}, "data_2602": {"class_id": "http://edamontology.org/data_2602", "definitions": "Data concerning a particular genotype, phenotype or a genotype / phenotype relation.", "label": "Genotype and phenotype data"}, "data_2603": {"class_id": "http://edamontology.org/data_2603", "definitions": "Image, hybridisation or some other data arising from a study of gene expression, typically profiling or quantification.", "label": "Gene expression data"}, "data_2600": {"class_id": "http://edamontology.org/data_2600", "definitions": "Primary data about a specific biological pathway or network (the nodes and connections within the pathway or network).", "label": "Pathway or network"}, "data_2601": {"class_id": "http://edamontology.org/data_2601", "definitions": "Data concerning one or more small molecules.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Small molecule data"}, "data_2606": {"class_id": "http://edamontology.org/data_2606", "definitions": "Name (not necessarily stable) an entry (RNA family) from the RFAM database.", "label": "RFAM name"}, "data_2605": {"class_id": "http://edamontology.org/data_2605", "definitions": "Unique identifier of a chemical compound from the KEGG database.", "label": "Compound ID (KEGG)"}, "data_2608": {"class_id": "http://edamontology.org/data_2608", "definitions": "Identifier of a biological reaction from the KEGG reactions database.", "label": "Reaction ID (KEGG)"}, "data_2609": {"class_id": "http://edamontology.org/data_2609", "definitions": "Unique identifier of a drug from the KEGG Drug database.", "label": "Drug ID (KEGG)"}, "operation_3629": {"class_id": "http://edamontology.org/operation_3629", "definitions": "The removal of isotope peaks in a spectrum, to represent the fragment ion as one data point.|Deisotoping is commonly done to reduce complexity, and done in conjunction with the charge state deconvolution.", "label": "Deisotoping"}, "operation_3628": {"class_id": "http://edamontology.org/operation_3628", "definitions": "Align multiple data sets using information from chromatography and/or peptide identification, from mass spectrometry experiments.", "label": "Chromatographic alignment"}, "operation_3625": {"class_id": "http://edamontology.org/operation_3625", "definitions": "Identify semantic relations among entities and concepts within a text, using text mining techniques.", "label": "Relation inference"}, "operation_3627": {"class_id": "http://edamontology.org/operation_3627", "definitions": "Re-adjust the output of mass spectrometry experiments with shifted ppm values.", "label": "Mass spectra calibration"}, "topic_3573": {"class_id": "http://edamontology.org/topic_3573", "definitions": "The study of organisms in freshwater ecosystems.", "label": "Freshwater biology"}, "topic_3572": {"class_id": "http://edamontology.org/topic_3572", "definitions": "The quality, integrity, and cleaning up of data. ", "label": "Data quality management"}, "operation_2574": {"class_id": "http://edamontology.org/operation_2574", "definitions": "Analyse the hydrophobic, hydrophilic or charge properties of a protein (from analysis of sequence or structural information).", "label": "Protein hydropathy calculation"}, "operation_2575": {"class_id": "http://edamontology.org/operation_2575", "definitions": "Identify or predict catalytic residues, active sites or other ligand-binding sites in protein sequences or structures.", "label": "Protein binding site prediction"}, "topic_3577": {"class_id": "http://edamontology.org/topic_3577", "definitions": "An approach to medicine whereby decisions, practices and are tailored to the individual patient based on their predicted response or risk of disease.", "label": "Personalised medicine"}, "topic_3576": {"class_id": "http://edamontology.org/topic_3576", "definitions": "Biotechnology applied to the medical sciences and the development of medicines.", "label": "Medical biotechnology"}, "topic_3575": {"class_id": "http://edamontology.org/topic_3575", "definitions": "Health problems that are prevalent in tropical and subtropical regions.", "label": "Tropical medicine"}, "topic_3574": {"class_id": "http://edamontology.org/topic_3574", "definitions": "The study of inheritatnce in human beings.", "label": "Human genetics"}, "format_1620": {"class_id": "http://edamontology.org/format_1620", "definitions": "Entry format for the dbSNP database.", "label": "dbSNP polymorphism report format"}, "format_1623": {"class_id": "http://edamontology.org/format_1623", "definitions": "Format of an entry from the OMIM database of genotypes and phenotypes.", "label": "OMIM entry format"}, "format_1196": {"class_id": "http://edamontology.org/format_1196", "definitions": "Chemical structure specified in Simplified Molecular Input Line Entry System (SMILES) line notation.", "label": "SMILES"}, "data_3086": {"class_id": "http://edamontology.org/data_3086", "definitions": "A report (typically a table) on character or word composition / frequency of nucleic acid sequence(s).", "label": "Nucleic acid sequence composition (report)"}, "data_3085": {"class_id": "http://edamontology.org/data_3085", "definitions": "A report (typically a table) on character or word composition / frequency of protein sequence(s).", "label": "Protein sequence composition"}, "format_1627": {"class_id": "http://edamontology.org/format_1627", "definitions": "Report format on PCR primers and hybridisation oligos as generated by Whitehead primer3 program.", "label": "Primer3 primer"}, "format_1628": {"class_id": "http://edamontology.org/format_1628", "definitions": "A format of raw sequence read data from an Applied Biosystems sequencing machine.", "label": "ABI"}, "format_1629": {"class_id": "http://edamontology.org/format_1629", "definitions": "Format of MIRA sequence trace information file.", "label": "mira"}, "format_1198": {"class_id": "http://edamontology.org/format_1198", "definitions": "Chemical structure specified by Molecular Formula (MF), including a count of each element in a compound.|The general MF query format consists of a series of valid atomic symbols, with an optional number or range.", "label": "mf"}, "format_1199": {"class_id": "http://edamontology.org/format_1199", "definitions": "The InChIKey (hashed InChI) is a fixed length (25 character) condensed digital representation of an InChI chemical structure specification. It uniquely identifies a chemical compound.|An InChIKey identifier is not human- nor machine-readable but is more suitable for web searches than an InChI chemical structure specification.", "label": "InChIKey"}, "operation_0456": {"class_id": "http://edamontology.org/operation_0456", "definitions": "Calculate and plot a DNA or DNA/RNA melting profile.|A melting profile is used to visualise and analyse partly melted DNA conformations.", "label": "Nucleic acid melting profile plotting"}, "data_2736": {"class_id": "http://edamontology.org/data_2736", "definitions": "A feature identifier as used in the SwissRegulon database.|This can be name of a gene, the ID of a TFBS, or genomic coordinates in form \"chr:start..end\".", "label": "Sequence feature ID (SwissRegulon)"}, "data_2737": {"class_id": "http://edamontology.org/data_2737", "definitions": "A unique identifier of gene in the NMPDR database.|A FIG ID consists of four parts: a prefix, genome id, locus type and id number.", "label": "FIG ID"}, "data_2734": {"class_id": "http://edamontology.org/data_2734", "definitions": "The name of a family of viruses.", "label": "Family name (virus)"}, "data_2735": {"class_id": "http://edamontology.org/data_2735", "definitions": "The name of a SwissRegulon database.", "label": "Database name (SwissRegulon)"}, "data_2732": {"class_id": "http://edamontology.org/data_2732", "definitions": "The name of a family of organism.", "label": "Family name"}, "data_2733": {"class_id": "http://edamontology.org/data_2733", "definitions": "The name of a genus of viruses.", "label": "Genus name (virus)"}, "data_2730": {"class_id": "http://edamontology.org/data_2730", "definitions": "Identifier of a unisequence from the COGEME database.|A unisequence is a single sequence assembled from ESTs.", "label": "COGEME unisequence ID"}, "data_2731": {"class_id": "http://edamontology.org/data_2731", "definitions": "Accession number of an entry (family) from the TIGRFam database.", "label": "Protein family ID (GeneFarm)"}, "topic_3372": {"class_id": "http://edamontology.org/topic_3372", "definitions": "The process that leads from an original formulation of a computing problem to executable programs.", "label": "Software engineering"}, "topic_3406": {"class_id": "http://edamontology.org/topic_3406", "definitions": "The branch of medicine that deals with the prevention, diagnosis, and treatment of disorders of the ear, nose and throat.", "label": "Ear, nose and throat medicine"}, "operation_3199": {"class_id": "http://edamontology.org/operation_3199", "definitions": "A varient of oligonucleotide mapping where a read is mapped to two separate locations because of possible structural variation.", "label": "Split read mapping"}, "operation_3198": {"class_id": "http://edamontology.org/operation_3198", "definitions": "Align short oligonucleotide sequences (reads) to a larger (genomic) sequence.|The purpose of read mapping is to identify the location of sequenced fragments within a reference genome and assumes that there is, in fact, at least local similarity between the fragment and reference sequences.", "label": "Read mapping"}, "topic_3403": {"class_id": "http://edamontology.org/topic_3403", "definitions": "The multidisciplinary that cares for patients with acute, life-threatening illness or injury.", "label": "Critical care medicine"}, "topic_3402": {"class_id": "http://edamontology.org/topic_3402", "definitions": "Anaesthesia and anaesthetics.", "label": "Anaesthesiology"}, "data_2738": {"class_id": "http://edamontology.org/data_2738", "definitions": "A unique identifier of gene in the Xenbase database.", "label": "Gene ID (Xenbase)"}, "data_2739": {"class_id": "http://edamontology.org/data_2739", "definitions": "A unique identifier of gene in the Genolist database.", "label": "Gene ID (Genolist)"}, "format_1998": {"class_id": "http://edamontology.org/format_1998", "definitions": "Phylip non-interleaved format for (aligned) sequences.", "label": "phylipnon"}, "format_1999": {"class_id": "http://edamontology.org/format_1999", "definitions": "Alignment format for score values for pairs of sequences.", "label": "scores format"}, "data_1278": {"class_id": "http://edamontology.org/data_1278", "definitions": "A map showing the relative positions of genetic markers in a nucleic acid sequence, based on estimation of non-physical distance such as recombination frequencies.|A genetic (linkage) map indicates the proximity of two genes on a chromosome, whether two genes are linked and the frequency they are transmitted together to an offspring. They are limited to genetic markers of traits observable only in whole organisms.", "label": "Genetic map"}, "data_1279": {"class_id": "http://edamontology.org/data_1279", "definitions": "A map of genetic markers in a contiguous, assembled genomic sequence, with the sizes and separation of markers measured in base pairs.|A sequence map typically includes annotation on significant subsequences such as contigs, haplotypes and genes. The contigs shown will (typically) be a set of small overlapping clones representing a complete chromosomal segment.", "label": "Sequence map"}, "data_1276": {"class_id": "http://edamontology.org/data_1276", "definitions": "An informative report on intrinsic positional features of a nucleotide sequence.|This includes nucleotide sequence feature annotation in any known sequence feature table format and any other report of nucleic acid features.", "label": "Nucleic acid features"}, "data_1277": {"class_id": "http://edamontology.org/data_1277", "definitions": "An informative report on intrinsic positional features of a protein sequence.|This includes protein sequence feature annotation in any known sequence feature table format and any other report of protein features.", "label": "Protein features"}, "data_1274": {"class_id": "http://edamontology.org/data_1274", "definitions": "A map of (typically one) DNA sequence annotated with positional or non-positional features.", "label": "Map"}, "format_1991": {"class_id": "http://edamontology.org/format_1991", "definitions": "Mega format for (typically aligned) sequences.", "label": "mega"}, "format_1996": {"class_id": "http://edamontology.org/format_1996", "definitions": "EMBOSS simple sequence pair alignment format.", "label": "pair"}, "format_1997": {"class_id": "http://edamontology.org/format_1997", "definitions": "Phylip format for (aligned) sequences.", "label": "PHYLIP format"}, "data_1270": {"class_id": "http://edamontology.org/data_1270", "definitions": "Annotation of positional sequence features, organised into a standard feature table.", "label": "Feature table"}, "format_1995": {"class_id": "http://edamontology.org/format_1995", "definitions": "Nexus/paup non-interleaved format for (aligned) sequences.", "label": "nexusnon alignment format"}, "operation_2482": {"class_id": "http://edamontology.org/operation_2482", "definitions": "Process (read and / or write) a molecular secondary structure.", "label": "Secondary structure processing"}, "format_1445": {"class_id": "http://edamontology.org/format_1445", "definitions": "Format for distances, such as Branch Score distance, between two or more phylogenetic trees as used by the Phylip package.", "label": "Phylip tree distance format"}, "operation_2480": {"class_id": "http://edamontology.org/operation_2480", "definitions": "Analyse known molecular tertiary structures.", "label": "Structure analysis"}, "operation_2481": {"class_id": "http://edamontology.org/operation_2481", "definitions": "Analyse nucleic acid tertiary structural data.", "label": "Nucleic acid structure analysis"}, "operation_2486": {"class_id": "http://edamontology.org/operation_2486", "definitions": "Render a topology diagram of protein secondary structure.", "label": "Topology diagram drawing"}, "operation_2487": {"class_id": "http://edamontology.org/operation_2487", "definitions": "Compare protein tertiary structures.|Methods might identify structural neighbors, find structural similarities or define a structural core.", "label": "Protein structure comparison"}, "topic_3375": {"class_id": "http://edamontology.org/topic_3375", "definitions": "The study of how a drug interacts with the body.", "label": "Drug metabolism"}, "operation_2485": {"class_id": "http://edamontology.org/operation_2485", "definitions": "Render a helical wheel representation of protein secondary structure.", "label": "Helical wheel drawing"}, "operation_2488": {"class_id": "http://edamontology.org/operation_2488", "definitions": "Compare protein secondary structures.", "label": "Protein secondary structure comparison"}, "operation_2489": {"class_id": "http://edamontology.org/operation_2489", "definitions": "Predict the subcellular localisation of a protein sequence.|The prediction might include subcellular localisation (nuclear, cytoplasmic, mitochondrial, chloroplast, plastid, membrane etc) or export (extracellular proteins) of a protein.", "label": "Protein subcellular localisation prediction"}, "operation_3289": {"class_id": "http://edamontology.org/operation_3289", "definitions": "Search for and retrieve a data identifier of some kind, e.g. a database entry accession.", "label": "ID retrieval"}, "operation_3283": {"class_id": "http://edamontology.org/operation_3283", "definitions": "Process data in such a way that makes it hard to trace to the person which the data concerns.", "label": "Anonymisation"}, "operation_3282": {"class_id": "http://edamontology.org/operation_3282", "definitions": "Map data identifiers to one another for example to establish a link between two biological databases for the purposes of data integration.|The mapping can be achieved by comparing identifier values or some other means, e.g. exact matches to a provided sequence.", "label": "ID mapping"}, "operation_3280": {"class_id": "http://edamontology.org/operation_3280", "definitions": "Recognise named entities, ontology concepts, tags, events, and dictionary terms within documents.", "label": "Named-entity and concept recognition"}, "format_2015": {"class_id": "http://edamontology.org/format_2015", "definitions": "Data format for a sequence-HMM profile alignment.", "label": "Sequence-profile alignment (HMM) format"}, "format_3706": {"class_id": "http://edamontology.org/format_3706", "definitions": "Data format for biodiversity data.", "label": "Biodiversity data format"}, "data_3269": {"class_id": "http://edamontology.org/data_3269", "definitions": "An informative report on gene homologues between species.", "label": "Gene homology (report)"}, "data_3268": {"class_id": "http://edamontology.org/data_3268", "definitions": "A label (text token) describing a type of sequence feature such as gene, transcript, cds, exon, repeat, simple, misc, variation, somatic variation, structural variation, somatic structural variation, constrained or regulatory.", "label": "Sequence feature type"}, "topic_2280": {"class_id": "http://edamontology.org/topic_2280", "definitions": "The comparison two or more nucleic acid (typically RNA) secondary or tertiary structures.|Use this concept for methods that are exclusively for nucleic acid structures.", "label": "Nucleic acid structure comparison"}, "data_3266": {"class_id": "http://edamontology.org/data_3266", "definitions": "Unique identifier of sequence assembly.", "label": "Sequence assembly ID"}, "data_3265": {"class_id": "http://edamontology.org/data_3265", "definitions": "Identifier of a HGMD database entry.", "label": "HGMD ID"}, "data_3264": {"class_id": "http://edamontology.org/data_3264", "definitions": "Identifier of a COSMIC database entry.", "label": "COSMIC ID"}, "format_3100": {"class_id": "http://edamontology.org/format_3100", "definitions": "Format of summary of domain classification information for a CATH domain.|The report (for example http://www.cathdb.info/domain/1cukA01) includes CATH codes for levels in the hierarchy for the domain, level descriptions and relevant data and links.", "label": "CATH domain report format"}, "operation_2435": {"class_id": "http://edamontology.org/operation_2435", "definitions": "Process (read and / or write) a gene expression profile.", "label": "Gene expression profile processing"}, "topic_2754": {"class_id": "http://edamontology.org/topic_2754", "definitions": "Introns in a nucleotide sequences.", "label": "Introns"}, "format_2542": {"class_id": "http://edamontology.org/format_2542", "definitions": "Format of a report on protein features (domain composition).", "label": "Protein features (domains) format"}, "data_1016": {"class_id": "http://edamontology.org/data_1016", "definitions": "A position of one or more points (base or residue) in a sequence, or part of such a specification.", "label": "Sequence position"}, "data_1017": {"class_id": "http://edamontology.org/data_1017", "definitions": "Specification of range(s) of sequence positions.", "label": "Sequence range"}, "data_1014": {"class_id": "http://edamontology.org/data_1014", "definitions": "A specification (partial or complete) of one or more positions or regions of a molecular sequence or map.", "label": "Sequence position specification"}, "data_1015": {"class_id": "http://edamontology.org/data_1015", "definitions": "A unique identifier of molecular sequence feature, for example an ID of a feature that is unique within the scope of the GFF file.", "label": "Sequence feature ID"}, "data_1012": {"class_id": "http://edamontology.org/data_1012", "definitions": "Name of an enzyme.", "label": "Enzyme name"}, "data_1013": {"class_id": "http://edamontology.org/data_1013", "definitions": "Name of a restriction enzyme.", "label": "Restriction enzyme name"}, "data_1010": {"class_id": "http://edamontology.org/data_1010", "definitions": "Name or other identifier of an enzyme or record from a database of enzymes.", "label": "Enzyme identifier"}, "data_1011": {"class_id": "http://edamontology.org/data_1011", "definitions": "An Enzyme Commission (EC) number of an enzyme.", "label": "EC number"}, "format_3556": {"class_id": "http://edamontology.org/format_3556", "definitions": "MIME  HTML format for Web pages, which can include external resources, including images, Flash animations and so on. ", "label": "MHTML"}, "data_1018": {"class_id": "http://edamontology.org/data_1018", "definitions": "Name or other identifier of an nucleic acid feature.", "label": "Nucleic acid feature identifier"}, "data_1019": {"class_id": "http://edamontology.org/data_1019", "definitions": "Name or other identifier of a protein feature.", "label": "Protein feature identifier"}, "data_2618": {"class_id": "http://edamontology.org/data_2618", "definitions": "Identifier of a protein modification catalogued in a database.", "label": "Protein modification ID"}, "topic_0110": {"class_id": "http://edamontology.org/topic_0110", "definitions": "The transcription of DNA into mRNA.", "label": "Transcription"}, "topic_3055": {"class_id": "http://edamontology.org/topic_3055", "definitions": "The genes and genetic mechanisms such as Mendelian inheritance that underly continuous phenotypic traits (such as height or weight).", "label": "Quantitative genetics"}, "format_3547": {"class_id": "http://edamontology.org/format_3547", "definitions": "Format used for images and image metadata.", "label": "Image format"}, "data_2288": {"class_id": "http://edamontology.org/data_2288", "definitions": "An identifier of protein sequence(s) or protein sequence database entries.", "label": "Sequence identifier (protein)"}, "data_2289": {"class_id": "http://edamontology.org/data_2289", "definitions": "An identifier of nucleotide sequence(s) or nucleotide sequence database entries.", "label": "Sequence identifier (nucleic acid)"}, "data_2285": {"class_id": "http://edamontology.org/data_2285", "definitions": "Identifier for genetic elements in MIPS database.", "label": "Gene ID (MIPS)"}, "topic_0174": {"class_id": "http://edamontology.org/topic_0174", "definitions": "The prediction of three-dimensional structure of a (typically protein) sequence from first principles, using a physics-based or empirical scoring function and without using explicit structural templates.", "label": "Ab initio structure prediction"}, "format_3613": {"class_id": "http://edamontology.org/format_3613", "definitions": "Human ENCODE narrow peak format.|Format that covers both the broad peak format and narrow peak format from ENCODE.", "label": "ENCODE narrow peak format"}, "format_3612": {"class_id": "http://edamontology.org/format_3612", "definitions": "Human ENCODE peak format.|Format that covers both the broad peak format and narrow peak format from ENCODE.", "label": "ENCODE peak format"}, "format_3611": {"class_id": "http://edamontology.org/format_3611", "definitions": "FASTQ format subset for Phred sequencing quality score data only (no sequences) from 454 sequencers.", "label": "qual454"}, "format_3610": {"class_id": "http://edamontology.org/format_3610", "definitions": "FASTQ format subset for Phred sequencing quality score data only (no sequences) for SOLiD data.|For SOLiD data, the sequence is in color space, except the first position. The quality values are those of the Sanger format.", "label": "qualsolid"}, "format_3617": {"class_id": "http://edamontology.org/format_3617", "definitions": "Data format for graph data.", "label": "Graph format"}, "format_3616": {"class_id": "http://edamontology.org/format_3616", "definitions": "TAB-delimited genome position file index format.", "label": "tabix"}, "format_3615": {"class_id": "http://edamontology.org/format_3615", "definitions": "Blocked GNU Zip format.|BAM files are compressed using a variant of GZIP (GNU ZIP), into a format called BGZF (Blocked GNU Zip Format).", "label": "bgzip"}, "format_3614": {"class_id": "http://edamontology.org/format_3614", "definitions": "Human ENCODE broad peak format.", "label": "ENCODE broad peak format"}, "data_2314": {"class_id": "http://edamontology.org/data_2314", "definitions": "A series of digits that are assigned consecutively to each sequence record processed by NCBI. The GI number bears no resemblance to the Accession number of the sequence record.|Nucleotide sequence GI number is shown in the VERSION field of the database record. Protein sequence GI number is shown in the CDS/db_xref field of a nucleotide database record, and the VERSION field of a protein database record.", "label": "GI number"}, "format_3619": {"class_id": "http://edamontology.org/format_3619", "definitions": "SIF (simple interaction file) Format - a network/pathway format used for instance in cytoscape.", "label": "sif"}, "format_3618": {"class_id": "http://edamontology.org/format_3618", "definitions": "XML-based format used to store graph descriptions within Galaxy.", "label": "xgmml"}, "data_0959": {"class_id": "http://edamontology.org/data_0959", "definitions": "Textual metadata on a submitted or completed job.", "label": "Job metadata"}, "data_1708": {"class_id": "http://edamontology.org/data_1708", "definitions": "Image of RNA secondary structure, knots, pseudoknots etc.", "label": "RNA secondary structure image"}, "data_1709": {"class_id": "http://edamontology.org/data_1709", "definitions": "Image of protein secondary structure.", "label": "Protein secondary structure image"}, "format_3550": {"class_id": "http://edamontology.org/format_3550", "definitions": "Text-based tagged file format for medical images generated using the MetaImage software package.", "label": "mhd"}, "data_2317": {"class_id": "http://edamontology.org/data_2317", "definitions": "The name of a cell line.", "label": "Cell line name (exact)"}, "data_1707": {"class_id": "http://edamontology.org/data_1707", "definitions": "An image (for viewing or printing) of a phylogenetic tree including (typically) a plot of rooted or unrooted phylogenies, cladograms, circular trees or phenograms and associated information.|See also 'Phylogenetic tree'", "label": "Phylogenetic tree image"}, "data_2318": {"class_id": "http://edamontology.org/data_2318", "definitions": "The name of a cell line.", "label": "Cell line name (truncated)"}, "data_2319": {"class_id": "http://edamontology.org/data_2319", "definitions": "The name of a cell line.", "label": "Cell line name (no punctuation)"}, "operation_2998": {"class_id": "http://edamontology.org/operation_2998", "definitions": "Compare two or more nucleic acids to identify similarities.", "label": "Nucleic acid comparison"}, "topic_0194": {"class_id": "http://edamontology.org/topic_0194", "definitions": "The integrated study of evolutionary relationships and whole genome data, for example, in the analysis of species trees, horizontal gene transfer and evolutionary reconstruction.", "label": "Phylogenomics"}, "topic_0195": {"class_id": "http://edamontology.org/topic_0195", "definitions": "Simulated polymerase chain reaction (PCR).", "label": "Virtual PCR"}, "topic_0196": {"class_id": "http://edamontology.org/topic_0196", "definitions": "The assembly of fragments of a DNA sequence to reconstruct the original sequence.|Assembly has two broad types, de-novo and re-sequencing. Re-sequencing is a specialised case of assembly, where an assembled (typically de-novo assembled) reference genome is available and is about 95% identical to the re-sequenced genome. All other cases of assembly are 'de-novo'.", "label": "Sequence assembly"}, "data_0952": {"class_id": "http://edamontology.org/data_0952", "definitions": "Resource definition for an EMBOSS database.", "label": "EMBOSS database resource definition"}, "topic_0660": {"class_id": "http://edamontology.org/topic_0660", "definitions": "One or more ribosomal RNA (rRNA) sequences.", "label": "rRNA"}, "data_0953": {"class_id": "http://edamontology.org/data_0953", "definitions": "Information on a version of software or data, for example name, version number and release date.|Development status / maturity may be part of the version information, for example in case of tools, standards, or some data records.", "label": "Version information"}, "topic_0199": {"class_id": "http://edamontology.org/topic_0199", "definitions": "Stable, naturally occuring mutations in a nucleotide sequence including alleles, naturally occurring mutations such as single base nucleotide substitutions, deletions and insertions, RFLPs and other polymorphisms.", "label": "Genetic variation"}, "format_3727": {"class_id": "http://edamontology.org/format_3727", "definitions": "Image file format used by the Open Microscopy Environment (OME).|An OME-TIFF dataset consists of one or more files in standard TIFF or BigTIFF format, with the file extension .ome.tif or .ome.tiff, and an identical (or in the case of multiple files, nearly identical) string of OME-XML metadata embedded in the ImageDescription tag of each file\u2019s first IFD (Image File Directory). BigTIFF file extensions are also permitted, with the file extension .ome.tf2, .ome.tf8 or .ome.btf, but note these file extensions are an addition to the original specification, and software using an older version of the specification may not be able to handle these file extensions.|OME develops open-source software and data format standards for the storage and manipulation of biological microscopy data. It is a joint project between universities, research establishments, industry and the software development community.", "label": "OME-TIFF"}, "data_1646": {"class_id": "http://edamontology.org/data_1646", "definitions": "Standard protonated molecular masses from trypsin (modified porcine trypsin, Promega) and keratin peptides, used in EMBOSS.", "label": "Molecular weights standard fingerprint"}, "format_3725": {"class_id": "http://edamontology.org/format_3725", "definitions": "Synthetic Biology Open Language (SBOL) is an XML format for the specification and exchange of biological design information in synthetic biology.|SBOL introduces a standardised format for the electronic exchange of information on the structural and functional aspects of biological designs.", "label": "SBOL"}, "data_1643": {"class_id": "http://edamontology.org/data_1643", "definitions": "Affymetrix library file of information about the probe sets such as the gene name with which the probe set is associated.", "label": "Affymetrix probe sets information library file"}, "data_1642": {"class_id": "http://edamontology.org/data_1642", "definitions": "Affymetrix library file of information about which probes belong to which probe set.", "label": "Affymetrix probe sets library file"}, "format_3729": {"class_id": "http://edamontology.org/format_3729", "definitions": "Input format used by the Database of Genotypes and Phenotypes (dbGaP).|The Database of Genotypes and Phenotypes (dbGaP) is a National Institutes of Health (NIH) sponsored repository charged to archive, curate and distribute information produced by studies investigating the interaction of genotype and phenotype.", "label": "dbGaP format"}, "format_3728": {"class_id": "http://edamontology.org/format_3728", "definitions": "The LocARNA PP format combines sequence or alignment information and (respectively, single or consensus) ensemble probabilities into an PP 2.0 record.|Format for multiple aligned or single sequences together with the probabilistic description of the (consensus) RNA secondary structure ensemble by probabilities of base pairs, base pair stackings, and base pairs and unpaired bases in the loop of base pairs.", "label": "LocARNA PP"}, "operation_3501": {"class_id": "http://edamontology.org/operation_3501", "definitions": "Analysis of a set of objects, such as genes, annotated with given categories, where eventual over-/under-representation of certain categories within the studied set of objects is revealed.|Categories from a relevant ontology can be used. The input is typically a set of genes or other biological objects, possibly represented by their identifiers, and the output of the analysis is typically a ranked list of categories, each associated with a statistical metric of over-/under-representation within the studied data.", "label": "Enrichment analysis"}, "operation_3502": {"class_id": "http://edamontology.org/operation_3502", "definitions": "Analyse a dataset with respect to concepts from an ontology of chemical structure.", "label": "Chemical class enrichment"}, "operation_2993": {"class_id": "http://edamontology.org/operation_2993", "definitions": "Process (read and / or write) molecular interaction data.", "label": "Molecular interaction data processing"}, "topic_0149": {"class_id": "http://edamontology.org/topic_0149", "definitions": "Protein-DNA/RNA interactions.", "label": "Protein-nucleic acid interactions"}, "operation_2996": {"class_id": "http://edamontology.org/operation_2996", "definitions": "Assign molecular structure(s) to a group or category.", "label": "Structure classification"}, "topic_0081": {"class_id": "http://edamontology.org/topic_0081", "definitions": "The curation, processing and analysis of the structure of biological molecules, typically proteins and nucleic acids and other macromolecules.|This includes related concepts such as structural properties, alignments and structural motifs.", "label": "Structure analysis"}, "data_1805": {"class_id": "http://edamontology.org/data_1805", "definitions": "Gene identifier used by WormBase database.", "label": "Gene ID (WormBase)"}, "data_1804": {"class_id": "http://edamontology.org/data_1804", "definitions": "Gene identifier from Sol Genomics Network.", "label": "Gene ID (SGN)"}, "data_1807": {"class_id": "http://edamontology.org/data_1807", "definitions": "The name of an open reading frame attributed by a sequencing project.", "label": "ORF name"}, "data_1806": {"class_id": "http://edamontology.org/data_1806", "definitions": "Any name (other than the recommended one) for a gene.", "label": "Gene synonym"}, "data_1801": {"class_id": "http://edamontology.org/data_1801", "definitions": "Gene identifier from Trypanosoma brucei GeneDB database.", "label": "Gene ID (GeneDB Trypanosoma brucei)"}, "data_1800": {"class_id": "http://edamontology.org/data_1800", "definitions": "Gene identifier from Schizosaccharomyces pombe GeneDB database.", "label": "Gene ID (GeneDB Schizosaccharomyces pombe)"}, "data_1803": {"class_id": "http://edamontology.org/data_1803", "definitions": "Gene identifier from Virginia Bioinformatics Institute microbial database.", "label": "Gene ID (Virginia microbial)"}, "data_1802": {"class_id": "http://edamontology.org/data_1802", "definitions": "Gene identifier from Gramene database.", "label": "Gene ID (Gramene)"}, "data_0860": {"class_id": "http://edamontology.org/data_0860", "definitions": "Data concering concerning specific or conserved pattern in molecular sequences and the classifiers used for their identification, including sequence motifs, profiles or other diagnostic element.|This can include metadata about a motif or sequence profile such as its name, length, technical details about the profile construction, and so on.", "label": "Sequence signature data"}, "data_0861": {"class_id": "http://edamontology.org/data_0861", "definitions": "Alignment of exact matches between subsequences (words) within two or more molecular sequences.", "label": "Sequence alignment (words)"}, "data_0862": {"class_id": "http://edamontology.org/data_0862", "definitions": "A dotplot of sequence similarities identified from word-matching or character comparison.", "label": "Dotplot"}, "data_0863": {"class_id": "http://edamontology.org/data_0863", "definitions": "Alignment of multiple molecular sequences.", "label": "Sequence alignment"}, "data_0864": {"class_id": "http://edamontology.org/data_0864", "definitions": "Some simple value controlling a sequence alignment (or similar 'match') operation.", "label": "Sequence alignment parameter"}, "data_0865": {"class_id": "http://edamontology.org/data_0865", "definitions": "A value representing molecular sequence similarity.", "label": "Sequence similarity score"}, "data_0866": {"class_id": "http://edamontology.org/data_0866", "definitions": "Report of general information on a sequence alignment, typically include a description, sequence identifiers and alignment score.", "label": "Sequence alignment metadata"}, "data_0867": {"class_id": "http://edamontology.org/data_0867", "definitions": "An informative report of molecular sequence alignment-derived data or metadata.|Use this for any computer-generated reports on sequence alignments, and for general information (metadata) on a sequence alignment, such as a description, sequence identifiers and alignment score.", "label": "Sequence alignment report"}, "data_0868": {"class_id": "http://edamontology.org/data_0868", "definitions": "A profile-profile alignment (each profile typically representing a sequence alignment).", "label": "Profile-profile alignment"}, "data_0869": {"class_id": "http://edamontology.org/data_0869", "definitions": "Alignment of one or more molecular sequence(s) to one or more sequence profile(s) (each profile typically representing a sequence alignment).|Data associated with the alignment might also be included, e.g. ranked list of best-scoring sequences and a graphical representation of scores.", "label": "Sequence-profile alignment"}, "format_3701": {"class_id": "http://edamontology.org/format_3701", "definitions": "A five-column, tab-delimited table of feature locations and qualifiers for importing annotation into an existing Sequin submission (an NCBI tool for submitting and updating GenBank entries). ", "label": "Sequin format"}, "data_2127": {"class_id": "http://edamontology.org/data_2127", "definitions": "An identifier of a genetic code.", "label": "Genetic code identifier"}, "data_2126": {"class_id": "http://edamontology.org/data_2126", "definitions": "Frame for translation of DNA (3 forward and 3 reverse frames relative to a chromosome).", "label": "Translation frame specification"}, "data_2129": {"class_id": "http://edamontology.org/data_2129", "definitions": "Name of a file format such as HTML, PNG, PDF, EMBL, GenBank and so on.", "label": "File format name"}, "data_2128": {"class_id": "http://edamontology.org/data_2128", "definitions": "Informal name for a genetic code, typically an organism name.", "label": "Genetic code name"}, "operation_0233": {"class_id": "http://edamontology.org/operation_0233", "definitions": "Convert a molecular sequence from one type to another.", "label": "Sequence conversion"}, "operation_0232": {"class_id": "http://edamontology.org/operation_0232", "definitions": "Merge two or more (typically overlapping) molecular sequences.", "label": "Sequence merging"}, "operation_0231": {"class_id": "http://edamontology.org/operation_0231", "definitions": "Edit or change a molecular sequence, either randomly or specifically.", "label": "Sequence editing"}, "operation_0230": {"class_id": "http://edamontology.org/operation_0230", "definitions": "Generate a molecular sequence by some means.", "label": "Sequence generation"}, "operation_0237": {"class_id": "http://edamontology.org/operation_0237", "definitions": "Find and/or analyse repeat sequences in (typically nucleotide) sequences.|Repeat sequences include tandem repeats, inverted or palindromic repeats, DNA microsatellites (Simple Sequence Repeats or SSRs), interspersed repeats, maximal duplications and reverse, complemented and reverse complemented repeats etc. Repeat units can be exact or imperfect, in tandem or dispersed, of specified or unspecified length.", "label": "Repeat sequence analysis"}, "operation_0236": {"class_id": "http://edamontology.org/operation_0236", "definitions": "Calculate character or word composition or frequency of a molecular sequence.", "label": "Sequence composition calculation"}, "operation_0235": {"class_id": "http://edamontology.org/operation_0235", "definitions": "Calculate sequence ambiguity, for example identity regions in protein or nucleotide sequences with many ambiguity codes.", "label": "Sequence ambiguity calculation"}, "operation_0234": {"class_id": "http://edamontology.org/operation_0234", "definitions": "Calculate sequence complexity, for example to find low-complexity regions in sequences.", "label": "Sequence complexity calculation"}, "operation_0239": {"class_id": "http://edamontology.org/operation_0239", "definitions": "Find (scan for) known motifs, patterns and regular expressions in molecular sequence(s).", "label": "Sequence motif recognition"}, "operation_0238": {"class_id": "http://edamontology.org/operation_0238", "definitions": "Discover new motifs or conserved patterns in sequences or sequence alignments (de-novo discovery).|Motifs and patterns might be conserved or over-represented (occur with improbable frequency).", "label": "Sequence motif discovery"}, "operation_3760": {"class_id": "http://edamontology.org/operation_3760", "definitions": "Operations concerning the handling and use of other tools.", "label": "Service management"}, "format_1500": {"class_id": "http://edamontology.org/format_1500", "definitions": "Format of a matrix of 3D-1D scores used by the EMBOSS Domainatrix applications.", "label": "Domainatrix 3D-1D scoring matrix format"}, "operation_0419": {"class_id": "http://edamontology.org/operation_0419", "definitions": "Predict catalytic residues, active sites or other ligand-binding sites in protein sequences.", "label": "Protein binding site prediction (from sequence)"}, "operation_0418": {"class_id": "http://edamontology.org/operation_0418", "definitions": "Detect or predict signal peptides and signal peptide cleavage sites in protein sequences.|Methods might use sequence motifs and features, amino acid composition, profiles, machine-learned classifiers, etc.", "label": "Protein signal peptide detection"}, "topic_3366": {"class_id": "http://edamontology.org/topic_3366", "definitions": "The combination and integration of data from different sources, for example into a central repository or warehouse, to provide users with a unified view of these data.", "label": "Data integration and warehousing"}, "format_1504": {"class_id": "http://edamontology.org/format_1504", "definitions": "Amino acid index format used by the AAindex database.", "label": "aaindex"}, "operation_3766": {"class_id": "http://edamontology.org/operation_3766", "definitions": "A data mining method typically used for studying biological networks based on pairwise correlations between variables.", "label": "Weighted correlation network analysis"}, "operation_3767": {"class_id": "http://edamontology.org/operation_3767", "definitions": "Identification of protein, for example from one or more peptide identifications by tandem mass spectrometry.", "label": "Protein identification"}, "operation_0413": {"class_id": "http://edamontology.org/operation_0413", "definitions": "Predict MHC class I or class II binding peptides, promiscuous binding peptides, immunogenicity etc.", "label": "MHC peptide immunogenicity prediction"}, "operation_0412": {"class_id": "http://edamontology.org/operation_0412", "definitions": "Detect or predict signal peptides (and typically predict subcellular localisation) of bacterial proteins.", "label": "Protein signal peptide detection (bacteria)"}, "operation_0411": {"class_id": "http://edamontology.org/operation_0411", "definitions": "Detect or predict signal peptides (and typically predict subcellular localisation) of eukaryotic proteins.", "label": "Protein signal peptide detection (eukaryotes)"}, "operation_0410": {"class_id": "http://edamontology.org/operation_0410", "definitions": "Predict crystallizability of a protein sequence.", "label": "Protein crystallizability prediction"}, "operation_0417": {"class_id": "http://edamontology.org/operation_0417", "definitions": "Predict post-translation modification sites in protein sequences.|Methods might predict sites of methylation, N-terminal myristoylation, N-terminal acetylation, sumoylation, palmitoylation, phosphorylation, sulfation, glycosylation, glycosylphosphatidylinositol (GPI) modification sites (GPI lipid anchor signals) etc.", "label": "Protein post-translation modification site prediction"}, "operation_0416": {"class_id": "http://edamontology.org/operation_0416", "definitions": "Predict antigenic determinant sites (epitopes) in protein sequences.|Epitope mapping is commonly done during vaccine design.", "label": "Epitope mapping"}, "operation_0415": {"class_id": "http://edamontology.org/operation_0415", "definitions": "Predict, recognise and identify features in nucleotide sequences such as functional sites or regions, typically by scanning for known motifs, patterns and regular expressions.|Methods typically involve scanning for known motifs, patterns and regular expressions.", "label": "Nucleic acid feature detection"}, "operation_0414": {"class_id": "http://edamontology.org/operation_0414", "definitions": "Predict, recognise and identify positional features in protein sequences such as functional sites or regions and secondary structure.|Methods typically involve scanning for known motifs, patterns and regular expressions.", "label": "Protein feature prediction (from sequence)"}, "data_3724": {"class_id": "http://edamontology.org/data_3724", "definitions": "Experimental determined parameter for the cultivation of an organism.", "label": "Cultivation parameter"}, "data_3722": {"class_id": "http://edamontology.org/data_3722", "definitions": "Experimentally determined parameter of the physiology of an organism, e.g. substrate spectrum.", "label": "Physiology parameter"}, "data_3723": {"class_id": "http://edamontology.org/data_3723", "definitions": "Experimentally determined parameter of the morphology of an organism, e.g. size & shape.", "label": "Morphology parameter"}, "data_3720": {"class_id": "http://edamontology.org/data_3720", "definitions": "A report about localisation of the isolaton of biological material e.g. country or coordinates.", "label": "Geographic location"}, "data_3721": {"class_id": "http://edamontology.org/data_3721", "definitions": "A report about any kind of isolation source of biological material e.g. blood, water, soil.", "label": "Isolation source"}, "data_2637": {"class_id": "http://edamontology.org/data_2637", "definitions": "A unique identifier for pathways, reactions, complexes and small molecules from the cPath (Pathway Commons) database.|These identifiers are unique within the cPath database, however, they are not stable between releases.", "label": "cPath ID"}, "data_2636": {"class_id": "http://edamontology.org/data_2636", "definitions": "A unique identifier of an interaction from the MatrixDB database.", "label": "MatrixDB interaction ID"}, "data_2635": {"class_id": "http://edamontology.org/data_2635", "definitions": "Identifier of a metabolite from the 3DMET database.", "label": "Compound ID (3DMET)"}, "data_2634": {"class_id": "http://edamontology.org/data_2634", "definitions": "The International Standard Book Number (ISBN) is for identifying printed books.", "label": "ISBN"}, "data_2633": {"class_id": "http://edamontology.org/data_2633", "definitions": "Unique identifier of a book.", "label": "Book ID"}, "data_2632": {"class_id": "http://edamontology.org/data_2632", "definitions": "Identifier of an entry from the Saccharomyces genome database (SGD).", "label": "SGD ID"}, "data_2631": {"class_id": "http://edamontology.org/data_2631", "definitions": "An identifier of a mobile genetic element from the Aclame database.", "label": "ACLAME ID"}, "data_2630": {"class_id": "http://edamontology.org/data_2630", "definitions": "An identifier of a mobile genetic element.", "label": "Mobile genetic element ID"}, "data_2639": {"class_id": "http://edamontology.org/data_2639", "definitions": "Identifier of an entry from the PubChem database.", "label": "PubChem ID"}, "data_2638": {"class_id": "http://edamontology.org/data_2638", "definitions": "Identifier of an assay from the PubChem database.", "label": "PubChem bioassay ID"}, "operation_3192": {"class_id": "http://edamontology.org/operation_3192", "definitions": "Cut (remove) the end from a molecular sequence.|This includes\n\nennd trimming\nTrim sequences (typically from an automated DNA sequencer) to remove misleading ends.\nFor example trim polyA tails, introns and primer sequence flanking the sequence of amplified exons, or other unwanted sequence.\n\ntrimming to a reference sequence, \nTrim sequences (typically from an automated DNA sequencer) to remove the sequence ends that extend beyond an assembled reference sequence.\n\nvector trimming\nTrim sequences (typically from an automated DNA sequencer) to remove sequence-specific end regions, typically contamination from vector sequences.", "label": "Sequence trimming"}, "operation_0529": {"class_id": "http://edamontology.org/operation_0529", "definitions": "Process (read and / or write) massively parallel signature sequencing (MPSS) data.", "label": "MPSS data processing"}, "data_3118": {"class_id": "http://edamontology.org/data_3118", "definitions": "topological domains such as cytoplasmic regions in a protein.", "label": "Protein features report (topological domains)"}, "operation_0527": {"class_id": "http://edamontology.org/operation_0527", "definitions": "Make sequence tag to gene assignments (tag mapping) of SAGE, MPSS and SBS data.|Sequence tag mapping assigns experimentally obtained sequence tags to known transcripts or annotate potential virtual sequence tags in a genome.", "label": "Sequence tag mapping"}, "operation_0526": {"class_id": "http://edamontology.org/operation_0526", "definitions": "Sequence assembly for EST sequences (transcribed mRNA).|Assemblers must handle (or be complicated by) alternative splicing, trans-splicing, single-nucleotide polymorphism (SNP), recoding, and post-transcriptional modification.", "label": "EST assembly"}, "operation_0525": {"class_id": "http://edamontology.org/operation_0525", "definitions": "The process of assembling many short DNA sequences together such thay they represent the original chromosomes from which the DNA originated.", "label": "Genome assembly"}, "operation_0524": {"class_id": "http://edamontology.org/operation_0524", "definitions": "Sequence assembly by combining fragments without the aid of a reference sequence or genome.|De-novo assemblers are much slower and more memory intensive than mapping assemblers.", "label": "De-novo assembly"}, "operation_0523": {"class_id": "http://edamontology.org/operation_0523", "definitions": "Sequence assembly by combining fragments using an existing backbone sequence, typically a reference genome.|The final sequence will resemble the backbone sequence. Mapping assemblers are usually much faster and less memory intensive than de-novo assemblers.", "label": "Mapping assembly"}, "operation_0522": {"class_id": "http://edamontology.org/operation_0522", "definitions": "Predict primers for methylation PCRs.", "label": "PCR primer design (for methylation PCRs)"}, "operation_0521": {"class_id": "http://edamontology.org/operation_0521", "definitions": "", "label": "PCR primer design (based on gene structure)"}, "operation_0520": {"class_id": "http://edamontology.org/operation_0520", "definitions": "Predict primers that are conserved across multiple genomes or species.", "label": "PCR primer design (for conserved primers)"}, "topic_3500": {"class_id": "http://edamontology.org/topic_3500", "definitions": "Animals, e.g. information on a specific animal genome including molecular sequences, genes and annotation.|The study of the animal kingdom.", "label": "Zoology"}, "format_1615": {"class_id": "http://edamontology.org/format_1615", "definitions": "Entry format of the Sanger GeneDB genome database.", "label": "GeneDB gene report format"}, "format_1614": {"class_id": "http://edamontology.org/format_1614", "definitions": "Entry format of the Saccharomyces Genome Database (SGD).", "label": "SGD gene report format"}, "format_1617": {"class_id": "http://edamontology.org/format_1617", "definitions": "Entry format of the WormBase genomes database.", "label": "WormBase gene report format"}, "format_1616": {"class_id": "http://edamontology.org/format_1616", "definitions": "Entry format of The Arabidopsis Information Resource (TAIR) genome database.", "label": "TAIR gene report format"}, "format_1611": {"class_id": "http://edamontology.org/format_1611", "definitions": "Entry format of the Maize genetics and genomics database (MaizeGDB).", "label": "MaizeGDB gene report format"}, "format_1610": {"class_id": "http://edamontology.org/format_1610", "definitions": "Entry format of KEGG GENES genome database.", "label": "KEGG GENES gene report format"}, "format_1613": {"class_id": "http://edamontology.org/format_1613", "definitions": "Entry format of the Rat Genome Database (RGD).", "label": "RGD gene report format"}, "format_1612": {"class_id": "http://edamontology.org/format_1612", "definitions": "Entry format of the Mouse Genome Database (MGD).", "label": "MGD gene report format"}, "format_1619": {"class_id": "http://edamontology.org/format_1619", "definitions": "Entry format of the TIGR genome database.", "label": "TIGR gene report format"}, "format_1618": {"class_id": "http://edamontology.org/format_1618", "definitions": "Entry format of the Zebrafish Information Network (ZFIN) genome database.", "label": "ZFIN gene report format"}, "format_3700": {"class_id": "http://edamontology.org/format_3700", "definitions": "Index file format used by the samtools package to index TAB-delimited genome position files.", "label": "Tabix index file format"}, "topic_3039": {"class_id": "http://edamontology.org/topic_3039", "definitions": "Nucleic acid (secondary or tertiary) structure, such as whole structures, structural features and associated annotation.", "label": "Nucleic acid structure"}, "data_2709": {"class_id": "http://edamontology.org/data_2709", "definitions": "An identifier of a gene expression profile from the CleanEx database.", "label": "CleanEx entry name"}, "data_2702": {"class_id": "http://edamontology.org/data_2702", "definitions": "Identifier of an enzyme from the CAZy enzymes database.", "label": "Enzyme ID (CAZy)"}, "data_2701": {"class_id": "http://edamontology.org/data_2701", "definitions": "A code number identifying a family from the CATH database.", "label": "CATH node ID (family)"}, "data_2700": {"class_id": "http://edamontology.org/data_2700", "definitions": "Identifier of a protein domain (or other node) from the CATH database.", "label": "CATH identifier"}, "data_2706": {"class_id": "http://edamontology.org/data_2706", "definitions": "Name of a chromosome as used in the BioCyc database.", "label": "Chromosome name (BioCyc)"}, "data_2705": {"class_id": "http://edamontology.org/data_2705", "definitions": "An identifier of a 'cellular component' concept from the Gene Ontology.", "label": "GO concept ID (cellular component)"}, "data_2704": {"class_id": "http://edamontology.org/data_2704", "definitions": "A unique identifier assigned by the I.M.A.G.E. consortium to a clone (cloned molecular sequence).", "label": "Clone ID (IMAGE)"}, "format_1295": {"class_id": "http://edamontology.org/format_1295", "definitions": "Report format for tandem repeats in a nucleotide sequence (format generated by the Sanger Centre quicktandem program).", "label": "quicktandem"}, "format_1297": {"class_id": "http://edamontology.org/format_1297", "definitions": "Report format for tandem repeats in a sequence (an EMBOSS report format).", "label": "EMBOSS repeat"}, "format_1296": {"class_id": "http://edamontology.org/format_1296", "definitions": "Report format for inverted repeats in a nucleotide sequence (format generated by the Sanger Centre inverted program).", "label": "Sanger inverted repeats"}, "topic_3168": {"class_id": "http://edamontology.org/topic_3168", "definitions": "The determination of complete (typically nucleotide) sequences, including those of genomes (full genome sequencing, de novo sequencing and resequencing), amplicons and transcriptomes.|Amplicon sequencing is the ultra-deep sequencing of PCR products (amplicons), usually for the purpose of efficient genetic variant identification and characterisation in specific genomic regions. ", "label": "Sequencing"}, "operation_3438": {"class_id": "http://edamontology.org/operation_3438", "definitions": "Mathematical determination of the value of something, typically a properly of a  molecule.", "label": "Calculation"}, "operation_3439": {"class_id": "http://edamontology.org/operation_3439", "definitions": "Predict a molecular pathway or network.", "label": "Pathway or network prediction"}, "data_1243": {"class_id": "http://edamontology.org/data_1243", "definitions": "A nucleotide sequence library of sequences to avoid during amplification (for example repetitive sequences, or possibly the sequences of genes in a gene family that should not be amplified. The file must is in a restricted FASTA format.", "label": "Primer3 mispriming library file"}, "data_1242": {"class_id": "http://edamontology.org/data_1242", "definitions": "A library of nucleotide sequences to avoid during hybridisation events. Hybridisation of the internal oligo to sequences in this library is avoided, rather than priming from them. The file is in a restricted FASTA format.", "label": "Primer3 internal oligo mishybridizing library"}, "data_1241": {"class_id": "http://edamontology.org/data_1241", "definitions": "File of sequence vectors used by EMBOSS vectorstrip application, or any file in same format.", "label": "vectorstrip cloning vector definition file"}, "data_1240": {"class_id": "http://edamontology.org/data_1240", "definitions": "Oligonucleotide primer(s) for PCR and DNA amplification, for example a minimal primer set.", "label": "PCR primers"}, "data_1246": {"class_id": "http://edamontology.org/data_1246", "definitions": "A cluster of nucleotide sequences.|The sequences are typically related, for example a family of sequences.", "label": "Sequence cluster (nucleic acid)"}, "data_1245": {"class_id": "http://edamontology.org/data_1245", "definitions": "A cluster of protein sequences.|The sequences are typically related, for example a family of sequences.", "label": "Sequence cluster (protein)"}, "data_1244": {"class_id": "http://edamontology.org/data_1244", "definitions": "File of one or more pairs of primer sequences, as used by EMBOSS primersearch application.", "label": "primersearch primer pairs sequence record"}, "topic_0613": {"class_id": "http://edamontology.org/topic_0613", "definitions": "The physicochemical, biochemical or structural properties of amino acids or peptides.", "label": "Peptides and amino acids"}, "data_2291": {"class_id": "http://edamontology.org/data_2291", "definitions": "An identifier of a polypeptide in the UniProt database.", "label": "UniProt ID"}, "data_1249": {"class_id": "http://edamontology.org/data_1249", "definitions": "The size (length) of a sequence, subsequence or region in a sequence, or range(s) of lengths.", "label": "Sequence length"}, "operation_2434": {"class_id": "http://edamontology.org/operation_2434", "definitions": "Retrieve a codon usage table and / or associated annotation.", "label": "Data retrieval (codon usage table)"}, "format_1475": {"class_id": "http://edamontology.org/format_1475", "definitions": "Format of an entry (or part of an entry) from the PDB database.", "label": "PDB database entry format"}, "format_1477": {"class_id": "http://edamontology.org/format_1477", "definitions": "Entry format of PDB database in mmCIF format.", "label": "mmCIF"}, "format_1476": {"class_id": "http://edamontology.org/format_1476", "definitions": "Entry format of PDB database in PDB format.", "label": "PDB"}, "operation_2491": {"class_id": "http://edamontology.org/operation_2491", "definitions": "Identify potential hydrogen bonds between amino acid residues.", "label": "Hydrogen bond calculation (inter-residue)"}, "operation_2490": {"class_id": "http://edamontology.org/operation_2490", "definitions": "Calculate contacts between residues in a protein structure.", "label": "Residue contact calculation (residue-residue)"}, "operation_2493": {"class_id": "http://edamontology.org/operation_2493", "definitions": "Process (read and / or write) codon usage data.", "label": "Codon usage data processing"}, "operation_2492": {"class_id": "http://edamontology.org/operation_2492", "definitions": "Predict the interactions of proteins with other molecules.", "label": "Protein interaction prediction"}, "operation_2495": {"class_id": "http://edamontology.org/operation_2495", "definitions": "The application of bioinformatics methods to the study of gene products, typically by processing gene expression data or profiles, interpreting them in functional terms.", "label": "Gene expression analysis"}, "data_2293": {"class_id": "http://edamontology.org/data_2293", "definitions": "Secondary (internal) identifier of a Gramene database entry.", "label": "Gramene secondary identifier"}, "operation_2497": {"class_id": "http://edamontology.org/operation_2497", "definitions": "Generate, process or analyse a biological pathway or network.", "label": "Pathway or network analysis"}, "format_3462": {"class_id": "http://edamontology.org/format_3462", "definitions": "Reference-based compression of alignment format", "label": "CRAM"}, "operation_2499": {"class_id": "http://edamontology.org/operation_2499", "definitions": "Analyse, characterize and model alternative splicing events, typically by comparing multiple nucleic acid sequences.", "label": "Splicing model analysis"}, "operation_2498": {"class_id": "http://edamontology.org/operation_2498", "definitions": "Analyse SAGE, MPSS or SBS experimental data, typically to identify or quantify mRNA transcripts.", "label": "Sequencing-based expression profile data analysis"}, "operation_2439": {"class_id": "http://edamontology.org/operation_2439", "definitions": "Process (read and / or write) RNA secondary structure data.", "label": "RNA secondary structure analysis"}, "operation_2438": {"class_id": "http://edamontology.org/operation_2438", "definitions": "Generate, analyse or handle a biological pathway or network.", "label": "Pathway or network processing"}, "topic_3361": {"class_id": "http://edamontology.org/topic_3361", "definitions": "The procedures used to conduct an experiment.", "label": "Laboratory techniques"}, "operation_2929": {"class_id": "http://edamontology.org/operation_2929", "definitions": "Calculate the molecular weight of a protein (or fragments) and compare it to another protein or reference data. Generally used for protein identification.", "label": "Protein fragment weight comparison"}, "operation_2928": {"class_id": "http://edamontology.org/operation_2928", "definitions": "Compare two or more entities, typically the sequence or structure (or derivatives) of macromolecules, to identify equivalent subunits.", "label": "Alignment"}, "operation_3432": {"class_id": "http://edamontology.org/operation_3432", "definitions": "Group together some data entities on the basis of similarities such that entities in the same group (cluster) are more similar to each other than to those in other groups (clusters).", "label": "Clustering"}, "operation_3545": {"class_id": "http://edamontology.org/operation_3545", "definitions": "Model some biological system using mathematical techniques including  dynamical systems, statistical models, differential equations, and game theoretic models.", "label": "Mathematical modelling"}, "topic_0173": {"class_id": "http://edamontology.org/topic_0173", "definitions": "The folding of nucleic acid molecules and the prediction or design of nucleic acid (typically RNA) sequences with specific conformations.", "label": "Nucleic acid structure prediction"}, "operation_3433": {"class_id": "http://edamontology.org/operation_3433", "definitions": "Construct some entity (typically a molecule sequence) from component pieces.", "label": "Assembly"}, "format_2064": {"class_id": "http://edamontology.org/format_2064", "definitions": "Format of a matrix of 3D-1D scores (amino acid environment probabilities).", "label": "3D-1D scoring matrix format"}, "data_1395": {"class_id": "http://edamontology.org/data_1395", "definitions": "Whether end gaps are scored or not.", "label": "Score end gaps control"}, "data_1394": {"class_id": "http://edamontology.org/data_1394", "definitions": "A simple floating point number defining the penalty for opening or extending a gap in an alignment.", "label": "Alignment score or penalty"}, "data_1397": {"class_id": "http://edamontology.org/data_1397", "definitions": "A penalty for opening a gap in an alignment.", "label": "Gap opening penalty"}, "data_1396": {"class_id": "http://edamontology.org/data_1396", "definitions": "Controls the order of sequences in an output sequence alignment.", "label": "Aligned sequence order"}, "data_1390": {"class_id": "http://edamontology.org/data_1390", "definitions": "Alignment of more than two protein sequences.", "label": "Multiple protein sequence alignment"}, "data_3274": {"class_id": "http://edamontology.org/data_3274", "definitions": "Identifier of an object from the MGI database.", "label": "MGI accession"}, "data_3275": {"class_id": "http://edamontology.org/data_3275", "definitions": "Name of a phenotype.", "label": "Phenotype name"}, "data_1399": {"class_id": "http://edamontology.org/data_1399", "definitions": "A penalty for gaps that are close together in an alignment.", "label": "Gap separation penalty"}, "data_1398": {"class_id": "http://edamontology.org/data_1398", "definitions": "A penalty for extending a gap in an alignment.", "label": "Gap extension penalty"}, "data_3272": {"class_id": "http://edamontology.org/data_3272", "definitions": "A phylogenetic tree that reflects phylogeny of the taxa from which the characters (used in calculating the tree) were sampled.", "label": "Species tree"}, "data_3273": {"class_id": "http://edamontology.org/data_3273", "definitions": "Name or other identifier of an entry from a biosample database.", "label": "Sample ID"}, "data_1029": {"class_id": "http://edamontology.org/data_1029", "definitions": "An NCBI UniGene unique identifier of a gene.", "label": "Gene identifier (NCBI UniGene)"}, "data_1028": {"class_id": "http://edamontology.org/data_1028", "definitions": "An NCBI RefSeq unique identifier of a gene.", "label": "Gene identifier (NCBI RefSeq)"}, "data_1023": {"class_id": "http://edamontology.org/data_1023", "definitions": "The name of a sequence feature-containing entity adhering to the standard feature naming scheme used by all EMBOSS applications.", "label": "EMBOSS Uniform Feature Object"}, "data_1022": {"class_id": "http://edamontology.org/data_1022", "definitions": "A name of a sequence feature, e.g. the name of a feature to be displayed to an end-user. Typically an EMBL or Swiss-Prot feature label.|A feature label identifies a feature of a sequence database entry. When used with the database name and the entry's primary accession number, it is a unique identifier of that feature.", "label": "Sequence feature label"}, "data_1021": {"class_id": "http://edamontology.org/data_1021", "definitions": "Typically one of the EMBL or Swiss-Prot feature qualifiers.|Feature qualifiers hold information about a feature beyond that provided by the feature key and location.", "label": "Sequence feature qualifier"}, "data_1020": {"class_id": "http://edamontology.org/data_1020", "definitions": "The type of a sequence feature, typically a term or accession from the Sequence Ontology, for example an EMBL or Swiss-Prot sequence feature key.|A feature key indicates the biological nature of the feature or information about changes to or versions of the sequence.", "label": "Sequence feature key"}, "data_1027": {"class_id": "http://edamontology.org/data_1027", "definitions": "An NCBI unique identifier of a gene.", "label": "Gene ID (NCBI)"}, "data_1026": {"class_id": "http://edamontology.org/data_1026", "definitions": "The short name of a gene; a single word that does not contain white space characters. It is typically derived from the gene name.", "label": "Gene symbol"}, "data_1025": {"class_id": "http://edamontology.org/data_1025", "definitions": "An identifier of a gene, such as a name/symbol or a unique identifier of a gene in a database.", "label": "Gene identifier"}, "data_1024": {"class_id": "http://edamontology.org/data_1024", "definitions": "String of one or more ASCII characters representing a codon.", "label": "Codon name"}, "owl:DeprecatedClass": {"class_id": "http://www.w3.org/2002/07/owl#DeprecatedClass", "definitions": "", "label": "DeprecatedClass"}, "topic_0769": {"class_id": "http://edamontology.org/topic_0769", "definitions": "Biological or biomedical analytical workflows or pipelines.", "label": "Workflows"}, "topic_3067": {"class_id": "http://edamontology.org/topic_3067", "definitions": "The form and function of the structures of living organisms.", "label": "Anatomy"}, "topic_3371": {"class_id": "http://edamontology.org/topic_3371", "definitions": "The use of chemistry to create new compounds.", "label": "Synthetic chemistry"}, "topic_3373": {"class_id": "http://edamontology.org/topic_3373", "definitions": "The process of bringing a new drug to market once a lead compounds has been identified through drug discovery.", "label": "Drug development"}, "format_3600": {"class_id": "http://edamontology.org/format_3600", "definitions": "RGB file format is the native raster graphics file format for Silicon Graphics workstations.", "label": "rgb"}, "format_3601": {"class_id": "http://edamontology.org/format_3601", "definitions": "The PBM format is a lowest common denominator monochrome file format. It serves as the common language of a large family of bitmap image conversion filters.", "label": "pbm"}, "format_3602": {"class_id": "http://edamontology.org/format_3602", "definitions": "The PGM format is a lowest common denominator grayscale file format.|It is designed to be extremely easy to learn and write programs for.", "label": "pgm"}, "format_3603": {"class_id": "http://edamontology.org/format_3603", "definitions": "PNG is a file format for image compression.|It iis expected to replace the Graphics Interchange Format (GIF).", "label": "PNG"}, "format_3604": {"class_id": "http://edamontology.org/format_3604", "definitions": "Scalable Vector Graphics (SVG) is an XML-based vector image format for two-dimensional graphics with support for interactivity and animation.| The SVG specification is an open standard developed by the World Wide Web Consortium (W3C) since 1999.", "label": "SVG"}, "format_3605": {"class_id": "http://edamontology.org/format_3605", "definitions": "Sun Raster is a raster graphics file format used on SunOS by Sun Microsystems| The SVG specification is an open standard developed by the World Wide Web Consortium (W3C) since 1999.", "label": "rast"}, "format_3606": {"class_id": "http://edamontology.org/format_3606", "definitions": "Textual report format for sequence quality for reports from sequencing machines.", "label": "Sequence quality report format (text)"}, "format_3607": {"class_id": "http://edamontology.org/format_3607", "definitions": "FASTQ format subset for Phred sequencing quality score data only (no sequences).|Phred quality scores  are defined as a property which is logarithmically related to the base-calling error probabilities.", "label": "qual"}, "format_3608": {"class_id": "http://edamontology.org/format_3608", "definitions": "FASTQ format subset for Phred sequencing quality score data only (no sequences) for Solexa/Illumina 1.0 format.|Solexa/Illumina 1.0 format can encode a Solexa/Illumina quality score from -5 to 62 using ASCII 59 to 126 (although in raw read data Solexa scores from -5 to 40 only are expected)", "label": "qualsolexa"}, "format_3609": {"class_id": "http://edamontology.org/format_3609", "definitions": "FASTQ format subset for Phred sequencing quality score data only (no sequences) from Illumina 1.5 and before Illumina 1.8.|Starting in Illumina 1.5 and before Illumina 1.8, the Phred scores 0 to 2 have a slightly different meaning. The values 0 and 1 are no longer used and the value 2, encoded by ASCII 66 \"B\", is used also at the end of reads as a Read Segment Quality Control Indicator.", "label": "qualillumina"}, "topic_1456": {"class_id": "http://edamontology.org/topic_1456", "definitions": "Trans- or intra-membrane regions of a protein, typically describing physicochemical properties of the secondary structure elements.", "label": "Protein membrane regions"}, "operation_3348": {"class_id": "http://edamontology.org/operation_3348", "definitions": "Generate a checksum of a molecular sequence.", "label": "Sequence checksum generation"}, "topic_3374": {"class_id": "http://edamontology.org/topic_3374", "definitions": "The process of formulating abd administering a pharmaceutical compound to achieve a therapeutic effect.", "label": "Biotherapeutics"}, "operation_3349": {"class_id": "http://edamontology.org/operation_3349", "definitions": "Construct a bibliography from the scientific literature.", "label": "Bibliography generation"}, "data_1738": {"class_id": "http://edamontology.org/data_1738", "definitions": "Information on a published article provided by the doc2loc program.|The doc2loc output includes the url, format, type and availability code of a document for every service provider.", "label": "doc2loc document information"}, "topic_3376": {"class_id": "http://edamontology.org/topic_3376", "definitions": "The discovery, development and approval of medicines.", "label": "Medicines research and development"}, "data_1733": {"class_id": "http://edamontology.org/data_1733", "definitions": "Reference for a concept from an ontology.", "label": "Ontology concept reference"}, "data_1732": {"class_id": "http://edamontology.org/data_1732", "definitions": "A comment on a concept from an ontology.", "label": "Ontology concept comment"}, "data_1731": {"class_id": "http://edamontology.org/data_1731", "definitions": "The definition of a concept from an ontology.", "label": "Ontology concept definition"}, "data_1730": {"class_id": "http://edamontology.org/data_1730", "definitions": "A relation type defined in an ontology.", "label": "Ontology relation type"}, "format_3288": {"class_id": "http://edamontology.org/format_3288", "definitions": "The PED/MAP file describes data used by the Plink package.", "label": "PED/MAP"}, "format_3284": {"class_id": "http://edamontology.org/format_3284", "definitions": "Standard flowgram format (SFF) is a binary file format used to encode results of pyrosequencing from the 454 Life Sciences platform for high-throughput sequencing.", "label": "SFF"}, "format_3285": {"class_id": "http://edamontology.org/format_3285", "definitions": "The MAP file describes SNPs and is used by the Plink package.", "label": "MAP"}, "format_3286": {"class_id": "http://edamontology.org/format_3286", "definitions": "The PED file describes individuals and genetic data and is used by the Plink package.", "label": "PED"}, "format_3287": {"class_id": "http://edamontology.org/format_3287", "definitions": "Data format for a metadata on an individual and their genetic data.", "label": "Individual genetic data format"}, "format_3281": {"class_id": "http://edamontology.org/format_3281", "definitions": "The A2M format is used as the primary format for multiple alignments of protein or nucleic-acid sequences in the SAM suite of tools. It is a small modification of FASTA format for sequences and is compatible with most tools that read FASTA.", "label": "A2M"}, "topic_0188": {"class_id": "http://edamontology.org/topic_0188", "definitions": "Sequence profiles; typically a positional, numerical matrix representing a sequence alignment.|Sequence profiles include position-specific scoring matrix (position weight matrix), hidden Markov models etc.", "label": "Sequence profiles and HMMs"}, "topic_0183": {"class_id": "http://edamontology.org/topic_0183", "definitions": "The superimposition of molecular tertiary structures or structural (3D) profiles (representing a structure or structure alignment).|This includes the generation, storage, analysis, rendering etc. of structure alignments.", "label": "Structure alignment"}, "topic_0182": {"class_id": "http://edamontology.org/topic_0182", "definitions": "The alignment of molecular sequences or sequence profiles (representing sequence alignments).|This includes the generation of alignments (the identification of equivalent sites), the analysis of alignments, editing, visualisation, alignment databases, the alignment (equivalence between sites) of sequence profiles (representing sequence alignments) and so on.", "label": "Sequence alignment"}, "format_3702": {"class_id": "http://edamontology.org/format_3702", "definitions": "Proprietary mass-spectrometry format of Thermo Scientific's ProteomeDiscoverer software.|This format corresponds to an SQLite database, and you can look into the files with e.g. SQLiteStudio3. There are also some readers (http://doi.org/10.1021/pr2005154) and converters (http://doi.org/10.1016/j.jprot.2015.06.015) for this format available, which re-engineered the database schema, but there is no official DB schema specification of Thermo Scientific for the format. ", "label": "MSF"}, "topic_0180": {"class_id": "http://edamontology.org/topic_0180", "definitions": "The recognition (prediction and assignment) of known protein structural domains or folds in protein sequence(s).", "label": "Protein fold recognition"}, "topic_0184": {"class_id": "http://edamontology.org/topic_0184", "definitions": "The alignment of molecular sequences to structures, structural (3D) profiles or templates (representing a structure or structure alignment).", "label": "Threading"}, "data_1658": {"class_id": "http://edamontology.org/data_1658", "definitions": "environmental information processing pathways.", "label": "Environmental information processing pathway report"}, "data_2386": {"class_id": "http://edamontology.org/data_2386", "definitions": "Identifier of an entry (promoter) from the EPD database.", "label": "EPD ID"}, "data_2385": {"class_id": "http://edamontology.org/data_2385", "definitions": "Accession number of a protein from the RefSeq database.", "label": "RefSeq accession (protein)"}, "data_2384": {"class_id": "http://edamontology.org/data_2384", "definitions": "Identifier of a protein entry catalogued in the International Protein Index (IPI) database.", "label": "IPI protein ID"}, "data_2383": {"class_id": "http://edamontology.org/data_2383", "definitions": "Identifier of an entry from the EGA database.", "label": "EGA accession"}, "data_2382": {"class_id": "http://edamontology.org/data_2382", "definitions": "Identifier of an entry from a database of genotype experiment metadata.", "label": "Genotype experiment ID"}, "data_2381": {"class_id": "http://edamontology.org/data_2381", "definitions": "Report of genotype experiment including case control, population, and family studies. These might use array based methods and re-sequencing methods.", "label": "Experiment report (genotyping)"}, "data_2380": {"class_id": "http://edamontology.org/data_2380", "definitions": "A unique identifier of an item from the CABRI database.", "label": "CABRI accession"}, "data_1656": {"class_id": "http://edamontology.org/data_1656", "definitions": "A report typically including a map (diagram) of a metabolic pathway.|This includes carbohydrate, energy, lipid, nucleotide, amino acid, glycan, PK/NRP, cofactor/vitamin, secondary metabolite, xenobiotics etc.", "label": "Metabolic pathway report"}, "data_1657": {"class_id": "http://edamontology.org/data_1657", "definitions": "genetic information processing pathways.", "label": "Genetic information processing pathway report"}, "operation_1818": {"class_id": "http://edamontology.org/operation_1818", "definitions": "Calculate the solvent accessibility ('accessible molecular surface') for each atom in a structure.|Waters are not considered.", "label": "Protein atom surface calculation (accessible molecular)"}, "operation_1819": {"class_id": "http://edamontology.org/operation_1819", "definitions": "Calculate the solvent accessibility ('accessible surface') for each residue in a structure.|Solvent accessibility might be calculated for the backbone, sidechain and total (backbone plus sidechain).", "label": "Protein residue surface calculation (accessible)"}, "topic_0611": {"class_id": "http://edamontology.org/topic_0611", "definitions": "The study of matter by studying the interference pattern from firing electrons at a sample, to analyse structures at resolutions higher than can be achieved using light.", "label": "Electron microscopy"}, "topic_3518": {"class_id": "http://edamontology.org/topic_3518", "definitions": "Microarray experiments including conditions, protocol, sample:data relationships etc.|This might specify which raw data file relates to which sample and information on hybridisations, e.g. which are technical and which are biological replicates.", "label": "Microarray experiment"}, "operation_0528": {"class_id": "http://edamontology.org/operation_0528", "definitions": "Process (read and / or write) serial analysis of gene expression (SAGE) data.", "label": "SAGE data processing"}, "topic_0141": {"class_id": "http://edamontology.org/topic_0141", "definitions": "Enzyme or chemical cleavage sites and proteolytic or mass calculations on a protein sequence.", "label": "Protein cleavage sites and proteolysis"}, "topic_0079": {"class_id": "http://edamontology.org/topic_0079", "definitions": "The structures of reactants or products of metabolism, for example small molecules such as including vitamins, polyols, nucleotides and amino acids.", "label": "Metabolites"}, "topic_0078": {"class_id": "http://edamontology.org/topic_0078", "definitions": "Archival, processing and analysis of protein data, typically molecular sequence and structural data.", "label": "Proteins"}, "topic_0077": {"class_id": "http://edamontology.org/topic_0077", "definitions": "The processing and analysis of nucleic acid sequence, structural and other data.", "label": "Nucleic acids"}, "data_3719": {"class_id": "http://edamontology.org/data_3719", "definitions": "Information about the biosafety classification of an organism according to corresponding law.", "label": "Biosafety classification"}, "operation_2284": {"class_id": "http://edamontology.org/operation_2284", "definitions": "Calculate a density plot (of base composition) for a nucleotide sequence.", "label": "Nucleic acid density plotting"}, "operation_1813": {"class_id": "http://edamontology.org/operation_1813", "definitions": "Query a sequence data resource (typically a database) and retrieve sequences and / or annotation.|This includes direct retrieval methods (e.g. the dbfetch program) but not those that perform calculations on the sequence.", "label": "Sequence retrieval"}, "data_2251": {"class_id": "http://edamontology.org/data_2251", "definitions": "A relax-NG schema.", "label": "Relax-NG schema"}, "data_2250": {"class_id": "http://edamontology.org/data_2250", "definitions": "An XML Schema.", "label": "XML Schema"}, "data_2253": {"class_id": "http://edamontology.org/data_2253", "definitions": "The name of a data type.", "label": "Data resource definition name"}, "data_2252": {"class_id": "http://edamontology.org/data_2252", "definitions": "An XSLT stylesheet.", "label": "XSLT stylesheet"}, "data_2254": {"class_id": "http://edamontology.org/data_2254", "definitions": "Name of an OBO file format such as OBO-XML, plain and so on.", "label": "OBO file format name"}, "operation_3197": {"class_id": "http://edamontology.org/operation_3197", "definitions": "Analyse a genetic variation, for example to annotate its location, alleles, classification, and effects on individual transcripts predicted for a gene model.|Genetic variation annotation provides contextual interpretation of coding SNP consequences in transcripts. It allows comparisons to be made between variation data in different populations or strains for the same transcript.", "label": "Genetic variation analysis"}, "operation_2428": {"class_id": "http://edamontology.org/operation_2428", "definitions": "Validate some data.", "label": "Validation"}, "operation_0398": {"class_id": "http://edamontology.org/operation_0398", "definitions": "Calculate the molecular weight of a protein sequence or fragments.", "label": "Protein molecular weight calculation"}, "operation_0399": {"class_id": "http://edamontology.org/operation_0399", "definitions": "Predict extinction coefficients or optical density of a protein sequence.", "label": "Protein extinction coefficient calculation"}, "operation_0394": {"class_id": "http://edamontology.org/operation_0394", "definitions": "Identify potential hydrogen bonds between amino acids and other groups.|The output might include the atoms involved in the bond, bond geometric parameters and bond enthalpy.", "label": "Hydrogen bond calculation"}, "operation_0395": {"class_id": "http://edamontology.org/operation_0395", "definitions": "Calculate non-canonical atomic interactions in protein structures.", "label": "Residue non-canonical interaction detection"}, "operation_0396": {"class_id": "http://edamontology.org/operation_0396", "definitions": "Calculate a Ramachandran plot of a protein structure.", "label": "Ramachandran plot calculation"}, "operation_0397": {"class_id": "http://edamontology.org/operation_0397", "definitions": "Validate a Ramachandran plot of a protein structure.", "label": "Ramachandran plot validation"}, "operation_0390": {"class_id": "http://edamontology.org/operation_0390", "definitions": "Decompose a structure into compact or globular fragments (protein peeling).", "label": "Protein peeling"}, "operation_0391": {"class_id": "http://edamontology.org/operation_0391", "definitions": "Calculate a matrix of distance between residues (for example the C-alpha atoms) in a protein structure.", "label": "Protein distance matrix calculation"}, "operation_0392": {"class_id": "http://edamontology.org/operation_0392", "definitions": "Calculate a residue contact map (typically all-versus-all inter-residue contacts) for a protein structure.", "label": "Protein contact map calculation"}, "operation_0393": {"class_id": "http://edamontology.org/operation_0393", "definitions": "Calculate clusters of contacting residues in protein structures.|This includes for example clusters of hydrophobic or charged residues, or clusters of contacting residues which have a key structural or functional role.", "label": "Residue cluster calculation"}, "data_2134": {"class_id": "http://edamontology.org/data_2134", "definitions": "A control of the order of data that is output, for example the order of sequences in an alignment.|Possible options including sorting by score, rank, by increasing P-value (probability, i.e. most statistically significant hits given first) and so on.", "label": "Results sort order"}, "data_2135": {"class_id": "http://edamontology.org/data_2135", "definitions": "A simple parameter that is a toggle (boolean value), typically a control for a modal tool.", "label": "Toggle"}, "data_2136": {"class_id": "http://edamontology.org/data_2136", "definitions": "The width of an output sequence or alignment.", "label": "Sequence width"}, "data_2137": {"class_id": "http://edamontology.org/data_2137", "definitions": "A penalty for introducing or extending a gap in an alignment.", "label": "Gap penalty"}, "data_2130": {"class_id": "http://edamontology.org/data_2130", "definitions": "A label (text token) describing a type of sequence profile such as frequency matrix, Gribskov profile, hidden Markov model etc.", "label": "Sequence profile type"}, "data_2131": {"class_id": "http://edamontology.org/data_2131", "definitions": "Name of a computer operating system such as Linux, PC or Mac.", "label": "Operating system name"}, "data_2132": {"class_id": "http://edamontology.org/data_2132", "definitions": "A type of point or block mutation, including insertion, deletion, change, duplication and moves.", "label": "Mutation type"}, "data_2133": {"class_id": "http://edamontology.org/data_2133", "definitions": "A logical operator such as OR, AND, XOR, and NOT.", "label": "Logical operator"}, "data_2139": {"class_id": "http://edamontology.org/data_2139", "definitions": "A temperature concerning nucleic acid denaturation, typically the temperature at which the two strands of a hybridised or double stranded nucleic acid (DNA or RNA/DNA) molecule separate.", "label": "Nucleic acid melting temperature"}, "format_3590": {"class_id": "http://edamontology.org/format_3590", "definitions": "HDF5 is a generic hierarchical data file format used to store and organize large amounts of data.|An HDF5 file appears to the user as a directed graph. The nodes of this graph are the higher-level HDF5 objects that are exposed by the HDF5 APIs: Groups, Datasets, Named datatypes.", "label": "hdf5"}, "format_2069": {"class_id": "http://edamontology.org/format_2069", "definitions": "Format of a sequence profile.", "label": "Sequence profile format"}, "format_2068": {"class_id": "http://edamontology.org/format_2068", "definitions": "Format of a sequence motif.", "label": "Sequence motif format"}, "topic_0612": {"class_id": "http://edamontology.org/topic_0612", "definitions": "The cell cycle including key genes and proteins.", "label": "Cell cycle"}, "format_2063": {"class_id": "http://edamontology.org/format_2063", "definitions": "Format of a report of general information about a specific enzyme.", "label": "Protein report (enzyme) format"}, "format_2062": {"class_id": "http://edamontology.org/format_2062", "definitions": "Format of a report of general information about a specific protein.", "label": "Protein report format"}, "format_2061": {"class_id": "http://edamontology.org/format_2061", "definitions": "Format of a report on PCR primers or hybridisation oligos in a nucleic acid sequence.", "label": "Nucleic acid features (primers) format"}, "format_2060": {"class_id": "http://edamontology.org/format_2060", "definitions": "Format of a map of (typically one) molecular sequence annotated with features.", "label": "Map format"}, "format_2067": {"class_id": "http://edamontology.org/format_2067", "definitions": "Format of a matrix of genetic distances between molecular sequences.", "label": "Sequence distance matrix format"}, "format_2066": {"class_id": "http://edamontology.org/format_2066", "definitions": "Format of a report on sequence hits and associated data from searching a sequence database.", "label": "Database hits (sequence) format"}, "format_2065": {"class_id": "http://edamontology.org/format_2065", "definitions": "Format of a report on the quality of a protein three-dimensional model.", "label": "Protein structure report (quality evaluation) format"}, "data_3479": {"class_id": "http://edamontology.org/data_3479", "definitions": "Multiple gene identifiers in a specific order.|Such data are often used for genome rearrangement tools and phylogenetic tree labeling.", "label": "Gene order"}, "data_0928": {"class_id": "http://edamontology.org/data_0928", "definitions": "Data quantifying the level of expression of (typically) multiple genes, derived for example from microarray experiments.", "label": "Gene expression profile"}, "topic_0610": {"class_id": "http://edamontology.org/topic_0610", "definitions": "The ecological and environmental sciences and especially the application of information technology (ecoinformatics).", "label": "Ecology"}, "data_0920": {"class_id": "http://edamontology.org/data_0920", "definitions": "An informative report on the set of genes (or allelic forms) present in an individual, organism or cell and associated with a specific physical characteristic, or a report concerning an organisms traits and phenotypes.", "label": "Genotype/phenotype report"}, "data_0923": {"class_id": "http://edamontology.org/data_0923", "definitions": "PCR experiments, e.g. quantitative real-time PCR.", "label": "PCR experiment report"}, "data_0922": {"class_id": "http://edamontology.org/data_0922", "definitions": "PCR primers and hybridisation oligos in a nucleic acid sequence.", "label": "Nucleic acid features report (primers)"}, "data_0925": {"class_id": "http://edamontology.org/data_0925", "definitions": "An assembly of fragments of a (typically genomic) DNA sequence.|Typically, an assembly is a collection of contigs (for example ESTs and genomic DNA fragments) that are ordered, aligned and merged. Annotation of the assembled sequence might be included.", "label": "Sequence assembly"}, "data_0924": {"class_id": "http://edamontology.org/data_0924", "definitions": "Fluorescence trace data generated by an automated DNA sequencer, which can be interprted as a molecular sequence (reads), given associated sequencing metadata such as base-call quality scores.|This is the raw data produced by a DNA sequencing machine.", "label": "Sequence trace"}, "data_0927": {"class_id": "http://edamontology.org/data_0927", "definitions": "An informative report on the linkage of alleles.|This includes linkage disequilibrium; the non-random association of alleles or polymorphisms at two or more loci (not necessarily on the same chromosome).", "label": "Genetic linkage report"}, "data_0926": {"class_id": "http://edamontology.org/data_0926", "definitions": "Radiation hybrid scores (RH) scores for one or more markers.|Radiation Hybrid (RH) scores are used in Radiation Hybrid mapping.", "label": "Radiation Hybrid (RH) scores"}, "format_1516": {"class_id": "http://edamontology.org/format_1516", "definitions": "Format of an entry from the withrefm section of the REBASE enzyme database.", "label": "REBASE withrefm enzyme report format"}, "topic_3370": {"class_id": "http://edamontology.org/topic_3370", "definitions": "The study of the separation, identification, and quantification of the chemical components of natural and artificial materials.", "label": "Analytical chemistry"}, "format_1514": {"class_id": "http://edamontology.org/format_1514", "definitions": "Format of an entry from the KEGG ENZYME database.", "label": "KEGG ENZYME enzyme report format"}, "format_1515": {"class_id": "http://edamontology.org/format_1515", "definitions": "Format of an entry from the proto section of the REBASE enzyme database.", "label": "REBASE proto enzyme report format"}, "format_1512": {"class_id": "http://edamontology.org/format_1512", "definitions": "Format of an entry from the BRENDA enzyme database.", "label": "BRENDA enzyme report format"}, "format_1513": {"class_id": "http://edamontology.org/format_1513", "definitions": "Format of an entry from the KEGG REACTION database of biochemical reactions.", "label": "KEGG REACTION enzyme report format"}, "topic_3377": {"class_id": "http://edamontology.org/topic_3377", "definitions": "The safety (or lack) of drugs and other medical interventions.", "label": "Safety sciences"}, "format_1511": {"class_id": "http://edamontology.org/format_1511", "definitions": "Format of an entry from IntEnz (The Integrated Relational Enzyme Database).|IntEnz is the master copy of the Enzyme Nomenclature, the recommendations of the NC-IUBMB on the Nomenclature and Classification of Enzyme-Catalysed Reactions.", "label": "IntEnz enzyme report format"}, "topic_3379": {"class_id": "http://edamontology.org/topic_3379", "definitions": "The testing of new medicines, vaccines or procedures on animals (preclinical) and humans (clinical) prior to their approval by regulatory authorities.", "label": "Preclinical and clinical studies"}, "topic_3378": {"class_id": "http://edamontology.org/topic_3378", "definitions": "The detection, assesment, understanding and prevention of adverse effects of medicines.|Pharmacovigilence concerns safety once a drug has gone to market.", "label": "Pharmacovigilence"}, "format_3597": {"class_id": "http://edamontology.org/format_3597", "definitions": "PSD (Photoshop Document) is a proprietary file that allows the user to work with the images\u2019 individual layers even after the file has been saved.", "label": "psd"}, "data_2628": {"class_id": "http://edamontology.org/data_2628", "definitions": "A unique identifier of an interaction from the BioGRID database.", "label": "BioGRID interaction ID"}, "data_2629": {"class_id": "http://edamontology.org/data_2629", "definitions": "Unique identifier of a peptidase enzyme from the MEROPS database.", "label": "Enzyme ID (MEROPS)"}, "topic_0625": {"class_id": "http://edamontology.org/topic_0625", "definitions": "The study of genetic constitution of a living entity, such as an individual, and organism, a cell and so on, typically with respect to a particular observable phenotypic traits, or resources concerning such traits, which might be an aspect of biochemistry, physiology, morphology, anatomy, development and so on.", "label": "Genotype and phenotype"}, "data_3717": {"class_id": "http://edamontology.org/data_3717", "definitions": "A report about any kind of isolation of biological material.", "label": "Isolation report"}, "data_3716": {"class_id": "http://edamontology.org/data_3716", "definitions": "A report about biosafety data.", "label": "Biosafety report"}, "data_2620": {"class_id": "http://edamontology.org/data_2620", "definitions": "Identifier of an entry from the RGD database.", "label": "RGD ID"}, "data_2621": {"class_id": "http://edamontology.org/data_2621", "definitions": "Identifier of a protein sequence from the TAIR database.", "label": "TAIR accession (protein)"}, "data_2622": {"class_id": "http://edamontology.org/data_2622", "definitions": "Identifier of a small molecule metabolite from the Human Metabolome Database (HMDB).", "label": "Compound ID (HMDB)"}, "data_3718": {"class_id": "http://edamontology.org/data_3718", "definitions": "Information about the ability of an organism to cause disease in a corresponding host.", "label": "Pathogenicity report"}, "data_2625": {"class_id": "http://edamontology.org/data_2625", "definitions": "Identifier of an entry from the LIPID MAPS database.", "label": "LIPID MAPS ID"}, "data_2626": {"class_id": "http://edamontology.org/data_2626", "definitions": "Identifier of a peptide from the PeptideAtlas peptide databases.", "label": "PeptideAtlas ID"}, "data_2627": {"class_id": "http://edamontology.org/data_2627", "definitions": "Identifier of a report of molecular interactions from a database (typically).", "label": "Molecular interaction ID"}, "topic_3674": {"class_id": "http://edamontology.org/topic_3674", "definitions": "Laboratory technique to sequence the methylated regions in DNA.", "label": "Methylated DNA immunoprecipitation"}, "topic_0751": {"class_id": "http://edamontology.org/topic_0751", "definitions": "Protein phosphorylation and phosphorylation sites in protein sequences.", "label": "Phosphorylation sites"}, "topic_0204": {"class_id": "http://edamontology.org/topic_0204", "definitions": "The regulation of gene expression.", "label": "Gene regulation"}, "topic_3676": {"class_id": "http://edamontology.org/topic_3676", "definitions": "Laboratory technique to sequence all the protein-coding regions in a genome, i.e., the exome.|Exome sequencing is considered a cheap alternative to whole genome sequencing.", "label": "Exome sequencing"}, "operation_0558": {"class_id": "http://edamontology.org/operation_0558", "definitions": "Annotate a phylogenetic tree with terms from a controlled vocabulary.", "label": "Phylogenetic tree annotation"}, "operation_0559": {"class_id": "http://edamontology.org/operation_0559", "definitions": "Predict and optimise peptide ligands that elicit an immunological response.", "label": "Immunogenicity prediction"}, "operation_0552": {"class_id": "http://edamontology.org/operation_0552", "definitions": "Apply bootstrapping or other measures to estimate confidence of a phylogenetic tree.", "label": "Phylogenetic tree bootstrapping"}, "operation_0553": {"class_id": "http://edamontology.org/operation_0553", "definitions": "Predict families of genes and gene function based on their position in a phylogenetic tree.", "label": "Phylogenetic tree analysis (gene family prediction)"}, "operation_0550": {"class_id": "http://edamontology.org/operation_0550", "definitions": "Identify a plausible model of DNA substitution that explains a molecular (DNA or protein) sequence alignment.", "label": "DNA substitution modelling"}, "operation_0551": {"class_id": "http://edamontology.org/operation_0551", "definitions": "Analyse the shape (topology) of a phylogenetic tree.", "label": "Phylogenetic tree analysis (shape)"}, "operation_0556": {"class_id": "http://edamontology.org/operation_0556", "definitions": "Compare two or more phylogenetic trees to detect subtrees or supertrees.", "label": "Phylogenetic sub/super tree detection"}, "operation_0557": {"class_id": "http://edamontology.org/operation_0557", "definitions": "Compare two or more phylogenetic trees to calculate distances between trees.", "label": "Phylogenetic tree distances calculation"}, "operation_0554": {"class_id": "http://edamontology.org/operation_0554", "definitions": "Analyse a phylogenetic tree to identify allele frequency distribution and change that is subject to evolutionary pressures (natural selection, genetic drift, mutation and gene flow). Identify type of natural selection (such as stabilizing, balancing or disruptive).|Stabilizing/purifying (directional) selection favors a single phenotype and tends to decrease genetic diversity as a population stabilizes on a particular trait, selecting out trait extremes or deleterious mutations. In contrast, balancing selection maintain genetic polymorphisms (or multiple alleles), whereas disruptive (or diversifying) selection favors individuals at both extremes of a trait.", "label": "Phylogenetic tree analysis (natural selection)"}, "operation_0555": {"class_id": "http://edamontology.org/operation_0555", "definitions": "Compare two or more phylogenetic trees to produce a consensus tree.|Methods typically test for topological similarity between trees using for example a congruence index.", "label": "Phylogenetic tree generation (consensus)"}, "topic_3515": {"class_id": "http://edamontology.org/topic_3515", "definitions": "Protein-drug interaction(s).", "label": "Protein-drug interactions"}, "data_1672": {"class_id": "http://edamontology.org/data_1672", "definitions": "Information on a version of the CATH database.", "label": "CATH version information"}, "format_1608": {"class_id": "http://edamontology.org/format_1608", "definitions": "Entry format of FlyBase genome database.", "label": "FlyBase gene report format"}, "format_1609": {"class_id": "http://edamontology.org/format_1609", "definitions": "Entry format of Gramene genome database.", "label": "Gramene gene report format"}, "topic_3511": {"class_id": "http://edamontology.org/topic_3511", "definitions": "The biology, archival, detection, prediction and analysis of positional features such as functional and other key sites, in nucleic acid sequences and the conserved patterns (motifs, profiles etc.) that may be used to describe them.|Sequence tagged sites are short DNA sequences that are unique within a genome and serve as a mapping landmark, detectable by PCR they allow a genome to be mapped via an ordering of STSs.", "label": "Nucleic acid sites, features and motifs"}, "topic_3510": {"class_id": "http://edamontology.org/topic_3510", "definitions": "The biology, archival, detection, prediction and analysis of positional features such as functional and other key sites, in protein sequences and the conserved patterns (motifs, profiles etc.) that may be used to describe them.|A signal peptide coding sequence encodes an N-terminal domain of a secreted protein, which is involved in attaching the polypeptide to a membrane leader sequence. A transit peptide coding sequence encodes an N-terminal domain of a nuclear-encoded organellar protein; which is involved in import of the protein into the organelle.", "label": "Protein sites, features and motifs"}, "data_1905": {"class_id": "http://edamontology.org/data_1905", "definitions": "Identifier of locus from MaizeGDB (Maize genome database).", "label": "Locus ID (MaizeGDB)"}, "format_1603": {"class_id": "http://edamontology.org/format_1603", "definitions": "Entry format of Ensembl genome database.", "label": "Ensembl gene report format"}, "topic_3673": {"class_id": "http://edamontology.org/topic_3673", "definitions": "Laboratory technique to sequence the complete DNA sequence of an organism's genome at a single time.", "label": "Whole genome sequencing"}, "format_1606": {"class_id": "http://edamontology.org/format_1606", "definitions": "Entry format of DragonDB genome database.", "label": "DragonDB gene report format"}, "data_1670": {"class_id": "http://edamontology.org/data_1670", "definitions": "Information on a database (or ontology) version, for example name, version number and release date.", "label": "Database version information"}, "format_1604": {"class_id": "http://edamontology.org/format_1604", "definitions": "Entry format of DictyBase genome database.", "label": "DictyBase gene report format"}, "format_1605": {"class_id": "http://edamontology.org/format_1605", "definitions": "Entry format of Candida Genome database.", "label": "CGD gene report format"}, "data_1671": {"class_id": "http://edamontology.org/data_1671", "definitions": "Information on an application version, for example name, version number and release date.", "label": "Tool version information"}, "data_1676": {"class_id": "http://edamontology.org/data_1676", "definitions": "The (typically numeric) unique identifier of a submitted job.", "label": "Job ID"}, "data_1677": {"class_id": "http://edamontology.org/data_1677", "definitions": "A label (text token) describing the type of job, for example interactive or non-interactive.", "label": "Job type"}, "data_3667": {"class_id": "http://edamontology.org/data_3667", "definitions": "Identifier of an entry from a database of disease.", "label": "Disease identifier"}, "data_2718": {"class_id": "http://edamontology.org/data_2718", "definitions": "Identifier of an oligonucleotide from a database.", "label": "Oligonucleotide ID"}, "data_2719": {"class_id": "http://edamontology.org/data_2719", "definitions": "Identifier of an oligonucleotide probe from the dbProbe database.", "label": "dbProbe ID"}, "operation_2952": {"class_id": "http://edamontology.org/operation_2952", "definitions": "Process (read and / or write) a molecular tertiary (3D) structure alignment.", "label": "Structure alignment processing"}, "data_2714": {"class_id": "http://edamontology.org/data_2714", "definitions": "Unique identifier of a position-specific scoring matrix from the CDD database.", "label": "CDD PSSM-ID"}, "data_2715": {"class_id": "http://edamontology.org/data_2715", "definitions": "Unique identifier for a protein from the CuticleDB database.", "label": "Protein ID (CuticleDB)"}, "data_2716": {"class_id": "http://edamontology.org/data_2716", "definitions": "Identifier of a predicted transcription factor from the DBD database.", "label": "DBD ID"}, "data_1675": {"class_id": "http://edamontology.org/data_1675", "definitions": "Metadata on the status of a submitted job.|Values for EBI services are 'DONE' (job has finished and the results can then be retrieved), 'ERROR' (the job failed or no results where found), 'NOT_FOUND' (the job id is no longer available; job results might be deleted, 'PENDING' (the job is in a queue waiting processing), 'RUNNING' (the job is currently being processed).", "label": "Job status"}, "data_2710": {"class_id": "http://edamontology.org/data_2710", "definitions": "An identifier of (typically a list of) gene expression experiments catalogued in the CleanEx database.", "label": "CleanEx dataset code"}, "data_2711": {"class_id": "http://edamontology.org/data_2711", "definitions": "An informative report of general information concerning a genome as a whole.", "label": "Genome report"}, "data_3669": {"class_id": "http://edamontology.org/data_3669", "definitions": "Some material that is used for educational (training) purposes.", "label": "Training material"}, "data_2713": {"class_id": "http://edamontology.org/data_2713", "definitions": "Unique identifier for a protein complex from the CORUM database.", "label": "Protein ID (CORUM)"}, "topic_3177": {"class_id": "http://edamontology.org/topic_3177", "definitions": "A topic concerning high-throughput sequencing of randomly fragmented genomic DNA, for example, to investigate whole-genome sequencing and resequencing, SNP discovery, identification of copy number variations and chromosomal rearrangements.", "label": "DNA-Seq"}, "topic_3176": {"class_id": "http://edamontology.org/topic_3176", "definitions": "DNA-histone complexes (chromatin), organisation of chromatin into nucleosomes and packaging into higher-order structures.", "label": "DNA packaging"}, "topic_3175": {"class_id": "http://edamontology.org/topic_3175", "definitions": "Variation in chromosome structure including microscopic and submicroscopic types of variation such as deletions, duplications, copy-number variants, insertions, inversions and translocations.", "label": "DNA structural variation"}, "topic_3174": {"class_id": "http://edamontology.org/topic_3174", "definitions": "The study of genetic material recovered from environmental samples, and associated environmental data.", "label": "Metagenomics"}, "topic_3173": {"class_id": "http://edamontology.org/topic_3173", "definitions": "The study of the epigenetic modifications of a whole cell, tissue, organism etc.|Epigenetics concerns the heritable changes in gene expression owing to mechanisms other than DNA sequence variation.", "label": "Epigenomics"}, "topic_3172": {"class_id": "http://edamontology.org/topic_3172", "definitions": "The systematic study of metabolites, the chemical processes they are involved, and the chemical fingerprints of specific cellular processes in a whole cell, tissue, organ or organism.", "label": "Metabolomics"}, "topic_3171": {"class_id": "http://edamontology.org/topic_3171", "definitions": "DNA methylation including bisulfite sequencing, methylation sites and analysis, for example of patterns and profiles of DNA methylation in a population, tissue etc.", "label": "DNA methylation"}, "operation_3778": {"class_id": "http://edamontology.org/operation_3778", "definitions": "Text annotation is the operation of adding notes, data and metadata, recognised entities and concepts, and their relations to a text (such as a scientific article).", "label": "Text annotation"}, "format_3758": {"class_id": "http://edamontology.org/format_3758", "definitions": "\"Raw\" result file from SEQUEST database search.", "label": "SEQUEST .out file"}, "topic_3179": {"class_id": "http://edamontology.org/topic_3179", "definitions": "Experimental techniques that combine chromatin immunoprecipitation ('ChIP') with microarray ('chip'). ChIP-on-chip is used for high-throughput study protein-DNA interactions.", "label": "ChIP-on-chip"}, "format_3549": {"class_id": "http://edamontology.org/format_3549", "definitions": "Medical image and metadata format of the Neuroimaging Informatics Technology Initiative.", "label": "nii"}, "data_1254": {"class_id": "http://edamontology.org/data_1254", "definitions": "An informative report about non-positional sequence features, typically a report on general molecular sequence properties derived from sequence analysis.", "label": "Sequence property"}, "data_1255": {"class_id": "http://edamontology.org/data_1255", "definitions": "Annotation of positional features of molecular sequence(s), i.e. that can be mapped to position(s) in the sequence.|This includes annotation of positional sequence features, organised into a standard feature table, or any other report of sequence features.  General feature reports are a source of sequence feature table information although internal conversion would be required.", "label": "Sequence features"}, "data_1256": {"class_id": "http://edamontology.org/data_1256", "definitions": "Comparative data on sequence features such as statistics, intersections (and data on intersections), differences etc.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Sequence features (comparative)"}, "data_1257": {"class_id": "http://edamontology.org/data_1257", "definitions": "A report of general sequence properties derived from protein sequence data.", "label": "Sequence property (protein)"}, "data_1250": {"class_id": "http://edamontology.org/data_1250", "definitions": "Size of a sequence word.|Word size is used for example in word-based sequence database search methods.", "label": "Word size"}, "data_1251": {"class_id": "http://edamontology.org/data_1251", "definitions": "Size of a sequence window.|A window is a region of fixed size but not fixed position over a molecular sequence. It is typically moved (computationally) over a sequence during scoring.", "label": "Window size"}, "data_1252": {"class_id": "http://edamontology.org/data_1252", "definitions": "Specification of range(s) of length of sequences.", "label": "Sequence length range"}, "data_1253": {"class_id": "http://edamontology.org/data_1253", "definitions": "Report on basic information about a molecular sequence such as name, accession number, type (nucleic or protein), length, description etc.", "label": "Sequence information report"}, "data_1258": {"class_id": "http://edamontology.org/data_1258", "definitions": "A report of general sequence properties derived from nucleotide sequence data.", "label": "Sequence property (nucleic acid)"}, "data_1259": {"class_id": "http://edamontology.org/data_1259", "definitions": "A report on sequence complexity, for example low-complexity or repeat regions in sequences.", "label": "Sequence complexity report"}, "topic_3516": {"class_id": "http://edamontology.org/topic_3516", "definitions": "Genotype experiment including case control, population, and family studies. These might use array based methods and re-sequencing methods.", "label": "Genotyping experiment"}, "format_3548": {"class_id": "http://edamontology.org/format_3548", "definitions": "Medical image format corresponding to the Digital Imaging and Communications in Medicine (DICOM) standard.", "label": "DICOM format"}, "topic_0617": {"class_id": "http://edamontology.org/topic_0617", "definitions": "Ribosomes, typically of ribosome-related genes and proteins.", "label": "Ribosomes"}, "operation_2265": {"class_id": "http://edamontology.org/operation_2265", "definitions": "Query a database and retrieve one or more data identifiers.", "label": "Data retrieval (identifier)"}, "topic_3512": {"class_id": "http://edamontology.org/topic_3512", "definitions": "Transcription of DNA into RNA and features of a messenger RNA (mRNA) molecules including precursor RNA, primary (unprocessed) transcript and fully processed molecules.|This includes 5'untranslated region (5'UTR), coding sequences (CDS), exons, intervening sequences (intron) and 3'untranslated regions (3'UTR).|This includes regions or sites in a eukaryotic and eukaryotic viral RNA sequence which directs endonuclease cleavage or polyadenylation of an RNA transcript.  A polyA signal is required for endonuclease cleavage of an RNA transcript that is followed by polyadenylation. A polyA site is a site on an RNA transcript to which adenine residues will be added during post-transcriptional polyadenylation.|This includes Introns, and protein-coding regions including coding sequences (CDS), exons, translation initiation sites and open reading frames.  Also expressed sequence tag (EST) or complementary DNA (cDNA) sequences.|This includes coding sequences for a signal or transit peptide.  A signal peptide coding sequence encodes an N-terminal domain of a secreted protein, which is involved in attaching the polypeptide to a membrane leader sequence. A transit peptide coding sequence encodes an N-terminal domain of a nuclear-encoded organellar protein; which is involved in import of the protein into the organelle.", "label": "Gene transcripts"}, "data_2400": {"class_id": "http://edamontology.org/data_2400", "definitions": "An informative report on a specific toxin.", "label": "Toxin annotation"}, "data_2401": {"class_id": "http://edamontology.org/data_2401", "definitions": "An informative report on a membrane protein.", "label": "Protein report (membrane protein)"}, "data_2402": {"class_id": "http://edamontology.org/data_2402", "definitions": "An informative report on tentative or known protein-drug interaction(s).", "label": "Protein-drug interaction report"}, "topic_2821": {"class_id": "http://edamontology.org/topic_2821", "definitions": "Unicellular eukaryotes, e.g. information on a unicellular eukaryote genome including molecular sequences, genes and annotation.|The resource may be specific to a unicellular eukaryote, a group of unicellular eukaryotes or all unicellular eukaryotes.", "label": "Unicellular eukaryotes"}, "operation_3809": {"class_id": "http://edamontology.org/operation_3809", "definitions": "Identify and assess specific genes or regulatory regions of interest that are differentially methylated.", "label": "Differentially-methylated region identification"}, "operation_3553": {"class_id": "http://edamontology.org/operation_3553", "definitions": "Annotate an image of some sort, typically with terms from a controlled vocabulary.", "label": "Image annotation"}, "operation_3552": {"class_id": "http://edamontology.org/operation_3552", "definitions": "Visualise images resulting from various types of microscopy.", "label": "Microscope image visualisation"}, "topic_2828": {"class_id": "http://edamontology.org/topic_2828", "definitions": "The study of matter and their structure by means of the diffraction of X-rays, typically the diffraction pattern caused by the regularly spaced atoms of a crystalline sample.", "label": "X-ray diffraction"}, "topic_2829": {"class_id": "http://edamontology.org/topic_2829", "definitions": "Conceptualisation, categorisation and naming of entities or phenomena within biology or bioinformatics.", "label": "Ontologies, nomenclature and classification"}, "operation_3557": {"class_id": "http://edamontology.org/operation_3557", "definitions": "Replace missing data with substituted values, usually by using some statistical or other mathematical approach.", "label": "Imputation"}, "operation_3800": {"class_id": "http://edamontology.org/operation_3800", "definitions": "Quantification of data arising from RNA-Seq high-throughput sequencing, typically the quantification of transcript abundances durnig transcriptome analysis in a gene expression study.", "label": "RNA-Seq quantification"}, "operation_3803": {"class_id": "http://edamontology.org/operation_3803", "definitions": "Mass spectra identification of compounds that are produced by living systems. Including polyketides, terpenoids, phenylpropanoids, alkaloids and antibiotics.", "label": "Natural product identification"}, "operation_3802": {"class_id": "http://edamontology.org/operation_3802", "definitions": "Sort a set of files or data items according to some property.", "label": "Sorting"}, "format_1938": {"class_id": "http://edamontology.org/format_1938", "definitions": "GFF feature file format with sequence in the header.", "label": "GFF2-seq"}, "format_1939": {"class_id": "http://edamontology.org/format_1939", "definitions": "GFF3 feature file format with sequence.", "label": "GFF3-seq"}, "topic_3519": {"class_id": "http://edamontology.org/topic_3519", "definitions": "PCR experiments, e.g. quantitative real-time PCR.", "label": "PCR experiment"}, "format_1607": {"class_id": "http://edamontology.org/format_1607", "definitions": "Entry format of EcoCyc genome database.", "label": "EcoCyc gene report format"}, "data_1386": {"class_id": "http://edamontology.org/data_1386", "definitions": "Alignment of exactly two nucleotide sequences.", "label": "Sequence alignment (nucleic acid pair)"}, "data_1387": {"class_id": "http://edamontology.org/data_1387", "definitions": "Alignment of exactly two protein sequences.", "label": "Sequence alignment (protein pair)"}, "data_1384": {"class_id": "http://edamontology.org/data_1384", "definitions": "Alignment of multiple protein sequences.", "label": "Sequence alignment (protein)"}, "data_1385": {"class_id": "http://edamontology.org/data_1385", "definitions": "Alignment of multiple molecular sequences of different types.|Hybrid sequence alignments include for example genomic DNA to EST, cDNA or mRNA.", "label": "Sequence alignment (hybrid)"}, "data_1382": {"class_id": "http://edamontology.org/data_1382", "definitions": "Alignment of more than two molecular sequences.", "label": "Sequence alignment (multiple)"}, "data_1383": {"class_id": "http://edamontology.org/data_1383", "definitions": "Alignment of multiple nucleotide sequences.", "label": "Sequence alignment (nucleic acid)"}, "data_1380": {"class_id": "http://edamontology.org/data_1380", "definitions": "A protein post-translational modification signature (sequence classifier) from the InterPro database.|A protein post-translational modification signature corresponds to sites that undergo modification of the primary structure, typically to activate or de-activate a function. For example, methylation, sumoylation, glycosylation etc. The modification might be permanent or reversible.", "label": "Protein post-translational modification signature"}, "data_1381": {"class_id": "http://edamontology.org/data_1381", "definitions": "Alignment of exactly two molecular sequences.", "label": "Sequence alignment (pair)"}, "data_1388": {"class_id": "http://edamontology.org/data_1388", "definitions": "Alignment of exactly two molecular sequences of different types.", "label": "Hybrid sequence alignment (pair)"}, "data_1389": {"class_id": "http://edamontology.org/data_1389", "definitions": "Alignment of more than two nucleotide sequences.", "label": "Multiple nucleotide sequence alignment"}, "format_1930": {"class_id": "http://edamontology.org/format_1930", "definitions": "FASTQ short read format ignoring quality scores.", "label": "FASTQ"}, "operation_2518": {"class_id": "http://edamontology.org/operation_2518", "definitions": "Compare nucleic acid tertiary structures.", "label": "Nucleic acid structure comparison"}, "format_1931": {"class_id": "http://edamontology.org/format_1931", "definitions": "FASTQ Illumina 1.3 short read format.", "label": "FASTQ-illumina"}, "operation_0475": {"class_id": "http://edamontology.org/operation_0475", "definitions": "Predict tertiary structure of DNA or RNA.|Methods might identify thermodynamically stable or evolutionarily conserved structures.", "label": "Nucleic acid structure prediction"}, "data_1038": {"class_id": "http://edamontology.org/data_1038", "definitions": "Identifier of a protein structural domain.|This is typically a character or string concatenated with a PDB identifier and a chain identifier.", "label": "Protein domain ID"}, "data_1039": {"class_id": "http://edamontology.org/data_1039", "definitions": "Identifier of a protein domain (or other node) from the SCOP database.", "label": "SCOP domain identifier"}, "operation_0474": {"class_id": "http://edamontology.org/operation_0474", "definitions": "Predict tertiary structure (backbone and side-chain conformation) of protein sequences.", "label": "Protein structure prediction"}, "data_1034": {"class_id": "http://edamontology.org/data_1034", "definitions": "Identifier of an entry from the SGD database.", "label": "Gene ID (SGD)"}, "data_1035": {"class_id": "http://edamontology.org/data_1035", "definitions": "Identifier of a gene from the GeneDB database.", "label": "Gene ID (GeneDB)"}, "data_1036": {"class_id": "http://edamontology.org/data_1036", "definitions": "Identifier of an entry from the TIGR database.", "label": "TIGR identifier"}, "data_1037": {"class_id": "http://edamontology.org/data_1037", "definitions": "Identifier of an gene from the TAIR database.", "label": "TAIR accession (gene)"}, "data_1030": {"class_id": "http://edamontology.org/data_1030", "definitions": "An Entrez unique identifier of a gene.", "label": "Gene identifier (Entrez)"}, "data_1031": {"class_id": "http://edamontology.org/data_1031", "definitions": "Identifier of a gene or feature from the CGD database.", "label": "Gene ID (CGD)"}, "data_1032": {"class_id": "http://edamontology.org/data_1032", "definitions": "Identifier of a gene from DictyBase.", "label": "Gene ID (DictyBase)"}, "data_1033": {"class_id": "http://edamontology.org/data_1033", "definitions": "Unique identifier for a gene (or other feature) from the Ensembl database.", "label": "Ensembl gene ID"}, "format_1568": {"class_id": "http://edamontology.org/format_1568", "definitions": "Entry format for the BIND database of protein interaction.", "label": "BIND entry format"}, "operation_0471": {"class_id": "http://edamontology.org/operation_0471", "definitions": "Predict cysteine bonding state and disulfide bond partners in protein sequences.", "label": "Disulfide bond prediction"}, "operation_0470": {"class_id": "http://edamontology.org/operation_0470", "definitions": "Predict open coils, non-regular secondary structure and intrinsically disordered / unstructured regions of protein sequences.", "label": "Protein secondary structure prediction (coils)"}, "operation_0473": {"class_id": "http://edamontology.org/operation_0473", "definitions": "Analyse G-protein coupled receptor proteins (GPCRs).", "label": "GPCR analysis"}, "topic_0616": {"class_id": "http://edamontology.org/topic_0616", "definitions": "A specific organelle, or organelles in general, typically the genes and proteins (or genome and proteome).", "label": "Organelles"}, "operation_3351": {"class_id": "http://edamontology.org/operation_3351", "definitions": "Analyse the surface properties of proteins or other macromolecules, including surface accessible pockets, interior inaccessible cavities etc.", "label": "Molecular surface analysis"}, "operation_0472": {"class_id": "http://edamontology.org/operation_0472", "definitions": "Predict G protein-coupled receptors (GPCR).", "label": "GPCR prediction"}, "topic_0172": {"class_id": "http://edamontology.org/topic_0172", "definitions": "The prediction, modelling, recognition or design of protein secondary or tertiary structure or other structural features.", "label": "Protein structure prediction"}, "operation_3350": {"class_id": "http://edamontology.org/operation_3350", "definitions": "Predict the structure of a multi-subunit protein and particularly how the subunits fit together.", "label": "Protein quaternary structure prediction"}, "operation_2950": {"class_id": "http://edamontology.org/operation_2950", "definitions": "Calculate contacts between residues, or between residues and other groups, in a protein structure, on the basis of distance calculations.|This includes identifying HET groups, which usually correspond to ligands, lipids, but might also (not consistently) include groups that are attached to amino acids. Each HET group is supposed to have a unique three letter code and a unique name which might be given in the output.  It can also include calculation of symmetry contacts, i.e. a contact between two atoms in different asymmetric unit.", "label": "Residue distance calculation"}, "operation_2951": {"class_id": "http://edamontology.org/operation_2951", "definitions": "Process (read and / or write) an alignment of two or more molecular sequences, structures or derived data.", "label": "Alignment processing"}, "data_1720": {"class_id": "http://edamontology.org/data_1720", "definitions": "A term from the Plant Ontology (PO).", "label": "Plant ontology term"}, "data_1721": {"class_id": "http://edamontology.org/data_1721", "definitions": "A term from the UMLS vocabulary.", "label": "UMLS"}, "data_1722": {"class_id": "http://edamontology.org/data_1722", "definitions": "A term from Foundational Model of Anatomy.|Classifies anatomical entities according to their shared characteristics (genus) and distinguishing characteristics (differentia). Specifies the part-whole and spatial relationships of the entities, morphological transformation of the entities during prenatal development and the postnatal life cycle and principles, rules and definitions according to which classes and relationships in the other three components of FMA are represented.", "label": "FMA"}, "data_1723": {"class_id": "http://edamontology.org/data_1723", "definitions": "A term from the EMAP mouse ontology.", "label": "EMAP"}, "data_1724": {"class_id": "http://edamontology.org/data_1724", "definitions": "A term from the ChEBI ontology.", "label": "ChEBI"}, "data_1725": {"class_id": "http://edamontology.org/data_1725", "definitions": "A term from the MGED ontology.", "label": "MGED"}, "data_1726": {"class_id": "http://edamontology.org/data_1726", "definitions": "A term from the myGrid ontology.|The ontology is provided as two components, the service ontology and the domain ontology. The domain ontology acts provides concepts for core bioinformatics data types and their relations. The service ontology describes the physical and operational features of web services.", "label": "myGrid"}, "data_1727": {"class_id": "http://edamontology.org/data_1727", "definitions": "A term definition for a biological process from the Gene Ontology (GO).|Data Type is an enumerated string.", "label": "GO (biological process)"}, "data_1728": {"class_id": "http://edamontology.org/data_1728", "definitions": "A term definition for a molecular function from the Gene Ontology (GO).|Data Type is an enumerated string.", "label": "GO (molecular function)"}, "data_1729": {"class_id": "http://edamontology.org/data_1729", "definitions": "A term definition for a cellular component from the Gene Ontology (GO).|Data Type is an enumerated string.", "label": "GO (cellular component)"}, "data_2398": {"class_id": "http://edamontology.org/data_2398", "definitions": "Unique identifier for a protein from the Ensembl database.", "label": "Ensembl protein ID"}, "data_2399": {"class_id": "http://edamontology.org/data_2399", "definitions": "transcription of DNA into RNA including the regulation of transcription.", "label": "Gene transcriptional features report"}, "format_2094": {"class_id": "http://edamontology.org/format_2094", "definitions": "Alphabet for molecular sequence with possible unknown positions but without non-sequence characters.", "label": "pure"}, "data_2717": {"class_id": "http://edamontology.org/data_2717", "definitions": "General annotation on an oligonucleotide probe, or a set of probes.", "label": "Oligonucleotide probe annotation"}, "data_2390": {"class_id": "http://edamontology.org/data_2390", "definitions": "Identifier of an entry from the UNITE database.", "label": "UNITE accession"}, "data_2391": {"class_id": "http://edamontology.org/data_2391", "definitions": "Identifier of an entry from the UTR database.", "label": "UTR accession"}, "data_2392": {"class_id": "http://edamontology.org/data_2392", "definitions": "Accession number of a UniParc (protein sequence) database entry.", "label": "UniParc accession"}, "data_2393": {"class_id": "http://edamontology.org/data_2393", "definitions": "Identifier of an entry from the Rouge or HUGE databases.", "label": "mFLJ/mKIAA number"}, "topic_0644": {"class_id": "http://edamontology.org/topic_0644", "definitions": "A specific proteome including protein sequences and annotation.", "label": "Proteome"}, "data_2395": {"class_id": "http://edamontology.org/data_2395", "definitions": "An informative report on a specific fungus.", "label": "Fungi annotation"}, "data_2396": {"class_id": "http://edamontology.org/data_2396", "definitions": "An informative report on a specific fungus anamorph.", "label": "Fungi annotation (anamorph)"}, "data_2397": {"class_id": "http://edamontology.org/data_2397", "definitions": "exons in a nucleotide sequences.", "label": "Gene features report (exon)"}, "format_3709": {"class_id": "http://edamontology.org/format_3709", "definitions": "Tab-delimited text files of GenePattern that contain a column for each sample, a row for each gene, and an expression value for each gene in each sample.", "label": "GCT/Res format"}, "format_3708": {"class_id": "http://edamontology.org/format_3708", "definitions": "Exchange format of the Access to Biological Collections Data (ABCD) Schema; a standard for the access to and exchange of data about specimens and observations (primary biodiversity data).", "label": "ABCD format"}, "format_3253": {"class_id": "http://edamontology.org/format_3253", "definitions": "A syntax for writing OWL class expressions.|This format was influenced by the OWL Abstract Syntax and the DL style syntax.", "label": "Manchester OWL Syntax"}, "format_3252": {"class_id": "http://edamontology.org/format_3252", "definitions": "A human-readable encoding for the Web Ontology Language (OWL).", "label": "OWL Functional Syntax"}, "data_3668": {"class_id": "http://edamontology.org/data_3668", "definitions": "The name of some disease.", "label": "Disease name"}, "data_1621": {"class_id": "http://edamontology.org/data_1621", "definitions": "Data on the influence of genotype on drug response.|The report might correlate gene expression or single-nucleotide polymorphisms with drug efficacy or toxicity.", "label": "Pharmacogenomic test report"}, "format_3256": {"class_id": "http://edamontology.org/format_3256", "definitions": "A plain text serialisation format for RDF (Resource Description Framework) graphs, and a subset of the Turtle (Terse RDF Triple Language) format.|N-Triples should not be confused with Notation 3 which is a superset of Turtle.", "label": "N-Triples"}, "format_3255": {"class_id": "http://edamontology.org/format_3255", "definitions": "The Terse RDF Triple Language (Turtle) is a human-friendly serialisation format for RDF (Resource Description Framework) graphs.|The SPARQL Query Language incorporates a very similar syntax.", "label": "Turtle"}, "data_1622": {"class_id": "http://edamontology.org/data_1622", "definitions": "An informative report on a specific disease.|For example, an informative report on a specific tumor including nature and origin of the sample, anatomic site, organ or tissue, tumor type, including morphology and/or histologic type, and so on.", "label": "Disease report"}, "topic_0147": {"class_id": "http://edamontology.org/topic_0147", "definitions": "Protein-protein interactions, individual interactions and networks, protein complexes, protein functional coupling etc.", "label": "Protein-protein interactions"}, "topic_3139": {"class_id": "http://edamontology.org/topic_3139", "definitions": "Sequence tagged sites (STS) in nucleic acid sequences.", "label": "Sequence tagged sites"}, "format_2034": {"class_id": "http://edamontology.org/format_2034", "definitions": "Data format for a biological model.", "label": "Biological model format"}, "format_2035": {"class_id": "http://edamontology.org/format_2035", "definitions": "Text format of a chemical formula.", "label": "Chemical formula format"}, "topic_0217": {"class_id": "http://edamontology.org/topic_0217", "definitions": "The processing and analysis of the bioinformatics literature and bibliographic data, such as literature search and query.", "label": "Literature analysis"}, "topic_3170": {"class_id": "http://edamontology.org/topic_3170", "definitions": "A topic concerning high-throughput sequencing of cDNA to measure the RNA content (transcriptome) of a sample, for example, to investigate how different alleles of a gene are expressed, detect post-transcriptional mutations or identify gene fusions.|This includes small RNA profiling (small RNA-Seq), for example to find novel small RNAs, characterize mutations and analyze expression of small RNAs.", "label": "RNA-Seq"}, "data_1403": {"class_id": "http://edamontology.org/data_1403", "definitions": "This is the threshold drop in score at which extension of word alignment is halted.", "label": "Drop off score"}, "topic_0082": {"class_id": "http://edamontology.org/topic_0082", "definitions": "The prediction of molecular structure, including the prediction, modelling, recognition or design of protein secondary or tertiary structure or other structural features, and the folding of nucleic acid molecules and the prediction or design of nucleic acid (typically RNA) sequences with specific conformations.|This includes the recognition (prediction and assignment) of known protein structural domains or folds in protein sequence(s), for example by threading, or the alignment of molecular sequences to structures, structural (3D) profiles or templates (representing a structure or structure alignment).", "label": "Structure prediction"}, "topic_0083": {"class_id": "http://edamontology.org/topic_0083", "definitions": "The alignment (equivalence between sites) of molecular sequences, structures or profiles (representing a sequence or structure alignment).", "label": "Alignment"}, "topic_0080": {"class_id": "http://edamontology.org/topic_0080", "definitions": "The archival, processing and analysis of molecular sequences (monomer composition of polymers) including molecular sequence data resources, sequence sites, alignments, motifs and profiles.", "label": "Sequence analysis"}, "data_1402": {"class_id": "http://edamontology.org/data_1402", "definitions": "The score (penalty) for a 'mismatch' used in various alignment and sequence database search applications with simple scoring schemes.", "label": "Mismatch penalty score"}, "topic_0084": {"class_id": "http://edamontology.org/topic_0084", "definitions": "The study of evolutionary relationships amongst organisms.|This includes diverse phylogenetic methods, including phylogenetic tree construction, typically from molecular sequence or morphological data, methods that simulate DNA sequence evolution, a phylogenetic tree or the underlying data, or which estimate or use molecular clock and stratigraphic (age) data, methods for studying gene evolution etc.", "label": "Phylogeny"}, "topic_0085": {"class_id": "http://edamontology.org/topic_0085", "definitions": "The study of gene or protein functions and their interactions in totality in a given organism, tissue, cell etc.", "label": "Functional genomics"}, "format_2030": {"class_id": "http://edamontology.org/format_2030", "definitions": "Format of a report on a chemical compound.", "label": "Small molecule report format"}, "topic_0089": {"class_id": "http://edamontology.org/topic_0089", "definitions": "The conceptualisation, categorisation and nomenclature (naming) of entities or phenomena within biology or bioinformatics. This includes formal ontologies, controlled vocabularies, structured glossary, symbols and terminology or other related resource.", "label": "Ontology and terminology"}, "format_2031": {"class_id": "http://edamontology.org/format_2031", "definitions": "Format of a report on a particular locus, gene, gene system or groups of genes.", "label": "Gene annotation format"}, "data_1531": {"class_id": "http://edamontology.org/data_1531", "definitions": "The extinction coefficient of a protein.", "label": "Protein extinction coefficient"}, "data_1530": {"class_id": "http://edamontology.org/data_1530", "definitions": "The hydrogen exchange rate of a protein.", "label": "Protein hydrogen exchange rate"}, "data_1533": {"class_id": "http://edamontology.org/data_1533", "definitions": "An informative report on protein subcellular localisation (nuclear, cytoplasmic, mitochondrial, chloroplast, plastid, membrane etc) or destination (exported / extracellular proteins).", "label": "Protein subcellular localisation"}, "data_1532": {"class_id": "http://edamontology.org/data_1532", "definitions": "The optical density of a protein.", "label": "Protein optical density"}, "data_1534": {"class_id": "http://edamontology.org/data_1534", "definitions": "An report on allergenicity / immunogenicity of peptides and proteins.|This includes data on peptide ligands that elicit an immune response (immunogens), allergic cross-reactivity, predicted antigenicity (Hopp and Woods plot) etc. These data are useful in the development of peptide-specific antibodies or multi-epitope vaccines. Methods might use sequence data (for example motifs) and / or structural data.", "label": "Peptide immunogenicity data"}, "data_1537": {"class_id": "http://edamontology.org/data_1537", "definitions": "Annotation about, or structural information derived from, one or more specific protein 3D structure(s) or structural domains.|This includes for example reports on the surface properties (shape, hydropathy, electrostatic patches etc) of a protein structure, protein flexibility or motion, and protein architecture (spatial arrangement of secondary structure).", "label": "Protein structure report"}, "data_1536": {"class_id": "http://edamontology.org/data_1536", "definitions": "A report on the immunogenicity of MHC class I or class II binding peptides.", "label": "MHC peptide immunogenicity report"}, "data_1539": {"class_id": "http://edamontology.org/data_1539", "definitions": "Report on the quality of a protein three-dimensional model.|Model validation might involve checks for atomic packing, steric clashes, agreement with electron density maps etc.", "label": "Protein structural quality report"}, "data_2240": {"class_id": "http://edamontology.org/data_2240", "definitions": "An informative report on the types of small molecules or 'heterogens' (non-protein groups) that are represented in PDB files.", "label": "Heterogen annotation"}, "format_2033": {"class_id": "http://edamontology.org/format_2033", "definitions": "Data format for a molecular tertiary structure.", "label": "Tertiary structure format"}, "topic_0178": {"class_id": "http://edamontology.org/topic_0178", "definitions": "The prediction of secondary or supersecondary structure of protein sequences.", "label": "Protein secondary structure prediction"}, "data_2247": {"class_id": "http://edamontology.org/data_2247", "definitions": "A consensus phylogenetic tree derived from comparison of multiple trees.", "label": "Phylogenetic consensus tree"}, "data_2245": {"class_id": "http://edamontology.org/data_2245", "definitions": "A collection of sequences output from a bootstrapping (resampling) procedure.|Bootstrapping is often performed in phylogenetic analysis.", "label": "Sequence set (bootstrapped)"}, "topic_0114": {"class_id": "http://edamontology.org/topic_0114", "definitions": "Gene structure, regions which make an RNA product and features such as promoters, coding regions, gene fusion, splice sites etc.|This includes the study of promoters, coding regions etc.|This incudes operons (operators, promoters and genes) from a bacterial genome.  For example the operon leader and trailer gene, gene composition of the operon and associated information.", "label": "Gene structure"}, "format_2036": {"class_id": "http://edamontology.org/format_2036", "definitions": "Format of raw (unplotted) phylogenetic data.", "label": "Phylogenetic character data format"}, "topic_0144": {"class_id": "http://edamontology.org/topic_0144", "definitions": "The processing and analysis of inter-atomic or inter-residue interactions in protein (3D) structures.", "label": "Protein residue interactions"}, "topic_3178": {"class_id": "http://edamontology.org/topic_3178", "definitions": "The alignment of sequences of (typically millions) of short reads to a reference genome.  This is a specialised topic within sequence alignment, especially because of complications arising from RNA splicing.", "label": "RNA-Seq alignment"}, "topic_3514": {"class_id": "http://edamontology.org/topic_3514", "definitions": "Protein-ligand (small molecule) interaction(s).", "label": "Protein-ligand interactions"}, "operation_0389": {"class_id": "http://edamontology.org/operation_0389", "definitions": "Analyse RNA or DNA-binding sites in protein structure.", "label": "Protein-nucleic acid binding site analysis"}, "operation_0388": {"class_id": "http://edamontology.org/operation_0388", "definitions": "Identify or predict catalytic residues, active sites or other ligand-binding sites in protein structures.", "label": "Protein binding site prediction (from structure)"}, "operation_0383": {"class_id": "http://edamontology.org/operation_0383", "definitions": "Analyse the hydrophobic, hydrophilic or charge properties of a protein structure.", "label": "Protein hydropathy calculation (from structure)"}, "format_2038": {"class_id": "http://edamontology.org/format_2038", "definitions": "Format of phylogenetic discrete states data.", "label": "Phylogenetic discrete states format"}, "operation_0380": {"class_id": "http://edamontology.org/operation_0380", "definitions": "Analyse repeat sequence organisation such as periodicity.", "label": "Repeat sequence organisation analysis"}, "operation_0387": {"class_id": "http://edamontology.org/operation_0387", "definitions": "Calculate the molecular surface area in proteins and other macromolecules.", "label": "Molecular surface calculation"}, "operation_0386": {"class_id": "http://edamontology.org/operation_0386", "definitions": "Calculate whether a protein structure has an unusually large net charge (dipole moment).", "label": "Protein dipole moment calculation"}, "operation_0385": {"class_id": "http://edamontology.org/operation_0385", "definitions": "Identify clusters of hydrophobic or charged residues in a protein structure.", "label": "Protein hydropathy cluster calculation"}, "format_2039": {"class_id": "http://edamontology.org/format_2039", "definitions": "Format of phylogenetic cliques data.", "label": "Phylogenetic tree report (cliques) format"}, "format_2037": {"class_id": "http://edamontology.org/format_2037", "definitions": "Format of phylogenetic continuous quantitative character data.", "label": "Phylogenetic continuous quantitative character format"}, "data_3449": {"class_id": "http://edamontology.org/data_3449", "definitions": "An image from a cell migration track assay.", "label": "Cell migration track image"}, "topic_0143": {"class_id": "http://edamontology.org/topic_0143", "definitions": "The comparison of two or more protein structures.|Use this concept for methods that are exclusively for protein structure.", "label": "Protein structure comparison"}, "data_3442": {"class_id": "http://edamontology.org/data_3442", "definitions": "An imaging technique that uses magnetic fields and radiowaves to form images, typically to investigate the anatomy and physiology of the human body.", "label": "MRI image"}, "format_2078": {"class_id": "http://edamontology.org/format_2078", "definitions": "Format used to specify range(s) of sequence positions.", "label": "Sequence range format"}, "format_2072": {"class_id": "http://edamontology.org/format_2072", "definitions": "Format of a hidden Markov model.", "label": "Hidden Markov model format"}, "operation_2415": {"class_id": "http://edamontology.org/operation_2415", "definitions": "Analyse protein folding, typically by processing sequence and / or structural data, and write an informative report.", "label": "Protein folding analysis"}, "format_2074": {"class_id": "http://edamontology.org/format_2074", "definitions": "Data format of a dirichlet distribution.", "label": "Dirichlet distribution format"}, "format_2075": {"class_id": "http://edamontology.org/format_2075", "definitions": "Data format for the emission and transition counts of a hidden Markov model.", "label": "HMM emission and transition counts format"}, "format_2076": {"class_id": "http://edamontology.org/format_2076", "definitions": "Format for secondary structure (predicted or real) of an RNA molecule.", "label": "RNA secondary structure format"}, "format_2077": {"class_id": "http://edamontology.org/format_2077", "definitions": "Format for secondary structure (predicted or real) of a protein molecule.", "label": "Protein secondary structure format"}, "data_0938": {"class_id": "http://edamontology.org/data_0938", "definitions": "Protein nuclear magnetic resonance (NMR) raw data.", "label": "Protein NMR data"}, "data_0939": {"class_id": "http://edamontology.org/data_0939", "definitions": "Protein secondary structure from protein coordinate or circular dichroism (CD) spectroscopic data.", "label": "Protein circular dichroism (CD) spectroscopic data"}, "data_0932": {"class_id": "http://edamontology.org/data_0932", "definitions": "Data on oligonucleotide probes (typically for use with DNA microarrays).", "label": "Oligonucleotide probe data"}, "data_0933": {"class_id": "http://edamontology.org/data_0933", "definitions": "Output from a serial analysis of gene expression (SAGE) experiment.", "label": "SAGE experimental data"}, "data_0931": {"class_id": "http://edamontology.org/data_0931", "definitions": "microarray experiments including conditions, protocol, sample:data relationships etc.", "label": "Microarray experiment report"}, "data_0936": {"class_id": "http://edamontology.org/data_0936", "definitions": "Tag to gene assignments (tag mapping) of SAGE, MPSS and SBS data. Typically this is the sequencing-based expression profile annotated with gene identifiers.", "label": "Sequence tag profile (with gene assignment)"}, "data_0937": {"class_id": "http://edamontology.org/data_0937", "definitions": "X-ray crystallography data.", "label": "Protein X-ray crystallographic data"}, "data_0934": {"class_id": "http://edamontology.org/data_0934", "definitions": "Massively parallel signature sequencing (MPSS) data.", "label": "MPSS experimental data"}, "data_0935": {"class_id": "http://edamontology.org/data_0935", "definitions": "Sequencing by synthesis (SBS) data.", "label": "SBS experimental data"}, "topic_3344": {"class_id": "http://edamontology.org/topic_3344", "definitions": "Topic concerning biological science that is (typically) performed in the context of medicine.", "label": "Biomedical science"}, "topic_3345": {"class_id": "http://edamontology.org/topic_3345", "definitions": "Topic concerning the identity of biological entities, or reports on such entities, and the mapping of entities and records in different databases.", "label": "Data identity and mapping"}, "topic_3346": {"class_id": "http://edamontology.org/topic_3346", "definitions": "The search and retrieval from a database on the basis of molecular sequence similarity.", "label": "Sequence search"}, "topic_3340": {"class_id": "http://edamontology.org/topic_3340", "definitions": "Collections of cells grown under laboratory conditions, specifically, cells from multi-cellular eukaryotes and especially animal cells.", "label": "Cell culture collection"}, "topic_3341": {"class_id": "http://edamontology.org/topic_3341", "definitions": "Collections of DNA, including both collections of cloned molecules, and populations of micro-organisms that store and propagate cloned DNA.", "label": "Clone library"}, "topic_3342": {"class_id": "http://edamontology.org/topic_3342", "definitions": "'translating' the output of basic and biomedical research into better diagnostic tools, medicines, medical procedures, policies and advice.", "label": "Translational medicine"}, "topic_3343": {"class_id": "http://edamontology.org/topic_3343", "definitions": "Collections of chemicals, typically for use in high-throughput screening experiments.", "label": "Compound libraries and screening"}, "operation_2414": {"class_id": "http://edamontology.org/operation_2414", "definitions": "Analyse protein function, typically by processing protein sequence and/or structural data, and generate an informative report.", "label": "Protein function analysis"}, "topic_2399": {"class_id": "http://edamontology.org/topic_2399", "definitions": "Transcription of DNA into RNA including the regulation of transcription.", "label": "Gene transcription"}, "topic_3540": {"class_id": "http://edamontology.org/topic_3540", "definitions": "Key residues involved in protein folding.", "label": "Protein key folding sites"}, "topic_3679": {"class_id": "http://edamontology.org/topic_3679", "definitions": "The design of an experiment involving non-human animals.", "label": "Animal study"}, "data_3707": {"class_id": "http://edamontology.org/data_3707", "definitions": "A report about biodiversity data.", "label": "Biodiversity report"}, "data_2659": {"class_id": "http://edamontology.org/data_2659", "definitions": "Identifier of a pathway from the Small Molecule Pathway Database (SMPDB).", "label": "Pathway ID (SMPDB)"}, "data_2658": {"class_id": "http://edamontology.org/data_2658", "definitions": "Identifier of a chemical from the ChemIDplus database.", "label": "Compound ID (ChemIDplus)"}, "data_2655": {"class_id": "http://edamontology.org/data_2655", "definitions": "A unique identifier of a type or group of cells.", "label": "Cell type identifier"}, "data_2654": {"class_id": "http://edamontology.org/data_2654", "definitions": "Identifier of a target protein from the Therapeutic Target Database (TTD).", "label": "Target ID (TTD)"}, "data_2657": {"class_id": "http://edamontology.org/data_2657", "definitions": "A unique identifier of a neuron from the NeuroMorpho database.", "label": "NeuroMorpho ID"}, "data_2656": {"class_id": "http://edamontology.org/data_2656", "definitions": "A unique identifier of a neuron from the NeuronDB database.", "label": "NeuronDB ID"}, "data_2651": {"class_id": "http://edamontology.org/data_2651", "definitions": "Identifier of a disease from the pharmacogenetics and pharmacogenomics knowledge base (PharmGKB).", "label": "Disease ID (PharmGKB)"}, "data_2650": {"class_id": "http://edamontology.org/data_2650", "definitions": "Identifier of a pathway from the pharmacogenetics and pharmacogenomics knowledge base (PharmGKB).", "label": "Pathway ID (PharmGKB)"}, "data_2653": {"class_id": "http://edamontology.org/data_2653", "definitions": "Identifier of a drug from the Therapeutic Target Database (TTD).", "label": "Drug ID (TTD)"}, "data_2652": {"class_id": "http://edamontology.org/data_2652", "definitions": "Identifier of a drug from the pharmacogenetics and pharmacogenomics knowledge base (PharmGKB).", "label": "Drug ID (PharmGKB)"}, "data_2099": {"class_id": "http://edamontology.org/data_2099", "definitions": "A name of a thing, which need not necessarily uniquely identify it.", "label": "Name"}, "data_0874": {"class_id": "http://edamontology.org/data_0874", "definitions": "Matrix of integer or floating point numbers for amino acid or nucleotide sequence comparison.|The comparison matrix might include matrix name, optional comment, height and width (or size) of matrix, an index row/column (of characters) and data rows/columns (of integers or floats).", "label": "Comparison matrix"}, "data_2093": {"class_id": "http://edamontology.org/data_2093", "definitions": "Reference to a dataset (or a cross-reference between two datasets), typically one or more entries in a biological database or ontology.|A list of database accessions or identifiers are usually included.", "label": "Data reference"}, "data_2092": {"class_id": "http://edamontology.org/data_2092", "definitions": "single nucleotide polymorphism (SNP) in a DNA sequence.", "label": "SNP"}, "data_2091": {"class_id": "http://edamontology.org/data_2091", "definitions": "A persistent (stable) and unique identifier, typically identifying an object (entry) from a database.", "label": "Accession"}, "data_2090": {"class_id": "http://edamontology.org/data_2090", "definitions": "Information on a database (or ontology) entry version, such as name (or other identifier) or parent database, unique identifier of entry, data, author and so on.", "label": "Database entry version information"}, "operation_0549": {"class_id": "http://edamontology.org/operation_0549", "definitions": "Construct a phylogenetic tree by using artificial-intelligence methods, for example genetic algorithms.", "label": "Phylogenetic tree generation (AI methods)"}, "operation_0548": {"class_id": "http://edamontology.org/operation_0548", "definitions": "Construct a phylogenetic tree by computing four-taxon trees (4-trees) and searching for the phylogeny that matches most closely.", "label": "Phylogenetic tree generation (quartet methods)"}, "topic_0140": {"class_id": "http://edamontology.org/topic_0140", "definitions": "The study of how proteins are transported within and without the cell, including signal peptides, protein subcellular localisation and export.", "label": "Protein targeting and localisation"}, "format_1569": {"class_id": "http://edamontology.org/format_1569", "definitions": "Entry format for the IntAct database of protein interaction.", "label": "IntAct entry format"}, "operation_0541": {"class_id": "http://edamontology.org/operation_0541", "definitions": "Phylogenetic tree construction from continuous quantitative character data.", "label": "Phylogenetic tree generation (from continuous quantitative characters)"}, "operation_0540": {"class_id": "http://edamontology.org/operation_0540", "definitions": "Phylogenetic tree construction from molecular sequences.|Methods typically compare multiple molecular sequence and estimate evolutionary distances and relationships to infer gene families or make functional predictions.", "label": "Phylogenetic tree generation (from molecular sequences)"}, "operation_0543": {"class_id": "http://edamontology.org/operation_0543", "definitions": "Phylogenetic tree construction from polymorphism data including microsatellites, RFLP (restriction fragment length polymorphisms), RAPD (random-amplified polymorphic DNA) and AFLP (amplified fragment length polymorphisms) data.", "label": "Phylogenetic tree generation (from polymorphism data)"}, "operation_0542": {"class_id": "http://edamontology.org/operation_0542", "definitions": "Phylogenetic tree construction from gene frequency data.", "label": "Phylogenetic tree generation (from gene frequencies)"}, "operation_0545": {"class_id": "http://edamontology.org/operation_0545", "definitions": "Construct a phylogenetic tree by computing a sequence alignment and searching for the tree with the fewest number of character-state changes from the alignment.|This includes evolutionary parsimony (invariants) methods.", "label": "Phylogenetic tree generation (parsimony methods)"}, "operation_0544": {"class_id": "http://edamontology.org/operation_0544", "definitions": "Construct a phylogenetic species tree, for example, from a genome-wide sequence comparison.", "label": "Phylogenetic species tree construction"}, "operation_0547": {"class_id": "http://edamontology.org/operation_0547", "definitions": "Construct a phylogenetic tree by relating sequence data to a hypothetical tree topology using a model of sequence evolution.|Maximum likelihood methods search for a tree that maximizes a likelihood function, i.e. that is most likely given the data and model. Bayesian analysis estimate the probability of tree for branch lengths and topology, typically using a Monte Carlo algorithm.", "label": "Phylogenetic tree generation (maximum likelihood and Bayesian methods)"}, "operation_0546": {"class_id": "http://edamontology.org/operation_0546", "definitions": "Construct a phylogenetic tree by computing (or using precomputed) distances between sequences and searching for the tree with minimal discrepancies between pairwise distances.|This includes neighbor joining (NJ) clustering method.", "label": "Phylogenetic tree generation (minimum distance methods)"}, "operation_1848": {"class_id": "http://edamontology.org/operation_1848", "definitions": "Reformat (a file or other report of) tertiary structure data.", "label": "Structure formatting"}, "data_3670": {"class_id": "http://edamontology.org/data_3670", "definitions": "A training course available for use on the Web.", "label": "Online course"}, "data_3671": {"class_id": "http://edamontology.org/data_3671", "definitions": "Any free or plain text, typically for human consumption and in English. Can instantiate also as a textual search query.", "label": "Text"}, "operation_2943": {"class_id": "http://edamontology.org/operation_2943", "definitions": "Generate a box plot, i.e. a depiction of groups of numerical data through their quartiles.", "label": "Box-Whisker plot plotting"}, "operation_2942": {"class_id": "http://edamontology.org/operation_2942", "definitions": "Visualise gene expression data after hierarchical clustering for representing hierarchical relationships.", "label": "Treemap visualisation"}, "operation_2941": {"class_id": "http://edamontology.org/operation_2941", "definitions": "Visualise gene expression data where each band (or line graph) corresponds to a sample.", "label": "Whole microarray graph plotting"}, "operation_2940": {"class_id": "http://edamontology.org/operation_2940", "definitions": "Render a graph in which the values of two variables are plotted along two axes; the pattern of the points reveals any correlation.", "label": "Scatter plot plotting"}, "operation_2947": {"class_id": "http://edamontology.org/operation_2947", "definitions": "Analyse a body of scientific text (typically a full text article from a scientific journal.)", "label": "Article analysis"}, "operation_2946": {"class_id": "http://edamontology.org/operation_2946", "definitions": "Process or analyse an alignment of molecular sequences or structures.", "label": "Alignment analysis"}, "operation_2945": {"class_id": "http://edamontology.org/operation_2945", "definitions": "Apply analytical methods to existing data of a specific type.|This excludes non-analytical methods that read and write the same basic type of data (for that, see 'Data handling').", "label": "Analysis"}, "operation_2944": {"class_id": "http://edamontology.org/operation_2944", "definitions": "Generate a physical (sequence) map of a DNA sequence showing the physical distance (base pairs) between features or landmarks such as restriction sites, cloned DNA fragments, genes and other genetic markers.", "label": "Physical mapping"}, "operation_2949": {"class_id": "http://edamontology.org/operation_2949", "definitions": "Analyse known protein-protein, protein-DNA/RNA or protein-ligand interactions.", "label": "Protein interaction analysis"}, "operation_2948": {"class_id": "http://edamontology.org/operation_2948", "definitions": "Analyse the interactions of two or more molecules (or parts of molecules) that are known to interact.", "label": "Molecular interaction analysis"}, "operation_1843": {"class_id": "http://edamontology.org/operation_1843", "definitions": "Identify poorly packed residues in protein structures.", "label": "Residue packing validation"}, "operation_1842": {"class_id": "http://edamontology.org/operation_1842", "definitions": "Calculate for each position in a protein structure the chance that a proline, when introduced at this position, would increase the stability of the whole protein.", "label": "Proline mutation value calculation"}, "operation_3703": {"class_id": "http://edamontology.org/operation_3703", "definitions": "Identification of the best reference for mapping for a specific dataset from a list of potential references, when performing genetic variation analysis.", "label": "Reference identification"}, "operation_3704": {"class_id": "http://edamontology.org/operation_3704", "definitions": "Label-free quantification by integration of ion current (ion counting).", "label": "Ion counting"}, "operation_3705": {"class_id": "http://edamontology.org/operation_3705", "definitions": "Chemical tagging free amino groups of intact proteins with stable isotopes.", "label": "Isotope-coded protein label"}, "format_1703": {"class_id": "http://edamontology.org/format_1703", "definitions": "The format of an entry from Chemical Entities of Biological Interest (ChEBI).|ChEBI includes an ontological classification defining relations between entities or classes of entities.", "label": "ChEBI entry format"}, "format_1702": {"class_id": "http://edamontology.org/format_1702", "definitions": "The format of an entry from a database of chemical structures and property predictions.", "label": "ChemSpider entry format"}, "format_1701": {"class_id": "http://edamontology.org/format_1701", "definitions": "The format of an entry from PubChem.", "label": "PubChem entry format"}, "format_1700": {"class_id": "http://edamontology.org/format_1700", "definitions": "The format of an entry from the KEGG GLYCAN database.", "label": "KEGG GLYCAN entry format"}, "format_1706": {"class_id": "http://edamontology.org/format_1706", "definitions": "The format of an entry from the KEGG DRUG database.", "label": "KEGG DRUG entry format"}, "format_1705": {"class_id": "http://edamontology.org/format_1705", "definitions": "The format of an entry from the HET group dictionary (HET groups from PDB files).", "label": "HET group dictionary entry format"}, "format_1704": {"class_id": "http://edamontology.org/format_1704", "definitions": "The format of an entry from the MSDchem ligand dictionary.", "label": "MSDchem ligand dictionary entry format"}, "topic_3407": {"class_id": "http://edamontology.org/topic_3407", "definitions": "The branch of medicine dealing with diseases of endocrine organs, hormone systems, their target organs, and disorders of the pathways of glucose and lipid metabolism.", "label": "Endocrinology and metabolism"}, "operation_2521": {"class_id": "http://edamontology.org/operation_2521", "definitions": "Process (read and / or write) a DNA map of some type.", "label": "Map data processing"}, "topic_3528": {"class_id": "http://edamontology.org/topic_3528", "definitions": "Disease pathways, typically of human disease.", "label": "Disease pathways"}, "topic_2830": {"class_id": "http://edamontology.org/topic_2830", "definitions": "Immunity-related genes, proteins and their ligands.|This includes T cell receptors (TR), major histocompatibility complex (MHC), immunoglobulin superfamily (IgSF) / antibodies, major histocompatibility complex superfamily (MhcSF), etc.\"", "label": "Immunoproteins, genes and antigens"}, "operation_3023": {"class_id": "http://edamontology.org/operation_3023", "definitions": "Predict, recognise, detect or identify some properties of proteins.", "label": "Prediction and recognition (protein)"}, "topic_2839": {"class_id": "http://edamontology.org/topic_2839", "definitions": "Specific molecules, including large molecules built from repeating subunits (macromolecules) and small molecules of biological significance.", "label": "Molecules"}, "topic_3521": {"class_id": "http://edamontology.org/topic_3521", "definitions": "Two-dimensional gel electrophoresis experiments, gels or spots in a gel.", "label": "2D PAGE experiment"}, "topic_3522": {"class_id": "http://edamontology.org/topic_3522", "definitions": "Northern Blot experiments.", "label": "Northern blot experiment"}, "topic_3523": {"class_id": "http://edamontology.org/topic_3523", "definitions": "RNAi experiments.", "label": "RNAi experiment"}, "topic_3524": {"class_id": "http://edamontology.org/topic_3524", "definitions": "Biological computational model experiments (simulation), for example the minimum information required in order to permit its correct interpretation and reproduction.", "label": "Simulation experiment"}, "topic_3323": {"class_id": "http://edamontology.org/topic_3323", "definitions": "The study of metabolic diseases.", "label": "Metabolic disease"}, "topic_3526": {"class_id": "http://edamontology.org/topic_3526", "definitions": "Protein-protein interaction(s), including interactions between protein domains.", "label": "Protein-protein interactions"}, "topic_3527": {"class_id": "http://edamontology.org/topic_3527", "definitions": "Cellular process pathways.", "label": "Cellular process pathways"}, "operation_3237": {"class_id": "http://edamontology.org/operation_3237", "definitions": "Remove forward and/or reverse primers from nucleic acid sequences (typically PCR products).", "label": "Primer removal"}, "operation_3232": {"class_id": "http://edamontology.org/operation_3232", "definitions": "Combine classical quantitative trait loci (QTL) analysis with gene expression profiling, for example, to describe describe cis- and trans-controlling elements for the expression of phenotype associated genes.", "label": "Gene expression QTL analysis"}, "operation_3233": {"class_id": "http://edamontology.org/operation_3233", "definitions": "Estimate the number of copies of loci of particular gene(s) in DNA sequences typically from gene-expression profiling technology based on microarray hybridisation-based experiments. For example, estimate copy number (or marker dosage) of a dominant marker in samples from polyploid plant cells or tissues, or chromosomal gains and losses in tumors.|Methods typically implement some statistical model for hypothesis testing, and methods estimate total copy number, i.e. do not distinguish the two inherited chromosomes quantities (specific copy number).", "label": "Copy number estimation"}, "operation_3230": {"class_id": "http://edamontology.org/operation_3230", "definitions": "Analyse mapping density (read depth) of (typically) short reads from sequencing platforms, for example, to detect deletions and duplications.", "label": "Read depth analysis"}, "operation_2419": {"class_id": "http://edamontology.org/operation_2419", "definitions": "Predict oligonucleotide primers or probes.", "label": "Primer and probe design"}, "format_1810": {"class_id": "http://edamontology.org/format_1810", "definitions": "Format of a report on Escherichia coli genes, proteins and molecules from the CyberCell Database (CCDB).", "label": "ColiCard report format"}, "operation_3559": {"class_id": "http://edamontology.org/operation_3559", "definitions": "Visualise, format or render data from an ontology, typically a tree of terms.", "label": "Ontology visualisation"}, "topic_2826": {"class_id": "http://edamontology.org/topic_2826", "definitions": "Protein secondary or tertiary structure alignments.", "label": "Protein structure alignment"}, "format_2329": {"class_id": "http://edamontology.org/format_2329", "definitions": "Format of a report on a gene from the GeneCards database.", "label": "GeneCards gene report format"}, "operation_2411": {"class_id": "http://edamontology.org/operation_2411", "definitions": "Process (read and / or write) one or more structural (3D) profile(s) or template(s) of some type.", "label": "Structural profile processing"}, "operation_2413": {"class_id": "http://edamontology.org/operation_2413", "definitions": "Process (read and / or write) some type of sequence profile.", "label": "Sequence profile processing"}, "format_1929": {"class_id": "http://edamontology.org/format_1929", "definitions": "FASTA format including NCBI-style IDs.", "label": "FASTA"}, "format_1928": {"class_id": "http://edamontology.org/format_1928", "definitions": "Staden experiment file format.", "label": "Staden experiment format"}, "data_1289": {"class_id": "http://edamontology.org/data_1289", "definitions": "Image of the restriction enzyme cleavage sites (restriction sites) in a nucleic acid sequence.", "label": "Restriction map"}, "data_1288": {"class_id": "http://edamontology.org/data_1288", "definitions": "Sequence map of a whole genome.", "label": "Genome map"}, "format_1923": {"class_id": "http://edamontology.org/format_1923", "definitions": "ACEDB sequence format.", "label": "acedb"}, "data_1286": {"class_id": "http://edamontology.org/data_1286", "definitions": "Sequence map of a plasmid (circular DNA).", "label": "Plasmid map"}, "data_1285": {"class_id": "http://edamontology.org/data_1285", "definitions": "Sequence map of a single gene annotated with genetic features such as introns, exons, untranslated regions, polyA signals, promoters, enhancers and (possibly) mutations defining alleles of a gene.", "label": "Gene map"}, "data_1284": {"class_id": "http://edamontology.org/data_1284", "definitions": "A gene map showing distances between loci based on relative cotransduction frequencies.", "label": "DNA transduction map"}, "data_1283": {"class_id": "http://edamontology.org/data_1283", "definitions": "A map showing banding patterns derived from direct observation of a stained chromosome.|This is the lowest-resolution physical map and can provide only rough estimates of physical (base pair) distances.  Like a genetic map, they are limited to genetic markers of traits observable only in whole organisms.", "label": "Cytogenetic map"}, "data_2868": {"class_id": "http://edamontology.org/data_2868", "definitions": "microsatellite polymorphism in a DNA sequence.", "label": "Nucleic acid features report (microsatellite)"}, "data_1281": {"class_id": "http://edamontology.org/data_1281", "definitions": "Image of a sequence with matches to signatures, motifs or profiles.", "label": "Sequence signature map"}, "data_1280": {"class_id": "http://edamontology.org/data_1280", "definitions": "A map of DNA (linear or circular) annotated with physical features or landmarks such as restriction sites, cloned DNA fragments, genes or genetic markers, along with the physical distances between them.|Distance in a physical map is measured in base pairs. A physical map might be ordered relative to a reference map (typically a genetic map) in the process of genome sequencing.", "label": "Physical map"}, "data_2869": {"class_id": "http://edamontology.org/data_2869", "definitions": "restriction fragment length polymorphisms (RFLP) in a DNA sequence.", "label": "Nucleic acid features report (RFLP)"}, "operation_2417": {"class_id": "http://edamontology.org/operation_2417", "definitions": "Process (read and / or write) data on the physicochemical property of a molecule.", "label": "Physicochemical property data processing"}, "operation_2416": {"class_id": "http://edamontology.org/operation_2416", "definitions": "Analyse protein secondary structure data.", "label": "Protein secondary structure analysis"}, "format_3622": {"class_id": "http://edamontology.org/format_3622", "definitions": "Data format used by the SQLite database conformant to the Gemini schema.", "label": "Gemini SQLite format"}, "format_3623": {"class_id": "http://edamontology.org/format_3623", "definitions": "Format of a data index of some type.", "label": "Index format"}, "format_3620": {"class_id": "http://edamontology.org/format_3620", "definitions": "MS Excel spreadsheet format consisting of a set of XML documents stored in a ZIP-compressed file.", "label": "xlsx"}, "format_3621": {"class_id": "http://edamontology.org/format_3621", "definitions": "Data format used by the SQLite database.", "label": "SQLite format"}, "format_3626": {"class_id": "http://edamontology.org/format_3626", "definitions": "Binary format used by MATLAB files to store workspace variables.", "label": "MAT"}, "format_3624": {"class_id": "http://edamontology.org/format_3624", "definitions": "An index of a genome database, indexed for use by the snpeff tool.", "label": "snpeffdb"}, "topic_2640": {"class_id": "http://edamontology.org/topic_2640", "definitions": "The study of cancer, for example, genes and proteins implicated in cancer.", "label": "Oncology"}, "data_1755": {"class_id": "http://edamontology.org/data_1755", "definitions": "Data on a single atom from a protein structure.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Protein atom"}, "data_1757": {"class_id": "http://edamontology.org/data_1757", "definitions": "Name of an atom.", "label": "Atom name"}, "data_1756": {"class_id": "http://edamontology.org/data_1756", "definitions": "Data on a single amino acid residue position in a protein structure.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Protein residue"}, "topic_0210": {"class_id": "http://edamontology.org/topic_0210", "definitions": "Information on a specific fish genome including molecular sequences, genes and annotation.", "label": "Fish"}, "operation_3801": {"class_id": "http://edamontology.org/operation_3801", "definitions": "Match experimentally measured mass spectrum to a spectrum in a spectral library or database.", "label": "Spectral library search"}, "data_1759": {"class_id": "http://edamontology.org/data_1759", "definitions": "Identifier of a model structure from a PDB file.", "label": "PDB model number"}, "data_1758": {"class_id": "http://edamontology.org/data_1758", "definitions": "Three-letter amino acid residue names as used in PDB files.", "label": "PDB residue name"}, "data_2908": {"class_id": "http://edamontology.org/data_2908", "definitions": "An accession of annotation on a (group of) organisms (catalogued in a database).", "label": "Organism accession"}, "data_2893": {"class_id": "http://edamontology.org/data_2893", "definitions": "Accession of a type or group of cells (catalogued in a database).", "label": "Cell type accession"}, "data_2892": {"class_id": "http://edamontology.org/data_2892", "definitions": "The name of a type or group of cells.", "label": "Cell type name"}, "data_2891": {"class_id": "http://edamontology.org/data_2891", "definitions": "Accession of a mathematical model, typically an entry from a database.", "label": "Biological model accession"}, "operation_0227": {"class_id": "http://edamontology.org/operation_0227", "definitions": "Generate an index of (typically a file of) biological data.", "label": "Indexing"}, "data_2897": {"class_id": "http://edamontology.org/data_2897", "definitions": "Accession of a toxin (catalogued in a database).", "label": "Toxin accession"}, "data_2896": {"class_id": "http://edamontology.org/data_2896", "definitions": "Name of a toxin.", "label": "Toxin name"}, "data_2895": {"class_id": "http://edamontology.org/data_2895", "definitions": "Accession of a drug.", "label": "Drug accession"}, "data_2894": {"class_id": "http://edamontology.org/data_2894", "definitions": "Accession of an entry from a database of chemicals.", "label": "Compound accession"}, "data_2899": {"class_id": "http://edamontology.org/data_2899", "definitions": "Common name of a drug.", "label": "Drug name"}, "data_2898": {"class_id": "http://edamontology.org/data_2898", "definitions": "Accession of a monosaccharide (catalogued in a database).", "label": "Monosaccharide accession"}, "topic_0659": {"class_id": "http://edamontology.org/topic_0659", "definitions": "Non-coding or functional RNA sequences, including regulatory RNA sequences, ribosomal RNA (rRNA) and transfer RNA (tRNA).|Non-coding RNA includes  piwi-interacting RNA (piRNA), small nuclear RNA (snRNA) and small nucleolar RNA (snoRNA).  Regulatory RNA includes microRNA (miRNA) - short single stranded RNA molecules that regulate gene expression, and small interfering RNA (siRNA).", "label": "Functional, regulatory and non-coding RNA"}, "topic_0655": {"class_id": "http://edamontology.org/topic_0655", "definitions": "Protein-coding regions including coding sequences (CDS), exons, translation initiation sites and open reading frames", "label": "Coding RNA"}, "topic_0654": {"class_id": "http://edamontology.org/topic_0654", "definitions": "DNA sequences and structure, including processes such as methylation and replication.|The DNA sequences might be coding or non-coding sequences.", "label": "DNA"}, "data_2879": {"class_id": "http://edamontology.org/data_2879", "definitions": "Annotation on or information derived from one or more specific lipid 3D structure(s).", "label": "Lipid report"}, "data_2878": {"class_id": "http://edamontology.org/data_2878", "definitions": "3D coordinate and associated data for a protein (3D) structural motif; any group of contiguous or non-contiguous amino acid residues but typically those forming a feature with a structural or functional role.", "label": "Protein structural motif"}, "data_1636": {"class_id": "http://edamontology.org/data_1636", "definitions": "A graphical 2D tabular representation of gene expression data, typically derived from a DNA microarray experiment.|A heat map is a table where rows and columns correspond to different genes and contexts (for example, cells or samples) and the cell color represents the level of expression of a gene that context.", "label": "Heat map"}, "data_2870": {"class_id": "http://edamontology.org/data_2870", "definitions": "A map showing distance between genetic markers estimated by radiation-induced breaks in a chromosome.|The radiation method can break very closely linked markers providing a more detailed map.  Most genetic markers and subsequences may be located to a defined map position and with a more precise estimates of distance than a linkage map.", "label": "Radiation hybrid map"}, "data_1634": {"class_id": "http://edamontology.org/data_1634", "definitions": "A report on linkage disequilibrium; the non-random association of alleles or polymorphisms at two or more loci (not necessarily on the same chromosome).", "label": "Linkage disequilibrium (report)"}, "data_2872": {"class_id": "http://edamontology.org/data_2872", "definitions": "A simple list of data identifiers (such as database accessions), possibly with additional basic information on the addressed data.", "label": "ID list"}, "data_2875": {"class_id": "http://edamontology.org/data_2875", "definitions": "An entry (resource) from the DRCAT bioinformatics resource catalogue.", "label": "DRCAT resource"}, "data_2874": {"class_id": "http://edamontology.org/data_2874", "definitions": "A set of sub-sequences displaying some type of polymorphism, typically indicating the sequence in which they occur, their position and other metadata.", "label": "Sequence set (polymorphic)"}, "data_2877": {"class_id": "http://edamontology.org/data_2877", "definitions": "3D coordinate and associated data for a multi-protein complex; two or more polypeptides chains in a stable, functional association with one another.", "label": "Protein complex"}, "format_3247": {"class_id": "http://edamontology.org/format_3247", "definitions": "mzIdentML is the exchange format for peptides and proteins identified from mass spectra, standardised by HUPO PSI PI. It can be used for outputs of proteomics search engines.", "label": "mzIdentML"}, "operation_0228": {"class_id": "http://edamontology.org/operation_0228", "definitions": "Analyse an index of biological data.", "label": "Data index analysis"}, "operation_2233": {"class_id": "http://edamontology.org/operation_2233", "definitions": "Identify a representative sequence from a set of sequences, typically using scores from pair-wise alignment or other comparison of the sequences.", "label": "Representative sequence identification"}, "operation_0229": {"class_id": "http://edamontology.org/operation_0229", "definitions": "Retrieve basic information about a molecular sequence.", "label": "Annotation retrieval (sequence)"}, "format_3750": {"class_id": "http://edamontology.org/format_3750", "definitions": "YAML (YAML Ain't Markup Language) is a human-readable tree-structured data serialisation language.|Data in YAML format can be serialised into text, or binary format.|YAML version 1.2 is a superset of JSON; prior versions were \"not strictly compatible\".", "label": "YAML"}, "operation_3561": {"class_id": "http://edamontology.org/operation_3561", "definitions": "Compare the models or schemas used by two or more databases, or any other general comparison of databases rather than a detailed comparison of the entries themselves.", "label": "Database comparison"}, "format_2919": {"class_id": "http://edamontology.org/format_2919", "definitions": "Format of a sequence annotation track.", "label": "Sequence annotation track format"}, "format_3751": {"class_id": "http://edamontology.org/format_3751", "definitions": "Tabular data represented as values in a text file delimited by some character.", "label": "DSV"}, "topic_0091": {"class_id": "http://edamontology.org/topic_0091", "definitions": "The archival, curation, processing and analysis of complex biological data.|This includes data  processing in general, including basic handling of files and databases, datatypes, workflows and annotation.", "label": "Bioinformatics"}, "operation_2239": {"class_id": "http://edamontology.org/operation_2239", "definitions": "Calculate a 3D-1D scoring matrix from analysis of protein sequence and structural data.|A 3D-1D scoring matrix scores the probability of amino acids occurring in different structural environments.", "label": "3D-1D scoring matrix generation"}, "format_3586": {"class_id": "http://edamontology.org/format_3586", "definitions": "A BED file where each feature is described by all twelve columns.|Tab delimited data in strict BED format - no non-standard columns allowed; column count forced to 12", "label": "bed12"}, "topic_0092": {"class_id": "http://edamontology.org/topic_0092", "definitions": "Rendering (drawing on a computer screen) or visualisation of molecular sequences, structures or other biomolecular data.", "label": "Data visualisation"}, "topic_0094": {"class_id": "http://edamontology.org/topic_0094", "definitions": "The study of the thermodynamic properties of a nucleic acid.", "label": "Nucleic acid thermodynamics"}, "topic_0097": {"class_id": "http://edamontology.org/topic_0097", "definitions": "The archival, curation, processing and analysis of nucleic acid structural information, such as whole structures, structural features and alignments, and associated annotation.|Includes secondary and tertiary nucleic acid structural data, nucleic acid thermodynamic, thermal and conformational properties including DNA or DNA/RNA denaturation (melting) etc.", "label": "Nucleic acid structure analysis"}, "operation_2238": {"class_id": "http://edamontology.org/operation_2238", "definitions": "Perform a statistical data operation of some type, e.g. calibration or validation.", "label": "Statistical calculation"}, "topic_0099": {"class_id": "http://edamontology.org/topic_0099", "definitions": "RNA sequences and structures.", "label": "RNA"}, "data_1522": {"class_id": "http://edamontology.org/data_1522", "definitions": "A protein sequence with annotation on hydrophobic or hydrophilic / charged regions, hydrophobicity plot etc.|Hydrophobic moment is a peptides hydrophobicity measured for different angles of rotation.", "label": "Protein sequence hydropathy plot"}, "data_1523": {"class_id": "http://edamontology.org/data_1523", "definitions": "A plot of the mean charge of the amino acids within a window of specified length as the window is moved along a protein sequence.", "label": "Protein charge plot"}, "data_1520": {"class_id": "http://edamontology.org/data_1520", "definitions": "Report on the hydrophobic moment of a polypeptide sequence.|Hydrophobic moment is a peptides hydrophobicity measured for different angles of rotation.", "label": "Peptide hydrophobic moment"}, "data_1521": {"class_id": "http://edamontology.org/data_1521", "definitions": "The aliphatic index of a protein.|The aliphatic index is the relative protein volume occupied by aliphatic side chains.", "label": "Protein aliphatic index"}, "data_1526": {"class_id": "http://edamontology.org/data_1526", "definitions": "Data on the stability, intrinsic disorder or globularity of a protein sequence.", "label": "Protein globularity"}, "data_1527": {"class_id": "http://edamontology.org/data_1527", "definitions": "The titration curve of a protein.", "label": "Protein titration curve"}, "data_1524": {"class_id": "http://edamontology.org/data_1524", "definitions": "The solubility or atomic solvation energy of a protein sequence or structure.", "label": "Protein solubility"}, "data_1525": {"class_id": "http://edamontology.org/data_1525", "definitions": "Data on the crystallizability of a protein sequence.", "label": "Protein crystallizability"}, "topic_1302": {"class_id": "http://edamontology.org/topic_1302", "definitions": "Regions or sites in a eukaryotic and eukaryotic viral RNA sequence which directs endonuclease cleavage or polyadenylation of an RNA transcript.", "label": "PolyA signal or sites"}, "data_0832": {"class_id": "http://edamontology.org/data_0832", "definitions": "Controlled vocabulary for gene names (symbols) from HUGO Gene Nomenclature Committee.", "label": "HGNC vocabulary"}, "data_1528": {"class_id": "http://edamontology.org/data_1528", "definitions": "The isoelectric point of one proteins.", "label": "Protein isoelectric point"}, "data_1529": {"class_id": "http://edamontology.org/data_1529", "definitions": "The pKa value of a protein.", "label": "Protein pKa value"}, "topic_0151": {"class_id": "http://edamontology.org/topic_0151", "definitions": "G-protein coupled receptors (GPCRs).", "label": "G protein-coupled receptors (GPCR)"}, "topic_1307": {"class_id": "http://edamontology.org/topic_1307", "definitions": "Splice sites in a nucleotide sequence or alternative RNA splicing events.", "label": "Splice sites"}, "data_0835": {"class_id": "http://edamontology.org/data_0835", "definitions": "Compendium of controlled vocabularies for the biomedical domain (Unified Medical Language System).", "label": "UMLS vocabulary"}, "topic_1305": {"class_id": "http://edamontology.org/topic_1305", "definitions": "Restriction enzyme recognition sites (restriction sites) in a nucleic acid sequence.", "label": "Restriction sites"}, "topic_3397": {"class_id": "http://edamontology.org/topic_3397", "definitions": "Topic concerning the branch of medicine that deals with the prevention, diagnosis, and treatment of disease, disorder and injury in animals.", "label": "Veterinary medicine"}, "topic_3383": {"class_id": "http://edamontology.org/topic_3383", "definitions": "The use of imaging techniques to understand biology.", "label": "Biological imaging"}, "topic_3396": {"class_id": "http://edamontology.org/topic_3396", "definitions": "An interdisciplinary field of study that looks at the dynamic systems of the human body as part of an integrted whole, incoporating biochemical, physiological, and environmental interactions that sustain life.", "label": "Systems medicine"}, "topic_3395": {"class_id": "http://edamontology.org/topic_3395", "definitions": "Biomedical approaches to clinical interventions that involve the use of stem cells.", "label": "Regnerative medicine"}, "operation_0377": {"class_id": "http://edamontology.org/operation_0377", "definitions": "Calculate base frequency or word composition of a nucleotide sequence.", "label": "Sequence composition calculation (nucleic acid)"}, "topic_3394": {"class_id": "http://edamontology.org/topic_3394", "definitions": "The protection of public health by controlling the safety and efficacy of products in areas including pharmaceuticals, veterinary medicine, medical devices, pesticides, agrochemicals, cosmetics, and complementary medicines.", "label": "Regulatory affairs"}, "operation_0372": {"class_id": "http://edamontology.org/operation_0372", "definitions": "Transcribe a nucleotide sequence into mRNA sequence(s).", "label": "DNA transcription"}, "operation_0370": {"class_id": "http://edamontology.org/operation_0370", "definitions": "Create (or remove) restriction sites in sequences, for example using silent mutations.", "label": "Restriction site creation"}, "operation_0371": {"class_id": "http://edamontology.org/operation_0371", "definitions": "Translate a DNA sequence into protein.", "label": "DNA translation"}, "topic_3393": {"class_id": "http://edamontology.org/topic_3393", "definitions": "The processes that need to be in place to ensure the quality of products for human or animal use.", "label": "Quality affairs"}, "topic_0152": {"class_id": "http://edamontology.org/topic_0152", "definitions": "Carbohydrates, typically including structural information.", "label": "Carbohydrates"}, "operation_0378": {"class_id": "http://edamontology.org/operation_0378", "definitions": "Calculate amino acid frequency or word composition of a protein sequence.", "label": "Sequence composition calculation (protein)"}, "operation_0379": {"class_id": "http://edamontology.org/operation_0379", "definitions": "Find (and possibly render) short repetitive subsequences (repeat sequences) in (typically nucleotide) sequences.", "label": "Repeat sequence detection"}, "format_1861": {"class_id": "http://edamontology.org/format_1861", "definitions": "Map of a plasmid (circular DNA) in PlasMapper TextMap format.", "label": "PlasMapper TextMap"}, "topic_3391": {"class_id": "http://edamontology.org/topic_3391", "definitions": "The collective characterisation and quantification of pools of biological molecules that translate into the structure, function, and dynamics of an organism or organisms.", "label": "Omics"}, "format_1369": {"class_id": "http://edamontology.org/format_1369", "definitions": "Format of the model of random sequences used by MEME.", "label": "MEME background Markov model"}, "topic_3390": {"class_id": "http://edamontology.org/topic_3390", "definitions": "The study of the effects of food components on the metabolism, health, performance and disease resistance of humans and animals.  It also includes the study of human behaviours related to food choices.", "label": "Nutritional science"}, "data_3451": {"class_id": "http://edamontology.org/data_3451", "definitions": "Rate of association of a protein with another protein or some other molecule.", "label": "Rate of association"}, "format_2049": {"class_id": "http://edamontology.org/format_2049", "definitions": "Format for phylogenetic tree distance data.", "label": "Phylogenetic tree report (tree distances) format"}, "format_2045": {"class_id": "http://edamontology.org/format_2045", "definitions": "Annotation format for electron microscopy models.", "label": "Electron microscopy model format"}, "topic_0158": {"class_id": "http://edamontology.org/topic_0158", "definitions": "Conserved patterns (motifs) in molecular sequences, that (typically) describe functional or other key sites.", "label": "Sequence motifs"}, "format_2040": {"class_id": "http://edamontology.org/format_2040", "definitions": "Format of phylogenetic invariants data.", "label": "Phylogenetic tree report (invariants) format"}, "data_0909": {"class_id": "http://edamontology.org/data_0909", "definitions": "The maximum initial velocity or rate of a reaction. It is the limiting velocity as substrate concentrations get very large.", "label": "Vmax"}, "data_0907": {"class_id": "http://edamontology.org/data_0907", "definitions": "An informative report on a specific protein family or other classification or group of protein sequences or structures.", "label": "Protein family report"}, "data_0906": {"class_id": "http://edamontology.org/data_0906", "definitions": "An informative report on interactions (predicted or known) within or between a protein, structural domain or part of a protein. This includes intra- and inter-residue contacts and distances, as well as interactions with other proteins and non-protein entities such as nucleic acid, metal atoms, water, ions etc. \n\n \n", "label": "Protein interaction report"}, "data_0905": {"class_id": "http://edamontology.org/data_0905", "definitions": "Protein-protein interaction data from for example yeast two-hybrid analysis, protein microarrays, immunoaffinity chromatography followed by mass spectrometry, phage display etc.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Protein interaction raw data"}, "data_0904": {"class_id": "http://edamontology.org/data_0904", "definitions": "Data on the effect of (typically point) mutation on protein folding, stability, structure and function.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Protein features (mutation)"}, "data_0903": {"class_id": "http://edamontology.org/data_0903", "definitions": "A report on an analysis or model of protein folding properties, folding pathways, residues or sites that are key to protein folding, nucleation or stabilisation centers etc.", "label": "Protein folding report"}, "data_0902": {"class_id": "http://edamontology.org/data_0902", "definitions": "An informative report on architecture (spatial arrangement of secondary structure) of a protein structure.", "label": "Protein architecture report"}, "data_0901": {"class_id": "http://edamontology.org/data_0901", "definitions": "structural domains or 3D folds in a protein or polypeptide chain.", "label": "Protein features report (domains)"}, "data_0900": {"class_id": "http://edamontology.org/data_0900", "definitions": "Data concerning the classification of the sequences and/or structures of protein structural domain(s).", "label": "Protein domain classification"}, "operation_0282": {"class_id": "http://edamontology.org/operation_0282", "definitions": "Generate a genetic (linkage) map of a DNA sequence (typically a chromosome) showing the relative positions of genetic markers based on estimation of non-physical distances.|Mapping involves ordering genetic loci along a chromosome and estimating the physical distance between loci. A genetic map shows the relative (not physical) position of known genes and genetic markers.|This includes mapping of the genetic architecture of dynamic complex traits (functional mapping), e.g. by characterisation of the underlying quantitative trait loci (QTLs) or nucleotides (QTNs).", "label": "Genetic mapping"}, "operation_0283": {"class_id": "http://edamontology.org/operation_0283", "definitions": "Analyse genetic linkage.|For example, estimate how close two genes are on a chromosome by calculating how often they are transmitted together to an offspring, ascertain whether two genes are linked and parental linkage, calculate linkage map distance etc.", "label": "Linkage analysis"}, "operation_0280": {"class_id": "http://edamontology.org/operation_0280", "definitions": "Retrieve information on restriction enzymes or restriction enzyme sites.", "label": "Data retrieval (restriction enzyme annotation)"}, "operation_0281": {"class_id": "http://edamontology.org/operation_0281", "definitions": "Identify genetic markers in DNA sequences.|A genetic marker is any DNA sequence of known chromosomal location that is associated with and specific to a particular gene or trait. This includes short sequences surrounding a SNP, Sequence-Tagged Sites (STS) which are well suited for PCR amplification, a longer minisatellites sequence etc.", "label": "Genetic marker identification"}, "operation_0286": {"class_id": "http://edamontology.org/operation_0286", "definitions": "Process (read and / or write) codon usage data, e.g. analyse codon usage tables or codon usage in molecular sequences.", "label": "Codon usage analysis"}, "operation_0287": {"class_id": "http://edamontology.org/operation_0287", "definitions": "Identify and plot third base position variability in a nucleotide sequence.", "label": "Base position variability plotting"}, "data_2648": {"class_id": "http://edamontology.org/data_2648", "definitions": "Identifier of a biological reaction (kinetics entry) from the SABIO-RK reactions database.", "label": "Reaction kinetics ID (SABIO-RK)"}, "data_2649": {"class_id": "http://edamontology.org/data_2649", "definitions": "Identifier of an entry from the pharmacogenetics and pharmacogenomics knowledge base (PharmGKB).", "label": "PharmGKB ID"}, "data_2646": {"class_id": "http://edamontology.org/data_2646", "definitions": "Identifier of a small molecular from the ChEMBL database.", "label": "Compound ID (ChEMBL)"}, "data_2647": {"class_id": "http://edamontology.org/data_2647", "definitions": "Unique identifier of an entry from the Ligand-gated ion channel (LGICdb) database.", "label": "LGICdb identifier"}, "data_2644": {"class_id": "http://edamontology.org/data_2644", "definitions": "Identifier of an enzyme-catalysed reaction from the Rhea database.", "label": "Reaction ID (Rhea)"}, "data_2645": {"class_id": "http://edamontology.org/data_2645", "definitions": "Identifier of a biological pathway from the Unipathway database.", "label": "Pathway ID (Unipathway)"}, "data_2642": {"class_id": "http://edamontology.org/data_2642", "definitions": "Identifier for a gene from the miRBase database.", "label": "Gene ID (miRBase)"}, "data_2643": {"class_id": "http://edamontology.org/data_2643", "definitions": "Identifier for a gene from the Zebrafish information network genome (ZFIN) database.", "label": "Gene ID (ZFIN)"}, "data_2641": {"class_id": "http://edamontology.org/data_2641", "definitions": "Identifier of an enzyme reaction mechanism from the MACie database.", "label": "Reaction ID (MACie)"}, "data_2088": {"class_id": "http://edamontology.org/data_2088", "definitions": "Structural data for DNA base pairs or runs of bases, such as energy or angle data.", "label": "DNA base structural data"}, "data_2084": {"class_id": "http://edamontology.org/data_2084", "definitions": "An informative human-readable report about one or more specific nucleic acid molecules, derived from analysis of primary (sequence or structural) data.", "label": "Nucleic acid report"}, "data_2085": {"class_id": "http://edamontology.org/data_2085", "definitions": "An informative report on general information, properties or features of one or more molecular tertiary (3D) structures.", "label": "Structure report"}, "data_2086": {"class_id": "http://edamontology.org/data_2086", "definitions": "A report on nucleic acid structure-derived data, describing structural properties of a DNA molecule, or any other annotation or information about specific nucleic acid 3D structure(s).|This includes reports on the stiffness, curvature, twist/roll data or other conformational parameters or properties.", "label": "Nucleic acid structure data"}, "data_2087": {"class_id": "http://edamontology.org/data_2087", "definitions": "A report on the physical (e.g. structural) or chemical properties of molecules, or parts of a molecule.", "label": "Molecular property"}, "data_2080": {"class_id": "http://edamontology.org/data_2080", "definitions": "A report of hits from searching a database of some type.", "label": "Database search results"}, "data_2081": {"class_id": "http://edamontology.org/data_2081", "definitions": "The secondary structure assignment (predicted or real) of a nucleic acid or protein.", "label": "Secondary structure"}, "data_2082": {"class_id": "http://edamontology.org/data_2082", "definitions": "An array of numerical values.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Matrix"}, "data_2083": {"class_id": "http://edamontology.org/data_2083", "definitions": "Data concerning, extracted from, or derived from the analysis of molecular alignment of some type.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Alignment data"}, "operation_0574": {"class_id": "http://edamontology.org/operation_0574", "definitions": "Render a sequence with motifs.", "label": "Sequence motif rendering"}, "operation_0575": {"class_id": "http://edamontology.org/operation_0575", "definitions": "Draw or visualise restriction maps in DNA sequences.", "label": "Restriction map drawing"}, "operation_0577": {"class_id": "http://edamontology.org/operation_0577", "definitions": "Draw a linear maps of DNA.", "label": "DNA linear map rendering"}, "operation_0570": {"class_id": "http://edamontology.org/operation_0570", "definitions": "Visualise or render molecular structure, for example a high-quality static picture or animation.  This includes secondary structure such as knots, pseudoknots etc. as well as tertiary and quaternary structure.", "label": "Structure visualisation"}, "operation_0571": {"class_id": "http://edamontology.org/operation_0571", "definitions": "Visualise microarray or other expression data.", "label": "Expression data visualisation"}, "operation_0572": {"class_id": "http://edamontology.org/operation_0572", "definitions": "Identify and analyse networks of protein interactions.", "label": "Protein interaction network visualisation"}, "operation_0573": {"class_id": "http://edamontology.org/operation_0573", "definitions": "Draw or visualise a DNA map.", "label": "Map drawing"}, "operation_0578": {"class_id": "http://edamontology.org/operation_0578", "definitions": "Draw a circular maps of DNA, for example a plasmid map.", "label": "Plasmid map drawing"}, "operation_0579": {"class_id": "http://edamontology.org/operation_0579", "definitions": "Visualise operon structure etc.", "label": "Operon drawing"}, "operation_3455": {"class_id": "http://edamontology.org/operation_3455", "definitions": "A technique used to construct an atomic model of an unknown structure from diffraction data, based upon an atomic model of a known structure, either a related protein or the same protein from a different crystal form.|The technique solves the phase problem, i.e. retrieve information concern phases of the structure.", "label": "Molecular replacement"}, "operation_0476": {"class_id": "http://edamontology.org/operation_0476", "definitions": "Predict tertiary structure of protein sequence(s) without homologs of known structure.", "label": "Ab initio structure prediction"}, "format_1665": {"class_id": "http://edamontology.org/format_1665", "definitions": "Format of Taverna workflows.", "label": "Taverna workflow format"}, "format_1666": {"class_id": "http://edamontology.org/format_1666", "definitions": "Format of mathematical models from the BioModel database.|Models are annotated and linked to relevant data resources, such as publications, databases of compounds and pathways, controlled vocabularies, etc.", "label": "BioModel mathematical model format"}, "topic_2259": {"class_id": "http://edamontology.org/topic_2259", "definitions": "The holistic modelling and analysis of complex biological systems and the interactions therein.|This includes databases of models and methods to construct or analyse a model.", "label": "Systems biology"}, "oboInOwl:ObsoleteClass": {"class_id": "http://www.geneontology.org/formats/oboInOwl#ObsoleteClass", "definitions": "An obsolete concept (redefined in EDAM).|Needed for conversion to the OBO format.", "label": "Obsolete concept (EDAM)"}, "operation_3481": {"class_id": "http://edamontology.org/operation_3481", "definitions": "Generate sequences from some probabilistic model, e.g. a model that simulates evolution.", "label": "Probabilistic sequence generation"}, "operation_3480": {"class_id": "http://edamontology.org/operation_3480", "definitions": "Generate some data from a choosen probibalistic model, possibly to evaluate algorithms.", "label": "Probabilistic data generation"}, "operation_3482": {"class_id": "http://edamontology.org/operation_3482", "definitions": "Identify or predict causes for antibiotic resistance from molecular sequence analysis.", "label": "Antimicrobial resistance prediction"}, "topic_0804": {"class_id": "http://edamontology.org/topic_0804", "definitions": "The application of information technology to immunology such as immunological processes, immunological genes, proteins and peptide ligands, antigens and so on.", "label": "Immunology"}, "operation_3715": {"class_id": "http://edamontology.org/operation_3715", "definitions": "Labeling all proteins and (possibly) all amino acids using C-13 or N-15 enriched grown medium or feed.|This includes N-15 metabolic labeling (labeling all proteins and (possibly) all amino acids using N-15 enriched grown medium or feed) and C-13 metabolic labeling (labeling all proteins and (possibly) all amino acids using C-13 enriched grown medium or feed).", "label": "Metabolic labeling"}, "operation_2446": {"class_id": "http://edamontology.org/operation_2446", "definitions": "Process (read and / or write) one or more molecular sequences and associated annotation.", "label": "Sequence processing"}, "operation_2447": {"class_id": "http://edamontology.org/operation_2447", "definitions": "Process (read and / or write) a protein sequence and associated annotation.", "label": "Sequence processing (protein)"}, "operation_2444": {"class_id": "http://edamontology.org/operation_2444", "definitions": "Process (read and / or write) protein secondary structure data.", "label": "Protein secondary structure processing"}, "operation_2445": {"class_id": "http://edamontology.org/operation_2445", "definitions": "Process (read and / or write) a network of protein interactions.", "label": "Protein interaction network processing"}, "operation_2442": {"class_id": "http://edamontology.org/operation_2442", "definitions": "Predict DNA tertiary structure.", "label": "DNA structure prediction"}, "operation_2443": {"class_id": "http://edamontology.org/operation_2443", "definitions": "Generate, process or analyse phylogenetic tree or trees.", "label": "Phylogenetic tree processing"}, "operation_2440": {"class_id": "http://edamontology.org/operation_2440", "definitions": "Process (read and / or write) RNA tertiary structure data.", "label": "Structure processing (RNA)"}, "operation_2441": {"class_id": "http://edamontology.org/operation_2441", "definitions": "Predict RNA tertiary structure.", "label": "RNA structure prediction"}, "operation_2990": {"class_id": "http://edamontology.org/operation_2990", "definitions": "Assign molecular sequences, structures or other biological data to a specific group or category according to qualities it shares with that group or category.", "label": "Classification"}, "format_3579": {"class_id": "http://edamontology.org/format_3579", "definitions": "Joint Picture Group file format for lossy graphics file.|Sequence of segments with markers. Begins with byte of 0xFF and follows by marker type.", "label": "JPG"}, "operation_2995": {"class_id": "http://edamontology.org/operation_2995", "definitions": "Assign molecular sequence(s) to a group or category.", "label": "Sequence classification"}, "operation_2448": {"class_id": "http://edamontology.org/operation_2448", "definitions": "Process (read and / or write) a nucleotide sequence and associated annotation.", "label": "Sequence processing (nucleic acid)"}, "operation_2997": {"class_id": "http://edamontology.org/operation_2997", "definitions": "Compare two or more proteins (or some aspect) to identify similarities.", "label": "Protein comparison"}, "data_1456": {"class_id": "http://edamontology.org/data_1456", "definitions": "trans- or intra-membrane regions of a protein, typically describing physicochemical properties of the secondary structure elements.", "label": "Protein features report (membrane regions)"}, "format_1247": {"class_id": "http://edamontology.org/format_1247", "definitions": "Format of an entry from the COG database of clusters of (related) protein sequences.", "label": "COG sequence cluster format"}, "format_1248": {"class_id": "http://edamontology.org/format_1248", "definitions": "Format for sequence positions (feature location) as used in DDBJ/EMBL/GenBank database.", "label": "EMBL feature location"}, "data_3111": {"class_id": "http://edamontology.org/data_3111", "definitions": "Data generated from processing and analysis of probe set data from a microarray experiment.|Such data as found in Affymetrix .CHP files or data from other software such as RMA or dChip.", "label": "Processed microarray data"}, "data_3110": {"class_id": "http://edamontology.org/data_3110", "definitions": "Raw data (typically MIAME-compliant) for hybridisations from a microarray experiment.|Such data as found in Affymetrix CEL or GPR files.", "label": "Raw microarray data"}, "data_3113": {"class_id": "http://edamontology.org/data_3113", "definitions": "Annotation on a biological sample, for example experimental factors and their values.|This might include compound and dose in a dose response experiment.", "label": "Sample annotation"}, "data_3112": {"class_id": "http://edamontology.org/data_3112", "definitions": "The final processed (normalised) data for a set of hybridisations in a microarray experiment.|This combines data from all hybridisations.", "label": "Gene expression matrix"}, "data_3115": {"class_id": "http://edamontology.org/data_3115", "definitions": "Annotation on the array itself used in a microarray experiment.|This might include gene identifiers, genomic coordinates, probe oligonucleotide sequences etc.", "label": "Microarray metadata"}, "data_3117": {"class_id": "http://edamontology.org/data_3117", "definitions": "Data concerning the hybridisations measured during a microarray experiment.", "label": "Microarray hybridisation data"}, "data_3116": {"class_id": "http://edamontology.org/data_3116", "definitions": "Annotation on laboratory and/or data processing protocols used in an microarray experiment.|This might describe e.g. the normalisation methods used to process the raw data.", "label": "Microarray protocol annotation"}, "data_3119": {"class_id": "http://edamontology.org/data_3119", "definitions": "A report of regions in a molecular sequence that are biased to certain characters.", "label": "Sequence features (compositionally-biased regions)"}, "data_1452": {"class_id": "http://edamontology.org/data_1452", "definitions": "Matrix of integer numbers for amino acid comparison.", "label": "Amino acid comparison matrix (integers)"}, "operation_1812": {"class_id": "http://edamontology.org/operation_1812", "definitions": "Prepare or load a user-specified data file so that it is available for use.", "label": "Loading"}, "topic_0787": {"class_id": "http://edamontology.org/topic_0787", "definitions": "Rice-specific data.", "label": "Rice"}, "data_1453": {"class_id": "http://edamontology.org/data_1453", "definitions": "Matrix of floating point numbers for amino acid comparison.", "label": "Amino acid comparison matrix (floats)"}, "topic_2807": {"class_id": "http://edamontology.org/topic_2807", "definitions": "A topic concerning primarily bioinformatics software tools, typically the broad function or purpose of a tool.", "label": "Tool topic"}, "data_1450": {"class_id": "http://edamontology.org/data_1450", "definitions": "Matrix of integer numbers for nucleotide comparison.", "label": "Nucleotide comparison matrix (integers)"}, "topic_3539": {"class_id": "http://edamontology.org/topic_3539", "definitions": "Structural domains or 3D folds in a protein or polypeptide chain.", "label": "Protein domains"}, "topic_3538": {"class_id": "http://edamontology.org/topic_3538", "definitions": "Disordered structure in a protein.", "label": "Protein disordered structure"}, "topic_3537": {"class_id": "http://edamontology.org/topic_3537", "definitions": "Chemical modification of a protein.", "label": "Protein chemical modifications"}, "topic_3536": {"class_id": "http://edamontology.org/topic_3536", "definitions": "Cleavage sites (for a proteolytic enzyme or agent) in a protein sequence.", "label": "Protein cleavage sites"}, "topic_3535": {"class_id": "http://edamontology.org/topic_3535", "definitions": "RNA and DNA-binding proteins and binding sites in protein sequences.", "label": "Protein-nucleic acid binding sites"}, "data_1451": {"class_id": "http://edamontology.org/data_1451", "definitions": "Matrix of floating point numbers for nucleotide comparison.", "label": "Nucleotide comparison matrix (floats)"}, "topic_3533": {"class_id": "http://edamontology.org/topic_3533", "definitions": "Catalytic residues (active site) of an enzyme.", "label": "Protein active sites"}, "format_1391": {"class_id": "http://edamontology.org/format_1391", "definitions": "FASTA-style format for multiple sequences aligned by HMMER package to an HMM.", "label": "HMMER-aln"}, "format_1392": {"class_id": "http://edamontology.org/format_1392", "definitions": "Format of multiple sequences aligned by DIALIGN package.", "label": "DIALIGN format"}, "format_1393": {"class_id": "http://edamontology.org/format_1393", "definitions": "EMBASSY 'domain alignment file' (DAF) format, containing a sequence alignment of protein domains belonging to the same SCOP or CATH family.|The format is clustal-like and includes annotation of domain family classification information.", "label": "daf"}, "format_3477": {"class_id": "http://edamontology.org/format_3477", "definitions": "Format of the cytoscape input file of gene expression ratios or values are specified over one or more experiments.", "label": "Cytoscape input file format"}, "topic_0786": {"class_id": "http://edamontology.org/topic_0786", "definitions": "Arabidopsis-specific data.", "label": "Arabidopsis"}, "operation_3229": {"class_id": "http://edamontology.org/operation_3229", "definitions": "Anaylse sequencing data from experiments aiming to selectively sequence the coding regions of the genome.", "label": "Exome assembly"}, "operation_3228": {"class_id": "http://edamontology.org/operation_3228", "definitions": "Detect large regions in a genome subject to copy-number variation, or other structural variations in genome(s).|Methods might involve analysis of whole-genome array comparative genome hybridisation or single-nucleotide polymorphism arrays, paired-end mapping of sequencing data, or from analysis of short reads from new sequencing technologies.", "label": "Structural variation detection"}, "operation_3225": {"class_id": "http://edamontology.org/operation_3225", "definitions": "Classify variants based on their potential effect on genes, especially functional effects on the expressed proteins.|Variants are typically classified by their position (intronic, exonic, etc.) in a gene transcript and (for variants in coding exons) by their effect on the protein sequence (synonymous, non-synonymous, frameshifting, etc.)", "label": "Variant classification"}, "operation_3224": {"class_id": "http://edamontology.org/operation_3224", "definitions": "Analyse gene expression patterns (typically from DNA microarray datasets) to identify sets of genes that are associated with a specific trait, condition, clinical outcome etc.|Gene sets can be defined beforehand by biological function, chromosome locations and so on.", "label": "Gene set testing"}, "operation_3227": {"class_id": "http://edamontology.org/operation_3227", "definitions": "Identify and map genomic alterations, including single nucleotide polymorphisms, short indels and structural variants, in a genome sequence.|Somatic variant calling is the detection of variations established in somatic cells and hence not inherited as a germ line variant.|Variant detection|Methods often utilise a database of aligned reads.", "label": "Variant calling"}, "operation_3226": {"class_id": "http://edamontology.org/operation_3226", "definitions": "Identify biologically interesting variants by prioritizing individual variants, for example, homozygous variants absent in control genomes.|Variant prioritisation can be used for example to produce a list of variants responsible for 'knocking out' genes in specific genomes. Methods amino acid substitution, aggregative approaches, probabilistic approach, inheritance and unified likelihood-frameworks.", "label": "Variant prioritisation"}, "operation_3221": {"class_id": "http://edamontology.org/operation_3221", "definitions": "Estimate the frequencies of different species from analysis of the molecular sequences, typically of DNA recovered from environmental samples.", "label": "Species frequency estimation"}, "operation_3223": {"class_id": "http://edamontology.org/operation_3223", "definitions": "Identify from molecular sequence analysis (typically from analysis of microarray or RNA-seq data) genes whose expression levels are significantly different between two sample groups.|Differential gene expression analysis is used, for example, to identify which genes are up-regulated (increased expression) or down-regulated (decreased expression) between a group treated with a drug and a control groups.", "label": "Differential gene expression analysis"}, "operation_3222": {"class_id": "http://edamontology.org/operation_3222", "definitions": "Identify putative protein-binding regions in a genome sequence from analysis of Chip-sequencing data or ChIP-on-chip data.|Chip-sequencing combines chromatin immunoprecipitation (ChIP) with massively parallel DNA sequencing to generate a set of reads, which are aligned to a genome sequence.  The enriched areas contain the binding sites of DNA-associated proteins.  For example, a transcription factor binding site. ChIP-on-chip in contrast combines chromatin immunoprecipitation ('ChIP') with microarray ('chip').  \"Peak-pair calling\" is similar to \"Peak calling\" in the context of ChIP-exo.", "label": "Peak calling"}, "format_1927": {"class_id": "http://edamontology.org/format_1927", "definitions": "EMBL entry format.", "label": "EMBL format"}, "topic_2869": {"class_id": "http://edamontology.org/topic_2869", "definitions": "Restriction fragment length polymorphisms (RFLP) in a DNA sequence.", "label": "RFLP"}, "operation_2412": {"class_id": "http://edamontology.org/operation_2412", "definitions": "Process (read and / or write) an index of (typically a file of) biological data.", "label": "Data index processing"}, "format_1808": {"class_id": "http://edamontology.org/format_1808", "definitions": "Report format for biological functions associated with a gene name and its alternative names (synonyms, homonyms), as generated by the GeneIlluminator service.|This includes a gene name and abbreviation of the name which may be in a name space indicating the gene status and relevant organisation.", "label": "GeneIlluminator gene report format"}, "format_1809": {"class_id": "http://edamontology.org/format_1809", "definitions": "Format of a report on the DNA and protein sequences for a given gene label from a bacterial chromosome maps from the BacMap database.", "label": "BacMap gene card format"}, "data_3025": {"class_id": "http://edamontology.org/data_3025", "definitions": "Identifier of a concept in an ontology of biological or bioinformatics concepts and relations.", "label": "Ontology concept identifier"}, "format_3770": {"class_id": "http://edamontology.org/format_3770", "definitions": "UniProtKB XML sequence features format is an XML format available for downloading UniProt entries.", "label": "UniProtKB XML"}, "data_3027": {"class_id": "http://edamontology.org/data_3027", "definitions": "The name of a concept for a molecular function from the GO ontology.", "label": "GO concept name (molecular function)"}, "data_3026": {"class_id": "http://edamontology.org/data_3026", "definitions": "The name of a concept for a biological process from the GO ontology.", "label": "GO concept name (biological process)"}, "data_3021": {"class_id": "http://edamontology.org/data_3021", "definitions": "Accession number of a UniProt (protein sequence) database entry.", "label": "UniProt accession"}, "data_3022": {"class_id": "http://edamontology.org/data_3022", "definitions": "Identifier of a genetic code in the NCBI list of genetic codes.", "label": "NCBI genetic code ID"}, "data_3029": {"class_id": "http://edamontology.org/data_3029", "definitions": "EMBL/GENBANK/DDBJ coding feature protein identifier, issued by International collaborators.|This qualifier consists of a stable ID portion (3+5 format with 3 position letters and 5 numbers) plus a version number after the decimal point. When the protein sequence encoded by the CDS changes, only the version number of the /protein_id value is incremented; the stable part of the /protein_id remains unchanged and as a result will permanently be associated with a given protein; this qualifier is valid only on CDS features which translate into a valid protein.", "label": "Protein ID (EMBL/GenBank/DDBJ)"}, "data_3028": {"class_id": "http://edamontology.org/data_3028", "definitions": "Data concerning the classification, identification and naming of organisms.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Taxonomy"}, "format_1926": {"class_id": "http://edamontology.org/format_1926", "definitions": "Fasta format variant with database name before ID.", "label": "dbid"}, "operation_3478": {"class_id": "http://edamontology.org/operation_3478", "definitions": "Reconstructing the inner node labels of a phylogenetic tree from its leafes.|Note that this is somewhat different from simply analysing an existing tree or constructing a completely new one.", "label": "Phylogenetic tree reconstruction"}, "format_3774": {"class_id": "http://edamontology.org/format_3774", "definitions": "BioJSON is a JSON format of single multiple sequence alignments, with their annotations, features, and custom visualisation and application settings for the Jalview workbench.", "label": "BioJSON (Jalview)"}, "topic_3571": {"class_id": "http://edamontology.org/topic_3571", "definitions": "The control of data entry and maintenance to ensure the data meets defined standards, qualities or constraints.", "label": "Data governance"}, "format_3775": {"class_id": "http://edamontology.org/format_3775", "definitions": "GSuite is a tabular format for collections of genome or sequence feature tracks, suitable for integrative multi-track analysis. GSuite contains links to genome/sequence tracks, with additional metadata.", "label": "GSuite"}, "topic_3570": {"class_id": "http://edamontology.org/topic_3570", "definitions": "The study of abstract mathematical concepts.", "label": "Pure mathematics"}, "format_3782": {"class_id": "http://edamontology.org/format_3782", "definitions": "BioC is a standardised XML format for sharing and integrating text data and annotations.", "label": "BioC"}, "data_1692": {"class_id": "http://edamontology.org/data_1692", "definitions": "The name of a person.", "label": "Person name"}, "operation_2264": {"class_id": "http://edamontology.org/operation_2264", "definitions": "Query a biological pathways database and retrieve annotation on one or more pathways.", "label": "Data retrieval (pathway or network)"}, "data_0988": {"class_id": "http://edamontology.org/data_0988", "definitions": "Identifier of a peptide chain.", "label": "Peptide identifier"}, "data_1290": {"class_id": "http://edamontology.org/data_1290", "definitions": "Image showing matches between protein sequence(s) and InterPro Entries.|The sequence(s) might be screened against InterPro, or be the sequences from the InterPro entry itself. Each protein is represented as a scaled horizontal line with colored bars indicating the position of the matches.", "label": "InterPro compact match image"}, "data_1291": {"class_id": "http://edamontology.org/data_1291", "definitions": "Image showing detailed information on matches between protein sequence(s) and InterPro Entries.|The sequence(s) might be screened against InterPro, or be the sequences from the InterPro entry itself.", "label": "InterPro detailed match image"}, "data_1292": {"class_id": "http://edamontology.org/data_1292", "definitions": "Image showing the architecture of InterPro domains in a protein sequence.|The sequence(s) might be screened against InterPro, or be the sequences from the InterPro entry itself. Domain architecture is shown as a series of non-overlapping domains in the protein.", "label": "InterPro architecture image"}, "data_1293": {"class_id": "http://edamontology.org/data_1293", "definitions": "SMART protein schematic in PNG format.", "label": "SMART protein schematic"}, "data_1294": {"class_id": "http://edamontology.org/data_1294", "definitions": "Images based on GlobPlot prediction of intrinsic disordered regions and globular domains in protein sequences.", "label": "GlobPlot domain image"}, "data_1298": {"class_id": "http://edamontology.org/data_1298", "definitions": "Report on the location of matches to profiles, motifs (conserved or functional patterns) or other signatures in one or more sequences.", "label": "Sequence motif matches"}, "data_1299": {"class_id": "http://edamontology.org/data_1299", "definitions": "Location of short repetitive subsequences (repeat sequences) in (typically nucleotide) sequences.|The report might include derived data map such as classification, annotation, organisation, periodicity etc.", "label": "Sequence features (repeats)"}, "format_1932": {"class_id": "http://edamontology.org/format_1932", "definitions": "FASTQ short read format with phred quality.", "label": "FASTQ-sanger"}, "format_1933": {"class_id": "http://edamontology.org/format_1933", "definitions": "FASTQ Solexa/Illumina 1.0 short read format.", "label": "FASTQ-solexa"}, "format_1934": {"class_id": "http://edamontology.org/format_1934", "definitions": "Fitch program format.", "label": "fitch program"}, "format_1935": {"class_id": "http://edamontology.org/format_1935", "definitions": "GCG sequence file format.| GCG SSF (single sequence file) file format.", "label": "GCG"}, "format_1936": {"class_id": "http://edamontology.org/format_1936", "definitions": "Genbank entry format.", "label": "GenBank format"}, "format_1937": {"class_id": "http://edamontology.org/format_1937", "definitions": "Genpept protein entry format.|Currently identical to refseqp format", "label": "genpept"}, "format_1747": {"class_id": "http://edamontology.org/format_1747", "definitions": "Format of an ATOM record (describing data for an individual atom) from a PDB file.", "label": "PDB atom record format"}, "operation_3471": {"class_id": "http://edamontology.org/operation_3471", "definitions": "Prediction of nucleic-acid folding using sequence alignments as a source of data.", "label": "Nucleic acid folding prediction (alignment-based)"}, "operation_3472": {"class_id": "http://edamontology.org/operation_3472", "definitions": "Count k-mers (substrings of length k) in DNA sequence data.|k-mer counting is used in genome and transcriptome assembly, metagenomic sequencing, and for error correction of sequence reads. ", "label": "k-mer counting"}, "operation_3503": {"class_id": "http://edamontology.org/operation_3503", "definitions": "Plot an incident curve such as a survival curve, death curve, mortality curve.", "label": "Incident curve plotting"}, "format_3657": {"class_id": "http://edamontology.org/format_3657", "definitions": "Graphical Pathway Markup Language (GPML) is an XML format used for exchanging biological pathways.", "label": "GPML"}, "format_3655": {"class_id": "http://edamontology.org/format_3655", "definitions": "Open data format for the storage, exchange, and processing of peptide sequence assignments of MS/MS scans, intended to provide a common data output format for many different MS/MS search engines and subsequent peptide-level analyses.", "label": "pepXML"}, "format_3654": {"class_id": "http://edamontology.org/format_3654", "definitions": "Common file format for proteomics mass spectrometric data developed at the Seattle Proteome Center/Institute for Systems Biology.", "label": "mzXML"}, "format_3653": {"class_id": "http://edamontology.org/format_3653", "definitions": "Spectral data file similar to dta.|Differ from .dta only in subtleties of the header line format and content and support the added feature of being able to.", "label": "pkl"}, "format_3652": {"class_id": "http://edamontology.org/format_3652", "definitions": "Spectral data format file where each spectrum is written to a separate file.|Each file contains one header line for the known or assumed charge and the mass of the precursor peptide ion, calculated from the measured *m*/*z* and the charge. This one line was then followed by all the *m*/*z*, intensity pairs that represent the spectrum.", "label": "dta"}, "format_3651": {"class_id": "http://edamontology.org/format_3651", "definitions": "Mascot Generic Format.  Encodes multiple MS/MS spectra in a single file.|Files includes *m*/*z*, intensity pairs separated by headers; headers can contain a bit more information, including search engine instructions.", "label": "MGF"}, "format_3650": {"class_id": "http://edamontology.org/format_3650", "definitions": "Format used by netCDF software  library for writing and reading chromatography-MS data files.", "label": "netCDF"}, "format_3099": {"class_id": "http://edamontology.org/format_3099", "definitions": "Format of raw CATH domain classification data files.|These are the parsable data files provided by CATH.", "label": "Raw CATH domain classification format"}, "format_3098": {"class_id": "http://edamontology.org/format_3098", "definitions": "Format of raw SCOP domain classification data files.|These are the parsable data files provided by SCOP.", "label": "Raw SCOP domain classification format"}, "format_1741": {"class_id": "http://edamontology.org/format_1741", "definitions": "OSCAR format of annotated chemical text.|OSCAR (Open-Source Chemistry Analysis Routines) software performs chemistry-specific parsing of chemical documents. It attempts to identify chemical names, ontology concepts, and chemical data from a document.", "label": "OSCAR format"}, "topic_2661": {"class_id": "http://edamontology.org/topic_2661", "definitions": "Structural and associated data for toxic chemical substances.", "label": "Toxins and targets"}, "format_1740": {"class_id": "http://edamontology.org/format_1740", "definitions": "The format of iHOP (Information Hyperlinked over Proteins) text-mining result.", "label": "iHOP format"}, "format_3097": {"class_id": "http://edamontology.org/format_3097", "definitions": "Format of data concerning the classification of the sequences and/or structures of protein structural domain(s).", "label": "Protein domain classification format"}, "data_1746": {"class_id": "http://edamontology.org/data_1746", "definitions": "Cartesian z coordinate of an atom (in a molecular structure).", "label": "Atomic z coordinate"}, "data_1744": {"class_id": "http://edamontology.org/data_1744", "definitions": "Cartesian x coordinate of an atom (in a molecular structure).", "label": "Atomic x coordinate"}, "data_1745": {"class_id": "http://edamontology.org/data_1745", "definitions": "Cartesian y coordinate of an atom (in a molecular structure).", "label": "Atomic y coordinate"}, "data_1742": {"class_id": "http://edamontology.org/data_1742", "definitions": "A residue identifier (a string) from a PDB file.", "label": "PDB residue number"}, "data_1743": {"class_id": "http://edamontology.org/data_1743", "definitions": "Cartesian coordinate of an atom (in a molecular structure).", "label": "Atomic coordinate"}, "data_1748": {"class_id": "http://edamontology.org/data_1748", "definitions": "Identifier (a string) of a specific atom from a PDB file for a molecular structure.", "label": "PDB atom name"}, "data_2884": {"class_id": "http://edamontology.org/data_2884", "definitions": "Biological data that has been plotted as a graph of some type.", "label": "Plot"}, "data_2885": {"class_id": "http://edamontology.org/data_2885", "definitions": "DNA polymorphism.", "label": "Nucleic acid features report (polymorphism)"}, "data_2886": {"class_id": "http://edamontology.org/data_2886", "definitions": "A protein sequence and associated metadata.", "label": "Protein sequence record"}, "data_2887": {"class_id": "http://edamontology.org/data_2887", "definitions": "A nucleic acid sequence and associated metadata.", "label": "Nucleic acid sequence record"}, "data_2880": {"class_id": "http://edamontology.org/data_2880", "definitions": "Image of one or more molecular secondary structures.", "label": "Secondary structure image"}, "data_2881": {"class_id": "http://edamontology.org/data_2881", "definitions": "An informative report on general information, properties or features of one or more molecular secondary structures.", "label": "Secondary structure report"}, "data_2882": {"class_id": "http://edamontology.org/data_2882", "definitions": "DNA sequence-specific feature annotation (not in a feature table).", "label": "DNA features"}, "data_2883": {"class_id": "http://edamontology.org/data_2883", "definitions": "Features concerning RNA or regions of DNA that encode an RNA molecule.", "label": "RNA features report"}, "format_1624": {"class_id": "http://edamontology.org/format_1624", "definitions": "Format of a record from the HGVbase database of genotypes and phenotypes.", "label": "HGVbase entry format"}, "data_2888": {"class_id": "http://edamontology.org/data_2888", "definitions": "A protein sequence and comprehensive metadata (such as a feature table), typically corresponding to a full entry from a molecular sequence database.", "label": "Protein sequence record (full)"}, "data_2889": {"class_id": "http://edamontology.org/data_2889", "definitions": "A nucleic acid sequence and comprehensive metadata (such as a feature table), typically corresponding to a full entry from a molecular sequence database.", "label": "Nucleic acid sequence record (full) "}, "format_1197": {"class_id": "http://edamontology.org/format_1197", "definitions": "Chemical structure specified in IUPAC International Chemical Identifier (InChI) line notation.", "label": "InChI"}, "operation_0365": {"class_id": "http://edamontology.org/operation_0365", "definitions": "Generate digest fragments for a nucleotide sequence containing restriction sites.", "label": "Nucleic acid restriction digest"}, "format_1626": {"class_id": "http://edamontology.org/format_1626", "definitions": "Format of an entry from the KEGG DISEASE database.", "label": "KEGG DISEASE entry format"}, "operation_3278": {"class_id": "http://edamontology.org/operation_3278", "definitions": "Calculate similarity between 2 or more documents.", "label": "Document similarity calculation"}, "data_1602": {"class_id": "http://edamontology.org/data_1602", "definitions": "The differences in codon usage fractions between two codon usage tables.", "label": "Codon usage fraction difference"}, "data_1601": {"class_id": "http://edamontology.org/data_1601", "definitions": "The effective number of codons used in a gene sequence.  This reflects how far codon usage of a gene departs from equal usage of synonymous codons.", "label": "Nc statistic"}, "data_1600": {"class_id": "http://edamontology.org/data_1600", "definitions": "A plot of the synonymous codon usage calculated for windows over a nucleotide sequence.", "label": "Codon usage bias plot"}, "topic_0153": {"class_id": "http://edamontology.org/topic_0153", "definitions": "Lipids and their structures.", "label": "Lipids"}, "data_2866": {"class_id": "http://edamontology.org/data_2866", "definitions": "Northern Blot experiments.", "label": "Northern blot report"}, "data_2867": {"class_id": "http://edamontology.org/data_2867", "definitions": "variable number of tandem repeat (VNTR) polymorphism in a DNA sequence.", "label": "Nucleic acid features report (VNTR)"}, "data_2865": {"class_id": "http://edamontology.org/data_2865", "definitions": "A numerical measure of differences in the frequency of occurrence of synonymous codons in DNA sequences.", "label": "Codon usage bias"}, "format_1582": {"class_id": "http://edamontology.org/format_1582", "definitions": "A report format for the kinetics of enzyme-catalysed reaction(s) in a format generated by EMBOSS findkm. This includes Michaelis Menten plot, Hanes Woolf plot, Michaelis Menten constant (Km) and maximum velocity (Vmax).", "label": "findkm"}, "data_3497": {"class_id": "http://edamontology.org/data_3497", "definitions": "A raw DNA sequence.", "label": "DNA sequence (raw)"}, "topic_0798": {"class_id": "http://edamontology.org/topic_0798", "definitions": "Mobile genetic elements, such as transposons, Plasmids, Bacteriophage elements and Group II introns.", "label": "Mobile genetic elements"}, "topic_0796": {"class_id": "http://edamontology.org/topic_0796", "definitions": "Informatics resources that aim to identify, map or analyse genetic markers in DNA sequences, for example to produce a genetic (linkage) map of a chromosome or genome or to analyse genetic linkage and synteny.", "label": "Genetic mapping and linkage"}, "topic_0797": {"class_id": "http://edamontology.org/topic_0797", "definitions": "The study (typically comparison) of the sequence, structure or function of multiple genomes.", "label": "Comparative genomics"}, "operation_0284": {"class_id": "http://edamontology.org/operation_0284", "definitions": "Calculate codon usage statistics and create a codon usage table.", "label": "Codon usage table generation"}, "data_1517": {"class_id": "http://edamontology.org/data_1517", "definitions": "An informative report on a specific restriction enzyme such as enzyme reference data.|This might include name of enzyme, organism, isoschizomers, methylation, source, suppliers, literature references, or data on restriction enzyme patterns such as name of enzyme, recognition site, length of pattern, number of cuts made by enzyme, details of blunt or sticky end cut etc.", "label": "Restriction enzyme report"}, "format_3476": {"class_id": "http://edamontology.org/format_3476", "definitions": "Format of a file of gene expression data, e.g. a gene expression matrix or profile.", "label": "Gene expression data format"}, "format_3475": {"class_id": "http://edamontology.org/format_3475", "definitions": "Tabular data represented as tab-separated values in a text file.", "label": "TSV"}, "topic_0222": {"class_id": "http://edamontology.org/topic_0222", "definitions": "Annotation of a genome.", "label": "Genome annotation"}, "topic_0220": {"class_id": "http://edamontology.org/topic_0220", "definitions": "The management and manipulation of digital documents, including database records, files and reports.", "label": "Document, record and content management"}, "topic_0221": {"class_id": "http://edamontology.org/topic_0221", "definitions": "Annotation of a molecular sequence.", "label": "Sequence annotation"}, "topic_1311": {"class_id": "http://edamontology.org/topic_1311", "definitions": "Operons (operators, promoters and genes) from a bacterial genome.", "label": "Operon"}, "topic_1312": {"class_id": "http://edamontology.org/topic_1312", "definitions": "Whole promoters or promoter elements (transcription start sites, RNA polymerase binding site, transcription factor binding sites, promoter enhancers etc) in a DNA sequence.", "label": "Promoters"}, "operation_3792": {"class_id": "http://edamontology.org/operation_3792", "definitions": "The analysis of microRNAs (miRNAs) : short, highly conserved small noncoding RNA molecules that are naturally occurring plant and animal genomes.", "label": "miRNA expression analysis"}, "data_1519": {"class_id": "http://edamontology.org/data_1519", "definitions": "List of molecular weight(s) of one or more proteins or peptides, for example cut by proteolytic enzymes or reagents.|The report might include associated data such as frequency of peptide fragment molecular weights.", "label": "Peptide molecular weights"}, "operation_0285": {"class_id": "http://edamontology.org/operation_0285", "definitions": "Compare two or more codon usage tables.", "label": "Codon usage table comparison"}, "format_3748": {"class_id": "http://edamontology.org/format_3748", "definitions": "A linked data format enables publishing structured data as linked data (Linked Data), so that the data can be interlinked and become more useful through semantic queries.", "label": "Linked data format"}, "operation_0364": {"class_id": "http://edamontology.org/operation_0364", "definitions": "Generate a random sequence, for example, with a specific character composition.", "label": "Random sequence generation"}, "operation_0367": {"class_id": "http://edamontology.org/operation_0367", "definitions": "Mutate a molecular sequence a specified amount or shuffle it to produce a randomised sequence with the same overall composition.", "label": "Sequence mutation and randomisation"}, "operation_0366": {"class_id": "http://edamontology.org/operation_0366", "definitions": "Cleave a protein sequence into peptide fragments (corresponding to enzymatic or chemical cleavage).", "label": "Protein sequence cleavage"}, "operation_0361": {"class_id": "http://edamontology.org/operation_0361", "definitions": "Annotate a molecular sequence record with terms from a controlled vocabulary.", "label": "Sequence annotation"}, "operation_0360": {"class_id": "http://edamontology.org/operation_0360", "definitions": "Search a database of molecular structure and retrieve structures that are similar to a query structure.", "label": "Structural similarity search"}, "operation_0363": {"class_id": "http://edamontology.org/operation_0363", "definitions": "Generate the reverse and / or complement of a nucleotide sequence.", "label": "Nucleic acid sequence reverse and complement"}, "operation_0362": {"class_id": "http://edamontology.org/operation_0362", "definitions": "Annotate a genome sequence with terms from a controlled vocabulary.", "label": "Genome annotation"}, "operation_3795": {"class_id": "http://edamontology.org/operation_3795", "definitions": "A technique whereby molecules with desired properties and function are isolated from libraries of random molecules, through iterative cycles of selection, amplification, and mutagenesis.", "label": "In vitro selection"}, "operation_0369": {"class_id": "http://edamontology.org/operation_0369", "definitions": "Cut (remove) characters or a region from a molecular sequence.", "label": "Sequence cutting"}, "operation_0368": {"class_id": "http://edamontology.org/operation_0368", "definitions": "Mask characters in a molecular sequence (replacing those characters with a mask character).|For example, SNPs or repeats in a DNA sequence might be masked.", "label": "Sequence masking"}, "format_2052": {"class_id": "http://edamontology.org/format_2052", "definitions": "Format for reports on a protein family.", "label": "Protein family report format"}, "format_2051": {"class_id": "http://edamontology.org/format_2051", "definitions": "Format for sequence polymorphism data.", "label": "Polymorphism report format"}, "format_2056": {"class_id": "http://edamontology.org/format_2056", "definitions": "Format for information about a microarray experimental per se (not the data generated from that experiment).", "label": "Microarray experiment data format"}, "format_2057": {"class_id": "http://edamontology.org/format_2057", "definitions": "Format for sequence trace data (i.e. including base call information).", "label": "Sequence trace format"}, "format_2054": {"class_id": "http://edamontology.org/format_2054", "definitions": "Format for molecular interaction data.", "label": "Protein interaction format"}, "format_2055": {"class_id": "http://edamontology.org/format_2055", "definitions": "Format for sequence assembly data.", "label": "Sequence assembly format"}, "topic_0133": {"class_id": "http://edamontology.org/topic_0133", "definitions": "Two-dimensional gel electrophoresis image and related data.", "label": "Two-dimensional gel electrophoresis"}, "format_2058": {"class_id": "http://edamontology.org/format_2058", "definitions": "Format of a file of gene expression data, e.g. a gene expression matrix or profile.", "label": "Gene expression report format"}, "format_2059": {"class_id": "http://edamontology.org/format_2059", "definitions": "Format of a report on genotype / phenotype information.", "label": "Genotype and phenotype annotation format"}, "format_3507": {"class_id": "http://edamontology.org/format_3507", "definitions": "Format of documents including word processor, spreadsheet and presentation.", "label": "Document format"}, "format_3506": {"class_id": "http://edamontology.org/format_3506", "definitions": "Microsoft Word format.", "label": "docx"}, "topic_0134": {"class_id": "http://edamontology.org/topic_0134", "definitions": "An analytical chemistry technique that measures the mass-to-charge ratio and abundance of ions in the gas phase.", "label": "Mass spectrometry"}, "topic_0135": {"class_id": "http://edamontology.org/topic_0135", "definitions": "Protein microarray data.", "label": "Protein microarrays"}, "data_0910": {"class_id": "http://edamontology.org/data_0910", "definitions": "Km is the concentration (usually in Molar units) of substrate that leads to half-maximal velocity of an enzyme-catalysed reaction.", "label": "Km"}, "data_0911": {"class_id": "http://edamontology.org/data_0911", "definitions": "An informative report about a specific nucleotide base.", "label": "Nucleotide base annotation"}, "data_0912": {"class_id": "http://edamontology.org/data_0912", "definitions": "A report of primarily non-positional data describing intrinsic physical, chemical or other properties of a nucleic acid molecule.|The report may be based on analysis of nucleic acid sequence or structural data. This is a broad data type and is used a placeholder for other, more specific types.", "label": "Nucleic acid property"}, "operation_3190": {"class_id": "http://edamontology.org/operation_3190", "definitions": "Trim sequences (typically from an automated DNA sequencer) to remove sequence-specific end regions, typically contamination from vector sequences.", "label": "Trim vector"}, "data_0914": {"class_id": "http://edamontology.org/data_0914", "definitions": "Data derived from analysis of codon usage (typically a codon usage table) of DNA sequences.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Codon usage data"}, "data_0916": {"class_id": "http://edamontology.org/data_0916", "definitions": "A report on predicted or actual gene structure, regions which make an RNA product and features such as promoters, coding regions, splice sites etc.|This includes any report on a particular locus or gene.  This might include the gene name, description, summary and so on. It can include details about the function of a gene, such as its encoded protein or a functional classification of the gene sequence along according to the encoded protein(s).", "label": "Gene report"}, "data_0917": {"class_id": "http://edamontology.org/data_0917", "definitions": "A report on the classification of nucleic acid / gene sequences according to the functional classification of their gene products.", "label": "Gene classification"}, "data_0918": {"class_id": "http://edamontology.org/data_0918", "definitions": "stable, naturally occuring mutations in a nucleotide sequence including alleles, naturally occurring mutations such as single base nucleotide substitutions, deletions and insertions, RFLPs and other polymorphisms.", "label": "DNA variation"}, "data_0919": {"class_id": "http://edamontology.org/data_0919", "definitions": "An informative report on a specific chromosome.|This includes basic information. e.g. chromosome number, length, karyotype features, chromosome sequence etc.", "label": "Chromosome report"}, "topic_0724": {"class_id": "http://edamontology.org/topic_0724", "definitions": "Primarily the classification of proteins (from sequence or structural data) into clusters, groups, families etc., curation of a particular protein or protein family, or any other proteins that have been classified as members of a common group.", "label": "Protein families"}, "operation_3196": {"class_id": "http://edamontology.org/operation_3196", "definitions": "Analyse DNA sequence data to identify differences between the genetic composition (genotype) of an individual compared to other individual's or a reference sequence.|Methods might consider cytogenetic analyses, copy number polymorphism (and calculate copy number calls for copy-number variation(CNV) regions), single nucleotide polymorphism (SNP), , rare copy number variation (CNV) identification, loss of heterozygosity data and so on.", "label": "Genotyping"}, "operation_3195": {"class_id": "http://edamontology.org/operation_3195", "definitions": "Detect errors in DNA sequences generated from sequencing projects).", "label": "Sequencing error detection"}, "operation_2475": {"class_id": "http://edamontology.org/operation_2475", "definitions": "Identify the architecture of a protein structure.|Includes methods that try to suggest the most likely biological unit for a given protein X-ray crystal structure based on crystal symmetry and scoring of putative protein-protein interfaces.", "label": "Protein architecture recognition"}, "data_2673": {"class_id": "http://edamontology.org/data_2673", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Canis familiaris' division).", "label": "Ensembl ID ('Canis familiaris')"}, "data_2672": {"class_id": "http://edamontology.org/data_2672", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Bos taurus' division).", "label": "Ensembl ID ('Bos taurus')"}, "data_2671": {"class_id": "http://edamontology.org/data_2671", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database (Homo sapiens division).", "label": "Ensembl ID (Homo sapiens)"}, "data_2670": {"class_id": "http://edamontology.org/data_2670", "definitions": "Identifier of a signaling pathway from the Database of Quantitative Cellular Signaling (DQCS).", "label": "Pathway ID (DQCS)"}, "data_2677": {"class_id": "http://edamontology.org/data_2677", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Danio rerio' division).", "label": "Ensembl ID ('Danio rerio')"}, "data_2676": {"class_id": "http://edamontology.org/data_2676", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Ciona savignyi' division).", "label": "Ensembl ID ('Ciona savignyi')"}, "data_2675": {"class_id": "http://edamontology.org/data_2675", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Ciona intestinalis' division).", "label": "Ensembl ID ('Ciona intestinalis')"}, "data_2674": {"class_id": "http://edamontology.org/data_2674", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Cavia porcellus' division).", "label": "Ensembl ID ('Cavia porcellus')"}, "operation_0299": {"class_id": "http://edamontology.org/operation_0299", "definitions": "Align structural (3D) profiles or templates (representing structures or structure alignments).", "label": "3D profile-to-3D profile alignment"}, "operation_0298": {"class_id": "http://edamontology.org/operation_0298", "definitions": "Align sequence profiles (representing sequence alignments).", "label": "Profile-profile alignment"}, "data_2679": {"class_id": "http://edamontology.org/data_2679", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Echinops telfairi' division).", "label": "Ensembl ID ('Echinops telfairi')"}, "data_2678": {"class_id": "http://edamontology.org/data_2678", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Dasypus novemcinctus' division).", "label": "Ensembl ID ('Dasypus novemcinctus')"}, "topic_3405": {"class_id": "http://edamontology.org/topic_3405", "definitions": "The study, diagnosis, prevention and treatments of disorders of the oral cavity, maxillofacial area and adjacent structures.", "label": "Dentistry"}, "data_2071": {"class_id": "http://edamontology.org/data_2071", "definitions": "An amino acid sequence motif.", "label": "Sequence motif (protein)"}, "data_2070": {"class_id": "http://edamontology.org/data_2070", "definitions": "A nucleotide sequence motif.", "label": "Sequence motif (nucleic acid)"}, "topic_3404": {"class_id": "http://edamontology.org/topic_3404", "definitions": "The branch of medicine that deals with prevention, diagnosis and treatment of disorders of the skin, scalp, hair and nails.", "label": "Dermatology"}, "data_2079": {"class_id": "http://edamontology.org/data_2079", "definitions": "Some simple value controlling a search operation, typically a search of a database.", "label": "Search parameter"}, "operation_0563": {"class_id": "http://edamontology.org/operation_0563", "definitions": "Reformat a codon usage table.", "label": "Codon usage table formatting"}, "operation_0562": {"class_id": "http://edamontology.org/operation_0562", "definitions": "Reformat (a file or other report of) molecular sequence alignment(s).", "label": "Sequence alignment formatting"}, "operation_0561": {"class_id": "http://edamontology.org/operation_0561", "definitions": "Reformat (a file or other report of) molecular sequence(s).", "label": "Sequence formatting"}, "operation_0560": {"class_id": "http://edamontology.org/operation_0560", "definitions": "Predict or optimise DNA to elicit (via DNA vaccination) an immunological response.", "label": "DNA vaccine design"}, "operation_0567": {"class_id": "http://edamontology.org/operation_0567", "definitions": "Render or visualise a phylogenetic tree.", "label": "Phylogenetic tree visualisation"}, "operation_0566": {"class_id": "http://edamontology.org/operation_0566", "definitions": "Visualise, format or render sequence clusters.", "label": "Sequence cluster visualisation"}, "operation_0565": {"class_id": "http://edamontology.org/operation_0565", "definitions": "Visualise, format or print a molecular sequence alignment.", "label": "Sequence alignment visualisation"}, "operation_0564": {"class_id": "http://edamontology.org/operation_0564", "definitions": "Visualise, format or render a molecular sequence or sequences such as a sequence alignment, possibly with sequence features or properties shown.", "label": "Sequence visualisation"}, "topic_0722": {"class_id": "http://edamontology.org/topic_0722", "definitions": "Classification of nucleic acid sequences and structures.", "label": "Nucleic acid classification"}, "operation_0569": {"class_id": "http://edamontology.org/operation_0569", "definitions": "Render and visualise protein secondary structure.", "label": "Protein secondary structure visualisation"}, "operation_0568": {"class_id": "http://edamontology.org/operation_0568", "definitions": "Visualise RNA secondary structure, knots, pseudoknots etc.", "label": "RNA secondary structure visualisation"}, "topic_0130": {"class_id": "http://edamontology.org/topic_0130", "definitions": "Protein stability, folding (in 3D space) and protein sequence-structure-function relationships.  This includes for example study of inter-atomic or inter-residue interactions in protein (3D) structures, the effect of mutation, and the design of proteins with specific properties, typically by designing changes (via site-directed mutagenesis) to an existing protein.", "label": "Protein folding, stability and design"}, "format_1651": {"class_id": "http://edamontology.org/format_1651", "definitions": "The format of an entry from the PATIKA biological pathways database.", "label": "PATIKA entry format"}, "format_1650": {"class_id": "http://edamontology.org/format_1650", "definitions": "The format of an entry from the INOH signal transduction pathways database.", "label": "INOH entry format"}, "format_1653": {"class_id": "http://edamontology.org/format_1653", "definitions": "The format of an entry from the aMAZE biological pathways and molecular interactions database.", "label": "aMAZE entry format"}, "format_1652": {"class_id": "http://edamontology.org/format_1652", "definitions": "The format of an entry from the reactome biological pathways database.", "label": "Reactome entry format"}, "format_1655": {"class_id": "http://edamontology.org/format_1655", "definitions": "The format of an entry from the Panther Pathways database.", "label": "Panther Pathways entry format"}, "format_1654": {"class_id": "http://edamontology.org/format_1654", "definitions": "The format of an entry from the CPDB database.", "label": "CPDB entry format"}, "data_0006": {"class_id": "http://edamontology.org/data_0006", "definitions": "Information, represented in an information artefact (data record) that is 'understandable' by dedicated computational tools that can use the data as input or produce it as output.", "label": "Data"}, "data_0007": {"class_id": "http://edamontology.org/data_0007", "definitions": "A bioinformatics package or tool, e.g. a standalone application or web service.", "label": "Tool"}, "topic_3038": {"class_id": "http://edamontology.org/topic_3038", "definitions": "Molecular secondary or tertiary (3D) structural data resources, typically of proteins and nucleic acids.", "label": "Structure databases"}, "data_0005": {"class_id": "http://edamontology.org/data_0005", "definitions": "A type of computational resource used in bioinformatics.", "label": "Resource type"}, "topic_0137": {"class_id": "http://edamontology.org/topic_0137", "definitions": "The study of the hydrophobic, hydrophilic and charge properties of a protein.", "label": "Protein hydropathy"}, "topic_3419": {"class_id": "http://edamontology.org/topic_3419", "definitions": "The branch of medicine that deals with the mangement of mental illness, emotional disturbance and abnormal behaviour.", "label": "Psychiatry"}, "format_2194": {"class_id": "http://edamontology.org/format_2194", "definitions": "Abstract format used by MedLine database.", "label": "medline"}, "operation_3798": {"class_id": "http://edamontology.org/operation_3798", "definitions": "An operation which groups reads or contigs and assigns them to operational taxonomic units.|Binning methods use one or a combination of compositional features or sequence similarity.", "label": "Read binning"}, "data_1852": {"class_id": "http://edamontology.org/data_1852", "definitions": "A component of a larger sequence assembly.", "label": "Sequence assembly component"}, "data_1853": {"class_id": "http://edamontology.org/data_1853", "definitions": "A report on a chromosome aberration such as abnormalities in chromosome structure.", "label": "Chromosome annotation (aberration)"}, "format_1992": {"class_id": "http://edamontology.org/format_1992", "definitions": "Mega non-interleaved format for (typically aligned) sequences.", "label": "meganon"}, "data_2180": {"class_id": "http://edamontology.org/data_2180", "definitions": "Two or more things.", "label": "2 or more"}, "format_1993": {"class_id": "http://edamontology.org/format_1993", "definitions": "MSF format for (aligned) sequences.", "label": "msf alignment format"}, "data_1856": {"class_id": "http://edamontology.org/data_1856", "definitions": "An insertion code (part of the residue number) for an amino acid residue from a PDB file.", "label": "PDB insertion code"}, "topic_3324": {"class_id": "http://edamontology.org/topic_3324", "definitions": "The branch of medicine that deals with the prevention, diagnosis and management of transmissable disease with clinically evident illness resulting from infection with pathogenic biological agents (viruses, bacteria, fungi, protozoa, parasites and prions).", "label": "Infectious disease"}, "topic_3325": {"class_id": "http://edamontology.org/topic_3325", "definitions": "The study of rare diseases.", "label": "Rare diseases"}, "topic_3322": {"class_id": "http://edamontology.org/topic_3322", "definitions": "The study of respiratory system.", "label": "Respiratory medicine"}, "format_1990": {"class_id": "http://edamontology.org/format_1990", "definitions": "Alignment format for start and end of matches between sequence pairs.", "label": "match"}, "topic_3320": {"class_id": "http://edamontology.org/topic_3320", "definitions": "RNA splicing; post-transcription RNA modification involving the removal of introns and joining of exons.|This includes the study of splice sites, splicing patterns, alternative splicing events and variants, isoforms, etc..", "label": "RNA splicing"}, "data_1857": {"class_id": "http://edamontology.org/data_1857", "definitions": "The fraction of an atom type present at a site in a molecular structure.|The sum of the occupancies of all the atom types at a site should not normally significantly exceed 1.0.", "label": "Atomic occupancy"}, "operation_2454": {"class_id": "http://edamontology.org/operation_2454", "definitions": "Detect, predict and identify genes or components of genes in DNA sequences, including promoters, coding regions, splice sites, etc. |Methods for gene prediction might be ab initio, based on phylogenetic comparisons, use motifs, sequence features, support vector machine, alignment etc.", "label": "Gene prediction"}, "operation_2457": {"class_id": "http://edamontology.org/operation_2457", "definitions": "Predict G-protein coupled receptor (GPCR) coupling selectivity.", "label": "GPCR coupling selectivity prediction"}, "operation_2456": {"class_id": "http://edamontology.org/operation_2456", "definitions": "Classify G-protein coupled receptors (GPCRs) into families and subfamilies.", "label": "GPCR classification"}, "operation_2451": {"class_id": "http://edamontology.org/operation_2451", "definitions": "Compare two or more molecular sequences.", "label": "Sequence comparison"}, "operation_2453": {"class_id": "http://edamontology.org/operation_2453", "definitions": "Process (read and / or write) a sequence feature table.", "label": "Feature table processing"}, "operation_2452": {"class_id": "http://edamontology.org/operation_2452", "definitions": "Process (read and / or write) a sequence cluster.", "label": "Sequence cluster processing"}, "operation_3456": {"class_id": "http://edamontology.org/operation_3456", "definitions": "A method used to refine a structure by moving the whole molecule or parts of it as a rigid unit, rather than moving individual atoms.|Rigid body refinement usually follows molecular replacement in the assignment of a structure from diffraction data.", "label": "Rigid body refinement"}, "operation_3457": {"class_id": "http://edamontology.org/operation_3457", "definitions": "An image processing technique that combines and analyze multiple images of a particulate sample, in order to produce an image with clearer features that are more easily interpreted.|Single particle analysis is used to improve the information that can be obtained by relatively low resolution techniques, , e.g. an image of a protein or virus from transmission electron microscopy (TEM).", "label": "Single particle analysis"}, "operation_3454": {"class_id": "http://edamontology.org/operation_3454", "definitions": "Phase a macromolecular crystal structure, for example by using molecular replacement or experimental phasing methods.", "label": "Phasing"}, "data_1855": {"class_id": "http://edamontology.org/data_1855", "definitions": "An identifier of a clone (cloned molecular sequence) from a database.", "label": "Clone ID"}, "operation_2459": {"class_id": "http://edamontology.org/operation_2459", "definitions": "Process (read and / or write) a protein tertiary structure.", "label": "Structure processing (protein)"}, "operation_3453": {"class_id": "http://edamontology.org/operation_3453", "definitions": "The evaluation of diffraction intensities and integration of diffraction maxima from a diffraction experiment.", "label": "Diffraction data integration"}, "operation_3450": {"class_id": "http://edamontology.org/operation_3450", "definitions": "Measurement of neurites; projections (axons or dendrites) from the cell body of a neuron, from analysis of neuron images.", "label": "Neurite measurement"}, "data_1588": {"class_id": "http://edamontology.org/data_1588", "definitions": "DNA base pair stacking energies data.", "label": "DNA base pair stacking energies data"}, "format_1994": {"class_id": "http://edamontology.org/format_1994", "definitions": "Nexus/paup format for (aligned) sequences.", "label": "nexus alignment format"}, "data_1589": {"class_id": "http://edamontology.org/data_1589", "definitions": "DNA base pair twist angle data.", "label": "DNA base pair twist angle data"}, "topic_3517": {"class_id": "http://edamontology.org/topic_3517", "definitions": "Genome-wide association study experiments.", "label": "GWAS study"}, "data_1858": {"class_id": "http://edamontology.org/data_1858", "definitions": "Isotropic B factor (atomic displacement parameter) for an atom from a PDB file.", "label": "Isotropic B factor"}, "data_3102": {"class_id": "http://edamontology.org/data_3102", "definitions": "Unique numerical identifier of chemicals in the scientific literature, as assigned by the Chemical Abstracts Service.", "label": "CAS number"}, "data_3103": {"class_id": "http://edamontology.org/data_3103", "definitions": "Unique identifier of a drug conforming to the Anatomical Therapeutic Chemical (ATC) Classification System, a drug classification system controlled by the WHO Collaborating Centre for Drug Statistics Methodology (WHOCC).", "label": "ATC code"}, "topic_2819": {"class_id": "http://edamontology.org/topic_2819", "definitions": "Invertebrates, e.g. information on a specific invertebrate genome including molecular sequences, genes and annotation.|The resource may be specific to an invertebrate, a group of invertebrates or all invertebrates.", "label": "Invertebrates"}, "data_1859": {"class_id": "http://edamontology.org/data_1859", "definitions": "A cytogenetic map showing chromosome banding patterns in mutant cell lines relative to the wild type.|A cytogenetic map is built from a set of mutant cell lines with sub-chromosomal deletions and a reference wild-type line ('genome deletion panel'). The panel is used to map markers onto the genome by comparing mutant to wild-type banding patterns. Markers are linked (occur in the same deleted region) if they share the same banding pattern (presence or absence) as the deletion panel.", "label": "Deletion map"}, "data_3106": {"class_id": "http://edamontology.org/data_3106", "definitions": "Metadata concerning the software, hardware or other aspects of a computer system.", "label": "System metadata"}, "data_3107": {"class_id": "http://edamontology.org/data_3107", "definitions": "A name of a sequence feature, e.g. the name of a feature to be displayed to an end-user.", "label": "Sequence feature name"}, "data_3104": {"class_id": "http://edamontology.org/data_3104", "definitions": "A unique, unambiguous, alphanumeric identifier of a chemical substance as catalogued by the Substance Registration System of the Food and Drug Administration (FDA).", "label": "UNII"}, "data_3105": {"class_id": "http://edamontology.org/data_3105", "definitions": "Basic information concerning geographical location or time.", "label": "Geotemporal metadata"}, "topic_2813": {"class_id": "http://edamontology.org/topic_2813", "definitions": "The genes, gene variations and proteins involved in one or more specific diseases.", "label": "Disease genes and proteins"}, "data_3108": {"class_id": "http://edamontology.org/data_3108", "definitions": "Raw data such as measurements or other results from laboratory experiments, as generated from laboratory hardware.|This is a broad data type and is used a placeholder for other, more specific types. It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Experimental measurement"}, "topic_2817": {"class_id": "http://edamontology.org/topic_2817", "definitions": "Yeast, e.g. information on a specific yeast genome including molecular sequences, genes and annotation.", "label": "Yeast"}, "topic_2816": {"class_id": "http://edamontology.org/topic_2816", "definitions": "Informatics resource (typically a database) primarily focussed on genes.", "label": "Gene resources"}, "topic_2815": {"class_id": "http://edamontology.org/topic_2815", "definitions": "The study of human beings in general, including the human genome and proteome.", "label": "Human biology"}, "topic_2814": {"class_id": "http://edamontology.org/topic_2814", "definitions": "Protein secondary or tertiary structural data and/or associated annotation.", "label": "Protein structure analysis"}, "data_2699": {"class_id": "http://edamontology.org/data_2699", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Xenopus tropicalis' division).", "label": "Ensembl ID ('Xenopus tropicalis')"}, "data_2698": {"class_id": "http://edamontology.org/data_2698", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Tupaia belangeri' division).", "label": "Ensembl ID ('Tupaia belangeri')"}, "data_2691": {"class_id": "http://edamontology.org/data_2691", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Oryctolagus cuniculus' division).", "label": "Ensembl ID ('Oryctolagus cuniculus')"}, "data_2690": {"class_id": "http://edamontology.org/data_2690", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Ornithorhynchus anatinus' division).", "label": "Ensembl ID (\"Ornithorhynchus anatinus\\\")"}, "data_2693": {"class_id": "http://edamontology.org/data_2693", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Otolemur garnettii' division).", "label": "Ensembl ID ('Otolemur garnettii')"}, "data_2692": {"class_id": "http://edamontology.org/data_2692", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Oryzias latipes' division).", "label": "Ensembl ID ('Oryzias latipes')"}, "data_2695": {"class_id": "http://edamontology.org/data_2695", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Rattus norvegicus' division).", "label": "Ensembl ID ('Rattus norvegicus')"}, "data_2694": {"class_id": "http://edamontology.org/data_2694", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Pan troglodytes' division).", "label": "Ensembl ID ('Pan troglodytes')"}, "data_2697": {"class_id": "http://edamontology.org/data_2697", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Takifugu rubripes' division).", "label": "Ensembl ID ('Takifugu rubripes')"}, "data_2696": {"class_id": "http://edamontology.org/data_2696", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Spermophilus tridecemlineatus' division).", "label": "Ensembl ID ('Spermophilus tridecemlineatus')"}, "data_1193": {"class_id": "http://edamontology.org/data_1193", "definitions": "The name of a FASTA tool.|This includes 'fasta3', 'fastx3', 'fasty3', 'fastf3', 'fasts3' and 'ssearch'.", "label": "Tool name (FASTA)"}, "data_1192": {"class_id": "http://edamontology.org/data_1192", "definitions": "The name of a BLAST tool.|This include 'blastn', 'blastp', 'blastx', 'tblastn' and 'tblastx'.", "label": "Tool name (BLAST)"}, "data_1191": {"class_id": "http://edamontology.org/data_1191", "definitions": "The unique name of a signature (sequence classifier) method.|Signature methods from http://www.ebi.ac.uk/Tools/InterProScan/help.html#results include BlastProDom, FPrintScan, HMMPIR, HMMPfam, HMMSmart, HMMTigr, ProfileScan, ScanRegExp, SuperFamily and HAMAP.", "label": "Tool name (signature)"}, "data_1190": {"class_id": "http://edamontology.org/data_1190", "definitions": "The name of a computer package, application, method or function.", "label": "Tool name"}, "data_1195": {"class_id": "http://edamontology.org/data_1195", "definitions": "The name of an EMBASSY package.", "label": "Tool name (EMBASSY package)"}, "data_1194": {"class_id": "http://edamontology.org/data_1194", "definitions": "The name of an EMBOSS application.", "label": "Tool name (EMBOSS)"}, "operation_3205": {"class_id": "http://edamontology.org/operation_3205", "definitions": "Determine cytosine methylation status of specific positions in a nucleic acid sequences.", "label": "Methylation calling"}, "topic_0164": {"class_id": "http://edamontology.org/topic_0164", "definitions": "The comparison and grouping together of molecular sequences on the basis of their similarities.|This includes systems that generate, process and analyse sequence clusters.", "label": "Sequence clustering"}, "data_0886": {"class_id": "http://edamontology.org/data_0886", "definitions": "Alignment (superimposition) of molecular tertiary (3D) structures.|A tertiary structure alignment will include the untransformed coordinates of one macromolecule, followed by the second (or subsequent) structure(s) with all the coordinates transformed (by rotation / translation) to give a superposition.", "label": "Structure alignment"}, "operation_3218": {"class_id": "http://edamontology.org/operation_3218", "definitions": "Raw sequence data quality control.|Analyse raw sequence data from a sequencing pipeline and identify (and possiby fix) problems.", "label": "Sequencing quality control"}, "operation_3219": {"class_id": "http://edamontology.org/operation_3219", "definitions": "Pre-process sequence reads to ensure (or improve) quality and reliability.|For example process paired end reads to trim low quality ends remove short sequences, identify sequence inserts, detect chimeric reads, or remove low quality sequnces including vector, adaptor, low complexity and contaminant sequences. Sequences might come from genomic DNA library, EST libraries, SSH library and so on.", "label": "Read pre-processing"}, "data_1353": {"class_id": "http://edamontology.org/data_1353", "definitions": "Any specific or conserved pattern (typically expressed as a regular expression) in a molecular sequence.", "label": "Sequence motif"}, "data_1352": {"class_id": "http://edamontology.org/data_1352", "definitions": "Regular expression pattern.", "label": "Regular expression"}, "data_1355": {"class_id": "http://edamontology.org/data_1355", "definitions": "An informative report about a specific or conserved protein sequence pattern.", "label": "Protein signature"}, "data_1354": {"class_id": "http://edamontology.org/data_1354", "definitions": "Some type of statistical model representing a (typically multiple) sequence alignment.", "label": "Sequence profile"}, "topic_2257": {"class_id": "http://edamontology.org/topic_2257", "definitions": "Visualise a phylogeny, for example, render a phylogenetic tree.", "label": "Phylogeny visualisation"}, "data_1359": {"class_id": "http://edamontology.org/data_1359", "definitions": "A protein regular expression pattern from the Prosite database.", "label": "Prosite protein pattern"}, "data_1358": {"class_id": "http://edamontology.org/data_1358", "definitions": "A nucleotide regular expression pattern from the Prosite database.", "label": "Prosite nucleotide pattern"}, "format_1341": {"class_id": "http://edamontology.org/format_1341", "definitions": "Results format for searches of the InterPro database.", "label": "InterPro hits format"}, "operation_3213": {"class_id": "http://edamontology.org/operation_3213", "definitions": "Generate an index of a genome sequence using a suffix arrays algorithm.|A suffix array consists of the lexicographically sorted list of suffixes of a genome.", "label": "Genome indexing (suffix arrays)"}, "operation_3214": {"class_id": "http://edamontology.org/operation_3214", "definitions": "Analyse one or more spectra from mass spectrometry (or other) experiments.", "label": "Spectral analysis"}, "operation_3215": {"class_id": "http://edamontology.org/operation_3215", "definitions": "Identify peaks in a spectrum from a mass spectrometry, NMR, or some other spectrum-generating experiment.", "label": "Peak detection"}, "operation_3216": {"class_id": "http://edamontology.org/operation_3216", "definitions": "Link together a non-contiguous series of genomic sequences into a scaffold, consisting of sequences separated by gaps of known length.  The sequences that are linked are typically typically contigs; contiguous sequences corresponding to read overlaps.|Scaffold may be positioned along a chromosome physical map to create a \"golden path\".", "label": "Scaffolding"}, "operation_3217": {"class_id": "http://edamontology.org/operation_3217", "definitions": "Fill the gaps in a sequence assembly (scaffold) by merging in additional sequences.|Different techniques are used to generate gap sequences to connect contigs, depending on the size of the gap. For small (5-20kb) gaps, PCR amplification and sequencing is used.  For large (>20kb) gaps, fragments are cloned (e.g. in BAC (Bacterial artificial chromosomes) vectors) and then sequenced.", "label": "Scaffold gap completion"}, "operation_3279": {"class_id": "http://edamontology.org/operation_3279", "definitions": "Cluster (group) documents on the basis of their calculated similarity.", "label": "Document clustering"}, "topic_0167": {"class_id": "http://edamontology.org/topic_0167", "definitions": "The processing, analysis or use of some type of structural (3D) profile or template; a computational entity (typically a numerical matrix) that is derived from and represents a structure or structure alignment.", "label": "Structural (3D) profiles"}, "data_0887": {"class_id": "http://edamontology.org/data_0887", "definitions": "An informative report of molecular tertiary structure alignment-derived data.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Structure alignment report"}, "data_3036": {"class_id": "http://edamontology.org/data_3036", "definitions": "An identifier of an array of numerical values, such as a comparison matrix.", "label": "Matrix identifier"}, "data_3034": {"class_id": "http://edamontology.org/data_3034", "definitions": "Name or other identifier of molecular sequence feature(s).", "label": "Sequence feature identifier"}, "data_3035": {"class_id": "http://edamontology.org/data_3035", "definitions": "An identifier of a molecular tertiary structure, typically an entry from a structure database.", "label": "Structure identifier"}, "data_2529": {"class_id": "http://edamontology.org/data_2529", "definitions": "An informative report on a specific molecule.", "label": "Molecule report"}, "data_2528": {"class_id": "http://edamontology.org/data_2528", "definitions": "Data concerning a specific type of molecule.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Molecular data"}, "data_2527": {"class_id": "http://edamontology.org/data_2527", "definitions": "Typically a simple numerical or string value that controls the operation of a tool.", "label": "Parameter"}, "data_2526": {"class_id": "http://edamontology.org/data_2526", "definitions": "Data concerning, extracted from, or derived from the analysis of a scientific text (or texts) such as a full text article from a scientific journal.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation. It includes concepts that are best described as scientific text or closely concerned with or derived from text.", "label": "Text data"}, "data_2525": {"class_id": "http://edamontology.org/data_2525", "definitions": "Data concerning one or more nucleic acid molecules.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Nucleic acid data"}, "data_2524": {"class_id": "http://edamontology.org/data_2524", "definitions": "Data concerning one or more protein molecules.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Protein data"}, "data_2523": {"class_id": "http://edamontology.org/data_2523", "definitions": "Data concerning phylogeny, typically of molecular sequences, including reports of information concerning or derived from a phylogenetic tree, or from comparing two or more phylogenetic trees.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Phylogenetic data"}, "data_2522": {"class_id": "http://edamontology.org/data_2522", "definitions": "Data concerning a map of molecular sequence(s).|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Map data"}, "topic_0166": {"class_id": "http://edamontology.org/topic_0166", "definitions": "Structural features or common 3D motifs within protein structures, including the surface of a protein structure, such as biological interfaces with other molecules.|This includes conformation of conserved substructures, conserved geometry (spatial arrangement) of secondary structure or protein backbone, solvent-exposed surfaces, internal cavities, the analysis of shape, hydropathy, electrostatic patches, role and functions etc.", "label": "Protein structural motifs and surfaces"}, "data_1550": {"class_id": "http://edamontology.org/data_1550", "definitions": "Non-canonical atomic interactions in protein structures.", "label": "Protein non-canonical interactions"}, "operation_0291": {"class_id": "http://edamontology.org/operation_0291", "definitions": "Build clusters of similar sequences, typically using scores from pair-wise alignment or other comparison of the sequences.|The clusters may be output or used internally for some other purpose.", "label": "Sequence clustering"}, "operation_0290": {"class_id": "http://edamontology.org/operation_0290", "definitions": "Compare two or more molecular sequences, identify and remove redundant sequences based on some criteria.", "label": "Sequence redundancy removal"}, "operation_0293": {"class_id": "http://edamontology.org/operation_0293", "definitions": "Align two or more molecular sequences of different types (for example genomic DNA to EST, cDNA or mRNA).", "label": "Hybrid sequence alignment construction"}, "operation_0292": {"class_id": "http://edamontology.org/operation_0292", "definitions": "Align (identify equivalent sites within) molecular sequences.", "label": "Sequence alignment"}, "operation_0295": {"class_id": "http://edamontology.org/operation_0295", "definitions": "Align (superimpose) molecular tertiary structures.", "label": "Structure alignment"}, "operation_0294": {"class_id": "http://edamontology.org/operation_0294", "definitions": "Align molecular sequences using sequence and structural information.", "label": "Structure-based sequence alignment"}, "operation_3201": {"class_id": "http://edamontology.org/operation_3201", "definitions": "Identify single nucleotide change in base positions in sequencing data that differ from a reference genome and which might, especially by reference to population frequency or functional data, indicate a polymorphism.", "label": "SNP calling"}, "format_2096": {"class_id": "http://edamontology.org/format_2096", "definitions": "Alphabet for a molecular sequence with possible unknown positions but without ambiguity characters.", "label": "unambiguous sequence"}, "topic_0160": {"class_id": "http://edamontology.org/topic_0160", "definitions": "The archival, detection, prediction and analysis of positional features such as functional and other key sites, in molecular sequences and the conserved patterns (motifs, profiles etc.) that may be used to describe them.", "label": "Sequence sites, features and motifs"}, "operation_0297": {"class_id": "http://edamontology.org/operation_0297", "definitions": "Generate some type of structural (3D) profile or template from a structure or structure alignment.", "label": "3D profile generation"}, "format_2097": {"class_id": "http://edamontology.org/format_2097", "definitions": "Alphabet for a molecular sequence with possible unknown positions and possible ambiguity characters.", "label": "ambiguous"}, "operation_0296": {"class_id": "http://edamontology.org/operation_0296", "definitions": "Generate some type of sequence profile (for example a hidden Markov model) from a sequence alignment.", "label": "Sequence profile generation"}, "format_2848": {"class_id": "http://edamontology.org/format_2848", "definitions": "Format of a bibliographic reference.", "label": "Bibliographic reference format"}, "format_2095": {"class_id": "http://edamontology.org/format_2095", "definitions": "Alphabet for a molecular sequence with possible unknown positions but possibly with non-sequence characters.", "label": "unpure"}, "topic_3263": {"class_id": "http://edamontology.org/topic_3263", "definitions": "The protection of data, such as patient health data, from damage or unwanted access from unauthorised users.", "label": "Data security"}, "data_1772": {"class_id": "http://edamontology.org/data_1772", "definitions": "A numerical value, that is some type of scored value arising for example from a prediction method.", "label": "Score"}, "data_1771": {"class_id": "http://edamontology.org/data_1771", "definitions": "Information on an molecular sequence version.", "label": "Sequence version"}, "data_1776": {"class_id": "http://edamontology.org/data_1776", "definitions": "Report on general functional properties of specific protein(s).|For properties that can be mapped to a sequence, use 'Sequence report' instead.", "label": "Protein report (function)"}, "data_1179": {"class_id": "http://edamontology.org/data_1179", "definitions": "A stable unique identifier for each taxon (for a species, a family, an order, or any other group in the NCBI taxonomy database.", "label": "NCBI taxonomy ID"}, "data_1178": {"class_id": "http://edamontology.org/data_1178", "definitions": "An identifier of a concept from the HGNC controlled vocabulary.", "label": "HGNC concept ID"}, "data_1171": {"class_id": "http://edamontology.org/data_1171", "definitions": "Unique identifier of an entry from the BioModel database.", "label": "BioModel ID"}, "data_1170": {"class_id": "http://edamontology.org/data_1170", "definitions": "Name of a biological (mathematical) model.", "label": "Biological model name"}, "data_1173": {"class_id": "http://edamontology.org/data_1173", "definitions": "Identifier of an entry from the ChemSpider database.", "label": "ChemSpider ID"}, "data_1172": {"class_id": "http://edamontology.org/data_1172", "definitions": "Chemical structure specified in PubChem Compound Identification (CID), a non-zero integer identifier for a unique chemical structure.", "label": "PubChem CID"}, "data_1175": {"class_id": "http://edamontology.org/data_1175", "definitions": "An identifier of a concept from the BioPax ontology.", "label": "BioPax concept ID"}, "data_1174": {"class_id": "http://edamontology.org/data_1174", "definitions": "Identifier of an entry from the ChEBI database.", "label": "ChEBI ID"}, "data_1177": {"class_id": "http://edamontology.org/data_1177", "definitions": "An identifier of a concept from the MeSH vocabulary.", "label": "MeSH concept ID"}, "data_1176": {"class_id": "http://edamontology.org/data_1176", "definitions": "An identifier of a concept from The Gene Ontology.", "label": "GO concept ID"}, "operation_0449": {"class_id": "http://edamontology.org/operation_0449", "definitions": "Analyse correlations between sites in a molecular sequence alignment.|This is typically done to identify possible covarying positions and predict contacts or structural constraints in protein structures.", "label": "Sequence alignment analysis (site correlation)"}, "topic_0640": {"class_id": "http://edamontology.org/topic_0640", "definitions": "The archival, processing and analysis of nucleotide sequences and and sequence-based entities such as alignments, motifs and profiles.", "label": "Nucleic acid sequence analysis"}, "topic_0641": {"class_id": "http://edamontology.org/topic_0641", "definitions": "The repetitive nature of molecular sequences.", "label": "Repeat sequences"}, "topic_0642": {"class_id": "http://edamontology.org/topic_0642", "definitions": "The (character) complexity of molecular sequences, particularly regions of low complexity.", "label": "Low complexity sequences"}, "format_2607": {"class_id": "http://edamontology.org/format_2607", "definitions": "Alphabet for any protein sequence without unknown positions, ambiguity or non-sequence characters.", "label": "completely unambiguous pure protein"}, "format_3262": {"class_id": "http://edamontology.org/format_3262", "definitions": "OWL ontology XML serialisation format.", "label": "OWL/XML"}, "operation_1778": {"class_id": "http://edamontology.org/operation_1778", "definitions": "Compare the functional properties of two or more proteins.", "label": "Protein function comparison"}, "format_3261": {"class_id": "http://edamontology.org/format_3261", "definitions": "Resource Description Framework (RDF) XML format.|RDF/XML is a serialisation syntax for OWL DL, but not for OWL Full.", "label": "RDF/XML"}, "data_2858": {"class_id": "http://edamontology.org/data_2858", "definitions": "A concept from a biological ontology.|This includes any fields from the concept definition such as concept name, definition, comments and so on.", "label": "Ontology concept"}, "data_2857": {"class_id": "http://edamontology.org/data_2857", "definitions": "Bibliographic data concerning scientific article(s).", "label": "Article metadata"}, "data_2856": {"class_id": "http://edamontology.org/data_2856", "definitions": "Distances (values representing similarity) between a group of molecular structures.", "label": "Structural distance matrix"}, "data_2855": {"class_id": "http://edamontology.org/data_2855", "definitions": "A matrix of distances between molecular entities, where a value (distance) is (typically) derived from comparison of two entities and reflects their similarity.", "label": "Distance matrix"}, "data_2854": {"class_id": "http://edamontology.org/data_2854", "definitions": "A simple matrix of numbers, where each value (or column of values) is derived derived from analysis of the corresponding position in a sequence alignment.", "label": "Position-specific scoring matrix"}, "data_2852": {"class_id": "http://edamontology.org/data_2852", "definitions": "3D coordinate and associated data for the (3D) structure of a toxin.", "label": "Toxin structure"}, "data_2851": {"class_id": "http://edamontology.org/data_2851", "definitions": "3D coordinate and associated data for the (3D) structure of a drug.", "label": "Drug structure"}, "data_2850": {"class_id": "http://edamontology.org/data_2850", "definitions": "3D coordinate and associated data for a lipid structure.", "label": "Lipid structure"}, "format_3312": {"class_id": "http://edamontology.org/format_3312", "definitions": "Format for the Genetic Data Environment (GDE).", "label": "GDE"}, "format_3313": {"class_id": "http://edamontology.org/format_3313", "definitions": "A multiple alignment in vertical format, as used in the AMPS (Alignment of Multiple Protein Sequences) pacakge.", "label": "BLC"}, "data_0583": {"class_id": "http://edamontology.org/data_0583", "definitions": "A directory on disk from which files are read.", "label": "Directory metadata"}, "data_0582": {"class_id": "http://edamontology.org/data_0582", "definitions": "An ontology of biological or bioinformatics concepts and relations, a controlled vocabulary, structured glossary etc.", "label": "Ontology"}, "data_0581": {"class_id": "http://edamontology.org/data_0581", "definitions": "A digital data archive typically based around a relational model but sometimes using an object-oriented, tree or graph-based model.", "label": "Database"}, "format_3310": {"class_id": "http://edamontology.org/format_3310", "definitions": "XRNA old input style format.", "label": "SS"}, "format_3311": {"class_id": "http://edamontology.org/format_3311", "definitions": "RNA Markup Language.", "label": "RNAML"}, "data_2989": {"class_id": "http://edamontology.org/data_2989", "definitions": "key residues involved in protein folding.", "label": "Protein features report (key folding sites)"}, "data_2985": {"class_id": "http://edamontology.org/data_2985", "definitions": "A thermodynamic or kinetic property of a nucleic acid molecule.", "label": "Nucleic acid thermodynamic data"}, "data_0899": {"class_id": "http://edamontology.org/data_0899", "definitions": "3D structural motifs in a protein.", "label": "Protein structural motifs and surfaces"}, "data_2987": {"class_id": "http://edamontology.org/data_2987", "definitions": "A report on a classification of molecular sequences, structures or other entities.|This can include an entire classification, components such as classifiers, assignments of entities to a classification and so on.", "label": "Classification report"}, "data_2986": {"class_id": "http://edamontology.org/data_2986", "definitions": "Data concerning the classification of nucleic acid sequences or structures.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Nucleic acid classification"}, "data_2981": {"class_id": "http://edamontology.org/data_2981", "definitions": "Data concerning specific or conserved pattern in molecular sequences.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Sequence motif data"}, "data_2980": {"class_id": "http://edamontology.org/data_2980", "definitions": "An informative report concerning the classification of protein sequences or structures.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Protein classification"}, "data_2983": {"class_id": "http://edamontology.org/data_2983", "definitions": "Data concerning a specific biological pathway or network.", "label": "Pathway or network data"}, "data_2982": {"class_id": "http://edamontology.org/data_2982", "definitions": "Data concerning models representing a (typically multiple) sequence alignment.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Sequence profile data"}, "data_2969": {"class_id": "http://edamontology.org/data_2969", "definitions": "Image of a molecular sequence, possibly with sequence features or properties shown.", "label": "Sequence image"}, "data_1543": {"class_id": "http://edamontology.org/data_1543", "definitions": "Data on the surface properties (shape, hydropathy, electrostatic patches etc) of a protein structure.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Protein surface report"}, "data_1508": {"class_id": "http://edamontology.org/data_1508", "definitions": "Van der Waals radii of atoms for different amino acid residues.", "label": "Amino acid index (van der Waals radii)"}, "data_1509": {"class_id": "http://edamontology.org/data_1509", "definitions": "An informative report on a specific enzyme.", "label": "Enzyme report"}, "data_0894": {"class_id": "http://edamontology.org/data_0894", "definitions": "An informative report about a specific amino acid.", "label": "Amino acid annotation"}, "data_1501": {"class_id": "http://edamontology.org/data_1501", "definitions": "A table of 20 numerical values which quantify a property (e.g. physicochemical or biochemical) of the common amino acids.", "label": "Amino acid index"}, "data_1502": {"class_id": "http://edamontology.org/data_1502", "definitions": "Chemical classification (small, aliphatic, aromatic, polar, charged etc) of amino acids.", "label": "Amino acid index (chemical classes)"}, "data_1503": {"class_id": "http://edamontology.org/data_1503", "definitions": "Statistical protein contact potentials.", "label": "Amino acid pair-wise contact potentials"}, "data_0897": {"class_id": "http://edamontology.org/data_0897", "definitions": "A report of primarily non-positional data describing intrinsic physical, chemical or other properties of a protein molecule or model.|The report may be based on analysis of nucleic acid sequence or structural data. This is a broad data type and is used a placeholder for other, more specific types.", "label": "Protein property"}, "data_1506": {"class_id": "http://edamontology.org/data_1506", "definitions": "Hydrophobic, hydrophilic or charge properties of amino acids.", "label": "Amino acid index (hydropathy)"}, "data_1507": {"class_id": "http://edamontology.org/data_1507", "definitions": "Experimental free energy values for the water-interface and water-octanol transitions for the amino acids.", "label": "Amino acid index (White-Wimley data)"}, "data_2215": {"class_id": "http://edamontology.org/data_2215", "definitions": "An informative report on the functional properties of mutant proteins including transcriptional activities, promotion of cell growth and tumorigenicity, dominant negative effects, capacity to induce apoptosis, cell-cycle arrest or checkpoints in human cells and so on.", "label": "Mutation annotation (functional)"}, "data_2214": {"class_id": "http://edamontology.org/data_2214", "definitions": "An informative report on mutation prognostic data, such as information on patient cohort, the study settings and the results of the study.", "label": "Mutation annotation (prognostic)"}, "data_2217": {"class_id": "http://edamontology.org/data_2217", "definitions": "An informative report on a specific tumor including nature and origin of the sample, anatomic site, organ or tissue, tumor type, including morphology and/or histologic type, and so on.", "label": "Tumor annotation"}, "data_2216": {"class_id": "http://edamontology.org/data_2216", "definitions": "The number of a codon, for instance, at which a mutation is located.", "label": "Codon number"}, "data_2213": {"class_id": "http://edamontology.org/data_2213", "definitions": "An informative report on the prevalence of mutation(s), including data on samples and mutation prevalence (e.g. by tumour type)..", "label": "Mutation annotation (prevalence)"}, "data_2212": {"class_id": "http://edamontology.org/data_2212", "definitions": "Information describing the mutation itself, the organ site, tissue and type of lesion where the mutation has been identified, description of the patient origin and life-style.", "label": "Mutation annotation (basic)"}, "data_0891": {"class_id": "http://edamontology.org/data_0891", "definitions": "An alignment of a sequence to a 3D profile (representing structures or a structure alignment).", "label": "Sequence-3D profile alignment"}, "data_2219": {"class_id": "http://edamontology.org/data_2219", "definitions": "The name of a field in a database.", "label": "Database field name"}, "data_2218": {"class_id": "http://edamontology.org/data_2218", "definitions": "Basic information about a server on the web, such as an SRS server.", "label": "Server metadata"}, "data_0890": {"class_id": "http://edamontology.org/data_0890", "definitions": "A 3D profile-3D profile alignment (each profile representing structures or a structure alignment).", "label": "Structural (3D) profile alignment"}, "data_2961": {"class_id": "http://edamontology.org/data_2961", "definitions": "A report typically including a map (diagram) of a gene regulatory network.", "label": "Gene regulatory network report"}, "format_3249": {"class_id": "http://edamontology.org/format_3249", "definitions": "GelML is the format for describing the process of gel electrophoresis, standardised by HUPO PSI PS.", "label": "GelML"}, "data_0892": {"class_id": "http://edamontology.org/data_0892", "definitions": "Matrix of values used for scoring sequence-structure compatibility.", "label": "Protein sequence-structure scoring matrix"}, "operation_0350": {"class_id": "http://edamontology.org/operation_0350", "definitions": "Search a sequence database and retrieve sequences that are similar to a query sequence using a word-based method.|Word-based methods (for example BLAST, gapped BLAST, MEGABLAST, WU-BLAST etc.) are usually quicker than alignment-based methods. They may or may not handle gaps.", "label": "Sequence database search (by sequence using word-based methods)"}, "operation_0351": {"class_id": "http://edamontology.org/operation_0351", "definitions": "Search a sequence database and retrieve sequences that are similar to a query sequence using a sequence profile-based method, or with a supplied profile as query.|This includes tools based on PSI-BLAST.", "label": "Sequence database search (by sequence using profile-based methods)"}, "operation_0352": {"class_id": "http://edamontology.org/operation_0352", "definitions": "Search a sequence database for sequences that are similar to a query sequence using a local alignment-based method.|This includes tools based on the Smith-Waterman algorithm or FASTA.", "label": "Sequence database search (by sequence using local alignment-based methods)"}, "operation_0353": {"class_id": "http://edamontology.org/operation_0353", "definitions": "Search sequence(s) or a sequence database for sequences that are similar to a query sequence using a global alignment-based method.|This includes tools based on the Needleman and Wunsch algorithm.", "label": "Sequence database search (by sequence using global alignment-based methods)"}, "operation_0354": {"class_id": "http://edamontology.org/operation_0354", "definitions": "Search a DNA database (for example a database of conserved sequence tags) for matches to Sequence-Tagged Site (STS) primer sequences.|STSs are genetic markers that are easily detected by the polymerase chain reaction (PCR) using specific primers.", "label": "Sequence database search (by sequence for primer sequences)"}, "operation_0355": {"class_id": "http://edamontology.org/operation_0355", "definitions": "Search sequence(s) or a sequence database for sequences which match a set of peptide masses, for example a peptide mass fingerprint from mass spectrometry.", "label": "Sequence database search (by molecular weight)"}, "operation_0356": {"class_id": "http://edamontology.org/operation_0356", "definitions": "Search sequence(s) or a sequence database for sequences of a given isoelectric point.", "label": "Sequence database search (by isoelectric point)"}, "operation_0357": {"class_id": "http://edamontology.org/operation_0357", "definitions": "Query a tertiary structure database and retrieve entries with a given entry code or accession number.", "label": "Structure retrieval (by code)"}, "operation_0358": {"class_id": "http://edamontology.org/operation_0358", "definitions": "Query a tertiary structure database and retrieve entries containing a given keyword.", "label": "Structure retrieval (by keyword)"}, "operation_0359": {"class_id": "http://edamontology.org/operation_0359", "definitions": "Search a tertiary structure database and retrieve structures with a sequence similar to a query sequence.", "label": "Structure database search (by sequence)"}, "format_2027": {"class_id": "http://edamontology.org/format_2027", "definitions": "Data format for reports on enzyme kinetics.", "label": "Enzyme kinetics report format"}, "topic_0128": {"class_id": "http://edamontology.org/topic_0128", "definitions": "Protein-protein, protein-DNA/RNA and protein-ligand interactions, including analysis of known interactions and prediction of putative interactions.|This includes experimental (e.g. yeast two-hybrid) and computational analysis techniques.", "label": "Protein interactions"}, "format_2021": {"class_id": "http://edamontology.org/format_2021", "definitions": "Data format of a report from text mining.", "label": "Text mining report format"}, "format_2020": {"class_id": "http://edamontology.org/format_2020", "definitions": "Data format for a full-text scientific article.", "label": "Article format"}, "data_2321": {"class_id": "http://edamontology.org/data_2321", "definitions": "A unique, persistent identifier of an enzyme.", "label": "Enzyme ID"}, "data_2320": {"class_id": "http://edamontology.org/data_2320", "definitions": "The name of a cell line.", "label": "Cell line name (assonant)"}, "topic_0123": {"class_id": "http://edamontology.org/topic_0123", "definitions": "The study of the physical and biochemical properties of peptides and proteins, for example the hydrophobic, hydrophilic and charge properties of a protein.", "label": "Protein properties"}, "topic_0122": {"class_id": "http://edamontology.org/topic_0122", "definitions": "The elucidation of the three dimensional structure for all (available) proteins in a given organism.", "label": "Structural genomics"}, "data_2325": {"class_id": "http://edamontology.org/data_2325", "definitions": "Identifier of an enzyme from the REBASE enzymes database.", "label": "REBASE enzyme number"}, "format_3508": {"class_id": "http://edamontology.org/format_3508", "definitions": "Portable Document Format", "label": "PDF"}, "data_2327": {"class_id": "http://edamontology.org/data_2327", "definitions": "A unique identifier assigned to NCBI protein sequence records.|Nucleotide sequence GI number is shown in the VERSION field of the database record. Protein sequence GI number is shown in the CDS/db_xref field of a nucleotide database record, and the VERSION field of a protein database record.", "label": "GI number (protein)"}, "data_2326": {"class_id": "http://edamontology.org/data_2326", "definitions": "Unique identifier of a drug from the DrugBank database.", "label": "DrugBank ID"}, "data_0964": {"class_id": "http://edamontology.org/data_0964", "definitions": "An informative report about a specific scent.", "label": "Scent annotation"}, "data_0967": {"class_id": "http://edamontology.org/data_0967", "definitions": "Data concerning or derived from a concept from a biological ontology.", "label": "Ontology concept data"}, "data_0966": {"class_id": "http://edamontology.org/data_0966", "definitions": "A term (name) from an ontology.", "label": "Ontology term"}, "data_0960": {"class_id": "http://edamontology.org/data_0960", "definitions": "Textual metadata on a software author or end-user, for example a person or other software.", "label": "User metadata"}, "data_0963": {"class_id": "http://edamontology.org/data_0963", "definitions": "Report on a particular strain of organism cell line including plants, virus, fungi and bacteria. The data typically includes strain number, organism type, growth conditions, source and so on.", "label": "Cell line report"}, "data_0962": {"class_id": "http://edamontology.org/data_0962", "definitions": "An informative report on a specific chemical compound.", "label": "Small molecule report"}, "data_0968": {"class_id": "http://edamontology.org/data_0968", "definitions": "Keyword(s) or phrase(s) used (typically) for text-searching purposes.|Boolean operators (AND, OR and NOT) and wildcard characters may be allowed.", "label": "Keyword"}, "topic_3032": {"class_id": "http://edamontology.org/topic_3032", "definitions": "The design of primers for PCR and DNA amplification or the design of molecular probes.", "label": "Primer or probe design"}, "data_2664": {"class_id": "http://edamontology.org/data_2664", "definitions": "Identifier of an entry from the GlycomeDB database.", "label": "GlycomeDB ID"}, "data_2665": {"class_id": "http://edamontology.org/data_2665", "definitions": "Identifier of an entry from the LipidBank database.", "label": "LipidBank ID"}, "data_2666": {"class_id": "http://edamontology.org/data_2666", "definitions": "Identifier of a conserved domain from the Conserved Domain Database.", "label": "CDD ID"}, "data_2667": {"class_id": "http://edamontology.org/data_2667", "definitions": "An identifier of an entry from the MMDB database.", "label": "MMDB ID"}, "data_2660": {"class_id": "http://edamontology.org/data_2660", "definitions": "Identifier of an entry from the BioNumbers database of key numbers and associated data in molecular biology.", "label": "BioNumbers ID"}, "data_2662": {"class_id": "http://edamontology.org/data_2662", "definitions": "Unique identifier of a toxin from the Toxin and Toxin Target Database (T3DB) database.", "label": "T3DB ID"}, "data_2663": {"class_id": "http://edamontology.org/data_2663", "definitions": "Identifier of a carbohydrate.", "label": "Carbohydrate identifier"}, "data_2668": {"class_id": "http://edamontology.org/data_2668", "definitions": "Unique identifier of an entry from the iRefIndex database of protein-protein interactions.", "label": "iRefIndex ID"}, "data_2669": {"class_id": "http://edamontology.org/data_2669", "definitions": "Unique identifier of an entry from the ModelDB database.", "label": "ModelDB ID"}, "format_2333": {"class_id": "http://edamontology.org/format_2333", "definitions": "Binary format.|Only specific native binary formats are listed under 'Binary format' in EDAM. Generic binary formats - such as any data being zipped, or any XML data being serialised into the Efficient XML Interchange (EXI) format - are not modelled in EDAM. Refer to http://wsio.org/compression_004.", "label": "Binary format"}, "format_2332": {"class_id": "http://edamontology.org/format_2332", "definitions": "eXtensible Markup Language (XML) format.|Data in XML format can be serialised into text, or binary format.", "label": "XML"}, "format_2331": {"class_id": "http://edamontology.org/format_2331", "definitions": "HTML format.", "label": "HTML"}, "format_2330": {"class_id": "http://edamontology.org/format_2330", "definitions": "Textual format.|Data in text format can be compressed into binary format, or can be a value of an XML element or attribute. Markup formats are not considered textual (or more precisely, not plain-textual).", "label": "Textual format"}, "format_2334": {"class_id": "http://edamontology.org/format_2334", "definitions": "Typical textual representation of a URI.", "label": "URI format"}, "format_1647": {"class_id": "http://edamontology.org/format_1647", "definitions": "The format of an entry from the KEGG PATHWAY database of pathway maps for molecular interactions and reaction networks.", "label": "KEGG PATHWAY entry format"}, "format_1644": {"class_id": "http://edamontology.org/format_1644", "definitions": "Format of Affymetrix data file of information about (normalised) expression levels of the individual probes.", "label": "CHP"}, "format_1645": {"class_id": "http://edamontology.org/format_1645", "definitions": "Format of an entry from the Electron Microscopy DataBase (EMDB).", "label": "EMDB entry format"}, "topic_3796": {"class_id": "http://edamontology.org/topic_3796", "definitions": "Large-scale study (typically comparison) of DNA sequences of populations.", "label": "Population genomics"}, "format_1640": {"class_id": "http://edamontology.org/format_1640", "definitions": "Entry format for the ArrayExpress microarrays database.", "label": "ArrayExpress entry format"}, "format_1641": {"class_id": "http://edamontology.org/format_1641", "definitions": "Affymetrix data file format for information about experimental conditions and protocols.", "label": "affymetrix-exp"}, "operation_0004": {"class_id": "http://edamontology.org/operation_0004", "definitions": "A function that processes a set of inputs and results in a set of outputs, or associates arguments (inputs) with values (outputs).|Special cases are: a) An operation that consumes no input (has no input arguments). Such operation is either a constant function, or an operation depending only on the underlying state. b) An operation that may modify the underlying state but has no output. c) The singular-case operation with no input or output, that still may modify the underlying state.", "label": "Operation"}, "format_1648": {"class_id": "http://edamontology.org/format_1648", "definitions": "The format of an entry from the MetaCyc metabolic pathways database.", "label": "MetaCyc entry format"}, "format_1649": {"class_id": "http://edamontology.org/format_1649", "definitions": "The format of a report from the HumanCyc metabolic pathways database.", "label": "HumanCyc entry format"}, "data_1469": {"class_id": "http://edamontology.org/data_1469", "definitions": "3D coordinate and associated data for a protein tertiary (3D) structure (all atoms).", "label": "Protein structure (all atoms)"}, "operation_3731": {"class_id": "http://edamontology.org/operation_3731", "definitions": "The comparison of samples from a metagenomics study, for example, by comparison of metagenome shotgun reads or assembled contig sequences, by comparison of functional profiles, or some other method.", "label": "Sample comparison"}, "operation_3730": {"class_id": "http://edamontology.org/operation_3730", "definitions": "Construction of a single sequence assembly of all reads from different samples, typically as part of a comparative metagenomic analysis.", "label": "Cross-assembly"}, "data_2192": {"class_id": "http://edamontology.org/data_2192", "definitions": "Data on an error generated by computer system or tool.", "label": "Error"}, "data_2193": {"class_id": "http://edamontology.org/data_2193", "definitions": "Basic information on any arbitrary database entry.", "label": "Database entry metadata"}, "data_2190": {"class_id": "http://edamontology.org/data_2190", "definitions": "A fixed-size datum calculated (by using a hash function) for a molecular sequence, typically for purposes of error detection or indexing.", "label": "Sequence checksum"}, "data_2191": {"class_id": "http://edamontology.org/data_2191", "definitions": "chemical modification of a protein.", "label": "Protein features report (chemical modifications)"}, "topic_3335": {"class_id": "http://edamontology.org/topic_3335", "definitions": "The diseases and abnormalities of the heart and circulatory system.", "label": "Cardiology"}, "topic_3334": {"class_id": "http://edamontology.org/topic_3334", "definitions": "The branch of medicine that deals with the anatomy, functions and disorders of the nervous system.", "label": "Neurology"}, "topic_3337": {"class_id": "http://edamontology.org/topic_3337", "definitions": "Repositories of biological samples, typically human, for basic biological and clinical research.", "label": "Biobank"}, "topic_3336": {"class_id": "http://edamontology.org/topic_3336", "definitions": "The discovery and design of drugs or potential drug compounds.|This includes methods that search compound collections, generate or analyse drug 3D conformations, identify drug targets with structural docking etc.", "label": "Drug discovery"}, "data_2198": {"class_id": "http://edamontology.org/data_2198", "definitions": "A cluster of similar genes.", "label": "Gene cluster"}, "format_2032": {"class_id": "http://edamontology.org/format_2032", "definitions": "Format of a workflow.", "label": "Workflow format"}, "operation_2460": {"class_id": "http://edamontology.org/operation_2460", "definitions": "Calculate the solvent accessibility for each atom in a structure.|Waters are not considered.", "label": "Protein atom surface calculation"}, "operation_2461": {"class_id": "http://edamontology.org/operation_2461", "definitions": "Calculate the solvent accessibility for each residue in a structure.", "label": "Protein residue surface calculation"}, "operation_2462": {"class_id": "http://edamontology.org/operation_2462", "definitions": "Calculate the solvent accessibility of a structure as a whole.", "label": "Protein surface calculation"}, "format_1739": {"class_id": "http://edamontology.org/format_1739", "definitions": "Article format of the PubMed Central database.", "label": "pmc"}, "operation_2464": {"class_id": "http://edamontology.org/operation_2464", "definitions": "Identify or predict protein-protein interactions, interfaces, binding sites etc.", "label": "Protein-protein interaction prediction"}, "operation_2465": {"class_id": "http://edamontology.org/operation_2465", "definitions": "Process (read and / or write) a molecular tertiary structure.", "label": "Structure processing"}, "operation_2466": {"class_id": "http://edamontology.org/operation_2466", "definitions": "Annotate a DNA map of some type with terms from a controlled vocabulary.", "label": "Map annotation"}, "operation_2467": {"class_id": "http://edamontology.org/operation_2467", "definitions": "Retrieve information on a protein.", "label": "Data retrieval (protein annotation)"}, "operation_2468": {"class_id": "http://edamontology.org/operation_2468", "definitions": "Retrieve a phylogenetic tree from a data resource.", "label": "Data retrieval (phylogenetic tree)"}, "operation_2469": {"class_id": "http://edamontology.org/operation_2469", "definitions": "Retrieve information on a protein interaction.", "label": "Data retrieval (protein interaction annotation)"}, "operation_3447": {"class_id": "http://edamontology.org/operation_3447", "definitions": "Processing of diffraction data into a corrected, ordered, and simplified form.", "label": "Diffraction data reduction"}, "operation_3446": {"class_id": "http://edamontology.org/operation_3446", "definitions": "Analysis of cell migration images in order to study cell migration, typically in order to study the processes that play a role in the disease progression.", "label": "Cell migration analysis"}, "format_1736": {"class_id": "http://edamontology.org/format_1736", "definitions": "CiteXplore 'core' citation format including title, journal, authors and abstract.", "label": "CiteXplore-core"}, "format_1737": {"class_id": "http://edamontology.org/format_1737", "definitions": "CiteXplore 'all' citation format includes all known details such as Mesh terms and cross-references.", "label": "CiteXplore-all"}, "format_1734": {"class_id": "http://edamontology.org/format_1734", "definitions": "Format of bibliographic reference as used by the PubMed database.", "label": "PubMed citation"}, "format_1735": {"class_id": "http://edamontology.org/format_1735", "definitions": "Format for abstracts of scientific articles from the Medline database.|Bibliographic reference information including citation information is included", "label": "Medline Display Format"}, "format_1228": {"class_id": "http://edamontology.org/format_1228", "definitions": "Format of an entry from UniGene.|A UniGene entry includes a set of transcript sequences assigned to the same transcription locus (gene or expressed pseudogene), with information on protein similarities, gene expression, cDNA clone reagents, and genomic location.", "label": "UniGene entry format"}, "format_3771": {"class_id": "http://edamontology.org/format_3771", "definitions": "UniProtKB RDF sequence features format is an RDF format available for downloading UniProt entries (in RDF/XML).", "label": "UniProtKB RDF"}, "data_3137": {"class_id": "http://edamontology.org/data_3137", "definitions": "features of non-coding or functional RNA molecules, including tRNA and rRNA.", "label": "Non-coding RNA"}, "data_3135": {"class_id": "http://edamontology.org/data_3135", "definitions": "coding sequences for a signal or transit peptide.", "label": "Nucleic acid features report (signal or transit peptide)"}, "data_3134": {"class_id": "http://edamontology.org/data_3134", "definitions": "An informative report on features of a messenger RNA (mRNA) molecules including precursor RNA, primary (unprocessed) transcript and fully processed molecules. This includes reports on a specific gene transcript, clone or EST.|This includes 5'untranslated region (5'UTR), coding sequences (CDS), exons, intervening sequences (intron) and 3'untranslated regions (3'UTR).", "label": "Gene transcript report"}, "data_3133": {"class_id": "http://edamontology.org/data_3133", "definitions": "A report on stem loops in a DNA sequence.|A stem loop is a hairpin structure; a double-helical structure formed when two complementary regions of a single strand of RNA or DNA molecule form base-pairs.", "label": "Nucleic acid features (stem loop)"}, "data_3132": {"class_id": "http://edamontology.org/data_3132", "definitions": "A report on displacement loops in a mitochondrial DNA sequence.|A displacement loop is a region of mitochondrial DNA in which one of the strands is displaced by an RNA molecule.", "label": "Nucleic acid features (d-loop)"}, "data_3131": {"class_id": "http://edamontology.org/data_3131", "definitions": "Report on the location of matches to profiles, motifs (conserved or functional patterns) or other signatures in one or more nucleic acid sequences.", "label": "Sequence motif matches (nucleic acid)"}, "data_3130": {"class_id": "http://edamontology.org/data_3130", "definitions": "Report on the location of matches to profiles, motifs (conserved or functional patterns) or other signatures in one or more protein sequences.", "label": "Sequence motif matches (protein)"}, "topic_2867": {"class_id": "http://edamontology.org/topic_2867", "definitions": "Variable number of tandem repeat (VNTR) polymorphism in a DNA sequence.", "label": "VNTR"}, "format_1343": {"class_id": "http://edamontology.org/format_1343", "definitions": "Format of results of a search of the InterPro database showing matches between protein sequence(s) and signatures for an InterPro entry.|The table presents matches between query proteins (rows) and signature methods (columns) for this entry. Alternatively the sequence(s) might be from from the InterPro entry itself. The match position in the protein sequence and match status (true positive, false positive etc) are indicated.", "label": "InterPro match table format"}, "data_3139": {"class_id": "http://edamontology.org/data_3139", "definitions": "sequence tagged sites (STS) in nucleic acid sequences.", "label": "Nucleic acid features report (STS)"}, "data_3138": {"class_id": "http://edamontology.org/data_3138", "definitions": "Features concerning transcription of DNA into RNA including the regulation of transcription.|This includes promoters, CAAT signals, TATA signals, -35 signals, -10 signals, GC signals, primer binding sites for initiation of transcription or reverse transcription, enhancer, attenuator, terminators and ribosome binding sites.", "label": "Transcriptional features (report)"}, "operation_2514": {"class_id": "http://edamontology.org/operation_2514", "definitions": "Generate a protein sequence by some means.", "label": "Sequence generation (protein)"}, "operation_2515": {"class_id": "http://edamontology.org/operation_2515", "definitions": "Visualise, format or render a nucleic acid sequence.|Various nucleic acid sequence analysis methods might generate a sequence rendering but are not (for brevity) listed under here.", "label": "Nucleic acid sequence visualisation"}, "data_2688": {"class_id": "http://edamontology.org/data_2688", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Mus musculus' division).", "label": "Ensembl ID ('Mus musculus')"}, "data_2689": {"class_id": "http://edamontology.org/data_2689", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Myotis lucifugus' division).", "label": "Ensembl ID ('Myotis lucifugus')"}, "operation_2510": {"class_id": "http://edamontology.org/operation_2510", "definitions": "Back-translate a protein sequence into DNA.", "label": "DNA back-translation"}, "operation_2511": {"class_id": "http://edamontology.org/operation_2511", "definitions": "Edit or change a nucleic acid sequence, either randomly or specifically.", "label": "Sequence editing (nucleic acid)"}, "operation_2512": {"class_id": "http://edamontology.org/operation_2512", "definitions": "Edit or change a protein sequence, either randomly or specifically.", "label": "Sequence editing (protein)"}, "operation_2513": {"class_id": "http://edamontology.org/operation_2513", "definitions": "Generate a nucleic acid sequence by some means.", "label": "Sequence generation (nucleic acid)"}, "data_2682": {"class_id": "http://edamontology.org/data_2682", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Gallus gallus' division).", "label": "Ensembl ID ('Gallus gallus')"}, "data_2683": {"class_id": "http://edamontology.org/data_2683", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Gasterosteus aculeatus' division).", "label": "Ensembl ID ('Gasterosteus aculeatus')"}, "data_2680": {"class_id": "http://edamontology.org/data_2680", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Erinaceus europaeus' division).", "label": "Ensembl ID ('Erinaceus europaeus')"}, "data_2681": {"class_id": "http://edamontology.org/data_2681", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Felis catus' division).", "label": "Ensembl ID ('Felis catus')"}, "data_2686": {"class_id": "http://edamontology.org/data_2686", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Macaca mulatta' division).", "label": "Ensembl ID ('Macaca mulatta')"}, "data_2687": {"class_id": "http://edamontology.org/data_2687", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Monodelphis domestica' division).", "label": "Ensembl ID ('Monodelphis domestica')"}, "data_2684": {"class_id": "http://edamontology.org/data_2684", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Homo sapiens' division).", "label": "Ensembl ID ('Homo sapiens')"}, "data_2685": {"class_id": "http://edamontology.org/data_2685", "definitions": "Identifier of an entry (exon, gene, transcript or protein) from the Ensembl 'core' database ('Loxodonta africana' division).", "label": "Ensembl ID ('Loxodonta africana')"}, "data_1184": {"class_id": "http://edamontology.org/data_1184", "definitions": "An identifier of a concept from the ChEBI ontology.", "label": "ChEBI concept ID"}, "data_1185": {"class_id": "http://edamontology.org/data_1185", "definitions": "An identifier of a concept from the MGED ontology.", "label": "MGED concept ID"}, "data_1186": {"class_id": "http://edamontology.org/data_1186", "definitions": "An identifier of a concept from the myGrid ontology.|The ontology is provided as two components, the service ontology and the domain ontology. The domain ontology acts provides concepts for core bioinformatics data types and their relations. The service ontology describes the physical and operational features of web services.", "label": "myGrid concept ID"}, "data_1187": {"class_id": "http://edamontology.org/data_1187", "definitions": "PubMed unique identifier of an article.", "label": "PubMed ID"}, "data_1180": {"class_id": "http://edamontology.org/data_1180", "definitions": "An identifier of a concept from the Plant Ontology (PO).", "label": "Plant Ontology concept ID"}, "data_1181": {"class_id": "http://edamontology.org/data_1181", "definitions": "An identifier of a concept from the UMLS vocabulary.", "label": "UMLS concept ID"}, "data_1182": {"class_id": "http://edamontology.org/data_1182", "definitions": "An identifier of a concept from Foundational Model of Anatomy.|Classifies anatomical entities according to their shared characteristics (genus) and distinguishing characteristics (differentia). Specifies the part-whole and spatial relationships of the entities, morphological transformation of the entities during prenatal development and the postnatal life cycle and principles, rules and definitions according to which classes and relationships in the other three components of FMA are represented.", "label": "FMA concept ID"}, "data_1183": {"class_id": "http://edamontology.org/data_1183", "definitions": "An identifier of a concept from the EMAP mouse ontology.", "label": "EMAP concept ID"}, "format_1342": {"class_id": "http://edamontology.org/format_1342", "definitions": "Format of results of a search of the InterPro database showing matches of query protein sequence(s) to InterPro entries.|The report includes a classification of regions in a query protein sequence which are assigned to a known InterPro protein family or group.", "label": "InterPro protein view report format"}, "data_1188": {"class_id": "http://edamontology.org/data_1188", "definitions": "Digital Object Identifier (DOI) of a published article.", "label": "DOI"}, "data_1189": {"class_id": "http://edamontology.org/data_1189", "definitions": "Medline UI (unique identifier) of an article.|The use of Medline UI has been replaced by the PubMed unique identifier.", "label": "Medline UI"}, "operation_3207": {"class_id": "http://edamontology.org/operation_3207", "definitions": "Analysing the DNA methylation of specific genes or regions of interest.", "label": "Gene methylation analysis"}, "operation_3206": {"class_id": "http://edamontology.org/operation_3206", "definitions": "Measure the overall level of methyl cytosines in a genome from analysis of experimental data, typically from chromatographic methods and methyl accepting capacity assay.", "label": "Whole genome methylation analysis"}, "data_1340": {"class_id": "http://edamontology.org/data_1340", "definitions": "A report on the evaluation of the significance of sequence similarity scores from a sequence database search (for example a BLAST search).", "label": "Sequence database hits evaluation data"}, "operation_3204": {"class_id": "http://edamontology.org/operation_3204", "definitions": "Determine cytosine methylation states in nucleic acid sequences.", "label": "Methylation analysis"}, "data_1346": {"class_id": "http://edamontology.org/data_1346", "definitions": "File of directives for ordering and spacing of MEME motifs.", "label": "MEME motifs directive file"}, "data_1347": {"class_id": "http://edamontology.org/data_1347", "definitions": "Dirichlet distribution used by hidden Markov model analysis programs.", "label": "Dirichlet distribution"}, "data_1344": {"class_id": "http://edamontology.org/data_1344", "definitions": "Alphabet for the motifs (patterns) that MEME will search for.", "label": "MEME motif alphabet"}, "data_1345": {"class_id": "http://edamontology.org/data_1345", "definitions": "MEME background frequencies file.", "label": "MEME background frequencies file"}, "format_1350": {"class_id": "http://edamontology.org/format_1350", "definitions": "Dirichlet distribution MEME format.", "label": "MEME Dirichlet prior"}, "format_1351": {"class_id": "http://edamontology.org/format_1351", "definitions": "Format of a report from the HMMER package on the emission and transition counts of a hidden Markov model.", "label": "HMMER emission and transition"}, "data_1348": {"class_id": "http://edamontology.org/data_1348", "definitions": "Emission and transition counts of a hidden Markov model, generated once HMM has been determined, for example after residues/gaps have been assigned to match, delete and insert states.", "label": "HMM emission and transition counts"}, "topic_0176": {"class_id": "http://edamontology.org/topic_0176", "definitions": "The study and simulation of molecular (typically protein) conformation using a computational model of physical forces and computer simulation.|This includes resources concerning flexibility and motion in protein and other molecular structures.", "label": "Molecular dynamics"}, "topic_3534": {"class_id": "http://edamontology.org/topic_3534", "definitions": "Binding sites in proteins, including cleavage sites (for a proteolytic enzyme or agent), key residues involved in protein folding, catalytic residues (active site) of an enzyme, ligand-binding (non-catalytic) residues of a protein, such as sites that bind metal, prosthetic groups or lipids, RNA and DNA-binding proteins and binding sites etc.", "label": "Protein binding sites"}, "topic_2269": {"class_id": "http://edamontology.org/topic_2269", "definitions": "The application of statistical methods to biological problems.", "label": "Statistics and probability"}, "format_1356": {"class_id": "http://edamontology.org/format_1356", "definitions": "Format of a regular expression pattern from the Prosite database.", "label": "prosite-pattern"}, "format_1357": {"class_id": "http://edamontology.org/format_1357", "definitions": "Format of an EMBOSS sequence pattern.", "label": "EMBOSS sequence pattern"}, "data_3002": {"class_id": "http://edamontology.org/data_3002", "definitions": "Annotation of one particular positional feature on a biomolecular (typically genome) sequence, suitable for import and display in a genome browser.", "label": "Annotation track"}, "data_2530": {"class_id": "http://edamontology.org/data_2530", "definitions": "An informative report on a specific organism.", "label": "Organism report"}, "data_2531": {"class_id": "http://edamontology.org/data_2531", "definitions": "Annotation on a wet lab experiment, such as experimental conditions.", "label": "Experiment report"}, "data_2533": {"class_id": "http://edamontology.org/data_2533", "definitions": "DNA mutation.", "label": "Nucleic acid features report (mutation)"}, "data_2534": {"class_id": "http://edamontology.org/data_2534", "definitions": "An attribute of a molecular sequence, possibly in reference to some other sequence.", "label": "Sequence attribute"}, "data_2535": {"class_id": "http://edamontology.org/data_2535", "definitions": "Output from a serial analysis of gene expression (SAGE), massively parallel signature sequencing (MPSS) or sequencing by synthesis (SBS) experiment.  In all cases this is a list of short sequence tags and the number of times it is observed.|This includes tag to gene assignments (tag mapping) of SAGE, MPSS and SBS data. Typically this is the sequencing-based expression profile annotated with gene identifiers.|SAGE, MPSS and SBS experiments are usually performed to study gene expression.  The sequence tags are typically subsequently annotated (after a database search) with the mRNA (and therefore gene) the tag was extracted from.", "label": "Sequence tag profile"}, "data_2536": {"class_id": "http://edamontology.org/data_2536", "definitions": "Data concerning a mass spectrometry measurement.", "label": "Mass spectrometry data"}, "data_2537": {"class_id": "http://edamontology.org/data_2537", "definitions": "Raw data from experimental methods for determining protein structure.|This is a broad data type and is used a placeholder for other, more specific types.  It is primarily intended to help navigation of EDAM and would not typically be used for annotation.", "label": "Protein structure raw data"}, "data_2538": {"class_id": "http://edamontology.org/data_2538", "definitions": "An identifier of a mutation.", "label": "Mutation identifier"}, "data_2539": {"class_id": "http://edamontology.org/data_2539", "definitions": "Data concerning an alignment of two or more molecular sequences, structures or derived data.|This is a broad data type and is used a placeholder for other, more specific types.  This includes entities derived from sequences and structures such as motifs and profiles.", "label": "Alignment data"}, "data_0895": {"class_id": "http://edamontology.org/data_0895", "definitions": "An informative report about a specific peptide.", "label": "Peptide annotation"}, "operation_2089": {"class_id": "http://edamontology.org/operation_2089", "definitions": "Refine an existing sequence alignment.", "label": "Sequence alignment refinement"}, "topic_0922": {"class_id": "http://edamontology.org/topic_0922", "definitions": "PCR primers and hybridisation oligos in a nucleic acid sequence.", "label": "Primers"}, "operation_2241": {"class_id": "http://edamontology.org/operation_2241", "definitions": "Visualise transmembrane proteins, typically the transmembrane regions within a sequence.", "label": "Transmembrane protein visualisation"}, "operation_2246": {"class_id": "http://edamontology.org/operation_2246", "definitions": "An operation performing purely illustrative (pedagogical) purposes.", "label": "Demonstration"}, "format_1912": {"class_id": "http://edamontology.org/format_1912", "definitions": "Phylogenetic tree Nexus (text) format.", "label": "Nexus format"}, "format_1910": {"class_id": "http://edamontology.org/format_1910", "definitions": "Phylogenetic tree Newick (text) format.", "label": "newick"}, "format_1911": {"class_id": "http://edamontology.org/format_1911", "definitions": "Phylogenetic tree TreeCon (text) format.", "label": "TreeCon format"}, "format_1915": {"class_id": "http://edamontology.org/format_1915", "definitions": "A defined way or layout of representing and structuring data in a computer file, blob, string, message, or elsewhere.|The main focus in EDAM lies on formats as means of structuring data exchanged between different tools or resources. The serialisation, compression, or encoding of concrete data formats/models is not in scope of EDAM. Format 'is format of' Data.", "label": "Format"}, "format_1918": {"class_id": "http://edamontology.org/format_1918", "definitions": "Data format for an individual atom.", "label": "Atomic data format"}, "format_1919": {"class_id": "http://edamontology.org/format_1919", "definitions": "Data format for a molecular sequence record.", "label": "Sequence record format"}, "format_3776": {"class_id": "http://edamontology.org/format_3776", "definitions": "BTrack is an HDF5-based binary format for genome or sequence feature tracks and their collections, suitable for integrative multi-track analysis. BTrack is a binary, compressed alternative to the GTrack and GSuite formats.", "label": "BTrack"}, "format_3554": {"class_id": "http://edamontology.org/format_3554", "definitions": "File format used for scripts written in the R programming language for execution within the R software environment, typically for statistical computation and graphics.", "label": "R file format"}, "topic_2811": {"class_id": "http://edamontology.org/topic_2811", "definitions": "Biological nomenclature (naming), symbols and terminology.", "label": "Nomenclature"}, "format_3555": {"class_id": "http://edamontology.org/format_3555", "definitions": "File format used for scripts for the Statistical Package for the Social Sciences.", "label": "SPSS"}, "data_2248": {"class_id": "http://edamontology.org/data_2248", "definitions": "A data schema for organising or transforming data of some type.", "label": "Schema"}, "data_2249": {"class_id": "http://edamontology.org/data_2249", "definitions": "A DTD (document type definition).", "label": "DTD"}, "data_1764": {"class_id": "http://edamontology.org/data_1764", "definitions": "FASTA sequence database (based on ATOM records in PDB) for CATH domains (clustered at different levels of sequence identity).", "label": "CATH representative domain sequences (ATOM)"}, "data_1765": {"class_id": "http://edamontology.org/data_1765", "definitions": "FASTA sequence database (based on COMBS sequence data) for CATH domains (clustered at different levels of sequence identity).", "label": "CATH representative domain sequences (COMBS)"}, "data_1766": {"class_id": "http://edamontology.org/data_1766", "definitions": "FASTA sequence database for all CATH domains (based on PDB ATOM records).", "label": "CATH domain sequences (ATOM)"}, "data_1767": {"class_id": "http://edamontology.org/data_1767", "definitions": "FASTA sequence database for all CATH domains (based on COMBS sequence data).", "label": "CATH domain sequences (COMBS)"}, "data_1762": {"class_id": "http://edamontology.org/data_1762", "definitions": "Summary of domain classification information for a CATH domain.|The report (for example http://www.cathdb.info/domain/1cukA01) includes CATH codes for levels in the hierarchy for the domain, level descriptions and relevant data and links.", "label": "CATH domain report"}, "format_3551": {"class_id": "http://edamontology.org/format_3551", "definitions": "Nearly Raw Rasta Data format designed to support scientific visualisation and image processing involving N-dimensional raster data.", "label": "nrrd"}, "topic_0163": {"class_id": "http://edamontology.org/topic_0163", "definitions": "Search and retrieve molecular sequences that are similar to a sequence-based query (typically a simple sequence).|The query is a sequence-based entity such as another sequence, a motif or profile.", "label": "Sequence database search"}, "data_1162": {"class_id": "http://edamontology.org/data_1162", "definitions": "Unique identifier of a MIRIAM data resource.|This is the identifier used internally by MIRIAM for a data type.", "label": "MIRIAM identifier"}, "data_1163": {"class_id": "http://edamontology.org/data_1163", "definitions": "The name of a data type from the MIRIAM database.", "label": "MIRIAM data type name"}, "data_1160": {"class_id": "http://edamontology.org/data_1160", "definitions": "Identifier of an entry from the CPDB (ConsensusPathDB) biological pathways database, which is an identifier from an external database integrated into CPDB.|This concept refers to identifiers used by the databases collated in CPDB; CPDB identifiers are not independently defined.", "label": "Pathway ID (CPDB)"}, "data_1161": {"class_id": "http://edamontology.org/data_1161", "definitions": "Identifier of a biological pathway from the Panther Pathways database.", "label": "Pathway ID (Panther)"}, "data_1166": {"class_id": "http://edamontology.org/data_1166", "definitions": "A synonymous name of a data type from the MIRIAM database.|A synonymous name for a MIRIAM data type taken from a controlled vocabulary.", "label": "MIRIAM data type synonymous name"}, "data_1167": {"class_id": "http://edamontology.org/data_1167", "definitions": "Unique identifier of a Taverna workflow.", "label": "Taverna workflow ID"}, "data_1164": {"class_id": "http://edamontology.org/data_1164", "definitions": "The URI (URL or URN) of a data entity from the MIRIAM database.|A MIRIAM URI consists of the URI of the MIRIAM data type (PubMed, UniProt etc) followed by the identifier of an element of that data type, for example PMID for a publication or an accession number for a GO term.", "label": "MIRIAM URI"}, "data_1165": {"class_id": "http://edamontology.org/data_1165", "definitions": "The primary name of a data type from the MIRIAM database.|The primary name of a MIRIAM data type is taken from a controlled vocabulary.", "label": "MIRIAM data type primary name"}, "data_2242": {"class_id": "http://edamontology.org/data_2242", "definitions": "Phylogenetic property values data.", "label": "Phylogenetic property values"}, "topic_3048": {"class_id": "http://edamontology.org/topic_3048", "definitions": "Mammals, e.g. information on a specific mammal genome including molecular sequences, genes and annotation.", "label": "Mammals"}, "format_2566": {"class_id": "http://edamontology.org/format_2566", "definitions": "Alphabet for a molecular sequence without any unknown positions or ambiguity characters.", "label": "completely unambiguous"}, "data_2849": {"class_id": "http://edamontology.org/data_2849", "definitions": "An abstract of a scientific article.", "label": "Abstract"}, "operation_3458": {"class_id": "http://edamontology.org/operation_3458", "definitions": "Compare (align and classify) multiple particle images from a micrograph in order to produce a representative image of the particle. |A micrograph can include particles in multiple different orientations and/or conformations. Particles are compared and organised into sets based on their similarity. Typically iterations of classification and alignment and are performed to optimise the final 3D EM map.", "label": "Single particle alignment and classification"}, "operation_3459": {"class_id": "http://edamontology.org/operation_3459", "definitions": "Clustering of molecular sequences on the basis of their function, typically using information from an ontology of gene function, or some other measure of functional phenotype.", "label": "Functional clustering"}, "data_2994": {"class_id": "http://edamontology.org/data_2994", "definitions": "Weights for sequence positions or characters in phylogenetic analysis where zero is defined as unweighted.", "label": "Phylogenetic character weights"}, "data_2992": {"class_id": "http://edamontology.org/data_2992", "definitions": "An image of protein structure.", "label": "Protein structure image"}, "data_2991": {"class_id": "http://edamontology.org/data_2991", "definitions": "Geometry data for a protein structure, for example bond lengths, bond angles, torsion angles, chiralities, planaraties etc.", "label": "Protein geometry report"}, "format_2922": {"class_id": "http://edamontology.org/format_2922", "definitions": "Some variant of Pearson MARKX alignment format.", "label": "markx0 variant"}, "format_2923": {"class_id": "http://edamontology.org/format_2923", "definitions": "Some variant of Mega format for (typically aligned) sequences.", "label": "mega variant"}, "format_2920": {"class_id": "http://edamontology.org/format_2920", "definitions": "Data format for molecular sequence alignment information that can hold sequence alignment(s) of only 2 sequences.", "label": "Alignment format (pair only)"}, "format_2921": {"class_id": "http://edamontology.org/format_2921", "definitions": "Format of sequence variation annotation.", "label": "Sequence variation annotation format"}, "format_2924": {"class_id": "http://edamontology.org/format_2924", "definitions": "Some variant of Phylip format for (aligned) sequences.", "label": "Phylip format variant"}, "operation_3211": {"class_id": "http://edamontology.org/operation_3211", "definitions": "Generate an index of a genome sequence.|Many sequence alignment tasks involving many or very large sequences rely on a precomputed index of the sequence to accelerate the alignment.  The Burrows-Wheeler Transform (BWT) is a permutation of the genome based on a suffix array algorithm.  A suffix array consists of the lexicographically sorted list of suffixes of a genome.", "label": "Genome indexing"}, "format_1332": {"class_id": "http://edamontology.org/format_1332", "definitions": "Format of results of a sequence database search using FASTA.|This includes (typically) score data, alignment data and a histogram (of observed and expected distribution of E values.)", "label": "FASTA search results format"}, "format_1333": {"class_id": "http://edamontology.org/format_1333", "definitions": "Format of results of a sequence database search using some variant of BLAST.|This includes score data, alignment data and summary table.", "label": "BLAST results"}, "topic_0208": {"class_id": "http://edamontology.org/topic_0208", "definitions": "The influence of genotype on drug response, for example by correlating gene expression or single-nucleotide polymorphisms with drug efficacy or toxicity.", "label": "Pharmacogenomics"}, "topic_0209": {"class_id": "http://edamontology.org/topic_0209", "definitions": "The design and chemical synthesis of bioactive molecules, for example drugs or potential drug compounds, for medicinal purposes.|This includes methods that search compound collections, generate or analyse drug 3D conformations, identify drug targets with structural docking etc.", "label": "Medicinal chemistry"}, "format_3329": {"class_id": "http://edamontology.org/format_3329", "definitions": "HMMER profile HMM file for HMMER versions 3.x", "label": "HMMER3"}, "format_3328": {"class_id": "http://edamontology.org/format_3328", "definitions": "HMMER profile HMM file for HMMER versions 2.x", "label": "HMMER2"}, "topic_0200": {"class_id": "http://edamontology.org/topic_0200", "definitions": "Microarrays, for example, to process microarray data or design probes and experiments.", "label": "Microarrays"}, "topic_0753": {"class_id": "http://edamontology.org/topic_0753", "definitions": "Metabolic pathways.", "label": "Metabolic pathways"}, "topic_0202": {"class_id": "http://edamontology.org/topic_0202", "definitions": "The study of drugs and their effects or responses in living systems. ", "label": "Pharmacology"}, "topic_0203": {"class_id": "http://edamontology.org/topic_0203", "definitions": "The analysis of levels and patterns of synthesis of gene products (proteins and functional RNA) including interpretation in functional terms of gene expression data.|Gene expression levels are analysed by identifying, quantifying or comparing mRNA transcripts, for example using microarrays, RNA-seq, northern blots, gene-indexed expression profiles etc.|This includes the study of codon usage in nucleotide sequence(s), genetic codes and so on.", "label": "Gene expression"}, "format_3327": {"class_id": "http://edamontology.org/format_3327", "definitions": "BAM indexing format", "label": "BAI"}, "format_3326": {"class_id": "http://edamontology.org/format_3326", "definitions": "", "label": "Data index format"}, "topic_0754": {"class_id": "http://edamontology.org/topic_0754", "definitions": "Signaling pathways.", "label": "Signaling pathways"}, "format_3699": {"class_id": "http://edamontology.org/format_3699", "definitions": "VDB ('vertical database') is the native format used for export from the NCBI Sequence Read Archive.", "label": "VDB"}, "format_3698": {"class_id": "http://edamontology.org/format_3698", "definitions": " SRA archive format (SRA) is the archive format used for input to the NCBI Sequence Read Archive.", "label": "SRA format"}, "data_2201": {"class_id": "http://edamontology.org/data_2201", "definitions": "A molecular sequence and comprehensive metadata (such as a feature table), typically corresponding to a full entry from a molecular sequence database.", "label": "Sequence record full"}, "format_3693": {"class_id": "http://edamontology.org/format_3693", "definitions": "AGP is a tabular format for a sequence assembly (a contig, a scaffold/supercontig, or a chromosome).", "label": "AGP"}, "format_3692": {"class_id": "http://edamontology.org/format_3692", "definitions": "SBGN-ML is an XML format for Systems Biology Graphical Notation (SBGN) diagrams of biological pathways or networks.", "label": "SBGN-ML"}, "format_3691": {"class_id": "http://edamontology.org/format_3691", "definitions": "Biological Expression Language (BEL) is a textual format for representing scientific findings in life sciences in a computable form.", "label": "BEL"}, "format_3690": {"class_id": "http://edamontology.org/format_3690", "definitions": "Biological Dynamics Markup Language (BDML) is an XML format for quantitative data describing biological dynamics.", "label": "BDML"}, "format_3696": {"class_id": "http://edamontology.org/format_3696", "definitions": "PostScript format", "label": "PS"}, "data_2208": {"class_id": "http://edamontology.org/data_2208", "definitions": "An identifier of a plasmid in a database.", "label": "Plasmid identifier"}, "data_2209": {"class_id": "http://edamontology.org/data_2209", "definitions": "A unique identifier of a specific mutation catalogued in a database.", "label": "Mutation ID"}, "format_2585": {"class_id": "http://edamontology.org/format_2585", "definitions": "Systems Biology Markup Language (SBML), the standard XML format for models of biological processes such as for example metabolism, cell signaling, and gene regulation.", "label": "SBML"}, "data_1885": {"class_id": "http://edamontology.org/data_1885", "definitions": "Identifier of a gene from the GeneFarm database.", "label": "Gene ID (GeneFarm)"}, "data_1884": {"class_id": "http://edamontology.org/data_1884", "definitions": "A controlled vocabulary for words and phrases that can appear in the keywords field (KW line) of entries from the UniProt database.", "label": "UniProt keywords"}, "data_1887": {"class_id": "http://edamontology.org/data_1887", "definitions": "Identifier for genetic elements in MIPS Maize database.", "label": "Gene ID (MIPS Maize)"}, "data_1886": {"class_id": "http://edamontology.org/data_1886", "definitions": "The blattner identifier for a gene.", "label": "Blattner number"}, "data_1881": {"class_id": "http://edamontology.org/data_1881", "definitions": "Information on the authors of a published work.", "label": "Author ID"}, "data_1880": {"class_id": "http://edamontology.org/data_1880", "definitions": "A term which is likely to be misleading of its meaning.", "label": "Misnomer"}, "data_1883": {"class_id": "http://edamontology.org/data_1883", "definitions": "A URI along with annotation describing the data found at the address.", "label": "Annotated URI"}, "data_1882": {"class_id": "http://edamontology.org/data_1882", "definitions": "An identifier representing an author in the DragonDB database.", "label": "DragonDB author identifier"}, "data_1889": {"class_id": "http://edamontology.org/data_1889", "definitions": "The name of an Antirrhinum Gene from the DragonDB database.", "label": "Gene name (DragonDB)"}, "data_1888": {"class_id": "http://edamontology.org/data_1888", "definitions": "Identifier for genetic elements in MIPS Medicago database.", "label": "Gene ID (MIPS Medicago)"}, "data_3483": {"class_id": "http://edamontology.org/data_3483", "definitions": "The spectrum of frequencies of electromagnetic radiation emitted from a molecule as a result of some spectroscopy experiment.", "label": "Spectrum"}, "data_3488": {"class_id": "http://edamontology.org/data_3488", "definitions": "Spectral information for a molecule from a nuclear magnetic resonance experiment.", "label": "NMR spectrum"}, "data_1401": {"class_id": "http://edamontology.org/data_1401", "definitions": "The score for a 'match' used in various sequence database search applications with simple scoring schemes.", "label": "Match reward score"}, "data_1400": {"class_id": "http://edamontology.org/data_1400", "definitions": "A penalty for gaps at the termini of an alignment, either from the N/C terminal of protein or 5'/3' terminal of nucleotide sequences.", "label": "Terminal gap penalty"}, "data_2338": {"class_id": "http://edamontology.org/data_2338", "definitions": "Any arbitrary identifier of an ontology.", "label": "Ontology identifier"}, "data_2339": {"class_id": "http://edamontology.org/data_2339", "definitions": "The name of a concept in an ontology.", "label": "Ontology concept name"}, "data_1405": {"class_id": "http://edamontology.org/data_1405", "definitions": "A simple floating point number defining the penalty for opening a gap in an alignment.", "label": "Gap opening penalty (float)"}, "data_1404": {"class_id": "http://edamontology.org/data_1404", "definitions": "A simple floating point number defining the penalty for opening a gap in an alignment.", "label": "Gap opening penalty (integer)"}, "data_1407": {"class_id": "http://edamontology.org/data_1407", "definitions": "A simple floating point number defining the penalty for extending a gap in an alignment.", "label": "Gap extension penalty (float)"}, "data_1406": {"class_id": "http://edamontology.org/data_1406", "definitions": "A simple floating point number defining the penalty for extending a gap in an alignment.", "label": "Gap extension penalty (integer)"}, "data_1409": {"class_id": "http://edamontology.org/data_1409", "definitions": "A simple floating point number defining the penalty for gaps that are close together in an alignment.", "label": "Gap separation penalty (float)"}, "data_1408": {"class_id": "http://edamontology.org/data_1408", "definitions": "A simple floating point number defining the penalty for gaps that are close together in an alignment.", "label": "Gap separation penalty (integer)"}, "data_2336": {"class_id": "http://edamontology.org/data_2336", "definitions": "Phase for translation of DNA (0, 1 or 2) relative to a fragment of the coding sequence.", "label": "Translation phase specification"}, "data_2337": {"class_id": "http://edamontology.org/data_2337", "definitions": "Data concerning or describing some core computational resource, as distinct from primary data.  This includes metadata on the origin, source, history, ownership or location of some thing.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Resource metadata"}, "topic_0112": {"class_id": "http://edamontology.org/topic_0112", "definitions": "The folding (in 3D space) of nucleic acid molecules.", "label": "Nucleic acid folding"}, "data_2335": {"class_id": "http://edamontology.org/data_2335", "definitions": "A score derived from the alignment of two sequences, which is then normalised with respect to the scoring system.|Bit scores are normalised with respect to the scoring system and therefore can be used to compare alignment scores from different searches.", "label": "Bit score"}, "data_0976": {"class_id": "http://edamontology.org/data_0976", "definitions": "An identifier that identifies a particular type of data.|This concept exists only to assist EDAM maintenance and navigation in graphical browsers.  It does not add semantic information.  This branch provides an alternative organisation of the concepts nested under 'Accession' and 'Name'. All concepts under here are already included under 'Accession' or 'Name'.", "label": "Identifier (typed)"}, "data_0977": {"class_id": "http://edamontology.org/data_0977", "definitions": "An identifier of a bioinformatics tool, e.g. an application or web service.", "label": "Tool identifier"}, "data_0974": {"class_id": "http://edamontology.org/data_0974", "definitions": "An identifier of a biological entity or phenomenon.", "label": "Entity identifier"}, "data_0975": {"class_id": "http://edamontology.org/data_0975", "definitions": "An identifier of a data resource.", "label": "Data resource identifier"}, "data_0972": {"class_id": "http://edamontology.org/data_0972", "definitions": "An abstract of the results of text mining.|A text mining abstract will typically include an annotated a list of words or sentences extracted from one or more scientific articles.", "label": "Text mining report"}, "data_0970": {"class_id": "http://edamontology.org/data_0970", "definitions": "Bibliographic data that uniquely identifies a scientific article, book or other published material.|A bibliographic reference might include information such as authors, title, journal name, date and (possibly) a link to the abstract or full-text of the article if available.", "label": "Citation"}, "data_0971": {"class_id": "http://edamontology.org/data_0971", "definitions": "A scientific text, typically a full text article from a scientific journal.", "label": "Article"}, "data_0978": {"class_id": "http://edamontology.org/data_0978", "definitions": "Name or other identifier of a discrete entity (any biological thing with a distinct, discrete physical existence).", "label": "Discrete entity identifier"}, "data_0979": {"class_id": "http://edamontology.org/data_0979", "definitions": "Name or other identifier of an entity feature (a physical part or region of a discrete biological entity, or a feature that can be mapped to such a thing).", "label": "Entity feature identifier"}, "data_3101": {"class_id": "http://edamontology.org/data_3101", "definitions": "A node from a classification of protein structural domain(s).", "label": "Protein domain classification node"}, "operation_0384": {"class_id": "http://edamontology.org/operation_0384", "definitions": "Calculate solvent accessible or buried surface areas in protein or other molecular structures.|Solvent accessibility might be calculated for the backbone, sidechain and total (backbone plus sidechain).", "label": "Accessible surface calculation"}, "format_2328": {"class_id": "http://edamontology.org/format_2328", "definitions": "Format of a report on a gene from the PseudoCAP database.", "label": "PseudoCAP gene report format"}, "data_3558": {"class_id": "http://edamontology.org/data_3558", "definitions": "A report concerning a clinical trial.", "label": "Clinical trial report"}, "data_2053": {"class_id": "http://edamontology.org/data_2053", "definitions": "Data concerning molecular structural data.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Structural data"}, "topic_2818": {"class_id": "http://edamontology.org/topic_2818", "definitions": "Eukaryotes or data concerning eukaryotes, e.g. information on a specific eukaryote genome including molecular sequences, genes and annotation.|The resource may be specific to a eukaryote, a group of eukaryotes or all eukaryotes.", "label": "Eukaryotes"}, "data_2050": {"class_id": "http://edamontology.org/data_2050", "definitions": "General data for a molecule.", "label": "Molecular property (general)"}, "format_2322": {"class_id": "http://edamontology.org/format_2322", "definitions": "Format of an entry from the BioCyc enzyme database.", "label": "BioCyc enzyme report format"}, "format_2323": {"class_id": "http://edamontology.org/format_2323", "definitions": "Format of an entry from the Enzyme nomenclature database (ENZYME).", "label": "ENZYME enzyme report format"}, "operation_0349": {"class_id": "http://edamontology.org/operation_0349", "definitions": "Search a sequence database and retrieve sequences with a specified property, typically a physicochemical or compositional property.", "label": "Sequence database search (by property)"}, "operation_0348": {"class_id": "http://edamontology.org/operation_0348", "definitions": "Search a sequence database and retrieve sequences of a given amino acid composition.", "label": "Sequence database search (by amino acid composition)"}, "operation_0347": {"class_id": "http://edamontology.org/operation_0347", "definitions": "Search a sequence database and retrieve sequences matching a given sequence motif or pattern, such as a Prosite pattern or regular expression.", "label": "Sequence database search (by motif or pattern)"}, "operation_0346": {"class_id": "http://edamontology.org/operation_0346", "definitions": "Search a sequence database and retrieve sequences that are similar to a query sequence.", "label": "Sequence similarity search"}, "operation_0345": {"class_id": "http://edamontology.org/operation_0345", "definitions": "Query a database and retrieve sequences containing a given keyword.", "label": "Sequence retrieval (by keyword)"}, "operation_0344": {"class_id": "http://edamontology.org/operation_0344", "definitions": "Query a database and retrieve sequences with a given entry code or accession number.", "label": "Sequence retrieval (by code)"}, "operation_0343": {"class_id": "http://edamontology.org/operation_0343", "definitions": "Search a database of transmembrane proteins, for example for sequence or structural similarities.", "label": "Transmembrane protein database search"}, "operation_0342": {"class_id": "http://edamontology.org/operation_0342", "definitions": "Search a database of sequence profiles with a query sequence.", "label": "Sequence profile database search"}, "operation_0341": {"class_id": "http://edamontology.org/operation_0341", "definitions": "Screen a sequence against a motif or pattern database.", "label": "Motif database search"}, "operation_0340": {"class_id": "http://edamontology.org/operation_0340", "definitions": "Search a secondary protein database (of classification information) to assign a protein sequence(s) to a known protein family or group.", "label": "Protein secondary database search"}, "operation_3694": {"class_id": "http://edamontology.org/operation_3694", "definitions": "Visualise, format or render a mass spectrum.", "label": "Mass spectrum visualisation"}, "operation_3695": {"class_id": "http://edamontology.org/operation_3695", "definitions": "Filter a set of files or data items according to some property.", "label": "Filtering"}, "operation_1838": {"class_id": "http://edamontology.org/operation_1838", "definitions": "Calculate contacts between residues and ligands in a protein structure.", "label": "Residue contact calculation (residue-ligand)"}, "operation_1839": {"class_id": "http://edamontology.org/operation_1839", "definitions": "Calculate (and possibly score) salt bridges in a protein structure.|Salt bridges are interactions between oppositely charged atoms in different residues. The output might include the inter-atomic distance.", "label": "Salt bridge calculation"}, "operation_1836": {"class_id": "http://edamontology.org/operation_1836", "definitions": "Detect 'bumps' between residues in a structure, i.e. those with pairs of atoms whose Van der Waals' radii interpenetrate more than a defined distance.", "label": "Residue bump detection"}, "operation_1837": {"class_id": "http://edamontology.org/operation_1837", "definitions": "Calculate the number of symmetry contacts made by residues in a protein structure.|A symmetry contact is a contact between two atoms in different asymmetric unit.", "label": "Residue symmetry contact calculation"}, "operation_1834": {"class_id": "http://edamontology.org/operation_1834", "definitions": "Calculate protein residue contacts with metal in a structure.", "label": "Protein-metal contact calculation"}, "operation_1835": {"class_id": "http://edamontology.org/operation_1835", "definitions": "Calculate ion contacts in a structure (all ions for all side chain atoms).", "label": "Residue contact calculation (residue-negative ion)"}, "operation_1832": {"class_id": "http://edamontology.org/operation_1832", "definitions": "Calculate protein residue contacts with nucleic acids in a structure.", "label": "Residue contact calculation (residue-nucleic acid)"}, "topic_0179": {"class_id": "http://edamontology.org/topic_0179", "definitions": "The prediction of tertiary structure of protein sequences.", "label": "Protein tertiary structure prediction"}, "operation_1830": {"class_id": "http://edamontology.org/operation_1830", "definitions": "Detect free cysteines in a protein structure.|A free cysteine is neither involved in a cysteine bridge, nor functions as a ligand to a metal.", "label": "Free cysteine detection"}, "operation_1831": {"class_id": "http://edamontology.org/operation_1831", "definitions": "Detect cysteines that are bound to metal in a protein structure.", "label": "Metal-bound cysteine detection"}, "topic_2953": {"class_id": "http://edamontology.org/topic_2953", "definitions": "Topic for the design of nucleic acid sequences with specific conformations.", "label": "Nucleic acid design"}, "operation_0477": {"class_id": "http://edamontology.org/operation_0477", "definitions": "Build a three-dimensional protein model based on known (for example homologs) structures.|The model might be of a whole, part or aspect of protein structure. Molecular modelling methods might use sequence-structure alignment, structural templates, molecular dynamics, energy minimisation etc.", "label": "Protein modelling"}, "operation_3429": {"class_id": "http://edamontology.org/operation_3429", "definitions": "Construct some data entity.|For non-analytical operations, see the 'Processing' branch.", "label": "Generation"}, "topic_3300": {"class_id": "http://edamontology.org/topic_3300", "definitions": "The functions of living organisms and their constituent parts.", "label": "Physiology"}, "topic_3301": {"class_id": "http://edamontology.org/topic_3301", "definitions": "The biology of microorganisms.", "label": "Microbiology"}, "topic_3302": {"class_id": "http://edamontology.org/topic_3302", "definitions": "The biology of parasites.", "label": "Parasitology"}, "topic_3303": {"class_id": "http://edamontology.org/topic_3303", "definitions": "Research in support of healing by diagnosis, treatment, and prevention of disease.", "label": "Medicine"}, "topic_3304": {"class_id": "http://edamontology.org/topic_3304", "definitions": "The study of the nervous system and brain; its anatomy, physiology and function.", "label": "Neurobiology"}, "topic_3305": {"class_id": "http://edamontology.org/topic_3305", "definitions": "Topic concerning the the patterns, cause, and effect of disease within populations.", "label": "Public health and epidemiology"}, "topic_3306": {"class_id": "http://edamontology.org/topic_3306", "definitions": "The use of physics to study biological system.", "label": "Biophysics"}, "topic_3307": {"class_id": "http://edamontology.org/topic_3307", "definitions": "The development and application of theory, analytical methods, mathematical models and computational simulation of biological systems.|This includes the modeling and treatment of biological processes and systems in mathematical terms (theoretical biology).", "label": "Computational biology"}, "topic_3308": {"class_id": "http://edamontology.org/topic_3308", "definitions": "The analysis of transcriptomes, or a set of all the RNA molecules in a specific cell, tissue etc.", "label": "Transcriptomics"}, "format_1219": {"class_id": "http://edamontology.org/format_1219", "definitions": "Alphabet for any protein sequence with possible ambiguity and unknown positions but without non-sequence characters.", "label": "pure protein"}, "format_1218": {"class_id": "http://edamontology.org/format_1218", "definitions": "Alphabet for any protein sequence with possible unknown positions but without ambiguity or non-sequence characters.", "label": "unambiguous pure protein"}, "topic_3292": {"class_id": "http://edamontology.org/topic_3292", "definitions": "Chemical substances and physico-chemical processes and that occur within living organisms.", "label": "Biochemistry"}, "format_1211": {"class_id": "http://edamontology.org/format_1211", "definitions": "Alphabet for a nucleotide sequence (characters ACGTU only) with possible unknown positions but without ambiguity or non-sequence characters .", "label": "unambiguous pure nucleotide"}, "format_1210": {"class_id": "http://edamontology.org/format_1210", "definitions": "Alphabet for a nucleotide sequence with possible ambiguity and unknown positions but without non-sequence characters.", "label": "pure nucleotide"}, "format_1213": {"class_id": "http://edamontology.org/format_1213", "definitions": "Alphabet for an RNA sequence with possible ambiguity, unknown positions and non-sequence characters.", "label": "rna"}, "format_1212": {"class_id": "http://edamontology.org/format_1212", "definitions": "Alphabet for a DNA sequence with possible ambiguity, unknown positions and non-sequence characters.", "label": "dna"}, "format_1215": {"class_id": "http://edamontology.org/format_1215", "definitions": "Alphabet for a DNA sequence with possible ambiguity and unknown positions but without non-sequence characters.", "label": "pure dna"}, "format_1214": {"class_id": "http://edamontology.org/format_1214", "definitions": "Alphabet for a DNA sequence (characters ACGT only) with possible unknown positions but without ambiguity or non-sequence characters.", "label": "unambiguous pure dna"}, "format_1217": {"class_id": "http://edamontology.org/format_1217", "definitions": "Alphabet for an RNA sequence with possible ambiguity and unknown positions but without non-sequence characters.", "label": "pure rna"}, "format_1216": {"class_id": "http://edamontology.org/format_1216", "definitions": "Alphabet for an RNA sequence (characters ACGU only) with possible unknown positions but without ambiguity or non-sequence characters.", "label": "unambiguous pure rna sequence"}, "data_3128": {"class_id": "http://edamontology.org/data_3128", "definitions": "A report on regions within a nucleic acid sequence which form secondary or tertiary (3D) structures.", "label": "Nucleic acid structure report"}, "data_3129": {"class_id": "http://edamontology.org/data_3129", "definitions": "short repetitive subsequences (repeat sequences) in a protein sequence.", "label": "Protein features report (repeats)"}, "operation_2520": {"class_id": "http://edamontology.org/operation_2520", "definitions": "Generate a map of a DNA sequence annotated with positional or non-positional features of some type.", "label": "DNA mapping"}, "format_1581": {"class_id": "http://edamontology.org/format_1581", "definitions": "Entry format for the FSSP database.", "label": "FSSP entry format"}, "format_1580": {"class_id": "http://edamontology.org/format_1580", "definitions": "Entry format for the ProDom protein domain classification database.", "label": "ProDom entry format"}, "data_3122": {"class_id": "http://edamontology.org/data_3122", "definitions": "A report on features in a nucleic acid sequence that indicate changes to or differences between sequences.", "label": "Nucleic acid features (difference and change)"}, "data_3123": {"class_id": "http://edamontology.org/data_3123", "definitions": "regions within a nucleic acid sequence containing a signal that alters a biological function.", "label": "Nucleic acid features report (expression signal)"}, "data_3125": {"class_id": "http://edamontology.org/data_3125", "definitions": "nucleic acids binding to some other molecule.|This includes ribosome binding sites (Shine-Dalgarno sequence in prokaryotes).", "label": "Nucleic acid features report (binding)"}, "data_3126": {"class_id": "http://edamontology.org/data_3126", "definitions": "repetitive elements within a nucleic acid sequence.", "label": "Nucleic acid repeats (report)"}, "data_3127": {"class_id": "http://edamontology.org/data_3127", "definitions": "DNA replication or recombination.", "label": "Nucleic acid features report (replication and recombination)"}, "topic_3293": {"class_id": "http://edamontology.org/topic_3293", "definitions": "The study of evolutionary relationships amongst organisms from analysis of genetic information (typically gene or protein sequences).", "label": "Phylogenetics"}, "operation_2871": {"class_id": "http://edamontology.org/operation_2871", "definitions": "Generate a physical DNA map (sequence map) from analysis of sequence tagged sites (STS).|An STS is a short subsequence of known sequence and location that occurs only once in the chromosome or genome that is being mapped. Sources of STSs include 1. expressed sequence tags (ESTs), simple sequence length polymorphisms (SSLPs), and random genomic sequences from cloned genomic DNA or database sequences.", "label": "Sequence tagged site (STS) mapping"}, "operation_2503": {"class_id": "http://edamontology.org/operation_2503", "definitions": "Process (read and / or write) molecular sequence data.", "label": "Sequence data processing"}, "operation_2502": {"class_id": "http://edamontology.org/operation_2502", "definitions": "Process (read and / or write) protein sequence or structural data.", "label": "Protein analysis"}, "operation_2501": {"class_id": "http://edamontology.org/operation_2501", "definitions": "Process (read and / or write) nucleic acid sequence or structural data.", "label": "Nucleic acid analysis"}, "operation_2500": {"class_id": "http://edamontology.org/operation_2500", "definitions": "Analyse raw microarray data.", "label": "Microarray raw data analysis"}, "operation_2507": {"class_id": "http://edamontology.org/operation_2507", "definitions": "Analyse a protein sequence alignment, typically to detect features or make predictions.", "label": "Nucleic acid sequence alignment analysis"}, "operation_2506": {"class_id": "http://edamontology.org/operation_2506", "definitions": "Analyse a protein sequence alignment, typically to detect features or make predictions.", "label": "Protein sequence alignment analysis"}, "operation_2505": {"class_id": "http://edamontology.org/operation_2505", "definitions": "Process (read and / or write) text.", "label": "Text processing"}, "operation_2504": {"class_id": "http://edamontology.org/operation_2504", "definitions": "Process (read and / or write) molecular structural data.", "label": "Structural data processing"}, "operation_2509": {"class_id": "http://edamontology.org/operation_2509", "definitions": "Compare two or more protein sequences.", "label": "Protein sequence comparison"}, "operation_2508": {"class_id": "http://edamontology.org/operation_2508", "definitions": "Compare two or more nucleic acid sequences.", "label": "Nucleic acid sequence comparison"}, "topic_2278": {"class_id": "http://edamontology.org/topic_2278", "definitions": "Predict transmembrane domains and topology in protein sequences.", "label": "Transmembrane protein prediction"}, "topic_2271": {"class_id": "http://edamontology.org/topic_2271", "definitions": "Search for and retrieve molecular structures that are similar to a structure-based query (typically another structure or part of a structure).|The query is a structure-based entity such as another structure, a 3D (structural) motif, 3D profile or template.", "label": "Structure database search"}, "topic_2277": {"class_id": "http://edamontology.org/topic_2277", "definitions": "Single nucleotide polymorphisms (SNP) and associated data, for example, the discovery and annotation of SNPs.", "label": "SNP"}, "topic_2276": {"class_id": "http://edamontology.org/topic_2276", "definitions": "The prediction of functional properties of a protein.", "label": "Protein function prediction"}, "topic_2275": {"class_id": "http://edamontology.org/topic_2275", "definitions": "The construction, analysis, evaluation, refinement etc. of models of a molecules properties or behaviour, including the modelling the structure of proteins in complex with small molecules or other macromolecules (docking).", "label": "Molecular modelling"}, "data_1377": {"class_id": "http://edamontology.org/data_1377", "definitions": "A protein conserved site signature (sequence classifier) from the InterPro database.|A protein conserved site signature is any short sequence pattern that may contain one or more unique residues and is cannot be described as a active site, binding site or post-translational modification.", "label": "Protein conserved site signature"}, "data_1376": {"class_id": "http://edamontology.org/data_1376", "definitions": "A protein site signature (sequence classifier) from the InterPro database.|A protein site signature is a classifier for a specific site in a protein.", "label": "Protein site signature"}, "data_1375": {"class_id": "http://edamontology.org/data_1375", "definitions": "A protein repeat signature (sequence classifier) from the InterPro database.|A protein repeat signature is a repeated protein motif, that is not in single copy expected to independently fold into a globular domain.", "label": "Protein repeat signature"}, "data_1374": {"class_id": "http://edamontology.org/data_1374", "definitions": "A protein region signature (sequence classifier) from the InterPro database.|A protein region signature defines a region which cannot be described as a protein family or domain signature.", "label": "Protein region signature"}, "data_1373": {"class_id": "http://edamontology.org/data_1373", "definitions": "A protein domain signature (sequence classifier) from the InterPro database.|Protein domain signatures identify structural or functional domains or other units with defined boundaries.", "label": "Protein domain signature"}, "data_1372": {"class_id": "http://edamontology.org/data_1372", "definitions": "A protein family signature (sequence classifier) from the InterPro database.|Protein family signatures cover all domains in the matching proteins and span >80% of the protein length and with no adjacent protein domain signatures or protein region signatures.", "label": "Protein family signature"}, "data_1371": {"class_id": "http://edamontology.org/data_1371", "definitions": "NULL hidden Markov model representation used by the HMMER package.", "label": "HMMER NULL hidden Markov model"}, "format_1367": {"class_id": "http://edamontology.org/format_1367", "definitions": "A profile (sequence classifier) in the format used in the JASPAR database.", "label": "JASPAR format"}, "format_1366": {"class_id": "http://edamontology.org/format_1366", "definitions": "Sequence profile (sequence classifier) format used in the PROSITE database.", "label": "prosite-profile"}, "format_1360": {"class_id": "http://edamontology.org/format_1360", "definitions": "A motif in the format generated by the MEME program.", "label": "meme-motif"}, "data_1379": {"class_id": "http://edamontology.org/data_1379", "definitions": "A protein binding site signature (sequence classifier) from the InterPro database.|A protein binding site signature corresponds to a site that reversibly binds chemical compounds, which are not themselves substrates of the enzymatic reaction. This includes enzyme cofactors and residues involved in electron transport or protein structure modification.", "label": "Protein binding site signature"}, "data_1378": {"class_id": "http://edamontology.org/data_1378", "definitions": "A protein active site signature (sequence classifier) from the InterPro database.|A protein active site signature corresponds to an enzyme catalytic pocket. An active site typically includes non-contiguous residues, therefore multiple signatures may be required to describe an active site. ; residues involved in enzymatic reactions for which mutational data is typically available.", "label": "Protein active site signature"}, "format_1699": {"class_id": "http://edamontology.org/format_1699", "definitions": "The format of an entry from the KEGG PLANT database.", "label": "KEGG PLANT entry format"}, "format_1698": {"class_id": "http://edamontology.org/format_1698", "definitions": "The format of an entry from the KEGG COMPOUND database.", "label": "KEGG COMPOUND entry format"}, "format_1697": {"class_id": "http://edamontology.org/format_1697", "definitions": "The format of an entry from the KEGG LIGAND chemical database.", "label": "KEGG LIGAND entry format"}, "data_2540": {"class_id": "http://edamontology.org/data_2540", "definitions": "Data concerning an index of data.|This is a broad data type and is used a placeholder for other, more specific types.", "label": "Data index data"}, "operation_2479": {"class_id": "http://edamontology.org/operation_2479", "definitions": "Analyse a protein sequence (using methods that are only applicable to protein sequences).", "label": "Protein sequence analysis"}, "operation_2478": {"class_id": "http://edamontology.org/operation_2478", "definitions": "Analyse a nucleic acid sequence (using methods that are only applicable to nucleic acid sequences).", "label": "Nucleic acid sequence analysis"}, "operation_2476": {"class_id": "http://edamontology.org/operation_2476", "definitions": "Simulate molecular (typically protein) conformation using a computational model of physical forces and computer simulation.", "label": "Molecular dynamics simulation"}, "data_2955": {"class_id": "http://edamontology.org/data_2955", "definitions": "An informative report of information about molecular sequence(s), including basic information (metadata), and reports generated from molecular sequence analysis, including positional features and non-positional properties.", "label": "Sequence report"}, "operation_2474": {"class_id": "http://edamontology.org/operation_2474", "definitions": "Compare the architecture of two or more protein structures.", "label": "Protein architecture comparison"}, "operation_2473": {"class_id": "http://edamontology.org/operation_2473", "definitions": "Retrieve information on a specific genotype or phenotype.", "label": "Data retrieval (genotype and phenotype annotation)"}, "operation_2472": {"class_id": "http://edamontology.org/operation_2472", "definitions": "Retrieve information on a specific gene.", "label": "Data retrieval (gene annotation)"}, "operation_2471": {"class_id": "http://edamontology.org/operation_2471", "definitions": "Retrieve information on an RNA family.", "label": "Data retrieval (RNA family annotation)"}, "operation_2470": {"class_id": "http://edamontology.org/operation_2470", "definitions": "Retrieve information on a protein family.", "label": "Data retrieval (protein family annotation)"}, "topic_3360": {"class_id": "http://edamontology.org/topic_3360", "definitions": "Objective indicators of biological state often used to assess health, and determinate treatment.", "label": "Biomarkers"}, "data_1089": {"class_id": "http://edamontology.org/data_1089", "definitions": "Identifier of an object from the FlyBase database.", "label": "FlyBase ID"}, "data_1088": {"class_id": "http://edamontology.org/data_1088", "definitions": "Unique identifier of a scientific article.", "label": "Article ID"}, "data_1085": {"class_id": "http://edamontology.org/data_1085", "definitions": "Identifier of a mathematical model, typically an entry from a database.", "label": "Biological model ID"}, "data_1084": {"class_id": "http://edamontology.org/data_1084", "definitions": "Identifier of a data type definition from some provider.", "label": "Data resource definition ID"}, "data_1087": {"class_id": "http://edamontology.org/data_1087", "definitions": "A unique (typically numerical) identifier of a concept in an ontology of biological or bioinformatics concepts and relations.", "label": "Ontology concept ID"}, "data_1086": {"class_id": "http://edamontology.org/data_1086", "definitions": "Identifier of an entry from a database of chemicals.", "label": "Compound identifier"}, "data_1081": {"class_id": "http://edamontology.org/data_1081", "definitions": "Identifier of an entry from a database of genotypes and phenotypes.", "label": "Genotype and phenotype annotation ID"}, "data_1080": {"class_id": "http://edamontology.org/data_1080", "definitions": "Accession of a report of gene expression (e.g. a gene expression profile) from a database.", "label": "Gene expression report ID"}, "data_1083": {"class_id": "http://edamontology.org/data_1083", "definitions": "Identifier of a biological or biomedical workflow, typically from a database of workflows.", "label": "Workflow ID"}, "data_1082": {"class_id": "http://edamontology.org/data_1082", "definitions": "Identifier of an entry from a database of biological pathways or networks.", "label": "Pathway or network identifier"}, "format_1967": {"class_id": "http://edamontology.org/format_1967", "definitions": "DAS sequence (XML) format (any type).", "label": "DAS format"}, "format_1966": {"class_id": "http://edamontology.org/format_1966", "definitions": "NCBI ASN.1-based sequence format.", "label": "ASN.1 sequence format"}, "format_1965": {"class_id": "http://edamontology.org/format_1965", "definitions": "Treecon output sequence format.", "label": "treecon sequence format"}, "format_1964": {"class_id": "http://edamontology.org/format_1964", "definitions": "Plain text sequence format (essentially unformatted).", "label": "plain text format (unformatted)"}, "format_1963": {"class_id": "http://edamontology.org/format_1963", "definitions": "UniProtKB entry sequence format.", "label": "UniProtKB format"}, "format_1962": {"class_id": "http://edamontology.org/format_1962", "definitions": "DNA strider output sequence format.", "label": "strider format"}, "format_1961": {"class_id": "http://edamontology.org/format_1961", "definitions": "Stockholm multiple sequence alignment format (used by Pfam and Rfam).", "label": "Stockholm format"}, "format_1960": {"class_id": "http://edamontology.org/format_1960", "definitions": "Staden suite sequence format.", "label": "Staden format"}, "topic_2226": {"class_id": "http://edamontology.org/topic_2226", "definitions": "Experimental methods for biomolecular structure determination, such as X-ray crystallography, nuclear magnetic resonance (NMR), circular dichroism (CD) spectroscopy, microscopy etc., including the assignment or modelling of molecular structure from such data.", "label": "Structure determination"}, "format_1349": {"class_id": "http://edamontology.org/format_1349", "definitions": "Dirichlet distribution HMMER format.", "label": "HMMER Dirichlet prior"}, "format_1969": {"class_id": "http://edamontology.org/format_1969", "definitions": "EMBOSS debugging trace sequence format of full internal data content.", "label": "debug-seq"}, "format_1968": {"class_id": "http://edamontology.org/format_1968", "definitions": "DAS sequence (XML) format (nucleotide-only).|The use of this format is deprecated.", "label": "dasdna"}, "operation_3352": {"class_id": "http://edamontology.org/operation_3352", "definitions": "Compare two or more ontologies, e.g. identify differences.", "label": "Ontology comparison"}, "topic_0606": {"class_id": "http://edamontology.org/topic_0606", "definitions": "Data resources for the biological or biomedical literature, either a primary source of literature or some derivative.", "label": "Literature data resources"}, "format_3665": {"class_id": "http://edamontology.org/format_3665", "definitions": "A list of k-mers and their occurences in a dataset. Can also be used as an implicit De Bruijn graph.", "label": "K-mer countgraph"}, "operation_3357": {"class_id": "http://edamontology.org/operation_3357", "definitions": "Recognition of which format the given data is in.|'Format recognition' is not a bioinformatics-specific operation, but of great relevance in bioinformatics. Should be removed from EDAM if/when captured satisfactorily in a suitable domain-generic ontology.", "label": "Format detection"}, "topic_3697": {"class_id": "http://edamontology.org/topic_3697", "definitions": "The ecology of microorganisms including their relationship with one another and their environment.", "label": "Microbial ecology"}, "data_1791": {"class_id": "http://edamontology.org/data_1791", "definitions": "Symbol of a gene approved by the HUGO Gene Nomenclature Committee.", "label": "Gene name (HGNC)"}, "data_1790": {"class_id": "http://edamontology.org/data_1790", "definitions": "Symbol of a gene from E.coli Genetic Stock Center.", "label": "Gene name (CGSC)"}, "data_1793": {"class_id": "http://edamontology.org/data_1793", "definitions": "Symbol of a gene from Bacillus subtilis Genome Sequence Project.", "label": "Gene name (Bacillus subtilis)"}, "data_1792": {"class_id": "http://edamontology.org/data_1792", "definitions": "Symbol of a gene from the Mouse Genome Database.", "label": "Gene name (MGD)"}, "data_1795": {"class_id": "http://edamontology.org/data_1795", "definitions": "Identifier of a gene from EcoGene Database.", "label": "Gene ID (EcoGene)"}, "data_1794": {"class_id": "http://edamontology.org/data_1794", "definitions": "Identifier of a gene from PlasmoDB Plasmodium Genome Resource.", "label": "Gene ID (PlasmoDB)"}, "data_1797": {"class_id": "http://edamontology.org/data_1797", "definitions": "Gene identifier from Glossina morsitans GeneDB database.", "label": "Gene ID (GeneDB Glossina morsitans)"}, "data_1796": {"class_id": "http://edamontology.org/data_1796", "definitions": "Gene identifier from FlyBase database.", "label": "Gene ID (FlyBase)"}, "data_1799": {"class_id": "http://edamontology.org/data_1799", "definitions": "Gene identifier from Plasmodium falciparum GeneDB database.", "label": "Gene ID (GeneDB Plasmodium falciparum)"}, "data_1798": {"class_id": "http://edamontology.org/data_1798", "definitions": "Gene identifier from Leishmania major GeneDB database.", "label": "Gene ID (GeneDB Leishmania major)"}, "operation_3212": {"class_id": "http://edamontology.org/operation_3212", "definitions": "Generate an index of a genome sequence using the Burrows-Wheeler algorithm.|The Burrows-Wheeler Transform (BWT) is a permutation of the genome based on a suffix array algorithm.", "label": "Genome indexing (Burrows-Wheeler)"}, "format_3765": {"class_id": "http://edamontology.org/format_3765", "definitions": "Data table formatted such that it can be passed/streamed within the KNIME platform.", "label": "KNIME datatable format"}, "data_1159": {"class_id": "http://edamontology.org/data_1159", "definitions": "Identifier of an entry from the PATIKA database.", "label": "Pathway ID (PATIKA)"}, "data_1158": {"class_id": "http://edamontology.org/data_1158", "definitions": "Identifier of an entry from the INOH database.", "label": "Pathway ID (INOH)"}, "data_1157": {"class_id": "http://edamontology.org/data_1157", "definitions": "Identifier of an pathway from the BioCyc biological pathways database.", "label": "Pathway ID (BioCyc)"}, "data_1156": {"class_id": "http://edamontology.org/data_1156", "definitions": "Identifier of an entry from the aMAZE database.", "label": "Pathway ID (aMAZE)"}, "data_1155": {"class_id": "http://edamontology.org/data_1155", "definitions": "Identifier of an entry from the Reactome database.", "label": "Pathway ID (reactome)"}, "data_1154": {"class_id": "http://edamontology.org/data_1154", "definitions": "Unique identifier of an object from one of the KEGG databases (excluding the GENES division).", "label": "KEGG object identifier"}, "data_1153": {"class_id": "http://edamontology.org/data_1153", "definitions": "Identifier of an entry from the OMIM database.", "label": "OMIM ID"}, "data_1152": {"class_id": "http://edamontology.org/data_1152", "definitions": "Identifier of an entry from the HIVDB database.", "label": "HIVDB identifier"}, "data_1151": {"class_id": "http://edamontology.org/data_1151", "definitions": "Identifier of an entry from the HGVbase database.", "label": "HGVbase ID"}, "data_1150": {"class_id": "http://edamontology.org/data_1150", "definitions": "Accession number of an entry from a database of disease.", "label": "Disease ID"}, "operation_3024": {"class_id": "http://edamontology.org/operation_3024", "definitions": "Predict, recognise, detect or identify some properties of nucleic acids.", "label": "Prediction and recognition (nucleic acid)"}, "format_3764": {"class_id": "http://edamontology.org/format_3764", "definitions": "XML file format for files containing information about peptide identifications from mass spectrometry data analysis carried out with OpenMS.", "label": "idXML"}, "data_2835": {"class_id": "http://edamontology.org/data_2835", "definitions": "Identifier for a gene from the VBASE2 database.", "label": "Gene ID (VBASE2)"}, "data_2837": {"class_id": "http://edamontology.org/data_2837", "definitions": "Identifier of a pathway from the BioSystems pathway database.", "label": "Pathway ID (BioSystems)"}, "data_2836": {"class_id": "http://edamontology.org/data_2836", "definitions": "A unique identifier for a virus from the DPVweb database.", "label": "DPVweb ID"}, "data_2831": {"class_id": "http://edamontology.org/data_2831", "definitions": "A flat-file (textual) data archive.", "label": "Databank"}, "data_2832": {"class_id": "http://edamontology.org/data_2832", "definitions": "A web site providing data (web pages) on a common theme to a HTTP client.", "label": "Web portal"}, "data_2838": {"class_id": "http://edamontology.org/data_2838", "definitions": "Data concerning a proteomics experiment.", "label": "Experimental data (proteomics)"}, "topic_3401": {"class_id": "http://edamontology.org/topic_3401", "definitions": "The prevention of pain and the evaluation, treatment and rehabilitation of persons in pain.", "label": "Pain medicine"}, "topic_3298": {"class_id": "http://edamontology.org/topic_3298", "definitions": "Phenomes, or the study of the change in phenotype (the physical and biochemical traits of organisms) in response to genetic and environmental factors. ", "label": "Phenomics"}, "topic_0663": {"class_id": "http://edamontology.org/topic_0663", "definitions": "One or more transfer RNA (tRNA) sequences.", "label": "tRNA"}};
